% !TEX root = msc_thesis.tex

\chapter{Discussion}

\section{Limitations}

Cystic fibrosis

  - Existing approaches remain limited in their ability to predict disease-causing variants. In a study of 1571 mutations of the CFTR gene causing cystic fibrosis, (SIFT, PolyPhen, PANTHER) \cite{dorfman_common_2010}

Long QT syndrome

  - Assessment of the predictive accuracy of five in silico prediction tools, alone or in combination, and two metaservers to classify long QT syndrome gene mutations.

  - http://www.ncbi.nlm.nih.gov/pubmed/25967940



\section{Future Directions}

% Improve domain definitions and alignments

eSCOP

Gene3D

- Use sequence profiles (e.g. Pfam or Gene3D) to guide the alignment.


\subsection{Better features}


% Better features

- Use covariation between amino acids in addition tho the conservation score to predict the impact of mutations, as described by Kowarsch et. al. \cite{kowarsch_correlated_2010}. \\


% Replace FoldX with MODELLER

- Use multiple templates when building the homology models. \\
- Create multiple models and choose the one with the highest DOPE score. \\
- Refine the model using molecular dynamics. \\

Long-term MD is not useful for optimizing structures in most cases \cite{raval_refinement_2012}.



\subsection{Mutation affecting multiple amino acids}

ELASPIC can easily be extended to calculate the $\Delta \Delta G$ for mutations invovling multiple amino acids. The tricky part is that the number of features changes with the number of amino acids that are mutated. We could address this by treating a mutation affecting multiple amino acids as a set of single amino acid mutations. For example, we could use the following recursive strategy:

\begin{enumerate}
    \item Introdue each of the single amino acid mutations, one at a time.
    \item Select the single amino acid mutation with the most stabilising effect.
    \item Repeat for the remaining mutations, using the structure containing the mutation selected in Step 2.
\end{enumerate}

About one third on mutations in the Protherm and Skempi databases affect multiple amino acids. We could include those mutations in the training set by dividing them into single amino acid mutations and assigning to them a $\Delta \Delta G$ proportional to their contribution to the overall mutation score, as determined by the multiple amino acid substitution version of ELASPIC. This would require ``bootstrapping'' the ELASPIC predictor using single amin acid mutations, using the ``bootstrapped'' predictor to approximate the contribution of single amino acid mutaitons to the $\Delta \Delta G$ affecting mulitple amino acids, adding those mutations to the training set, and repeating.

It is likely that the performance of the ELASPIC predictor would be lower for mutations affecting multiple amino acids than for mutations affecting a single amino acids, as the former is more likely to induce changes in the conformation of the protein that are not modelled by ELASPIC. The performance of ELASPIC could potentially be improved by including a backbone relaxation step between each mutation, using molecular dynamics \cite{abraham_gromacs:_2015}, backrub \cite{smith_predicting_2011}, or other algorithms \cite{sun_protein_2016}.

If the ELASPIC preictor can achieve reasonable results for mutations affecting multiple amino acids, it could be used ``in reverse'' to design protein domains with increased stability and protein interfaces with increased affinity.


% Multiple mutations + insertions / deletions

- Predict homo-oligomers, since this is the predominant form of oligomerization in proteins. \\
- Multiple amino acid substitutions + insertions / deletions. \\
- Alternative splicing / aberrant splicing. \\


% Protein-protein interactions

Predict PPIs: PRISM: Protein interaction by structure matching.



% Protein-ligand interactions

- drugging protein-protein interfaces \cite{wells_reaching_2007}

Platinum: Protein-ligand affinity change upon mutation database.

  - http://bleoberis.bioc.cam.ac.uk/platinum/


BioLiP is a semi-manually curated database for high-quality, biologically relevant ligand-protein binding interactions.

  - http://zhanglab.ccmb.med.umich.edu/BioLiP/

  - The structure data are collected primarily from the Protein Data Bank, with biological insights mined from literature and other specific databases.



% Protein-DNA/RNA interactions

ProNIT



% Protein-peptide interactions

ELM
