% !TEX root = msc_thesis.tex

\chapter{Discussion} \label{ch:discussion}

The primary objective of this project was to extend ELASPIC and make it possible to predict the effect of mutations on protein stability and protein-protein interaction affinity on a genome-wide scale. We were able to meet this objective with a reasonable amount of success. We implemented ELASPIC as an easy to install and fully automated pipeline, which can scale from a single mutation in a user-provided PDB file to hundreds of thousands of mutation affecting most proteins in the genome. We calculated $\Delta \Delta G$ values for almost one million mutations implicated in human diseases or found in cancers, which, to our knowledge, makes this the first study to evaluate the structural impact of mutations at such a large scale. It is now possible to use ELASPIC as part of a toolbox for annotating variants discovered through high-throughput sequencing, since evaluating a mutation takes under several minutes if the Provean supporting set and homology models have been precalculated.  With the work of Daniel Witvliet \textit{et al.} \cite{witvliet_elaspic_2016}, it is also possible to use ELASPIC through a webserver.

The secondary objective of this project was to use structural information provided by ELASPIC to make accurate and informative predictions regarding the phenotypic effect of mutations. This objective was met with much less success. Provean remains more accurate than ELASPIC in predicting whether or not a mutation is associated with disease, as seen in the performance of the two tools on the validation and the test subsets of the Humsavar, ClinVar and COSMIC datasets (Figures \ref{fig:core_validation} and \ref{fig:interface_validation}). It seems that the sequence conservation score already encompasses not only the structural effect of mutations but also other effects that can make a mutation deleterious. We found no evidence that ``edgetic mutations'', or mutations which affect only one of multiple interactions mediated by a protein, are more likely to be deleterious than mutations that destroy all interactions \cite{sahni_widespread_2015}. We did find that mutations falling inside protein-protein interaction interfaces are more likely to be involved in disease (data not shown). However, this may be solely due to the fact that we are more likely to make a homology model of protein-protein interactions involved in disease, since those interactions are better studied and have better structural representation in the PDB. We considered using ELASPIC in combination with a kinetic models of cell metabolism \cite{bordbar_personalized_2015} or cell growth and replication \cite{karr_whole-cell_2012} in order to predict the effect of mutations on cell function. However, even when used as part of a kinetic model, the $\Delta \Delta G$ predicted by ELASPIC may not be more informative than a deleteriousness score predicted by Provean or another sequence-based tool, especially as the accuracy of the models and ELASPIC predictions remains limited.

Overall, sequence-based tools such as Provean appear to be much better predictors of mutation deleteriousness than features describing the structure of the wildtype and mutant residues or the energetic effect of the mutation. This result is consistent with previous reports. For example, an algorithm published in 2016 called Variant Interpretation and Prediction Using Rosetta (VIPUR) \cite{baugh_robust_2016}, which combines Provean score, $\Delta \Delta G$ predicted by Rosetta and other structural and sequential features, can predict whether or not a mutation is involved in a disease with an area under the receiver operating characteristic (ROC) curve of 0.831. Provean achieves an area under the ROC curve of 0.819 on the same dataset.

It seems that the additional complexity required to evaluate the structural impact of mutations does not justify the marginal increase in performance that is observed. The most promising avenue for future work would be in trying to use the data that was generated though the genome-wide, structural analysis of mutations, in order to improve the accuracy of ELASPIC predictors. The performance of ELASPIC core and interface predictors on the training sets is highly correlated with their performance on the validation sets, as was shown in Sections \ref{sec:gridsearch} and \ref{sec:feature_elimination}. Since the validation sets contain 10 to 100 times more mutations than the training sets, and are much more diverse, they could contribute much information to the ELASPIC predictors. Some possible ways in which this could expand and incorporate more information into the ELASPIC training set are discussed in Chapter \ref{ch:future_directions}.
