% !TEX root = msc_thesis.tex

\chapter{Discussion} \label{chap:discussion}



In the set of features selected through feature elimination, there are both sequence-based features and structure-based features. The most important sequence-based feature is the Provean score

Out of the remaining featues,
Results of feature elimination support the view that electrostatics, Van der Waals forces and entropy are the main forces determining the effect of mutations, as proposed by Benedix \textit{et al.} in the Concoord/Poisson-Boltzmann surface area model (Equation \ref{eq:benedix_et_al}).

Out of the features that remain after feature elimination.

\begin{equation} \label{eq:benedix_et_al}
    \Delta G_{CC/PBSA} = \Delta G_{electrostatic} + \Delta G_{van\ der\ Waals} + \Delta G_{entropy}
\end{equation}

``By weighting the individual averaged energy contributions (sepa-
rately for folding free energies and protein-protein binding affinities)
water contributions are implicitly taken into account.''


- Use covariation between amino acids in addition to the conservation score to predict the impact of mutations, as described by Kowarsch et. al. \cite{kowarsch_correlated_2010}.

- Standard conservation metrics, such as Provean, may predict a certain substitution to be benign because it occurs in other organisms. However, this does not take into account any potentially covarying mutations that mask the deleterious effect of the mutation in question.


% Replace FoldX with MODELLER

- Use multiple templates when building the homology models.
- Create multiple models and choose the one with the highest DOPE score.
- Refine the model using molecular dynamics, although it has been reported that long-term MD is not useful for optimizing structures in most cases \cite{raval_refinement_2012}.




Cystic fibrosis



Long QT syndrome

  - Assessment of the predictive accuracy of five \textit{in-silico} prediction tools, alone or in combination, and two meta-servers to classify long QT syndrome gene mutations.

  - \url{http://www.ncbi.nlm.nih.gov/pubmed/25967940}


% === Published post factum ===

Since the publication of the ELASPIC pipeline \cite{berliner_combining_2014} and webserver \cite{witvliet_elaspic_2016}, several other algorithms have been published which use a similar approach as ELASPIC to either predict mutation deleteriousness \cite{baugh_robust_2016} or the $\Delta \Delta G$.




VIPUR \cite{baugh_robust_2016}

% ddG

MutaBind \cite{li_mutabind_2016}.




\section{Protein science}
