% !TEX root = msc_thesis.tex

\mychapter{1}{Introduction}

The central dogma of biology is that DNA is transcribed into RNA which is translated into Protein.

% Sequence-based

Advances in DNA sequencing technology have led to an enormous growth in the number of genome sequences that are available. Millions of single nucleotide polymorphisms (SNPs) have been implicated in thousands of diseases. However, the etiology by which the mutations cause or contribute to a disease are often unknown.

Interpreting the variation in those genome sequences in order arrive at actionable results remains a challenge. Evaluating experimentally the effect of all discovered mutations is not feasible both in terms of the time and the cost that would be required. Computational techniques have been developed to predict the effect of mutations and prioritize them for experimental validation.

Sequence-based tools are the de-facto standard for predicting whether variants in genome sequences are deleterious. Tools such as Ensembl VEP, ANNOVAR and SNPeff can leverage databases of pre-calculated scores (dbNSFP) to annotate VCF files with the score predicted by each of the tools.
Such algorithms generally fall into three categories:

Sequence-based approaches which predict the effect of mutation by using different metrics describing the conservation of a particular residue. Examples include Mutation Assessor, CONDEL (ensemble approach)

The most widely-used program is Sorting Intolerant from Tolerant or \textbf{SIFT}. SIFT creates an extensive multiple sequence alignment for every protein, and produces a conservation score based on the likelihood of the wildtype and mutant amino acids occurring at a given position. However, we had difficulty compiling and running SIFT in a cluster setting. Furthermore

Another widely-used mutation scoring tool is \textbf{PolyPhen-2}. However, it is trained on a dataset of deleterious and neutral human mutations. This would make it difficult for us to run benchmarks, since we would have to be meticulous to ensure that the validation set that we are using does not contain mutations that are in the PolyPhen-2 training set.

Another popular sequence-based algorithm is \textbf{Provean}. Provean is comparable to the leading mutation scoring programs, such as SIFT, PolyPhen-2, Mutation Assessor, and CONDEL \cite{choi_predicting_2012}. Furthermore, Provean is distributed under a GPLv3 license, and uses \textit{supporting sets} of at most 45 sequences which can be precalculated and stored. If a supporting set is available, calculating the Provean score takes several seconds per mutation.

Other sequence-based algorithms include FATHMM, CONDEL, MutationAssesor, MutPred, and others.

Despite the proliferation of tools predicting the deleteriousness of different SNPs, our ability to act on those predictions remains limited. One of the reasons is that while sequence-based tools achieve reasonably good performance at predicting whether or not a given mutation is going to be deleterious, they fall short in predicting \textit{why} that mutation is deleterious. This lack of actionable predictions limits the usability of the vast DNA sequencing data that has been generated.

% Structure-based

One reason for out lack of ability in interpreting is the focus on the sequence-level features, while in the majority of missense mutations, it is the alteration in the function of the transcribed protein which is responsible for the detrimental effect of mutations.

The field of protein science has generally been concerned with the broad questions of protein folding, protein design. Algorithms have been developed to predict the effect of mutations on protein folding and protein-protein affinity, but those tools are generally meant to be used on a case-by-case basis and have not been designed to be applied on a genome-wide scale to predict the effect of missense mutations from whole-genome sequencing studies.

While the growth in protein crystal structures has not seen the rapid rise that was observed in DNA sequencing, the number of resolved protein structures has also been increasing, with the Protein Data Bank (PDB) containing close to 125,000 structures as of 2016.

A related are of research is predicting the energetic effect of mutations.

The most accurate class of computational techniques are alchemical free energy calculations, which involve modelling the structural transition from the wildtype to the mutant protein and using the Bennett acceptance ratio (BAR) or thermodynamic integration (TI) to calculate the energetics of the transition \cite{monticelli_introduction_2013}. However, alchemical free energy calculations are computationally expansive, and are generally used only in cases where the experimental characterization of mutants is particularly difficult, as in the case of D-amino acid peptide design \cite{welch_potent_2007}.

Many algorithms have been developed which attempt to predict the effect of mutations on protein stability and / or on protein-protein interaction affinity. Those techniques generally use a rigid backbone representation of protein and use statistical potentials. For a review see XXX.

Mixed strategies which utilize both sequence- and structure-based approaches. Such algorithms include PoPMuSiC,

Structure-based tools which predict the effect of mutations on protein structure and / or function using features describing the three-dimensional structure of the protein. mCSM \cite{pires_mcsm:_2014} (graph-based signatures), MAESTRO \cite{laimer_maestro_2015} (multi-agent machine learning), CC/PBSA (Concoord/Poisson-Boltzmann surface area) \cite{benedix_predicting_2009},

Some algorithms rely on the conservation of the residue in multiple sequence alignments.

Predicting protein thermal stability changes upon point mutations using statistical potentials: Introducing HoTMuSiC

  - MAESTRO implements a multi-agent machine learning system.

  - Structure based tools AUTO-MUTE [7], CUPSAT [8], Dmutant [9], FoldX [10], Eris [11], PoPMuSiC [12], SDM [13] or mCSM [14] usually perform better than the sequence based counterparts. Recently, SDM and mCSM have been integrated into a new method called DUET [15].

INPS: predicting the impact of non-synonymous variations on protein stability from sequence

  - \url{http://bioinformatics.oxfordjournals.org/content/31/17/2816.long}

  - Here, we describe INPS, a novel approach for annotating the effect of non-synonymous mutations on the protein stability from its sequence.

  - \cite{fariselli_inps:_2015}

FoldX

PoPMuSiC

RosettaCM

mCSM: predicting the effects of mutations in proteins using graph-based signatures.

  - \url{http://www.ncbi.nlm.nih.gov/pubmed/24281696}

  - ``To understand the roles of mutations in disease, we have evaluated their impacts not only on protein stability but also on protein-protein and protein-nucleic acid interactions''.

  - \cite{pires_mcsm:_2014}


Predicting Binding Free Energy Change Caused by Point Mutations with Knowledge-Modified MM/PBSA Method

  - \url{http://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1004276}

  - ``The core of the SAAMBE method is a modified molecular mechanics Poisson-Boltzmann Surface Area (MM/PBSA) method with residue specific dielectric constant''.

  - \cite{petukh_predicting_2015}


\subsection{Goals and objectives}

\begin{itemize}
    \item Evaluate how well we can predict the deleteriousness of a mutation by measuring the effect of protein folding on protein stability.
    \item Assessing the impact of missense mutations.
    \item Protein engineering. For example generating biological therapeutics that are more thermostable and have a higher affinity for their target.
    \item Basic science: characterizing the forces that are most important in protein folding and binding, and the effect of mutations on those forces.
    \item In this work we examine how much sequence-based features can aid in the prediction of traditionally structural realms such as the prediction of $\Delta \Delta G$ scores of mutations, and how much structure-based features can aid with the prediction of mutation pathogenicity--a traditionally sequence based
\end{itemize}



\section{Published post factum}

% Deleteriousness

VIPUR \cite{baugh_robust_2016}

% ddG

MutaBind \cite{li_mutabind_2016}.



\section{Benchmarks}

Rosetta benchmark \cite{o_conchuir_web_2015}

Benchmark showing Rosetta doing poorly: \cite{potapov_assessing_2009}

I-Mutant2, DMutant, CUPSAT, FoldX \cite{khan_performance_2010}



\section{Acknowledgements}

This is a continuation of the work performed by Niklas Berliner \textit{et al.} \cite{berliner_combining_2014}. In \ref{chap:implementation} we discuss how we expand ELASPIC to work on the genome-wide scale. In \ref{chap:results} we discuss how we retrained ELASPIC while leveraging the information we extracted from genome-wide analysis.
