% !TEX root = msc_thesis.tex

\chapter{Introduction} \label{ch:introduction}

\section{Background}

Recent advances in DNA sequence technology have drastically lowered the cost and improved the accuracy of genome sequencing \cite{wetterstrand_dna_2016}. This has made exome and whole-genome sequencing a viable and cost-effective tool in both the laboratory and in the clinic to assist with the diagnosis and direct treatment of pediatric conditions \cite{chrystoja_whole_2014} and cancers \cite{nik-zainal_landscape_2016}, and has led to an enormous growth in the amount of genomic data that is being generated. However, interpreting such genomic data to produce meaningful and actionable results remains a challenge.

\textit{In vitro} and \textit{in vivo} experiments remain the gold standard in elucidating the effect of mutations. However, evaluating experimentally the effect of all discovered mutants is not feasible. Computational techniques have been developed to predict the effect of different mutations and to prioritize them for experimental validation. Those techniques generally use conservation score describing the likelihood that a particular amino acid being found in the particular position in orthologous proteins.

The most widely-used program for predicting the deleteriousness of a mutation is Sorting Intolerant from Tolerant (SIFT) \cite{ng_sift:_2003}. SIFT runs PSI-Blast to create a multiple sequence alignment for the query protein, and computes a conservation score by looking at the likelihood of the wildtype and mutant amino acids occurring at a given position in the alignment. While SIFT is a well-established tool in the field, it is difficult to compile and install on a local machine. Furthermore, multiple sequence alignments constructed by SIFT can be several megabytes in size, and caching this data for an entire proteome would require a non-trivial amount of storage space.

Another popular sequence-based algorithm is Provean \cite{choi_predicting_2012}. Provean also creates a multiple sequence alignment for the query protein. However, instead of using the entire alignment, Provean runs CD-HIT to select under 50 representative sequences, referred to as the ``supporting set'', which capture the diversity of the alignment. The supporting set for a particular protein can be precalculated and stored for future use. If a supporting set is available, calculating the Provean score takes several seconds per mutation. Provean is reported to achieve similar performance to SIFT \cite{choi_predicting_2012}. However, unlike SIFT, it is freely available under the GPLv3 license, it compiles easily and it runs on all modern Linux distributions.

Many other tools have been developed that offer various advantages over SIFT / Provean. PolyPhen-2 \cite{adzhubei_predicting_2001} uses support vector machines to combine a conservation score with different sequential and structural features of the wildtype and mutant residue. However, since PolyPhen-2 is trained on a dataset of human deleterious mutations, it is difficult to use in downstream applications, as one would have to make sure to exclude the PolyPhen-2 training set throughout the training and validation process. FATHMM \cite{shihab_ranking_2014} constructs a hidden Markov model based on the alignment, and is reported to achieve marginally higher accuracy than SIFT / Provean. Other techniques offering various advantages over SIFT / Provean include MutPred \cite{li_automated_2009}, MutationAssesor \cite{network_integrated_2011}, CADD \cite{kircher_general_2014}, CONDEL \cite{choi_predicting_2012}, and others. Each of those tools uses

Despite the proliferation of tools predicting the deleteriousness of different SNPs, those tools remain limited in their accuracy and the type of information that they can provide. While millions of single nucleotide polymorphisms (SNPs) have been implicated in thousands of diseases, approaches for predicting the phenotypic effect of newly-discovered mutations are still in their infancy. One of the reasons is that while sequence-based tools achieve reasonably good performance at predicting whether or not a given mutation is going to be deleterious, they fall short in predicting \textit{why} that mutation is deleterious. This lack of actionable predictions limits the usability of the vast DNA sequencing data that has been generated. However, the etiology by which the mutations cause or contribute to a disease are often unknown.



\section{Predicting the structural effect of mutations}

One reason for out lack of ability in interpreting is the focus on the sequence-level features, while in the majority of missense mutations, it is the alteration in the function of the transcribed protein which is responsible for the detrimental effect of mutations.

The field of protein science has generally been concerned with the broad questions of protein folding, protein design. Algorithms have been developed to predict the effect of mutations on protein folding and protein-protein affinity, but those tools are generally meant to be used on a case-by-case basis and have not been designed to be applied on a genome-wide scale to predict the effect of missense mutations from whole-genome sequencing studies.

While the growth in protein crystal structures has not seen the rapid rise that was observed in DNA sequencing, the number of resolved protein structures has also been increasing, with the Protein Data Bank (PDB) containing close to 125,000 structures as of 2016.

The most accurate class of computational techniques are alchemical free energy calculations, which involve modelling the structural transition from the wildtype to the mutant protein and using the Bennett acceptance ratio (BAR) or thermodynamic integration (TI) to calculate the energetics of the transition \cite{monticelli_introduction_2013}. However, alchemical free energy calculations are computationally expansive, and are generally used only in cases where the experimental characterization of mutants is particularly difficult, as in the case of D-amino acid peptide design \cite{welch_potent_2007}.

Many algorithms have been developed which attempt to predict the effect of mutations on protein stability and / or on protein-protein interaction affinity. Those techniques generally use a rigid backbone representation of protein and use statistical potentials. For a review see XXX.

Mixed strategies which utilize both sequence- and structure-based approaches. Such algorithms include PoPMuSiC,

Structure-based tools which predict the effect of mutations on protein structure and / or function using features describing the three-dimensional structure of the protein. mCSM \cite{pires_mcsm:_2014} (graph-based signatures), MAESTRO \cite{laimer_maestro_2015} (multi-agent machine learning), CC/PBSA (Concoord/Poisson-Boltzmann surface area) \cite{benedix_predicting_2009},



\section{Goals and objectives}

\begin{itemize}
    \item Evaluate how well we can predict the deleteriousness of a mutation by measuring the effect of protein folding on protein stability.
    \item Assessing the impact of missense mutations.
    \item Protein engineering. For example generating biological therapeutics that are more thermostable and have a higher affinity for their target.
    \item Basic science: characterizing the forces that are most important in protein folding and binding, and the effect of mutations on those forces.
    \item In this work we examine how much sequence-based features can aid in the prediction of traditionally structural realms such as the prediction of $\Delta \Delta G$ scores of mutations, and how much structure-based features can aid with the prediction of mutation pathogenicity--a traditionally sequence based
\end{itemize}



\section{Acknowledgements}

This is a continuation of the work performed by Niklas Berliner \textit{et al.} \cite{berliner_combining_2014}. In \ref{ch:implementation} we discuss how we expand ELASPIC to work on the genome-wide scale. In \ref{ch:results} we discuss how we retrained ELASPIC while leveraging the information we extracted from genome-wide analysis.
