<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ul.lst-kix_s8oehts9vu79-8{list-style-type:none}.lst-kix_krjlquulz7zd-7>li:before{content:"\0025cb  "}ul.lst-kix_s8oehts9vu79-7{list-style-type:none}.lst-kix_qffkk51zafur-4>li:before{content:"\0025cb  "}.lst-kix_qffkk51zafur-6>li:before{content:"\0025cf  "}ul.lst-kix_s8oehts9vu79-4{list-style-type:none}ul.lst-kix_s8oehts9vu79-3{list-style-type:none}ul.lst-kix_s8oehts9vu79-6{list-style-type:none}ul.lst-kix_s8oehts9vu79-5{list-style-type:none}ol.lst-kix_ur253ckj31qe-8.start{counter-reset:lst-ctn-kix_ur253ckj31qe-8 0}.lst-kix_krjlquulz7zd-3>li:before{content:"\0025cf  "}.lst-kix_qffkk51zafur-0>li:before{content:"\0025cf  "}.lst-kix_qffkk51zafur-8>li:before{content:"\0025a0  "}ol.lst-kix_4roymkz9vma-3.start{counter-reset:lst-ctn-kix_4roymkz9vma-3 0}.lst-kix_krjlquulz7zd-1>li:before{content:"\0025cb  "}.lst-kix_t6shmrihc7et-8>li:before{content:"\0025a0  "}ul.lst-kix_s8oehts9vu79-0{list-style-type:none}ul.lst-kix_s8oehts9vu79-2{list-style-type:none}ul.lst-kix_s8oehts9vu79-1{list-style-type:none}.lst-kix_jm1l4y2hlnt3-3>li:before{content:"\0025cf  "}.lst-kix_t6shmrihc7et-0>li:before{content:"\0025cf  "}.lst-kix_t6shmrihc7et-4>li:before{content:"\0025cb  "}.lst-kix_lm4lwqvvvsaj-5>li:before{content:"\0025a0  "}.lst-kix_jm1l4y2hlnt3-7>li:before{content:"\0025cb  "}.lst-kix_lux5kw5rxj0b-6>li:before{content:"\0025cf  "}.lst-kix_o1wsnr4piq1b-2>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-2}.lst-kix_lm4lwqvvvsaj-3>li:before{content:"\0025cf  "}.lst-kix_jm1l4y2hlnt3-1>li:before{content:"\0025cb  "}.lst-kix_lux5kw5rxj0b-8>li:before{content:"\0025a0  "}.lst-kix_t6shmrihc7et-6>li:before{content:"\0025cf  "}.lst-kix_1i1lr1oi4e9-2>li:before{content:"\0025a0  "}.lst-kix_1i1lr1oi4e9-0>li:before{content:"\0025cf  "}.lst-kix_1i1lr1oi4e9-4>li:before{content:"\0025cb  "}.lst-kix_qffkk51zafur-2>li:before{content:"\0025a0  "}.lst-kix_lm4lwqvvvsaj-7>li:before{content:"\0025cb  "}.lst-kix_jm1l4y2hlnt3-5>li:before{content:"\0025a0  "}.lst-kix_t6shmrihc7et-2>li:before{content:"\0025a0  "}ol.lst-kix_ur253ckj31qe-3.start{counter-reset:lst-ctn-kix_ur253ckj31qe-3 0}ul.lst-kix_4r80w4zgcdjq-0{list-style-type:none}.lst-kix_1i1lr1oi4e9-8>li:before{content:"\0025a0  "}.lst-kix_g84y3pe344o9-1>li:before{content:"\0025cb  "}.lst-kix_g84y3pe344o9-5>li:before{content:"\0025a0  "}ul.lst-kix_4r80w4zgcdjq-3{list-style-type:none}ul.lst-kix_4r80w4zgcdjq-4{list-style-type:none}.lst-kix_1i1lr1oi4e9-6>li:before{content:"\0025cf  "}ul.lst-kix_4r80w4zgcdjq-1{list-style-type:none}ul.lst-kix_4r80w4zgcdjq-2{list-style-type:none}.lst-kix_g84y3pe344o9-7>li:before{content:"\0025cb  "}ul.lst-kix_4r80w4zgcdjq-7{list-style-type:none}ul.lst-kix_4r80w4zgcdjq-8{list-style-type:none}ul.lst-kix_4r80w4zgcdjq-5{list-style-type:none}ul.lst-kix_4r80w4zgcdjq-6{list-style-type:none}.lst-kix_lm4lwqvvvsaj-1>li:before{content:"\0025cb  "}ol.lst-kix_4roymkz9vma-8.start{counter-reset:lst-ctn-kix_4roymkz9vma-8 0}ol.lst-kix_4fkgc27ss5mq-8.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-8 0}.lst-kix_eu4b93xn98yk-2>li:before{content:"\0025a0  "}ol.lst-kix_ontx89k1fqr9-0{list-style-type:none}.lst-kix_jduurcgheuo-3>li:before{content:"-  "}.lst-kix_eu4b93xn98yk-0>li:before{content:"\0025cf  "}.lst-kix_krjlquulz7zd-5>li:before{content:"\0025a0  "}.lst-kix_jduurcgheuo-5>li:before{content:"-  "}.lst-kix_g84y3pe344o9-3>li:before{content:"\0025cf  "}.lst-kix_mpnpv7ekeo7q-1>li:before{content:"\0025cb  "}.lst-kix_mpnpv7ekeo7q-3>li:before{content:"\0025cf  "}.lst-kix_eu4b93xn98yk-8>li:before{content:"\0025a0  "}.lst-kix_jza88dihngw3-3>li:before{content:"\0025cf  "}ul.lst-kix_l4298avyh9v6-7{list-style-type:none}.lst-kix_abgphud9refj-7>li:before{content:"\0025cb  "}.lst-kix_mpnpv7ekeo7q-5>li:before{content:"\0025a0  "}ul.lst-kix_l4298avyh9v6-8{list-style-type:none}ul.lst-kix_l4298avyh9v6-5{list-style-type:none}ul.lst-kix_l4298avyh9v6-6{list-style-type:none}.lst-kix_jduurcgheuo-1>li:before{content:"-  "}.lst-kix_jza88dihngw3-1>li:before{content:"\0025cb  "}.lst-kix_abgphud9refj-3>li:before{content:"\0025cf  "}ul.lst-kix_l4298avyh9v6-0{list-style-type:none}ul.lst-kix_l4298avyh9v6-3{list-style-type:none}.lst-kix_abgphud9refj-1>li:before{content:"\0025cb  "}.lst-kix_abgphud9refj-5>li:before{content:"\0025a0  "}.lst-kix_7ka97cx080q5-7>li{counter-increment:lst-ctn-kix_7ka97cx080q5-7}.lst-kix_mpnpv7ekeo7q-7>li:before{content:"\0025cb  "}ul.lst-kix_l4298avyh9v6-4{list-style-type:none}.lst-kix_eu4b93xn98yk-4>li:before{content:"\0025cb  "}ul.lst-kix_l4298avyh9v6-1{list-style-type:none}ul.lst-kix_l4298avyh9v6-2{list-style-type:none}.lst-kix_eu4b93xn98yk-6>li:before{content:"\0025cf  "}ul.lst-kix_1rk6yybn5non-0{list-style-type:none}ul.lst-kix_uh3r4fp09cs-8{list-style-type:none}.lst-kix_ejnvxy8b1e9w-4>li:before{content:"\0025cb  "}ul.lst-kix_uh3r4fp09cs-5{list-style-type:none}ul.lst-kix_uh3r4fp09cs-4{list-style-type:none}ul.lst-kix_uh3r4fp09cs-7{list-style-type:none}ul.lst-kix_uh3r4fp09cs-6{list-style-type:none}ul.lst-kix_uh3r4fp09cs-1{list-style-type:none}ul.lst-kix_uh3r4fp09cs-0{list-style-type:none}ul.lst-kix_uh3r4fp09cs-3{list-style-type:none}.lst-kix_ejnvxy8b1e9w-0>li:before{content:"\0025cf  "}.lst-kix_ejnvxy8b1e9w-2>li:before{content:"\0025a0  "}ul.lst-kix_uh3r4fp09cs-2{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-3.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-3 0}.lst-kix_jza88dihngw3-7>li:before{content:"\0025cb  "}ul.lst-kix_1rk6yybn5non-6{list-style-type:none}ul.lst-kix_1rk6yybn5non-5{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-4.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-4 0}ul.lst-kix_1rk6yybn5non-8{list-style-type:none}ul.lst-kix_1rk6yybn5non-7{list-style-type:none}ul.lst-kix_1rk6yybn5non-2{list-style-type:none}ul.lst-kix_1rk6yybn5non-1{list-style-type:none}ul.lst-kix_1rk6yybn5non-4{list-style-type:none}.lst-kix_jza88dihngw3-5>li:before{content:"\0025a0  "}ul.lst-kix_1rk6yybn5non-3{list-style-type:none}.lst-kix_tgtwnecgfrou-6>li:before{content:"+  "}ul.lst-kix_t6shmrihc7et-4{list-style-type:none}ul.lst-kix_t6shmrihc7et-3{list-style-type:none}.lst-kix_zhrwxtd2sngd-6>li:before{content:"\0025cf  "}.lst-kix_zhrwxtd2sngd-8>li:before{content:"\0025a0  "}ul.lst-kix_t6shmrihc7et-6{list-style-type:none}.lst-kix_4roymkz9vma-6>li{counter-increment:lst-ctn-kix_4roymkz9vma-6}ul.lst-kix_t6shmrihc7et-5{list-style-type:none}.lst-kix_tgtwnecgfrou-4>li:before{content:"+  "}.lst-kix_tgtwnecgfrou-8>li:before{content:"+  "}ul.lst-kix_t6shmrihc7et-8{list-style-type:none}ul.lst-kix_t6shmrihc7et-7{list-style-type:none}.lst-kix_zhrwxtd2sngd-4>li:before{content:"\0025cb  "}.lst-kix_i2rq8qzg0efa-2>li:before{content:"+  "}.lst-kix_uh3r4fp09cs-4>li:before{content:"\0025cb  "}.lst-kix_uh3r4fp09cs-6>li:before{content:"\0025cf  "}.lst-kix_tgtwnecgfrou-0>li:before{content:"+  "}.lst-kix_jqqhsyijt7h5-3>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-3,decimal) ". "}.lst-kix_i2rq8qzg0efa-0>li:before{content:"+  "}.lst-kix_tgtwnecgfrou-2>li:before{content:"+  "}.lst-kix_9ft7dqndoe5p-8>li:before{content:"\0025a0  "}.lst-kix_ejnvxy8b1e9w-6>li:before{content:"\0025cf  "}.lst-kix_uh3r4fp09cs-8>li:before{content:"\0025a0  "}.lst-kix_ontx89k1fqr9-0>li{counter-increment:lst-ctn-kix_ontx89k1fqr9-0}.lst-kix_tfsdypepg13u-0>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-1>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-1,lower-latin) ". "}.lst-kix_9ft7dqndoe5p-4>li:before{content:"\0025cb  "}.lst-kix_ejnvxy8b1e9w-8>li:before{content:"\0025a0  "}.lst-kix_tfsdypepg13u-2>li:before{content:"\0025a0  "}.lst-kix_tfsdypepg13u-4>li:before{content:"\0025cb  "}.lst-kix_9ft7dqndoe5p-2>li:before{content:"\0025a0  "}.lst-kix_9ft7dqndoe5p-6>li:before{content:"\0025cf  "}.lst-kix_s0bij7o2r512-6>li:before{content:"\0025cf  "}.lst-kix_uh3r4fp09cs-2>li:before{content:"\0025a0  "}.lst-kix_tfsdypepg13u-6>li:before{content:"\0025cf  "}.lst-kix_zhrwxtd2sngd-0>li:before{content:"\0025cf  "}.lst-kix_9ft7dqndoe5p-0>li:before{content:"\0025cf  "}.lst-kix_zhrwxtd2sngd-2>li:before{content:"\0025a0  "}.lst-kix_uh3r4fp09cs-0>li:before{content:"\0025cf  "}.lst-kix_tfsdypepg13u-8>li:before{content:"\0025a0  "}ul.lst-kix_t6shmrihc7et-0{list-style-type:none}.lst-kix_s0bij7o2r512-8>li:before{content:"\0025a0  "}ul.lst-kix_t6shmrihc7et-2{list-style-type:none}ul.lst-kix_t6shmrihc7et-1{list-style-type:none}.lst-kix_cm89sr4fo6m-8>li:before{content:"\0025a0  "}.lst-kix_54trtu6df9f8-5>li:before{content:"\0025a0  "}.lst-kix_7tkvl1c78ae3-5>li:before{content:"\0025a0  "}.lst-kix_4h47981ce058-0>li:before{content:"\0025cf  "}ul.lst-kix_eu4b93xn98yk-7{list-style-type:none}.lst-kix_cm89sr4fo6m-4>li:before{content:"\0025cb  "}ul.lst-kix_eu4b93xn98yk-6{list-style-type:none}.lst-kix_7tkvl1c78ae3-1>li:before{content:"\0025cb  "}ul.lst-kix_eu4b93xn98yk-8{list-style-type:none}ul.lst-kix_eu4b93xn98yk-3{list-style-type:none}.lst-kix_c85u756bkx0j-5>li:before{content:"\0025a0  "}.lst-kix_b96slznfb3gj-4>li:before{content:"-  "}ul.lst-kix_eu4b93xn98yk-2{list-style-type:none}ul.lst-kix_eu4b93xn98yk-5{list-style-type:none}.lst-kix_alje6q6hzub8-7>li:before{content:"-  "}ol.lst-kix_7ka97cx080q5-1.start{counter-reset:lst-ctn-kix_7ka97cx080q5-1 0}ul.lst-kix_eu4b93xn98yk-4{list-style-type:none}.lst-kix_4h47981ce058-4>li:before{content:"\0025cb  "}.lst-kix_cm89sr4fo6m-0>li:before{content:"\0025cf  "}.lst-kix_s0bij7o2r512-2>li:before{content:"\0025a0  "}.lst-kix_f3z068owedlc-4>li:before{content:"\0025cb  "}.lst-kix_f3z068owedlc-8>li:before{content:"\0025a0  "}.lst-kix_p4k358qyzf3y-3>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-3}ol.lst-kix_h9p9lo20tlpj-8.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-8 0}.lst-kix_4r80w4zgcdjq-5>li:before{content:"\0025a0  "}.lst-kix_e08effp12mqf-5>li:before{content:"\0025a0  "}.lst-kix_b96slznfb3gj-8>li:before{content:"-  "}.lst-kix_c85u756bkx0j-1>li:before{content:"\0025cb  "}.lst-kix_alje6q6hzub8-3>li:before{content:"-  "}.lst-kix_fyzn24yyf01g-3>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-7>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-7,lower-latin) ". "}ul.lst-kix_tcwoy9lfuqk4-2{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-1{list-style-type:none}ol.lst-kix_p4k358qyzf3y-7.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-7 0}ul.lst-kix_emh9tw6ivvnv-0{list-style-type:none}ul.lst-kix_jza88dihngw3-0{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-0{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-1{list-style-type:none}ul.lst-kix_jza88dihngw3-1{list-style-type:none}.lst-kix_f3z068owedlc-0>li:before{content:"\0025cf  "}.lst-kix_i2rq8qzg0efa-8>li:before{content:"+  "}ul.lst-kix_emh9tw6ivvnv-6{list-style-type:none}ul.lst-kix_jza88dihngw3-6{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-7{list-style-type:none}ul.lst-kix_jza88dihngw3-7{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-8{list-style-type:none}ul.lst-kix_jza88dihngw3-8{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-8{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-7{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-2{list-style-type:none}ul.lst-kix_jza88dihngw3-2{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-6{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-3{list-style-type:none}ul.lst-kix_jza88dihngw3-3{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-5{list-style-type:none}ul.lst-kix_emh9tw6ivvnv-4{list-style-type:none}ul.lst-kix_jza88dihngw3-4{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-4{list-style-type:none}.lst-kix_p4k358qyzf3y-5>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-5}ul.lst-kix_emh9tw6ivvnv-5{list-style-type:none}ul.lst-kix_jza88dihngw3-5{list-style-type:none}ul.lst-kix_tcwoy9lfuqk4-3{list-style-type:none}.lst-kix_1293ztc79klg-7>li:before{content:"\0025cb  "}.lst-kix_h9p9lo20tlpj-3>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-3}.lst-kix_i2rq8qzg0efa-4>li:before{content:"+  "}.lst-kix_jqqhsyijt7h5-4>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-4}.lst-kix_54trtu6df9f8-1>li:before{content:"\0025cb  "}.lst-kix_1293ztc79klg-3>li:before{content:"\0025cf  "}ol.lst-kix_nzxgmnmwjatv-2.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-2 0}ul.lst-kix_n2tc9z47p0ha-4{list-style-type:none}ul.lst-kix_n2tc9z47p0ha-3{list-style-type:none}ul.lst-kix_25j83v4xa4ig-8{list-style-type:none}.lst-kix_f7u4w07k18eh-1>li:before{content:"\0025cb  "}.lst-kix_icjmrj4tgidb-4>li:before{content:"-  "}ul.lst-kix_n2tc9z47p0ha-2{list-style-type:none}.lst-kix_s27gvee6ygqb-8>li:before{content:"\0025a0  "}ul.lst-kix_n2tc9z47p0ha-1{list-style-type:none}ul.lst-kix_n2tc9z47p0ha-8{list-style-type:none}ul.lst-kix_n2tc9z47p0ha-7{list-style-type:none}ul.lst-kix_n2tc9z47p0ha-6{list-style-type:none}ul.lst-kix_n2tc9z47p0ha-5{list-style-type:none}.lst-kix_yk52z9n95trk-1>li:before{content:"\0025cb  "}.lst-kix_aninp2c52sy3-8>li:before{content:"\0025a0  "}.lst-kix_x30zykguitxy-2>li:before{content:"\0025a0  "}.lst-kix_s27gvee6ygqb-0>li:before{content:"\0025cf  "}.lst-kix_icjmrj4tgidb-8>li:before{content:"-  "}.lst-kix_x3ptir2y8jgm-5>li:before{content:"\0025a0  "}.lst-kix_yk52z9n95trk-5>li:before{content:"\0025a0  "}ul.lst-kix_25j83v4xa4ig-1{list-style-type:none}.lst-kix_x30zykguitxy-6>li:before{content:"\0025cf  "}ul.lst-kix_25j83v4xa4ig-0{list-style-type:none}ul.lst-kix_25j83v4xa4ig-3{list-style-type:none}ul.lst-kix_25j83v4xa4ig-2{list-style-type:none}.lst-kix_o1wsnr4piq1b-1>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-1,lower-latin) ") "}ul.lst-kix_n2tc9z47p0ha-0{list-style-type:none}ul.lst-kix_25j83v4xa4ig-5{list-style-type:none}.lst-kix_aninp2c52sy3-4>li:before{content:"\0025cb  "}ul.lst-kix_25j83v4xa4ig-4{list-style-type:none}.lst-kix_v26ncletnxud-5>li:before{content:"\0025a0  "}ul.lst-kix_25j83v4xa4ig-7{list-style-type:none}ul.lst-kix_25j83v4xa4ig-6{list-style-type:none}.lst-kix_e08effp12mqf-1>li:before{content:"\0025cb  "}.lst-kix_x3ptir2y8jgm-1>li:before{content:"\0025cb  "}.lst-kix_ieipvto6nfrd-2>li:before{content:"-  "}.lst-kix_ieipvto6nfrd-6>li:before{content:"-  "}.lst-kix_4h47981ce058-8>li:before{content:"\0025a0  "}.lst-kix_2w2qcvvb39hd-7>li:before{content:"\0025cb  "}.lst-kix_4roymkz9vma-6>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) "." counter(lst-ctn-kix_4roymkz9vma-4,decimal) "." counter(lst-ctn-kix_4roymkz9vma-5,decimal) "." counter(lst-ctn-kix_4roymkz9vma-6,decimal) ". "}.lst-kix_fyzn24yyf01g-7>li:before{content:"\0025cb  "}.lst-kix_icjmrj4tgidb-0>li:before{content:"-  "}.lst-kix_s27gvee6ygqb-4>li:before{content:"\0025cb  "}.lst-kix_f7u4w07k18eh-5>li:before{content:"\0025a0  "}.lst-kix_2w2qcvvb39hd-3>li:before{content:"\0025cf  "}.lst-kix_b96slznfb3gj-0>li:before{content:"-  "}.lst-kix_4roymkz9vma-2>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) ". "}ul.lst-kix_yqx6kz4lvbt4-8{list-style-type:none}.lst-kix_qswbihp1i06v-7>li:before{content:"\0025cb  "}.lst-kix_xprjjxgruvg-0>li:before{content:"\0025cf  "}ul.lst-kix_yqx6kz4lvbt4-1{list-style-type:none}ul.lst-kix_yqx6kz4lvbt4-0{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-4.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-4 0}ul.lst-kix_yqx6kz4lvbt4-3{list-style-type:none}.lst-kix_4roymkz9vma-4>li{counter-increment:lst-ctn-kix_4roymkz9vma-4}.lst-kix_xwc7ibkfiucl-1>li:before{content:"\0025cb  "}ul.lst-kix_yqx6kz4lvbt4-2{list-style-type:none}ul.lst-kix_yqx6kz4lvbt4-5{list-style-type:none}ul.lst-kix_yqx6kz4lvbt4-4{list-style-type:none}.lst-kix_m3f8ler1ak4c-5>li:before{content:"\0025a0  "}ul.lst-kix_yqx6kz4lvbt4-7{list-style-type:none}ul.lst-kix_yqx6kz4lvbt4-6{list-style-type:none}.lst-kix_xprjjxgruvg-8>li:before{content:"\0025a0  "}.lst-kix_4tgbgh9bb7g-6>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-6}.lst-kix_fv8y17g7myim-5>li:before{content:"\0025a0  "}.lst-kix_xprjjxgruvg-4>li:before{content:"\0025cb  "}.lst-kix_o1wsnr4piq1b-5>li:before{content:"(" counter(lst-ctn-kix_o1wsnr4piq1b-5,lower-roman) ") "}.lst-kix_aninp2c52sy3-0>li:before{content:"\0025cf  "}.lst-kix_v26ncletnxud-1>li:before{content:"\0025cb  "}.lst-kix_m3f8ler1ak4c-1>li:before{content:"\0025cb  "}.lst-kix_ouaz0d8j7xj-7>li:before{content:"\0025cb  "}.lst-kix_qswbihp1i06v-3>li:before{content:"\0025cf  "}.lst-kix_oli7yc1w3gpg-6>li:before{content:"\0025cf  "}ol.lst-kix_4tgbgh9bb7g-6.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-6 0}.lst-kix_h9p9lo20tlpj-5>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-5}.lst-kix_ouaz0d8j7xj-3>li:before{content:"\0025cf  "}.lst-kix_4fkgc27ss5mq-3>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-3,decimal) ". "}.lst-kix_z9n4e0cpc149-0>li:before{content:"\0025cf  "}.lst-kix_25j83v4xa4ig-7>li:before{content:"\0025cb  "}.lst-kix_c4nzmsldypra-7>li:before{content:"\0025cb  "}ul.lst-kix_qffkk51zafur-0{list-style-type:none}.lst-kix_lux5kw5rxj0b-0>li:before{content:"\0025cf  "}.lst-kix_oli7yc1w3gpg-2>li:before{content:"\0025a0  "}ul.lst-kix_qffkk51zafur-1{list-style-type:none}.lst-kix_s8oehts9vu79-3>li:before{content:"\0025cf  "}ul.lst-kix_qffkk51zafur-4{list-style-type:none}.lst-kix_fv8y17g7myim-1>li:before{content:"\0025cb  "}.lst-kix_z9n4e0cpc149-4>li:before{content:"\0025cb  "}ul.lst-kix_qffkk51zafur-5{list-style-type:none}ul.lst-kix_qffkk51zafur-2{list-style-type:none}ul.lst-kix_qffkk51zafur-3{list-style-type:none}ul.lst-kix_qffkk51zafur-8{list-style-type:none}.lst-kix_k9nxvn8n4dkv-6>li:before{content:"\0025cf  "}ul.lst-kix_qffkk51zafur-6{list-style-type:none}.lst-kix_gayel7f32yde-8>li:before{content:"-  "}ul.lst-kix_qffkk51zafur-7{list-style-type:none}.lst-kix_c4nzmsldypra-3>li:before{content:"\0025cf  "}.lst-kix_lux5kw5rxj0b-4>li:before{content:"\0025cb  "}.lst-kix_s8oehts9vu79-7>li:before{content:"\0025cb  "}.lst-kix_4r80w4zgcdjq-1>li:before{content:"\0025cb  "}.lst-kix_k9nxvn8n4dkv-2>li:before{content:"\0025a0  "}.lst-kix_gayel7f32yde-0>li:before{content:"-  "}.lst-kix_gayel7f32yde-4>li:before{content:"-  "}.lst-kix_z9n4e0cpc149-8>li:before{content:"\0025a0  "}.lst-kix_n2tc9z47p0ha-1>li:before{content:"\0025cb  "}ul.lst-kix_eu4b93xn98yk-1{list-style-type:none}ul.lst-kix_eu4b93xn98yk-0{list-style-type:none}.lst-kix_xwc7ibkfiucl-5>li:before{content:"\0025a0  "}.lst-kix_4fkgc27ss5mq-7>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-7,lower-latin) ". "}.lst-kix_n2tc9z47p0ha-5>li:before{content:"\0025a0  "}ul.lst-kix_m3f8ler1ak4c-6{list-style-type:none}ul.lst-kix_m3f8ler1ak4c-7{list-style-type:none}ul.lst-kix_m3f8ler1ak4c-4{list-style-type:none}.lst-kix_o1wsnr4piq1b-0>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-0}ul.lst-kix_m3f8ler1ak4c-5{list-style-type:none}ul.lst-kix_m3f8ler1ak4c-2{list-style-type:none}ul.lst-kix_m3f8ler1ak4c-3{list-style-type:none}.lst-kix_o9ptu63d1d74-2>li:before{content:"\0025a0  "}.lst-kix_cpm4oaqfjy6b-4>li:before{content:"\0025cb  "}ul.lst-kix_m3f8ler1ak4c-0{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-8{list-style-type:none}ul.lst-kix_m3f8ler1ak4c-1{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-0{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-1{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-2{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-3{list-style-type:none}.lst-kix_cpm4oaqfjy6b-2>li:before{content:"\0025a0  "}ul.lst-kix_jm1l4y2hlnt3-4{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-5{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-6{list-style-type:none}.lst-kix_7ka97cx080q5-3>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-3,decimal) ". "}ol.lst-kix_h9p9lo20tlpj-0.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-0 0}.lst-kix_7ka97cx080q5-1>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-1,lower-latin) ". "}ul.lst-kix_m3f8ler1ak4c-8{list-style-type:none}.lst-kix_uhfsw7q6s458-3>li:before{content:"\0025cf  "}.lst-kix_o9ptu63d1d74-8>li:before{content:"\0025a0  "}.lst-kix_ontx89k1fqr9-1>li:before{content:"\0025cb  "}ul.lst-kix_gayel7f32yde-2{list-style-type:none}ul.lst-kix_gayel7f32yde-1{list-style-type:none}ul.lst-kix_gayel7f32yde-0{list-style-type:none}ul.lst-kix_gayel7f32yde-6{list-style-type:none}ul.lst-kix_gayel7f32yde-5{list-style-type:none}ul.lst-kix_gayel7f32yde-4{list-style-type:none}ul.lst-kix_gayel7f32yde-3{list-style-type:none}ul.lst-kix_gayel7f32yde-8{list-style-type:none}ul.lst-kix_gayel7f32yde-7{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-7{list-style-type:none}.lst-kix_dkq97x3e7jfk-8>li:before{content:"\0025a0  "}ol.lst-kix_nzxgmnmwjatv-6{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-5{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-4{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-3{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-2{list-style-type:none}.lst-kix_ontx89k1fqr9-3>li:before{content:"\0025cf  "}ol.lst-kix_nzxgmnmwjatv-1{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-0{list-style-type:none}.lst-kix_x7o2gv80my2k-3>li:before{content:"\0025cf  "}.lst-kix_x7o2gv80my2k-5>li:before{content:"\0025a0  "}.lst-kix_nr7rbh2hfyf0-0>li:before{content:"\0025cf  "}.lst-kix_nr7rbh2hfyf0-8>li:before{content:"\0025a0  "}.lst-kix_xiiy75ihy1il-1>li:before{content:"\0025cb  "}.lst-kix_xzp0xel2onox-8>li:before{content:"\0025a0  "}.lst-kix_nr7rbh2hfyf0-6>li:before{content:"\0025cf  "}ol.lst-kix_h9p9lo20tlpj-5.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-5 0}.lst-kix_qqf2twdkbiw0-0>li:before{content:"\0025cf  "}.lst-kix_xzp0xel2onox-0>li:before{content:"\0025cf  "}.lst-kix_uhfsw7q6s458-1>li:before{content:"\0025cb  "}.lst-kix_xzp0xel2onox-6>li:before{content:"\0025cf  "}.lst-kix_xiiy75ihy1il-3>li:before{content:"\0025cf  "}.lst-kix_bv0w79unf8ye-3>li:before{content:"\0025cf  "}.lst-kix_qqf2twdkbiw0-8>li:before{content:"\0025a0  "}.lst-kix_bv0w79unf8ye-5>li:before{content:"\0025a0  "}.lst-kix_vxbaz02t98k3-6>li:before{content:"\0025cf  "}.lst-kix_qqf2twdkbiw0-6>li:before{content:"\0025cf  "}.lst-kix_vxbaz02t98k3-4>li:before{content:"\0025cb  "}.lst-kix_o9ptu63d1d74-0>li:before{content:"\0025cf  "}ul.lst-kix_9tajp1agqiyw-5{list-style-type:none}.lst-kix_4h54ot51egiu-0>li:before{content:"\0025cf  "}.lst-kix_e21xjkb2e68h-4>li:before{content:"\0025cb  "}ul.lst-kix_9tajp1agqiyw-4{list-style-type:none}ul.lst-kix_9tajp1agqiyw-7{list-style-type:none}ul.lst-kix_9tajp1agqiyw-6{list-style-type:none}ul.lst-kix_9tajp1agqiyw-1{list-style-type:none}.lst-kix_9bzwod9gvkvp-7>li:before{content:"\0025cb  "}ul.lst-kix_9tajp1agqiyw-0{list-style-type:none}ul.lst-kix_9tajp1agqiyw-3{list-style-type:none}ul.lst-kix_9tajp1agqiyw-2{list-style-type:none}.lst-kix_9bzwod9gvkvp-5>li:before{content:"\0025a0  "}.lst-kix_ygjhinrwstz-3>li:before{content:"\0025cf  "}.lst-kix_4h54ot51egiu-2>li:before{content:"\0025a0  "}.lst-kix_e21xjkb2e68h-2>li:before{content:"\0025a0  "}ul.lst-kix_9tajp1agqiyw-8{list-style-type:none}ol.lst-kix_uthm435lclg8-0.start{counter-reset:lst-ctn-kix_uthm435lclg8-0 0}ul.lst-kix_abgphud9refj-5{list-style-type:none}ul.lst-kix_abgphud9refj-4{list-style-type:none}ul.lst-kix_abgphud9refj-3{list-style-type:none}ul.lst-kix_abgphud9refj-2{list-style-type:none}ul.lst-kix_abgphud9refj-1{list-style-type:none}ul.lst-kix_abgphud9refj-0{list-style-type:none}.lst-kix_ock170xmc4e5-3>li:before{content:"\0025cf  "}.lst-kix_4roymkz9vma-8>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) "." counter(lst-ctn-kix_4roymkz9vma-4,decimal) "." counter(lst-ctn-kix_4roymkz9vma-5,decimal) "." counter(lst-ctn-kix_4roymkz9vma-6,decimal) "." counter(lst-ctn-kix_4roymkz9vma-7,decimal) "." counter(lst-ctn-kix_4roymkz9vma-8,decimal) ". "}.lst-kix_ur253ckj31qe-0>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-0,decimal) ". "}.lst-kix_ygjhinrwstz-1>li:before{content:"\0025cb  "}.lst-kix_9tajp1agqiyw-0>li:before{content:"\0025cf  "}.lst-kix_ur253ckj31qe-2>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-2,lower-roman) ". "}ul.lst-kix_abgphud9refj-8{list-style-type:none}ul.lst-kix_abgphud9refj-7{list-style-type:none}.lst-kix_ock170xmc4e5-1>li:before{content:"\0025cb  "}ul.lst-kix_abgphud9refj-6{list-style-type:none}.lst-kix_a0ivi9b1qp8i-0>li:before{content:"\0025cf  "}.lst-kix_g6dv2r60jppe-3>li:before{content:"\0025cf  "}ul.lst-kix_tgtwnecgfrou-0{list-style-type:none}ul.lst-kix_tgtwnecgfrou-1{list-style-type:none}.lst-kix_4h54ot51egiu-8>li:before{content:"\0025a0  "}.lst-kix_l4298avyh9v6-2>li:before{content:"\0025a0  "}ul.lst-kix_tgtwnecgfrou-4{list-style-type:none}ul.lst-kix_tgtwnecgfrou-5{list-style-type:none}ul.lst-kix_tgtwnecgfrou-2{list-style-type:none}.lst-kix_9tajp1agqiyw-6>li:before{content:"\0025cf  "}ul.lst-kix_tgtwnecgfrou-3{list-style-type:none}ul.lst-kix_tgtwnecgfrou-8{list-style-type:none}ul.lst-kix_tgtwnecgfrou-6{list-style-type:none}.lst-kix_g6dv2r60jppe-5>li:before{content:"\0025a0  "}ul.lst-kix_tgtwnecgfrou-7{list-style-type:none}ol.lst-kix_7ka97cx080q5-3.start{counter-reset:lst-ctn-kix_7ka97cx080q5-3 0}.lst-kix_9tajp1agqiyw-8>li:before{content:"\0025a0  "}.lst-kix_k0c3ozmp0pnp-6>li:before{content:"\0025cf  "}ul.lst-kix_v26ncletnxud-2{list-style-type:none}ul.lst-kix_v26ncletnxud-3{list-style-type:none}ul.lst-kix_v26ncletnxud-0{list-style-type:none}ul.lst-kix_v26ncletnxud-1{list-style-type:none}ul.lst-kix_v26ncletnxud-8{list-style-type:none}ul.lst-kix_v26ncletnxud-6{list-style-type:none}ul.lst-kix_v26ncletnxud-7{list-style-type:none}ul.lst-kix_v26ncletnxud-4{list-style-type:none}ul.lst-kix_v26ncletnxud-5{list-style-type:none}.lst-kix_l4298avyh9v6-4>li:before{content:"\0025cb  "}ol.lst-kix_4fkgc27ss5mq-5.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-5 0}.lst-kix_ric95po2pkzv-1>li:before{content:"\0025cb  "}.lst-kix_gmyqb4k4mims-5>li:before{content:"\0025a0  "}.lst-kix_uthm435lclg8-7>li:before{content:"\0025cb  "}.lst-kix_gmyqb4k4mims-7>li:before{content:"\0025cb  "}.lst-kix_k0c3ozmp0pnp-0>li:before{content:"\0025cf  "}.lst-kix_a0ivi9b1qp8i-2>li:before{content:"\0025a0  "}.lst-kix_ric95po2pkzv-3>li:before{content:"\0025cf  "}ul.lst-kix_nr7rbh2hfyf0-4{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-3{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-2{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-1{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-8{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-7{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-6{list-style-type:none}ul.lst-kix_nr7rbh2hfyf0-5{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-3.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-3 0}.lst-kix_o1wsnr4piq1b-7>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-7}ul.lst-kix_nr7rbh2hfyf0-0{list-style-type:none}ul.lst-kix_qwci6b6x7du4-6{list-style-type:none}.lst-kix_a0ivi9b1qp8i-8>li:before{content:"\0025a0  "}ul.lst-kix_qwci6b6x7du4-5{list-style-type:none}ul.lst-kix_qwci6b6x7du4-4{list-style-type:none}ul.lst-kix_qwci6b6x7du4-3{list-style-type:none}ol.lst-kix_p4k358qyzf3y-5.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-5 0}ul.lst-kix_qwci6b6x7du4-8{list-style-type:none}ul.lst-kix_qwci6b6x7du4-7{list-style-type:none}.lst-kix_ur253ckj31qe-8>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-8,lower-roman) ". "}.lst-kix_4tgbgh9bb7g-8>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-8}.lst-kix_k0c3ozmp0pnp-8>li:before{content:"\0025a0  "}ul.lst-kix_qwci6b6x7du4-2{list-style-type:none}ul.lst-kix_qwci6b6x7du4-1{list-style-type:none}ul.lst-kix_qwci6b6x7du4-0{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-6.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-6 0}.lst-kix_7tkvl1c78ae3-3>li:before{content:"\0025cf  "}.lst-kix_54trtu6df9f8-7>li:before{content:"\0025cb  "}ol.lst-kix_4roymkz9vma-6.start{counter-reset:lst-ctn-kix_4roymkz9vma-6 0}.lst-kix_c85u756bkx0j-3>li:before{content:"\0025cf  "}.lst-kix_b96slznfb3gj-2>li:before{content:"-  "}.lst-kix_2w2qcvvb39hd-1>li:before{content:"\0025cb  "}.lst-kix_cm89sr4fo6m-6>li:before{content:"\0025cf  "}ul.lst-kix_uthm435lclg8-2{list-style-type:none}ul.lst-kix_uthm435lclg8-1{list-style-type:none}ul.lst-kix_uthm435lclg8-4{list-style-type:none}ul.lst-kix_uthm435lclg8-3{list-style-type:none}.lst-kix_4r80w4zgcdjq-7>li:before{content:"\0025cb  "}ul.lst-kix_n2keu386kfur-7{list-style-type:none}.lst-kix_alje6q6hzub8-5>li:before{content:"-  "}ul.lst-kix_n2keu386kfur-8{list-style-type:none}ul.lst-kix_n2keu386kfur-5{list-style-type:none}ul.lst-kix_n2keu386kfur-6{list-style-type:none}.lst-kix_h9p9lo20tlpj-1>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-1}ul.lst-kix_n2keu386kfur-3{list-style-type:none}ul.lst-kix_n2keu386kfur-4{list-style-type:none}.lst-kix_4tgbgh9bb7g-4>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-4,lower-latin) ". "}.lst-kix_s0bij7o2r512-0>li:before{content:"\0025cf  "}ul.lst-kix_n2keu386kfur-1{list-style-type:none}ul.lst-kix_uthm435lclg8-6{list-style-type:none}ul.lst-kix_n2keu386kfur-2{list-style-type:none}ul.lst-kix_uthm435lclg8-5{list-style-type:none}ul.lst-kix_uthm435lclg8-8{list-style-type:none}ul.lst-kix_n2keu386kfur-0{list-style-type:none}ul.lst-kix_uthm435lclg8-7{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-8{list-style-type:none}ul.lst-kix_yk52z9n95trk-4{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-7{list-style-type:none}ul.lst-kix_yk52z9n95trk-5{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-6{list-style-type:none}ul.lst-kix_yk52z9n95trk-6{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-5{list-style-type:none}ul.lst-kix_yk52z9n95trk-7{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-4{list-style-type:none}ul.lst-kix_yk52z9n95trk-0{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-3{list-style-type:none}ul.lst-kix_yk52z9n95trk-1{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-2{list-style-type:none}ul.lst-kix_yk52z9n95trk-2{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-1{list-style-type:none}.lst-kix_fyzn24yyf01g-1>li:before{content:"\0025cb  "}ul.lst-kix_yk52z9n95trk-3{list-style-type:none}ul.lst-kix_xm7ynypsh0z5-0{list-style-type:none}.lst-kix_f3z068owedlc-2>li:before{content:"\0025a0  "}.lst-kix_7ka97cx080q5-0>li{counter-increment:lst-ctn-kix_7ka97cx080q5-0}ul.lst-kix_yk52z9n95trk-8{list-style-type:none}.lst-kix_e08effp12mqf-7>li:before{content:"\0025cb  "}.lst-kix_4roymkz9vma-8>li{counter-increment:lst-ctn-kix_4roymkz9vma-8}.lst-kix_y5u0wek48eld-7>li:before{content:"-  "}.lst-kix_uthm435lclg8-1>li:before{content:"\0025cb  "}.lst-kix_1293ztc79klg-5>li:before{content:"\0025a0  "}.lst-kix_i2rq8qzg0efa-6>li:before{content:"+  "}ul.lst-kix_vlcqhp8hs5e3-4{list-style-type:none}ul.lst-kix_vlcqhp8hs5e3-3{list-style-type:none}.lst-kix_icjmrj4tgidb-2>li:before{content:"-  "}ul.lst-kix_vlcqhp8hs5e3-2{list-style-type:none}ul.lst-kix_vlcqhp8hs5e3-1{list-style-type:none}ul.lst-kix_vlcqhp8hs5e3-0{list-style-type:none}.lst-kix_f7u4w07k18eh-3>li:before{content:"\0025cf  "}ol.lst-kix_7ka97cx080q5-6.start{counter-reset:lst-ctn-kix_7ka97cx080q5-6 0}.lst-kix_4fkgc27ss5mq-0>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-0}.lst-kix_o1wsnr4piq1b-3>li:before{content:"(" counter(lst-ctn-kix_o1wsnr4piq1b-3,decimal) ") "}.lst-kix_aninp2c52sy3-2>li:before{content:"\0025a0  "}.lst-kix_hxvvdzewxqs2-1>li:before{content:"\0025cb  "}.lst-kix_v26ncletnxud-7>li:before{content:"\0025cb  "}.lst-kix_x3ptir2y8jgm-3>li:before{content:"\0025cf  "}.lst-kix_d1mamalvi3wu-8>li:before{content:"\0025a0  "}.lst-kix_x30zykguitxy-4>li:before{content:"\0025cb  "}ul.lst-kix_vlcqhp8hs5e3-8{list-style-type:none}ul.lst-kix_vlcqhp8hs5e3-7{list-style-type:none}ul.lst-kix_vlcqhp8hs5e3-6{list-style-type:none}.lst-kix_yk52z9n95trk-7>li:before{content:"\0025cb  "}ul.lst-kix_vlcqhp8hs5e3-5{list-style-type:none}ol.lst-kix_p4k358qyzf3y-2.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-2 0}.lst-kix_4roymkz9vma-0>li:before{content:"\0025cf  "}.lst-kix_4h47981ce058-6>li:before{content:"\0025cf  "}.lst-kix_ieipvto6nfrd-4>li:before{content:"-  "}.lst-kix_jqqhsyijt7h5-8>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-8}.lst-kix_s27gvee6ygqb-2>li:before{content:"\0025a0  "}.lst-kix_ur253ckj31qe-5>li{counter-increment:lst-ctn-kix_ur253ckj31qe-5}.lst-kix_a76b6w8pnkyv-6>li:before{content:"\0025cf  "}.lst-kix_wxhgjniia6a-5>li:before{content:"\0025a0  "}.lst-kix_ia5tg78e6pm4-3>li:before{content:"\0025cf  "}.lst-kix_d1mamalvi3wu-0>li:before{content:"\0025cf  "}.lst-kix_ns4c34lwftpb-2>li:before{content:"\0025a0  "}.lst-kix_k08o05q2crtr-5>li:before{content:"\0025a0  "}ol.lst-kix_7ka97cx080q5-8.start{counter-reset:lst-ctn-kix_7ka97cx080q5-8 0}ul.lst-kix_ia5tg78e6pm4-8{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-0.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-0 0}ul.lst-kix_ia5tg78e6pm4-7{list-style-type:none}ul.lst-kix_ia5tg78e6pm4-6{list-style-type:none}.lst-kix_jduurcgheuo-7>li:before{content:"-  "}ul.lst-kix_ia5tg78e6pm4-5{list-style-type:none}ul.lst-kix_ia5tg78e6pm4-4{list-style-type:none}ul.lst-kix_ia5tg78e6pm4-3{list-style-type:none}ul.lst-kix_ia5tg78e6pm4-2{list-style-type:none}ul.lst-kix_ia5tg78e6pm4-1{list-style-type:none}.lst-kix_qswbihp1i06v-5>li:before{content:"\0025a0  "}.lst-kix_mi7vyzrjrpd8-7>li:before{content:"\0025cb  "}ul.lst-kix_ia5tg78e6pm4-0{list-style-type:none}.lst-kix_emh9tw6ivvnv-7>li:before{content:"\0025cb  "}ul.lst-kix_4h47981ce058-6{list-style-type:none}ul.lst-kix_4h47981ce058-5{list-style-type:none}ul.lst-kix_4h47981ce058-4{list-style-type:none}.lst-kix_nzxgmnmwjatv-0>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-0}ul.lst-kix_4h47981ce058-3{list-style-type:none}.lst-kix_xwc7ibkfiucl-3>li:before{content:"\0025cf  "}.lst-kix_l9w5fm8md2s9-3>li:before{content:"\0025cf  "}ul.lst-kix_4h47981ce058-8{list-style-type:none}ul.lst-kix_4h47981ce058-7{list-style-type:none}ul.lst-kix_4h47981ce058-2{list-style-type:none}ul.lst-kix_4h47981ce058-1{list-style-type:none}ul.lst-kix_4h47981ce058-0{list-style-type:none}.lst-kix_fv8y17g7myim-7>li:before{content:"\0025cb  "}.lst-kix_xprjjxgruvg-6>li:before{content:"\0025cf  "}ol.lst-kix_p4k358qyzf3y-0.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-0 0}.lst-kix_wawbbtzgf6il-1>li:before{content:"\0025cb  "}.lst-kix_n2keu386kfur-5>li:before{content:"\0025a0  "}.lst-kix_xm7ynypsh0z5-7>li:before{content:"\0025cb  "}.lst-kix_s8oehts9vu79-1>li:before{content:"\0025cb  "}.lst-kix_6bb7qu9vjqr2-2>li:before{content:"-  "}.lst-kix_25j83v4xa4ig-5>li:before{content:"\0025a0  "}.lst-kix_nwfayb939eln-4>li:before{content:"\0025cb  "}.lst-kix_oli7yc1w3gpg-4>li:before{content:"\0025cb  "}ul.lst-kix_a76b6w8pnkyv-0{list-style-type:none}.lst-kix_cxongm456yyi-2>li:before{content:"\0025a0  "}.lst-kix_e5u94rrvoqs-5>li:before{content:"\0025a0  "}ul.lst-kix_a76b6w8pnkyv-1{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-2{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-3{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-4{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-5{list-style-type:none}.lst-kix_ouaz0d8j7xj-5>li:before{content:"\0025a0  "}ul.lst-kix_a76b6w8pnkyv-6{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-7{list-style-type:none}ul.lst-kix_a76b6w8pnkyv-8{list-style-type:none}.lst-kix_h9p9lo20tlpj-6>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-6,decimal) ". "}.lst-kix_z9n4e0cpc149-2>li:before{content:"\0025a0  "}.lst-kix_tcwoy9lfuqk4-7>li:before{content:"\0025cb  "}.lst-kix_c4nzmsldypra-1>li:before{content:"\0025cb  "}.lst-kix_k9nxvn8n4dkv-8>li:before{content:"\0025a0  "}.lst-kix_dkq97x3e7jfk-2>li:before{content:"\0025a0  "}.lst-kix_o6fhz26vvidv-5>li:before{content:"\0025a0  "}.lst-kix_w7vhjpenesq3-1>li:before{content:"\0025cb  "}.lst-kix_4fkgc27ss5mq-7>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-7}.lst-kix_4fkgc27ss5mq-1>li:before{content:"\0025cb  "}.lst-kix_7ev7v5cpgtkc-4>li:before{content:"\0025cb  "}.lst-kix_k9nxvn8n4dkv-0>li:before{content:"\0025cf  "}ul.lst-kix_hxvvdzewxqs2-0{list-style-type:none}.lst-kix_m3f8ler1ak4c-7>li:before{content:"\0025cb  "}.lst-kix_flatlsyq6jr0-2>li:before{content:"\0025a0  "}.lst-kix_nzxgmnmwjatv-7>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-7}.lst-kix_o1wsnr4piq1b-4>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-4}.lst-kix_fcckwvdxoljx-6>li:before{content:"-  "}ol.lst-kix_4roymkz9vma-5.start{counter-reset:lst-ctn-kix_4roymkz9vma-5 0}.lst-kix_zvvafajkk2a-3>li:before{content:"\0025cf  "}.lst-kix_gayel7f32yde-2>li:before{content:"-  "}ul.lst-kix_jm1l4y2hlnt3-7{list-style-type:none}ul.lst-kix_jm1l4y2hlnt3-8{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-7{list-style-type:none}.lst-kix_n2tc9z47p0ha-7>li:before{content:"\0025cb  "}ul.lst-kix_hxvvdzewxqs2-8{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-5{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-6{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-3{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-4{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-1{list-style-type:none}ul.lst-kix_hxvvdzewxqs2-2{list-style-type:none}.lst-kix_8rc65o4ywyr4-0>li:before{content:"\0025cf  "}.lst-kix_8f57fsr386yf-0>li:before{content:"\0025cf  "}ul.lst-kix_xwc7ibkfiucl-8{list-style-type:none}ul.lst-kix_xwc7ibkfiucl-7{list-style-type:none}ul.lst-kix_xwc7ibkfiucl-6{list-style-type:none}ul.lst-kix_xwc7ibkfiucl-5{list-style-type:none}.lst-kix_xqkxylhicw49-8>li:before{content:"\0025a0  "}.lst-kix_8rc65o4ywyr4-3>li:before{content:"\0025cf  "}.lst-kix_tczavuk46t7o-6>li:before{content:"\0025cf  "}.lst-kix_8rc65o4ywyr4-6>li:before{content:"\0025cf  "}.lst-kix_8f57fsr386yf-7>li:before{content:"\0025cb  "}ol.lst-kix_4fkgc27ss5mq-0{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-3{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-2{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-5{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-4{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-7{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-6{list-style-type:none}ul.lst-kix_xwc7ibkfiucl-0{list-style-type:none}.lst-kix_xqkxylhicw49-2>li:before{content:"\0025a0  "}ul.lst-kix_xwc7ibkfiucl-4{list-style-type:none}.lst-kix_8f57fsr386yf-3>li:before{content:"\0025cf  "}ul.lst-kix_xwc7ibkfiucl-3{list-style-type:none}ul.lst-kix_xwc7ibkfiucl-2{list-style-type:none}.lst-kix_8f57fsr386yf-4>li:before{content:"\0025cb  "}.lst-kix_xqkxylhicw49-5>li:before{content:"\0025a0  "}ul.lst-kix_xwc7ibkfiucl-1{list-style-type:none}.lst-kix_8f57fsr386yf-1>li:before{content:"\0025cb  "}.lst-kix_jqqhsyijt7h5-1>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-1}.lst-kix_1rk6yybn5non-4>li:before{content:"-  "}.lst-kix_nzxgmnmwjatv-8>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-8,lower-roman) ". "}ol.lst-kix_h9p9lo20tlpj-6.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-6 0}ol.lst-kix_o1wsnr4piq1b-4.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-4 0}.lst-kix_qwci6b6x7du4-5>li:before{content:"-  "}.lst-kix_vlcqhp8hs5e3-4>li:before{content:"\0025cb  "}.lst-kix_qwci6b6x7du4-6>li:before{content:"-  "}.lst-kix_1rk6yybn5non-5>li:before{content:"-  "}ul.lst-kix_aninp2c52sy3-4{list-style-type:none}.lst-kix_qwci6b6x7du4-2>li:before{content:"-  "}ul.lst-kix_aninp2c52sy3-5{list-style-type:none}.lst-kix_vlcqhp8hs5e3-7>li:before{content:"\0025cb  "}.lst-kix_vlcqhp8hs5e3-8>li:before{content:"\0025a0  "}ul.lst-kix_aninp2c52sy3-6{list-style-type:none}ul.lst-kix_aninp2c52sy3-7{list-style-type:none}ul.lst-kix_aninp2c52sy3-0{list-style-type:none}.lst-kix_qwci6b6x7du4-0>li:before{content:"-  "}.lst-kix_qwci6b6x7du4-3>li:before{content:"-  "}ul.lst-kix_aninp2c52sy3-1{list-style-type:none}.lst-kix_nzxgmnmwjatv-4>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-4,lower-latin) ". "}ul.lst-kix_aninp2c52sy3-2{list-style-type:none}ul.lst-kix_aninp2c52sy3-3{list-style-type:none}.lst-kix_nzxgmnmwjatv-7>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-7,lower-latin) ". "}.lst-kix_1rk6yybn5non-8>li:before{content:"-  "}.lst-kix_nzxgmnmwjatv-5>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-5,lower-roman) ". "}ul.lst-kix_aninp2c52sy3-8{list-style-type:none}.lst-kix_yqx6kz4lvbt4-7>li:before{content:"\0025cb  "}.lst-kix_yqx6kz4lvbt4-8>li:before{content:"\0025a0  "}ol.lst-kix_jqqhsyijt7h5-2.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-2 0}.lst-kix_nzxgmnmwjatv-1>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-1,lower-latin) ". "}.lst-kix_nzxgmnmwjatv-2>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-2,lower-roman) ". "}ul.lst-kix_l9w5fm8md2s9-6{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-7{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-8{list-style-type:none}.lst-kix_vlcqhp8hs5e3-1>li:before{content:"\0025cb  "}.lst-kix_1rk6yybn5non-1>li:before{content:"-  "}.lst-kix_1rk6yybn5non-2>li:before{content:"-  "}ul.lst-kix_e21xjkb2e68h-3{list-style-type:none}ul.lst-kix_e21xjkb2e68h-4{list-style-type:none}ul.lst-kix_e21xjkb2e68h-5{list-style-type:none}.lst-kix_o1wsnr4piq1b-6>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-6}ul.lst-kix_e21xjkb2e68h-6{list-style-type:none}.lst-kix_xm7ynypsh0z5-0>li:before{content:"\0025cf  "}.lst-kix_tiumq34nhjs2-5>li:before{content:"\0025a0  "}ul.lst-kix_e21xjkb2e68h-7{list-style-type:none}ul.lst-kix_e21xjkb2e68h-8{list-style-type:none}.lst-kix_tiumq34nhjs2-8>li:before{content:"\0025a0  "}.lst-kix_h9p9lo20tlpj-8>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-8}ul.lst-kix_e21xjkb2e68h-0{list-style-type:none}ul.lst-kix_e21xjkb2e68h-1{list-style-type:none}.lst-kix_mi7vyzrjrpd8-6>li:before{content:"\0025cf  "}ul.lst-kix_e21xjkb2e68h-2{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-4.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-4 0}.lst-kix_l9w5fm8md2s9-4>li:before{content:"\0025cb  "}.lst-kix_b6qjs7hmfdgs-7>li:before{content:"\0025cb  "}.lst-kix_mi7vyzrjrpd8-0>li:before{content:"\0025cf  "}.lst-kix_wawbbtzgf6il-8>li:before{content:"\0025a0  "}.lst-kix_l9w5fm8md2s9-0>li:before{content:"\0025cf  "}.lst-kix_np2y6n1468n7-0>li:before{content:"\0025cf  "}.lst-kix_yqx6kz4lvbt4-4>li:before{content:"\0025cb  "}.lst-kix_b6qjs7hmfdgs-1>li:before{content:"\0025cb  "}ol.lst-kix_jqqhsyijt7h5-7.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-7 0}.lst-kix_i0q09d67i790-3>li:before{content:"\0025cf  "}.lst-kix_b6qjs7hmfdgs-5>li:before{content:"\0025a0  "}.lst-kix_wawbbtzgf6il-2>li:before{content:"\0025a0  "}.lst-kix_xm7ynypsh0z5-6>li:before{content:"\0025cf  "}ul.lst-kix_icjmrj4tgidb-2{list-style-type:none}ul.lst-kix_icjmrj4tgidb-3{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-1.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-1 0}ul.lst-kix_icjmrj4tgidb-0{list-style-type:none}.lst-kix_np2y6n1468n7-4>li:before{content:"\0025cb  "}ul.lst-kix_icjmrj4tgidb-1{list-style-type:none}ul.lst-kix_icjmrj4tgidb-8{list-style-type:none}ul.lst-kix_icjmrj4tgidb-6{list-style-type:none}ul.lst-kix_icjmrj4tgidb-7{list-style-type:none}ul.lst-kix_icjmrj4tgidb-4{list-style-type:none}.lst-kix_i0q09d67i790-5>li:before{content:"\0025a0  "}ul.lst-kix_icjmrj4tgidb-5{list-style-type:none}.lst-kix_e5u94rrvoqs-2>li:before{content:"\0025a0  "}.lst-kix_e5u94rrvoqs-0>li:before{content:"\0025cf  "}.lst-kix_cxongm456yyi-3>li:before{content:"\0025cf  "}ul.lst-kix_f7u4w07k18eh-8{list-style-type:none}ul.lst-kix_f7u4w07k18eh-7{list-style-type:none}.lst-kix_tnmatnpjv4t2-8>li:before{content:"\0025a0  "}ol.lst-kix_p4k358qyzf3y-4.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-4 0}ul.lst-kix_f7u4w07k18eh-6{list-style-type:none}.lst-kix_p4k358qyzf3y-7>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-7,lower-latin) ". "}ul.lst-kix_f7u4w07k18eh-5{list-style-type:none}ul.lst-kix_f7u4w07k18eh-4{list-style-type:none}ul.lst-kix_f7u4w07k18eh-3{list-style-type:none}ul.lst-kix_f7u4w07k18eh-2{list-style-type:none}ul.lst-kix_f7u4w07k18eh-1{list-style-type:none}.lst-kix_e5u94rrvoqs-6>li:before{content:"\0025cf  "}ul.lst-kix_f7u4w07k18eh-0{list-style-type:none}.lst-kix_7ev7v5cpgtkc-5>li:before{content:"\0025a0  "}ol.lst-kix_7ka97cx080q5-1{list-style-type:none}.lst-kix_zvvafajkk2a-2>li:before{content:"\0025a0  "}ol.lst-kix_7ka97cx080q5-2{list-style-type:none}ol.lst-kix_7ka97cx080q5-0{list-style-type:none}ol.lst-kix_7ka97cx080q5-5{list-style-type:none}ol.lst-kix_7ka97cx080q5-6{list-style-type:none}ol.lst-kix_7ka97cx080q5-3{list-style-type:none}.lst-kix_flatlsyq6jr0-3>li:before{content:"\0025cf  "}.lst-kix_tnmatnpjv4t2-4>li:before{content:"\0025cb  "}ol.lst-kix_7ka97cx080q5-4{list-style-type:none}.lst-kix_fcckwvdxoljx-7>li:before{content:"-  "}.lst-kix_nzxgmnmwjatv-4>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-4}ol.lst-kix_7ka97cx080q5-7{list-style-type:none}ol.lst-kix_7ka97cx080q5-8{list-style-type:none}.lst-kix_zvvafajkk2a-8>li:before{content:"\0025a0  "}.lst-kix_fcckwvdxoljx-1>li:before{content:"-  "}ol.lst-kix_p4k358qyzf3y-6{list-style-type:none}.lst-kix_zvvafajkk2a-6>li:before{content:"\0025cf  "}ol.lst-kix_p4k358qyzf3y-7{list-style-type:none}ol.lst-kix_p4k358qyzf3y-8{list-style-type:none}.lst-kix_tczavuk46t7o-0>li:before{content:"\0025cf  "}ol.lst-kix_p4k358qyzf3y-2{list-style-type:none}ol.lst-kix_p4k358qyzf3y-3{list-style-type:none}.lst-kix_7ev7v5cpgtkc-1>li:before{content:"\0025cb  "}ol.lst-kix_p4k358qyzf3y-4{list-style-type:none}ol.lst-kix_p4k358qyzf3y-5{list-style-type:none}.lst-kix_fcckwvdxoljx-3>li:before{content:"-  "}ol.lst-kix_p4k358qyzf3y-0{list-style-type:none}ol.lst-kix_p4k358qyzf3y-1{list-style-type:none}.lst-kix_7ka97cx080q5-2>li{counter-increment:lst-ctn-kix_7ka97cx080q5-2}.lst-kix_4tgbgh9bb7g-2>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-2}.lst-kix_wxhgjniia6a-4>li:before{content:"\0025cb  "}.lst-kix_ur253ckj31qe-3>li{counter-increment:lst-ctn-kix_ur253ckj31qe-3}ul.lst-kix_s0bij7o2r512-0{list-style-type:none}.lst-kix_4tgbgh9bb7g-7>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-7,lower-latin) ". "}ul.lst-kix_s0bij7o2r512-3{list-style-type:none}ul.lst-kix_s0bij7o2r512-4{list-style-type:none}ul.lst-kix_s0bij7o2r512-1{list-style-type:none}.lst-kix_4tgbgh9bb7g-6>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-6,decimal) ". "}ul.lst-kix_s0bij7o2r512-2{list-style-type:none}ul.lst-kix_s0bij7o2r512-7{list-style-type:none}ul.lst-kix_s0bij7o2r512-8{list-style-type:none}ul.lst-kix_s0bij7o2r512-5{list-style-type:none}ul.lst-kix_s0bij7o2r512-6{list-style-type:none}.lst-kix_4tgbgh9bb7g-3>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-3,decimal) ". "}.lst-kix_y5u0wek48eld-1>li:before{content:"-  "}.lst-kix_y5u0wek48eld-5>li:before{content:"-  "}.lst-kix_p4k358qyzf3y-5>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-5,lower-roman) ". "}.lst-kix_y5u0wek48eld-4>li:before{content:"-  "}.lst-kix_p4k358qyzf3y-4>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-4,lower-latin) ". "}.lst-kix_p4k358qyzf3y-1>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-1,lower-roman) ". "}.lst-kix_cxongm456yyi-0>li:before{content:"\0025cf  "}.lst-kix_h9p9lo20tlpj-0>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-0,decimal) ". "}.lst-kix_h9p9lo20tlpj-1>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-1,lower-latin) ". "}.lst-kix_a76b6w8pnkyv-8>li:before{content:"\0025a0  "}ol.lst-kix_jqqhsyijt7h5-1.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-1 0}ul.lst-kix_s27gvee6ygqb-2{list-style-type:none}ul.lst-kix_s27gvee6ygqb-1{list-style-type:none}.lst-kix_ia5tg78e6pm4-8>li:before{content:"\0025a0  "}.lst-kix_4tgbgh9bb7g-4>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-4}ul.lst-kix_s27gvee6ygqb-0{list-style-type:none}ul.lst-kix_s27gvee6ygqb-6{list-style-type:none}.lst-kix_nzxgmnmwjatv-3>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-3}.lst-kix_hxvvdzewxqs2-2>li:before{content:"\0025a0  "}ul.lst-kix_s27gvee6ygqb-5{list-style-type:none}ul.lst-kix_s27gvee6ygqb-4{list-style-type:none}ul.lst-kix_s27gvee6ygqb-3{list-style-type:none}.lst-kix_k08o05q2crtr-0>li:before{content:"\0025cf  "}ul.lst-kix_s27gvee6ygqb-8{list-style-type:none}ul.lst-kix_s27gvee6ygqb-7{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-6{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-5{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-4{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-3{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-8{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-7{list-style-type:none}.lst-kix_d1mamalvi3wu-1>li:before{content:"\0025cb  "}.lst-kix_h9p9lo20tlpj-7>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-7}ul.lst-kix_cpm4oaqfjy6b-2{list-style-type:none}.lst-kix_jqqhsyijt7h5-2>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-2}ul.lst-kix_cpm4oaqfjy6b-1{list-style-type:none}ul.lst-kix_cpm4oaqfjy6b-0{list-style-type:none}.lst-kix_wxhgjniia6a-8>li:before{content:"\0025a0  "}.lst-kix_k08o05q2crtr-3>li:before{content:"\0025cf  "}.lst-kix_ns4c34lwftpb-0>li:before{content:"\0025cf  "}ol.lst-kix_p4k358qyzf3y-8.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-8 0}.lst-kix_ia5tg78e6pm4-5>li:before{content:"\0025a0  "}.lst-kix_wxhgjniia6a-7>li:before{content:"\0025cb  "}.lst-kix_p4k358qyzf3y-1>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-1}.lst-kix_ur253ckj31qe-7>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-7,lower-latin) ". "}ul.lst-kix_l9w5fm8md2s9-2{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-3{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-4{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-5{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-0{list-style-type:none}ul.lst-kix_l9w5fm8md2s9-1{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-0{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-1{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-2{list-style-type:none}.lst-kix_l9w5fm8md2s9-1>li:before{content:"\0025cb  "}.lst-kix_b6qjs7hmfdgs-8>li:before{content:"\0025a0  "}.lst-kix_i0q09d67i790-2>li:before{content:"\0025a0  "}.lst-kix_np2y6n1468n7-1>li:before{content:"\0025cb  "}.lst-kix_emh9tw6ivvnv-1>li:before{content:"\0025cb  "}ul.lst-kix_tnmatnpjv4t2-7{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-8{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-3{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-4{list-style-type:none}ul.lst-kix_tnmatnpjv4t2-5{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-7.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-7 0}.lst-kix_b6qjs7hmfdgs-4>li:before{content:"\0025cb  "}ul.lst-kix_tnmatnpjv4t2-6{list-style-type:none}.lst-kix_nwfayb939eln-2>li:before{content:"\0025a0  "}.lst-kix_emh9tw6ivvnv-5>li:before{content:"\0025a0  "}.lst-kix_ock170xmc4e5-8>li:before{content:"\0025a0  "}.lst-kix_i0q09d67i790-6>li:before{content:"\0025cf  "}.lst-kix_n2keu386kfur-7>li:before{content:"\0025cb  "}.lst-kix_6bb7qu9vjqr2-0>li:before{content:"-  "}ul.lst-kix_wawbbtzgf6il-2{list-style-type:none}ul.lst-kix_o6fhz26vvidv-8{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-8{list-style-type:none}.lst-kix_e5u94rrvoqs-3>li:before{content:"\0025cf  "}ul.lst-kix_wawbbtzgf6il-3{list-style-type:none}ul.lst-kix_o6fhz26vvidv-7{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-7{list-style-type:none}ul.lst-kix_wawbbtzgf6il-4{list-style-type:none}.lst-kix_gmyqb4k4mims-8>li:before{content:"\0025a0  "}ul.lst-kix_1i1lr1oi4e9-6{list-style-type:none}ul.lst-kix_wawbbtzgf6il-5{list-style-type:none}.lst-kix_ric95po2pkzv-8>li:before{content:"\0025a0  "}ul.lst-kix_1i1lr1oi4e9-5{list-style-type:none}ul.lst-kix_wawbbtzgf6il-6{list-style-type:none}ul.lst-kix_o6fhz26vvidv-4{list-style-type:none}.lst-kix_uthm435lclg8-6>li:before{content:"\0025cf  "}ul.lst-kix_wawbbtzgf6il-7{list-style-type:none}ul.lst-kix_o6fhz26vvidv-3{list-style-type:none}ul.lst-kix_wawbbtzgf6il-8{list-style-type:none}ul.lst-kix_o6fhz26vvidv-6{list-style-type:none}ul.lst-kix_o6fhz26vvidv-5{list-style-type:none}ul.lst-kix_o6fhz26vvidv-0{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-0{list-style-type:none}.lst-kix_h9p9lo20tlpj-4>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-4,lower-latin) ". "}ul.lst-kix_o6fhz26vvidv-2{list-style-type:none}.lst-kix_y5u0wek48eld-8>li:before{content:"-  "}.lst-kix_gmyqb4k4mims-4>li:before{content:"\0025cb  "}ul.lst-kix_o6fhz26vvidv-1{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-4{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-3{list-style-type:none}ul.lst-kix_wawbbtzgf6il-0{list-style-type:none}ul.lst-kix_1i1lr1oi4e9-2{list-style-type:none}ul.lst-kix_wawbbtzgf6il-1{list-style-type:none}.lst-kix_o6fhz26vvidv-7>li:before{content:"\0025cb  "}ul.lst-kix_1i1lr1oi4e9-1{list-style-type:none}.lst-kix_dkq97x3e7jfk-4>li:before{content:"\0025cb  "}.lst-kix_tnmatnpjv4t2-7>li:before{content:"\0025cb  "}.lst-kix_p4k358qyzf3y-8>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-8,lower-roman) ". "}.lst-kix_a0ivi9b1qp8i-3>li:before{content:"\0025cf  "}.lst-kix_w7vhjpenesq3-3>li:before{content:"\0025cf  "}.lst-kix_w7vhjpenesq3-7>li:before{content:"\0025cb  "}.lst-kix_fcckwvdxoljx-0>li:before{content:"-  "}.lst-kix_flatlsyq6jr0-0>li:before{content:"" counter(lst-ctn-kix_flatlsyq6jr0-0,decimal) ". "}.lst-kix_zvvafajkk2a-5>li:before{content:"\0025a0  "}.lst-kix_fcckwvdxoljx-4>li:before{content:"-  "}.lst-kix_7ev7v5cpgtkc-2>li:before{content:"\0025a0  "}.lst-kix_o9ptu63d1d74-1>li:before{content:"\0025cb  "}ul.lst-kix_e08effp12mqf-4{list-style-type:none}ul.lst-kix_e08effp12mqf-5{list-style-type:none}ul.lst-kix_e08effp12mqf-6{list-style-type:none}ul.lst-kix_e08effp12mqf-7{list-style-type:none}.lst-kix_cpm4oaqfjy6b-5>li:before{content:"\0025a0  "}ul.lst-kix_e08effp12mqf-0{list-style-type:none}ul.lst-kix_e08effp12mqf-1{list-style-type:none}ul.lst-kix_e08effp12mqf-2{list-style-type:none}ul.lst-kix_e08effp12mqf-3{list-style-type:none}.lst-kix_h9p9lo20tlpj-2>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-2}.lst-kix_o9ptu63d1d74-4>li:before{content:"\0025cb  "}.lst-kix_nzxgmnmwjatv-6>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-6}.lst-kix_7ka97cx080q5-2>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-2,lower-roman) ". "}.lst-kix_cpm4oaqfjy6b-8>li:before{content:"\0025a0  "}ul.lst-kix_e08effp12mqf-8{list-style-type:none}.lst-kix_ontx89k1fqr9-2>li:before{content:"\0025a0  "}.lst-kix_ontx89k1fqr9-5>li:before{content:"\0025a0  "}.lst-kix_4fkgc27ss5mq-5>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-5}.lst-kix_uhfsw7q6s458-7>li:before{content:"\0025cb  "}ul.lst-kix_g6dv2r60jppe-6{list-style-type:none}ul.lst-kix_g6dv2r60jppe-5{list-style-type:none}.lst-kix_7ka97cx080q5-5>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-5,lower-roman) ". "}ul.lst-kix_g6dv2r60jppe-4{list-style-type:none}ul.lst-kix_g6dv2r60jppe-3{list-style-type:none}ul.lst-kix_g6dv2r60jppe-8{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-5.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-5 0}ul.lst-kix_g6dv2r60jppe-7{list-style-type:none}ul.lst-kix_g6dv2r60jppe-2{list-style-type:none}ul.lst-kix_g6dv2r60jppe-1{list-style-type:none}ul.lst-kix_g6dv2r60jppe-0{list-style-type:none}.lst-kix_x7o2gv80my2k-1>li:before{content:"\0025cb  "}ul.lst-kix_x7o2gv80my2k-8{list-style-type:none}.lst-kix_7ka97cx080q5-5>li{counter-increment:lst-ctn-kix_7ka97cx080q5-5}ul.lst-kix_x7o2gv80my2k-7{list-style-type:none}.lst-kix_nr7rbh2hfyf0-2>li:before{content:"\0025a0  "}ul.lst-kix_x7o2gv80my2k-4{list-style-type:none}ul.lst-kix_x7o2gv80my2k-3{list-style-type:none}ul.lst-kix_x7o2gv80my2k-6{list-style-type:none}ul.lst-kix_x7o2gv80my2k-5{list-style-type:none}.lst-kix_bv0w79unf8ye-7>li:before{content:"\0025cb  "}.lst-kix_x7o2gv80my2k-4>li:before{content:"\0025cb  "}.lst-kix_25j83v4xa4ig-1>li:before{content:"\0025cb  "}.lst-kix_xiiy75ihy1il-0>li:before{content:"\0025cf  "}.lst-kix_x7o2gv80my2k-7>li:before{content:"\0025cb  "}.lst-kix_ontx89k1fqr9-8>li:before{content:"\0025a0  "}ul.lst-kix_x7o2gv80my2k-0{list-style-type:none}ul.lst-kix_x7o2gv80my2k-2{list-style-type:none}ul.lst-kix_x7o2gv80my2k-1{list-style-type:none}.lst-kix_nr7rbh2hfyf0-5>li:before{content:"\0025a0  "}.lst-kix_xzp0xel2onox-2>li:before{content:"\0025a0  "}.lst-kix_uhfsw7q6s458-0>li:before{content:"\0025cf  "}.lst-kix_xiiy75ihy1il-7>li:before{content:"\0025cb  "}.lst-kix_qqf2twdkbiw0-2>li:before{content:"\0025a0  "}ul.lst-kix_8rc65o4ywyr4-0{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-0{list-style-type:none}ul.lst-kix_g84y3pe344o9-4{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-2{list-style-type:none}ul.lst-kix_g84y3pe344o9-5{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-1{list-style-type:none}ul.lst-kix_g84y3pe344o9-2{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-4{list-style-type:none}.lst-kix_xzp0xel2onox-5>li:before{content:"\0025a0  "}ul.lst-kix_g84y3pe344o9-3{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-3{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-0.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-0 0}ul.lst-kix_g84y3pe344o9-0{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-6{list-style-type:none}ul.lst-kix_g84y3pe344o9-1{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-5{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-5.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-5 0}ul.lst-kix_9bzwod9gvkvp-8{list-style-type:none}ul.lst-kix_9bzwod9gvkvp-7{list-style-type:none}.lst-kix_bv0w79unf8ye-4>li:before{content:"\0025cb  "}.lst-kix_ygjhinrwstz-7>li:before{content:"\0025cb  "}ul.lst-kix_g84y3pe344o9-8{list-style-type:none}.lst-kix_bv0w79unf8ye-1>li:before{content:"\0025cb  "}ul.lst-kix_g84y3pe344o9-6{list-style-type:none}.lst-kix_vxbaz02t98k3-3>li:before{content:"\0025cf  "}ul.lst-kix_g84y3pe344o9-7{list-style-type:none}.lst-kix_qqf2twdkbiw0-5>li:before{content:"\0025a0  "}ul.lst-kix_8rc65o4ywyr4-5{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-6{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-7{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-8{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-1{list-style-type:none}.lst-kix_e21xjkb2e68h-6>li:before{content:"\0025cf  "}ul.lst-kix_8rc65o4ywyr4-2{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-3{list-style-type:none}ul.lst-kix_8rc65o4ywyr4-4{list-style-type:none}.lst-kix_9tajp1agqiyw-2>li:before{content:"\0025a0  "}.lst-kix_ur253ckj31qe-4>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-4,lower-latin) ". "}.lst-kix_7ka97cx080q5-8>li{counter-increment:lst-ctn-kix_7ka97cx080q5-8}.lst-kix_4h54ot51egiu-4>li:before{content:"\0025cb  "}.lst-kix_n2tc9z47p0ha-8>li:before{content:"\0025a0  "}.lst-kix_emh9tw6ivvnv-8>li:before{content:"\0025a0  "}.lst-kix_vxbaz02t98k3-0>li:before{content:"\0025cf  "}.lst-kix_ock170xmc4e5-5>li:before{content:"\0025a0  "}.lst-kix_qswbihp1i06v-6>li:before{content:"\0025cf  "}.lst-kix_xwc7ibkfiucl-4>li:before{content:"\0025cb  "}.lst-kix_e21xjkb2e68h-0>li:before{content:"\0025cf  "}.lst-kix_m3f8ler1ak4c-6>li:before{content:"\0025cf  "}ul.lst-kix_ns4c34lwftpb-0{list-style-type:none}.lst-kix_fv8y17g7myim-6>li:before{content:"\0025cf  "}.lst-kix_m3f8ler1ak4c-0>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-7>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-7}.lst-kix_tcwoy9lfuqk4-0>li:before{content:"\0025cf  "}.lst-kix_n2keu386kfur-4>li:before{content:"\0025cb  "}.lst-kix_nwfayb939eln-5>li:before{content:"\0025a0  "}.lst-kix_6bb7qu9vjqr2-3>li:before{content:"-  "}.lst-kix_25j83v4xa4ig-4>li:before{content:"\0025cb  "}.lst-kix_oli7yc1w3gpg-3>li:before{content:"\0025cf  "}.lst-kix_uthm435lclg8-3>li:before{content:"\0025cf  "}.lst-kix_ric95po2pkzv-5>li:before{content:"\0025a0  "}.lst-kix_ouaz0d8j7xj-6>li:before{content:"\0025cf  "}.lst-kix_k0c3ozmp0pnp-4>li:before{content:"\0025cb  "}.lst-kix_c4nzmsldypra-8>li:before{content:"\0025a0  "}.lst-kix_fv8y17g7myim-0>li:before{content:"\0025cf  "}.lst-kix_z9n4e0cpc149-3>li:before{content:"\0025cf  "}ul.lst-kix_fcckwvdxoljx-1{list-style-type:none}ul.lst-kix_tczavuk46t7o-0{list-style-type:none}.lst-kix_o6fhz26vvidv-4>li:before{content:"\0025cb  "}ul.lst-kix_fcckwvdxoljx-0{list-style-type:none}.lst-kix_w7vhjpenesq3-0>li:before{content:"\0025cf  "}.lst-kix_tcwoy9lfuqk4-6>li:before{content:"\0025cf  "}ul.lst-kix_fcckwvdxoljx-3{list-style-type:none}ul.lst-kix_tczavuk46t7o-2{list-style-type:none}.lst-kix_h9p9lo20tlpj-7>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-7,lower-latin) ". "}ul.lst-kix_fcckwvdxoljx-2{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-3.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-3 0}ul.lst-kix_tczavuk46t7o-1{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-8.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-8 0}ul.lst-kix_fcckwvdxoljx-5{list-style-type:none}.lst-kix_c4nzmsldypra-2>li:before{content:"\0025a0  "}ul.lst-kix_tczavuk46t7o-4{list-style-type:none}ul.lst-kix_fcckwvdxoljx-4{list-style-type:none}ul.lst-kix_tczavuk46t7o-3{list-style-type:none}ul.lst-kix_fcckwvdxoljx-7{list-style-type:none}ul.lst-kix_tczavuk46t7o-6{list-style-type:none}ul.lst-kix_fcckwvdxoljx-6{list-style-type:none}ul.lst-kix_tczavuk46t7o-5{list-style-type:none}ul.lst-kix_tczavuk46t7o-8{list-style-type:none}ul.lst-kix_fcckwvdxoljx-8{list-style-type:none}.lst-kix_l4298avyh9v6-8>li:before{content:"\0025a0  "}ul.lst-kix_tczavuk46t7o-7{list-style-type:none}.lst-kix_dkq97x3e7jfk-1>li:before{content:"\0025cb  "}.lst-kix_gayel7f32yde-1>li:before{content:"-  "}.lst-kix_a0ivi9b1qp8i-6>li:before{content:"\0025cf  "}.lst-kix_gmyqb4k4mims-1>li:before{content:"\0025cb  "}.lst-kix_9bzwod9gvkvp-1>li:before{content:"\0025cb  "}ul.lst-kix_nwfayb939eln-0{list-style-type:none}.lst-kix_c85u756bkx0j-7>li:before{content:"\0025cb  "}ul.lst-kix_nwfayb939eln-1{list-style-type:none}ul.lst-kix_nwfayb939eln-2{list-style-type:none}ul.lst-kix_nwfayb939eln-3{list-style-type:none}.lst-kix_wxhgjniia6a-1>li:before{content:"\0025cb  "}ul.lst-kix_nwfayb939eln-8{list-style-type:none}.lst-kix_alje6q6hzub8-6>li:before{content:"-  "}.lst-kix_c85u756bkx0j-4>li:before{content:"\0025cb  "}.lst-kix_7tkvl1c78ae3-2>li:before{content:"\0025a0  "}ul.lst-kix_nwfayb939eln-4{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-8.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-8 0}ul.lst-kix_nwfayb939eln-5{list-style-type:none}ul.lst-kix_nwfayb939eln-6{list-style-type:none}ul.lst-kix_nwfayb939eln-7{list-style-type:none}.lst-kix_f3z068owedlc-6>li:before{content:"\0025cf  "}.lst-kix_s0bij7o2r512-4>li:before{content:"\0025cb  "}.lst-kix_fyzn24yyf01g-5>li:before{content:"\0025a0  "}.lst-kix_54trtu6df9f8-8>li:before{content:"\0025a0  "}.lst-kix_fyzn24yyf01g-2>li:before{content:"\0025a0  "}.lst-kix_4h47981ce058-2>li:before{content:"\0025a0  "}.lst-kix_cm89sr4fo6m-2>li:before{content:"\0025a0  "}ul.lst-kix_flatlsyq6jr0-5{list-style-type:none}ul.lst-kix_flatlsyq6jr0-6{list-style-type:none}ul.lst-kix_flatlsyq6jr0-3{list-style-type:none}.lst-kix_4tgbgh9bb7g-0>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-0,decimal) ". "}ul.lst-kix_flatlsyq6jr0-4{list-style-type:none}ul.lst-kix_flatlsyq6jr0-1{list-style-type:none}ul.lst-kix_flatlsyq6jr0-2{list-style-type:none}.lst-kix_f3z068owedlc-3>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-5>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-5,lower-roman) ". "}.lst-kix_x30zykguitxy-8>li:before{content:"\0025a0  "}.lst-kix_hxvvdzewxqs2-8>li:before{content:"\0025a0  "}.lst-kix_1293ztc79klg-1>li:before{content:"\0025cb  "}.lst-kix_e08effp12mqf-8>li:before{content:"\0025a0  "}.lst-kix_4fkgc27ss5mq-2>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-2}.lst-kix_uthm435lclg8-0>li:before{content:"" counter(lst-ctn-kix_uthm435lclg8-0,decimal) ". "}.lst-kix_i2rq8qzg0efa-7>li:before{content:"+  "}ul.lst-kix_flatlsyq6jr0-7{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-7.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-7 0}ul.lst-kix_flatlsyq6jr0-8{list-style-type:none}.lst-kix_icjmrj4tgidb-1>li:before{content:"-  "}.lst-kix_f7u4w07k18eh-2>li:before{content:"\0025a0  "}.lst-kix_p4k358qyzf3y-7>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-7}.lst-kix_x3ptir2y8jgm-7>li:before{content:"\0025cb  "}ul.lst-kix_ieipvto6nfrd-8{list-style-type:none}.lst-kix_aninp2c52sy3-1>li:before{content:"\0025cb  "}ul.lst-kix_ieipvto6nfrd-7{list-style-type:none}.lst-kix_yk52z9n95trk-8>li:before{content:"\0025a0  "}.lst-kix_o1wsnr4piq1b-4>li:before{content:"(" counter(lst-ctn-kix_o1wsnr4piq1b-4,lower-latin) ") "}ul.lst-kix_ieipvto6nfrd-2{list-style-type:none}ul.lst-kix_ieipvto6nfrd-1{list-style-type:none}.lst-kix_hxvvdzewxqs2-5>li:before{content:"\0025a0  "}ul.lst-kix_ieipvto6nfrd-0{list-style-type:none}.lst-kix_x30zykguitxy-5>li:before{content:"\0025a0  "}ul.lst-kix_ieipvto6nfrd-6{list-style-type:none}.lst-kix_d1mamalvi3wu-7>li:before{content:"\0025cb  "}ul.lst-kix_ieipvto6nfrd-5{list-style-type:none}.lst-kix_x3ptir2y8jgm-4>li:before{content:"\0025cb  "}ul.lst-kix_ieipvto6nfrd-4{list-style-type:none}ul.lst-kix_ieipvto6nfrd-3{list-style-type:none}ul.lst-kix_ns4c34lwftpb-2{list-style-type:none}ol.lst-kix_nzxgmnmwjatv-7.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-7 0}ul.lst-kix_ns4c34lwftpb-1{list-style-type:none}.lst-kix_ns4c34lwftpb-3>li:before{content:"\0025cf  "}ul.lst-kix_ns4c34lwftpb-4{list-style-type:none}.lst-kix_d1mamalvi3wu-4>li:before{content:"\0025cb  "}ul.lst-kix_ns4c34lwftpb-3{list-style-type:none}ul.lst-kix_ns4c34lwftpb-6{list-style-type:none}ul.lst-kix_ns4c34lwftpb-5{list-style-type:none}ul.lst-kix_ns4c34lwftpb-8{list-style-type:none}ul.lst-kix_ns4c34lwftpb-7{list-style-type:none}.lst-kix_4roymkz9vma-4>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) "." counter(lst-ctn-kix_4roymkz9vma-4,decimal) ". "}.lst-kix_ia5tg78e6pm4-2>li:before{content:"\0025a0  "}.lst-kix_ieipvto6nfrd-5>li:before{content:"-  "}.lst-kix_4roymkz9vma-7>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) "." counter(lst-ctn-kix_4roymkz9vma-4,decimal) "." counter(lst-ctn-kix_4roymkz9vma-5,decimal) "." counter(lst-ctn-kix_4roymkz9vma-6,decimal) "." counter(lst-ctn-kix_4roymkz9vma-7,decimal) ". "}.lst-kix_ns4c34lwftpb-6>li:before{content:"\0025cf  "}.lst-kix_a76b6w8pnkyv-5>li:before{content:"\0025a0  "}.lst-kix_2w2qcvvb39hd-5>li:before{content:"\0025a0  "}.lst-kix_a76b6w8pnkyv-2>li:before{content:"\0025a0  "}.lst-kix_2w2qcvvb39hd-2>li:before{content:"\0025a0  "}.lst-kix_s27gvee6ygqb-6>li:before{content:"\0025cf  "}.lst-kix_k08o05q2crtr-6>li:before{content:"\0025cf  "}.lst-kix_ieipvto6nfrd-8>li:before{content:"-  "}.lst-kix_jza88dihngw3-2>li:before{content:"\0025a0  "}.lst-kix_4h54ot51egiu-1>li:before{content:"\0025cb  "}ul.lst-kix_z9n4e0cpc149-8{list-style-type:none}ul.lst-kix_z9n4e0cpc149-7{list-style-type:none}.lst-kix_mpnpv7ekeo7q-6>li:before{content:"\0025cf  "}ul.lst-kix_z9n4e0cpc149-6{list-style-type:none}ul.lst-kix_z9n4e0cpc149-5{list-style-type:none}ul.lst-kix_z9n4e0cpc149-4{list-style-type:none}.lst-kix_abgphud9refj-8>li:before{content:"\0025a0  "}ul.lst-kix_z9n4e0cpc149-3{list-style-type:none}ul.lst-kix_z9n4e0cpc149-2{list-style-type:none}.lst-kix_tiumq34nhjs2-2>li:before{content:"\0025a0  "}.lst-kix_nwfayb939eln-8>li:before{content:"\0025a0  "}.lst-kix_e21xjkb2e68h-3>li:before{content:"\0025cf  "}ul.lst-kix_z9n4e0cpc149-1{list-style-type:none}.lst-kix_4roymkz9vma-3>li{counter-increment:lst-ctn-kix_4roymkz9vma-3}ul.lst-kix_e5u94rrvoqs-0{list-style-type:none}.lst-kix_jduurcgheuo-0>li:before{content:"-  "}ul.lst-kix_z9n4e0cpc149-0{list-style-type:none}ul.lst-kix_e5u94rrvoqs-1{list-style-type:none}.lst-kix_mi7vyzrjrpd8-3>li:before{content:"\0025cf  "}ul.lst-kix_e5u94rrvoqs-4{list-style-type:none}ul.lst-kix_e5u94rrvoqs-5{list-style-type:none}ul.lst-kix_e5u94rrvoqs-2{list-style-type:none}ul.lst-kix_e5u94rrvoqs-3{list-style-type:none}ul.lst-kix_e5u94rrvoqs-8{list-style-type:none}ul.lst-kix_vxbaz02t98k3-5{list-style-type:none}ul.lst-kix_vxbaz02t98k3-6{list-style-type:none}ul.lst-kix_e5u94rrvoqs-6{list-style-type:none}ul.lst-kix_vxbaz02t98k3-7{list-style-type:none}ul.lst-kix_e5u94rrvoqs-7{list-style-type:none}ul.lst-kix_vxbaz02t98k3-8{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-3.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-3 0}.lst-kix_ock170xmc4e5-2>li:before{content:"\0025a0  "}ul.lst-kix_vxbaz02t98k3-1{list-style-type:none}ul.lst-kix_vxbaz02t98k3-2{list-style-type:none}ul.lst-kix_vxbaz02t98k3-3{list-style-type:none}.lst-kix_ur253ckj31qe-1>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-1,lower-latin) ". "}ul.lst-kix_vxbaz02t98k3-4{list-style-type:none}.lst-kix_flatlsyq6jr0-6>li:before{content:"\0025cf  "}.lst-kix_wawbbtzgf6il-5>li:before{content:"\0025a0  "}ul.lst-kix_vxbaz02t98k3-0{list-style-type:none}.lst-kix_m3f8ler1ak4c-3>li:before{content:"\0025cf  "}.lst-kix_6bb7qu9vjqr2-6>li:before{content:"-  "}.lst-kix_o1wsnr4piq1b-7>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-7,lower-latin) ". "}ol.lst-kix_o1wsnr4piq1b-0.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-0 0}.lst-kix_yqx6kz4lvbt4-1>li:before{content:"\0025cb  "}.lst-kix_l9w5fm8md2s9-7>li:before{content:"\0025cb  "}.lst-kix_xm7ynypsh0z5-3>li:before{content:"\0025cf  "}.lst-kix_np2y6n1468n7-7>li:before{content:"\0025cb  "}.lst-kix_k0c3ozmp0pnp-7>li:before{content:"\0025cb  "}.lst-kix_ur253ckj31qe-0>li{counter-increment:lst-ctn-kix_ur253ckj31qe-0}.lst-kix_zhrwxtd2sngd-3>li:before{content:"\0025cf  "}ol.lst-kix_o1wsnr4piq1b-1.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-1 0}ul.lst-kix_lm4lwqvvvsaj-4{list-style-type:none}.lst-kix_lux5kw5rxj0b-2>li:before{content:"\0025a0  "}ol.lst-kix_4fkgc27ss5mq-8{list-style-type:none}ul.lst-kix_lm4lwqvvvsaj-5{list-style-type:none}ul.lst-kix_lm4lwqvvvsaj-6{list-style-type:none}ul.lst-kix_lm4lwqvvvsaj-7{list-style-type:none}.lst-kix_ric95po2pkzv-2>li:before{content:"\0025a0  "}ul.lst-kix_lm4lwqvvvsaj-0{list-style-type:none}.lst-kix_cxongm456yyi-6>li:before{content:"\0025cf  "}.lst-kix_fv8y17g7myim-3>li:before{content:"\0025cf  "}ul.lst-kix_lm4lwqvvvsaj-1{list-style-type:none}ul.lst-kix_mi7vyzrjrpd8-0{list-style-type:none}ul.lst-kix_lm4lwqvvvsaj-2{list-style-type:none}ul.lst-kix_mi7vyzrjrpd8-1{list-style-type:none}.lst-kix_o6fhz26vvidv-1>li:before{content:"\0025cb  "}ul.lst-kix_lm4lwqvvvsaj-3{list-style-type:none}.lst-kix_s8oehts9vu79-5>li:before{content:"\0025a0  "}ul.lst-kix_mi7vyzrjrpd8-2{list-style-type:none}.lst-kix_c4nzmsldypra-5>li:before{content:"\0025a0  "}.lst-kix_7ev7v5cpgtkc-8>li:before{content:"\0025a0  "}ul.lst-kix_mi7vyzrjrpd8-3{list-style-type:none}ul.lst-kix_mi7vyzrjrpd8-4{list-style-type:none}.lst-kix_oli7yc1w3gpg-0>li:before{content:"\0025cf  "}.lst-kix_z9n4e0cpc149-6>li:before{content:"\0025cf  "}ul.lst-kix_mi7vyzrjrpd8-5{list-style-type:none}.lst-kix_jqqhsyijt7h5-2>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-2,lower-roman) ". "}ul.lst-kix_mi7vyzrjrpd8-6{list-style-type:none}.lst-kix_n2keu386kfur-1>li:before{content:"\0025cb  "}ul.lst-kix_mi7vyzrjrpd8-7{list-style-type:none}.lst-kix_ejnvxy8b1e9w-7>li:before{content:"\0025cb  "}ul.lst-kix_mi7vyzrjrpd8-8{list-style-type:none}.lst-kix_tcwoy9lfuqk4-3>li:before{content:"\0025cf  "}.lst-kix_tnmatnpjv4t2-1>li:before{content:"\0025cb  "}.lst-kix_s0bij7o2r512-7>li:before{content:"\0025cb  "}.lst-kix_tfsdypepg13u-3>li:before{content:"\0025cf  "}.lst-kix_k9nxvn8n4dkv-4>li:before{content:"\0025cb  "}ul.lst-kix_krjlquulz7zd-1{list-style-type:none}ul.lst-kix_krjlquulz7zd-0{list-style-type:none}ul.lst-kix_lm4lwqvvvsaj-8{list-style-type:none}ul.lst-kix_krjlquulz7zd-3{list-style-type:none}ul.lst-kix_krjlquulz7zd-2{list-style-type:none}ul.lst-kix_krjlquulz7zd-5{list-style-type:none}ul.lst-kix_krjlquulz7zd-4{list-style-type:none}ul.lst-kix_krjlquulz7zd-7{list-style-type:none}ul.lst-kix_krjlquulz7zd-6{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-2.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-2 0}ul.lst-kix_krjlquulz7zd-8{list-style-type:none}.lst-kix_xwc7ibkfiucl-7>li:before{content:"\0025cb  "}.lst-kix_tczavuk46t7o-3>li:before{content:"\0025cf  "}ul.lst-kix_ock170xmc4e5-0{list-style-type:none}.lst-kix_qffkk51zafur-5>li:before{content:"\0025a0  "}ul.lst-kix_ock170xmc4e5-1{list-style-type:none}ul.lst-kix_ock170xmc4e5-2{list-style-type:none}ol.lst-kix_ur253ckj31qe-1{list-style-type:none}ul.lst-kix_ock170xmc4e5-3{list-style-type:none}ol.lst-kix_ur253ckj31qe-0{list-style-type:none}ul.lst-kix_ock170xmc4e5-4{list-style-type:none}ol.lst-kix_ur253ckj31qe-3{list-style-type:none}ul.lst-kix_ock170xmc4e5-5{list-style-type:none}ol.lst-kix_ur253ckj31qe-2{list-style-type:none}ul.lst-kix_ock170xmc4e5-6{list-style-type:none}.lst-kix_krjlquulz7zd-4>li:before{content:"\0025cb  "}.lst-kix_krjlquulz7zd-8>li:before{content:"\0025a0  "}ol.lst-kix_ur253ckj31qe-5{list-style-type:none}ul.lst-kix_ock170xmc4e5-7{list-style-type:none}ol.lst-kix_ur253ckj31qe-4{list-style-type:none}.lst-kix_qffkk51zafur-7>li:before{content:"\0025cb  "}.lst-kix_krjlquulz7zd-2>li:before{content:"\0025a0  "}.lst-kix_krjlquulz7zd-0>li:before{content:"\0025cf  "}ul.lst-kix_ock170xmc4e5-8{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-0.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-0 0}.lst-kix_t6shmrihc7et-1>li:before{content:"\0025cb  "}.lst-kix_t6shmrihc7et-3>li:before{content:"\0025cf  "}.lst-kix_lm4lwqvvvsaj-6>li:before{content:"\0025cf  "}.lst-kix_jm1l4y2hlnt3-2>li:before{content:"\0025a0  "}.lst-kix_jm1l4y2hlnt3-6>li:before{content:"\0025cf  "}ul.lst-kix_fyzn24yyf01g-3{list-style-type:none}ul.lst-kix_fyzn24yyf01g-2{list-style-type:none}ul.lst-kix_fyzn24yyf01g-1{list-style-type:none}ul.lst-kix_fyzn24yyf01g-0{list-style-type:none}.lst-kix_t6shmrihc7et-5>li:before{content:"\0025a0  "}.lst-kix_t6shmrihc7et-7>li:before{content:"\0025cb  "}ul.lst-kix_fyzn24yyf01g-7{list-style-type:none}ul.lst-kix_fyzn24yyf01g-6{list-style-type:none}.lst-kix_lm4lwqvvvsaj-4>li:before{content:"\0025cb  "}ul.lst-kix_fyzn24yyf01g-5{list-style-type:none}.lst-kix_jm1l4y2hlnt3-0>li:before{content:"\0025cf  "}.lst-kix_lux5kw5rxj0b-7>li:before{content:"\0025cb  "}ul.lst-kix_fyzn24yyf01g-4{list-style-type:none}.lst-kix_jm1l4y2hlnt3-8>li:before{content:"\0025a0  "}.lst-kix_1i1lr1oi4e9-1>li:before{content:"\0025cb  "}ul.lst-kix_fyzn24yyf01g-8{list-style-type:none}.lst-kix_1i1lr1oi4e9-3>li:before{content:"\0025cf  "}.lst-kix_qffkk51zafur-1>li:before{content:"\0025cb  "}ol.lst-kix_ur253ckj31qe-7{list-style-type:none}ol.lst-kix_ur253ckj31qe-6{list-style-type:none}ol.lst-kix_ur253ckj31qe-8{list-style-type:none}.lst-kix_nzxgmnmwjatv-8>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-8}.lst-kix_qffkk51zafur-3>li:before{content:"\0025cf  "}.lst-kix_lm4lwqvvvsaj-8>li:before{content:"\0025a0  "}.lst-kix_jm1l4y2hlnt3-4>li:before{content:"\0025cb  "}.lst-kix_g84y3pe344o9-0>li:before{content:"\0025cf  "}.lst-kix_g84y3pe344o9-2>li:before{content:"\0025a0  "}.lst-kix_1i1lr1oi4e9-7>li:before{content:"\0025cb  "}ul.lst-kix_np2y6n1468n7-5{list-style-type:none}ul.lst-kix_np2y6n1468n7-4{list-style-type:none}.lst-kix_1i1lr1oi4e9-5>li:before{content:"\0025a0  "}.lst-kix_g84y3pe344o9-6>li:before{content:"\0025cf  "}ul.lst-kix_np2y6n1468n7-3{list-style-type:none}ul.lst-kix_np2y6n1468n7-2{list-style-type:none}ul.lst-kix_np2y6n1468n7-8{list-style-type:none}ul.lst-kix_np2y6n1468n7-7{list-style-type:none}ul.lst-kix_np2y6n1468n7-6{list-style-type:none}.lst-kix_g84y3pe344o9-8>li:before{content:"\0025a0  "}ul.lst-kix_np2y6n1468n7-1{list-style-type:none}ul.lst-kix_np2y6n1468n7-0{list-style-type:none}.lst-kix_lm4lwqvvvsaj-2>li:before{content:"\0025a0  "}.lst-kix_h9p9lo20tlpj-4>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-4}ol.lst-kix_nzxgmnmwjatv-6.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-6 0}.lst-kix_lm4lwqvvvsaj-0>li:before{content:"\0025cf  "}.lst-kix_eu4b93xn98yk-1>li:before{content:"\0025cb  "}.lst-kix_jqqhsyijt7h5-5>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-5}.lst-kix_jduurcgheuo-4>li:before{content:"-  "}.lst-kix_g84y3pe344o9-4>li:before{content:"\0025cb  "}.lst-kix_jduurcgheuo-6>li:before{content:"-  "}.lst-kix_p4k358qyzf3y-4>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-4}.lst-kix_krjlquulz7zd-6>li:before{content:"\0025cf  "}.lst-kix_jduurcgheuo-8>li:before{content:"-  "}ol.lst-kix_nzxgmnmwjatv-1.start{counter-reset:lst-ctn-kix_nzxgmnmwjatv-1 0}ol.lst-kix_h9p9lo20tlpj-4.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-4 0}ol.lst-kix_o1wsnr4piq1b-6.start{counter-reset:lst-ctn-kix_o1wsnr4piq1b-6 0}ol.lst-kix_p4k358qyzf3y-6.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-6 0}.lst-kix_xprjjxgruvg-1>li:before{content:"\0025cb  "}.lst-kix_4tgbgh9bb7g-7>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-7}.lst-kix_xprjjxgruvg-3>li:before{content:"\0025cf  "}.lst-kix_xprjjxgruvg-7>li:before{content:"\0025cb  "}.lst-kix_xprjjxgruvg-5>li:before{content:"\0025a0  "}.lst-kix_4roymkz9vma-5>li{counter-increment:lst-ctn-kix_4roymkz9vma-5}.lst-kix_7ka97cx080q5-6>li{counter-increment:lst-ctn-kix_7ka97cx080q5-6}.lst-kix_s8oehts9vu79-0>li:before{content:"\0025cf  "}.lst-kix_4fkgc27ss5mq-2>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-2,lower-roman) ". "}.lst-kix_4fkgc27ss5mq-0>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-0,decimal) ". "}.lst-kix_4fkgc27ss5mq-4>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-4,lower-latin) ". "}.lst-kix_4fkgc27ss5mq-6>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-6,decimal) ". "}.lst-kix_s8oehts9vu79-2>li:before{content:"\0025a0  "}ul.lst-kix_k0c3ozmp0pnp-8{list-style-type:none}.lst-kix_s8oehts9vu79-4>li:before{content:"\0025cb  "}.lst-kix_s8oehts9vu79-6>li:before{content:"\0025cf  "}ul.lst-kix_k0c3ozmp0pnp-4{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-5{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-6{list-style-type:none}.lst-kix_lux5kw5rxj0b-1>li:before{content:"\0025cb  "}.lst-kix_lux5kw5rxj0b-5>li:before{content:"\0025a0  "}.lst-kix_k9nxvn8n4dkv-5>li:before{content:"\0025a0  "}ul.lst-kix_k0c3ozmp0pnp-7{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-0{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-1{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-2{list-style-type:none}ul.lst-kix_k0c3ozmp0pnp-3{list-style-type:none}.lst-kix_s8oehts9vu79-8>li:before{content:"\0025a0  "}.lst-kix_lux5kw5rxj0b-3>li:before{content:"\0025cf  "}.lst-kix_k9nxvn8n4dkv-7>li:before{content:"\0025cb  "}ol.lst-kix_7ka97cx080q5-2.start{counter-reset:lst-ctn-kix_7ka97cx080q5-2 0}.lst-kix_4r80w4zgcdjq-2>li:before{content:"\0025a0  "}.lst-kix_k9nxvn8n4dkv-1>li:before{content:"\0025cb  "}.lst-kix_4r80w4zgcdjq-0>li:before{content:"\0025cf  "}.lst-kix_k9nxvn8n4dkv-3>li:before{content:"\0025cf  "}.lst-kix_4fkgc27ss5mq-8>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-8,lower-roman) ". "}.lst-kix_o1wsnr4piq1b-1>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-1}ul.lst-kix_o9ptu63d1d74-5{list-style-type:none}.lst-kix_7tkvl1c78ae3-4>li:before{content:"\0025cb  "}ul.lst-kix_8f57fsr386yf-0{list-style-type:none}ul.lst-kix_o9ptu63d1d74-6{list-style-type:none}ul.lst-kix_8f57fsr386yf-1{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-0{list-style-type:none}ul.lst-kix_o9ptu63d1d74-7{list-style-type:none}ul.lst-kix_8f57fsr386yf-2{list-style-type:none}ul.lst-kix_o9ptu63d1d74-8{list-style-type:none}ul.lst-kix_8f57fsr386yf-3{list-style-type:none}ul.lst-kix_o9ptu63d1d74-1{list-style-type:none}ul.lst-kix_8f57fsr386yf-4{list-style-type:none}.lst-kix_alje6q6hzub8-8>li:before{content:"-  "}ul.lst-kix_ouaz0d8j7xj-3{list-style-type:none}ul.lst-kix_o9ptu63d1d74-2{list-style-type:none}.lst-kix_c85u756bkx0j-6>li:before{content:"\0025cf  "}ul.lst-kix_8f57fsr386yf-5{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-4{list-style-type:none}ul.lst-kix_o9ptu63d1d74-3{list-style-type:none}ul.lst-kix_8f57fsr386yf-6{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-1{list-style-type:none}ul.lst-kix_o9ptu63d1d74-4{list-style-type:none}ul.lst-kix_8f57fsr386yf-7{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-2{list-style-type:none}.lst-kix_7tkvl1c78ae3-0>li:before{content:"\0025cf  "}.lst-kix_7tkvl1c78ae3-8>li:before{content:"\0025a0  "}ul.lst-kix_wxhgjniia6a-8{list-style-type:none}ul.lst-kix_wxhgjniia6a-7{list-style-type:none}ul.lst-kix_wxhgjniia6a-6{list-style-type:none}ul.lst-kix_wxhgjniia6a-5{list-style-type:none}ul.lst-kix_4fkgc27ss5mq-1{list-style-type:none}ul.lst-kix_wxhgjniia6a-4{list-style-type:none}.lst-kix_cm89sr4fo6m-5>li:before{content:"\0025a0  "}.lst-kix_54trtu6df9f8-6>li:before{content:"\0025cf  "}ul.lst-kix_wxhgjniia6a-3{list-style-type:none}ul.lst-kix_wxhgjniia6a-2{list-style-type:none}.lst-kix_2w2qcvvb39hd-0>li:before{content:"\0025cf  "}ul.lst-kix_wxhgjniia6a-1{list-style-type:none}ul.lst-kix_wxhgjniia6a-0{list-style-type:none}.lst-kix_b96slznfb3gj-5>li:before{content:"-  "}.lst-kix_c85u756bkx0j-2>li:before{content:"\0025a0  "}.lst-kix_alje6q6hzub8-0>li:before{content:"-  "}.lst-kix_alje6q6hzub8-4>li:before{content:"-  "}.lst-kix_e08effp12mqf-6>li:before{content:"\0025cf  "}.lst-kix_4h47981ce058-1>li:before{content:"\0025cb  "}.lst-kix_4h47981ce058-5>li:before{content:"\0025a0  "}ul.lst-kix_7ev7v5cpgtkc-0{list-style-type:none}ul.lst-kix_8f57fsr386yf-8{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-7{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-8{list-style-type:none}ul.lst-kix_ouaz0d8j7xj-5{list-style-type:none}ul.lst-kix_o9ptu63d1d74-0{list-style-type:none}.lst-kix_fyzn24yyf01g-4>li:before{content:"\0025cb  "}ul.lst-kix_ouaz0d8j7xj-6{list-style-type:none}.lst-kix_cm89sr4fo6m-1>li:before{content:"\0025cb  "}.lst-kix_f3z068owedlc-5>li:before{content:"\0025a0  "}.lst-kix_4r80w4zgcdjq-4>li:before{content:"\0025cb  "}.lst-kix_s0bij7o2r512-1>li:before{content:"\0025cb  "}ul.lst-kix_d1mamalvi3wu-1{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-0.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-0 0}.lst-kix_1293ztc79klg-0>li:before{content:"\0025cf  "}ul.lst-kix_d1mamalvi3wu-0{list-style-type:none}.lst-kix_jqqhsyijt7h5-8>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-8,lower-roman) ". "}.lst-kix_fyzn24yyf01g-0>li:before{content:"\0025cf  "}ul.lst-kix_b96slznfb3gj-3{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-1{list-style-type:none}ul.lst-kix_b96slznfb3gj-4{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-2{list-style-type:none}ul.lst-kix_d1mamalvi3wu-8{list-style-type:none}ul.lst-kix_b96slznfb3gj-5{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-3{list-style-type:none}ul.lst-kix_d1mamalvi3wu-7{list-style-type:none}.lst-kix_x30zykguitxy-7>li:before{content:"\0025cb  "}.lst-kix_jqqhsyijt7h5-4>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-4,lower-latin) ". "}ul.lst-kix_b96slznfb3gj-6{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-4{list-style-type:none}ul.lst-kix_d1mamalvi3wu-6{list-style-type:none}ul.lst-kix_ygjhinrwstz-8{list-style-type:none}ul.lst-kix_b96slznfb3gj-7{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-5{list-style-type:none}ul.lst-kix_d1mamalvi3wu-5{list-style-type:none}.lst-kix_f3z068owedlc-1>li:before{content:"\0025cb  "}ul.lst-kix_b96slznfb3gj-8{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-6{list-style-type:none}ul.lst-kix_d1mamalvi3wu-4{list-style-type:none}ul.lst-kix_ygjhinrwstz-6{list-style-type:none}ul.lst-kix_7ev7v5cpgtkc-7{list-style-type:none}ul.lst-kix_d1mamalvi3wu-3{list-style-type:none}ul.lst-kix_ygjhinrwstz-7{list-style-type:none}.lst-kix_4r80w4zgcdjq-8>li:before{content:"\0025a0  "}ul.lst-kix_7ev7v5cpgtkc-8{list-style-type:none}ul.lst-kix_d1mamalvi3wu-2{list-style-type:none}ul.lst-kix_ygjhinrwstz-4{list-style-type:none}ul.lst-kix_ygjhinrwstz-5{list-style-type:none}.lst-kix_1293ztc79klg-8>li:before{content:"\0025a0  "}ul.lst-kix_ygjhinrwstz-2{list-style-type:none}ul.lst-kix_ygjhinrwstz-3{list-style-type:none}ul.lst-kix_ygjhinrwstz-0{list-style-type:none}ul.lst-kix_ygjhinrwstz-1{list-style-type:none}ul.lst-kix_b96slznfb3gj-0{list-style-type:none}ul.lst-kix_b96slznfb3gj-1{list-style-type:none}ul.lst-kix_b96slznfb3gj-2{list-style-type:none}.lst-kix_1293ztc79klg-4>li:before{content:"\0025cb  "}.lst-kix_i2rq8qzg0efa-5>li:before{content:"+  "}.lst-kix_54trtu6df9f8-2>li:before{content:"\0025a0  "}.lst-kix_icjmrj4tgidb-3>li:before{content:"-  "}.lst-kix_f7u4w07k18eh-0>li:before{content:"\0025cf  "}.lst-kix_s27gvee6ygqb-7>li:before{content:"\0025cb  "}.lst-kix_4fkgc27ss5mq-6>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-6}.lst-kix_aninp2c52sy3-7>li:before{content:"\0025cb  "}.lst-kix_yk52z9n95trk-2>li:before{content:"\0025a0  "}.lst-kix_v26ncletnxud-8>li:before{content:"\0025a0  "}.lst-kix_x3ptir2y8jgm-6>li:before{content:"\0025cf  "}.lst-kix_aninp2c52sy3-3>li:before{content:"\0025cf  "}.lst-kix_x30zykguitxy-3>li:before{content:"\0025cf  "}.lst-kix_yk52z9n95trk-6>li:before{content:"\0025cf  "}.lst-kix_icjmrj4tgidb-7>li:before{content:"-  "}.lst-kix_x3ptir2y8jgm-2>li:before{content:"\0025a0  "}.lst-kix_v26ncletnxud-4>li:before{content:"\0025cb  "}.lst-kix_o1wsnr4piq1b-2>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-2,lower-roman) ") "}.lst-kix_o1wsnr4piq1b-3>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-3}ul.lst-kix_tiumq34nhjs2-0{list-style-type:none}ul.lst-kix_tiumq34nhjs2-1{list-style-type:none}.lst-kix_ieipvto6nfrd-3>li:before{content:"-  "}.lst-kix_4roymkz9vma-5>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) "." counter(lst-ctn-kix_4roymkz9vma-4,decimal) "." counter(lst-ctn-kix_4roymkz9vma-5,decimal) ". "}ul.lst-kix_tiumq34nhjs2-2{list-style-type:none}ul.lst-kix_tiumq34nhjs2-3{list-style-type:none}ul.lst-kix_tiumq34nhjs2-4{list-style-type:none}.lst-kix_e08effp12mqf-2>li:before{content:"\0025a0  "}ul.lst-kix_tiumq34nhjs2-5{list-style-type:none}ul.lst-kix_tiumq34nhjs2-6{list-style-type:none}ol.lst-kix_ur253ckj31qe-4.start{counter-reset:lst-ctn-kix_ur253ckj31qe-4 0}.lst-kix_2w2qcvvb39hd-8>li:before{content:"\0025a0  "}.lst-kix_f7u4w07k18eh-8>li:before{content:"\0025a0  "}.lst-kix_fyzn24yyf01g-8>li:before{content:"\0025a0  "}.lst-kix_s27gvee6ygqb-3>li:before{content:"\0025cf  "}ul.lst-kix_tiumq34nhjs2-7{list-style-type:none}.lst-kix_f7u4w07k18eh-4>li:before{content:"\0025cb  "}ul.lst-kix_tiumq34nhjs2-8{list-style-type:none}.lst-kix_b96slznfb3gj-1>li:before{content:"-  "}.lst-kix_ieipvto6nfrd-7>li:before{content:"-  "}.lst-kix_4roymkz9vma-1>li:before{content:"\0025cb  "}.lst-kix_2w2qcvvb39hd-4>li:before{content:"\0025cb  "}.lst-kix_9tajp1agqiyw-1>li:before{content:"\0025cb  "}.lst-kix_jza88dihngw3-0>li:before{content:"\0025cf  "}ul.lst-kix_9ft7dqndoe5p-0{list-style-type:none}ul.lst-kix_9ft7dqndoe5p-1{list-style-type:none}ul.lst-kix_9ft7dqndoe5p-2{list-style-type:none}ul.lst-kix_9ft7dqndoe5p-3{list-style-type:none}.lst-kix_e21xjkb2e68h-1>li:before{content:"\0025cb  "}.lst-kix_e21xjkb2e68h-5>li:before{content:"\0025a0  "}ul.lst-kix_9ft7dqndoe5p-4{list-style-type:none}ul.lst-kix_9ft7dqndoe5p-5{list-style-type:none}.lst-kix_abgphud9refj-6>li:before{content:"\0025cf  "}ul.lst-kix_9ft7dqndoe5p-6{list-style-type:none}ul.lst-kix_9ft7dqndoe5p-7{list-style-type:none}.lst-kix_9tajp1agqiyw-5>li:before{content:"\0025a0  "}ul.lst-kix_9ft7dqndoe5p-8{list-style-type:none}.lst-kix_jduurcgheuo-2>li:before{content:"-  "}.lst-kix_9bzwod9gvkvp-4>li:before{content:"\0025cb  "}.lst-kix_mpnpv7ekeo7q-4>li:before{content:"\0025cb  "}.lst-kix_4h54ot51egiu-3>li:before{content:"\0025cf  "}.lst-kix_ygjhinrwstz-4>li:before{content:"\0025cb  "}.lst-kix_abgphud9refj-2>li:before{content:"\0025a0  "}.lst-kix_eu4b93xn98yk-5>li:before{content:"\0025a0  "}.lst-kix_9bzwod9gvkvp-8>li:before{content:"\0025a0  "}.lst-kix_mpnpv7ekeo7q-8>li:before{content:"\0025a0  "}.lst-kix_ygjhinrwstz-0>li:before{content:"\0025cf  "}.lst-kix_g6dv2r60jppe-2>li:before{content:"\0025a0  "}.lst-kix_ejnvxy8b1e9w-5>li:before{content:"\0025a0  "}.lst-kix_4fkgc27ss5mq-8>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-8}.lst-kix_ejnvxy8b1e9w-1>li:before{content:"\0025cb  "}.lst-kix_jza88dihngw3-8>li:before{content:"\0025a0  "}.lst-kix_l4298avyh9v6-1>li:before{content:"\0025cb  "}.lst-kix_g6dv2r60jppe-6>li:before{content:"\0025cf  "}.lst-kix_jza88dihngw3-4>li:before{content:"\0025cb  "}.lst-kix_4h54ot51egiu-7>li:before{content:"\0025cb  "}.lst-kix_zhrwxtd2sngd-5>li:before{content:"\0025a0  "}.lst-kix_k0c3ozmp0pnp-5>li:before{content:"\0025a0  "}.lst-kix_tgtwnecgfrou-5>li:before{content:"+  "}.lst-kix_ur253ckj31qe-6>li{counter-increment:lst-ctn-kix_ur253ckj31qe-6}.lst-kix_jqqhsyijt7h5-0>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-0,decimal) ". "}.lst-kix_i2rq8qzg0efa-1>li:before{content:"+  "}ol.lst-kix_flatlsyq6jr0-0.start{counter-reset:lst-ctn-kix_flatlsyq6jr0-0 0}.lst-kix_l4298avyh9v6-5>li:before{content:"\0025a0  "}.lst-kix_k0c3ozmp0pnp-1>li:before{content:"\0025cb  "}.lst-kix_tgtwnecgfrou-1>li:before{content:"+  "}.lst-kix_uh3r4fp09cs-7>li:before{content:"\0025cb  "}.lst-kix_tfsdypepg13u-1>li:before{content:"\0025cb  "}ul.lst-kix_mpnpv7ekeo7q-6{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-5{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-8{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-7{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-2{list-style-type:none}.lst-kix_nzxgmnmwjatv-1>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-1}ul.lst-kix_mpnpv7ekeo7q-1{list-style-type:none}.lst-kix_9ft7dqndoe5p-1>li:before{content:"\0025cb  "}.lst-kix_9ft7dqndoe5p-5>li:before{content:"\0025a0  "}ul.lst-kix_mpnpv7ekeo7q-4{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-3{list-style-type:none}.lst-kix_uh3r4fp09cs-3>li:before{content:"\0025cf  "}.lst-kix_tfsdypepg13u-5>li:before{content:"\0025a0  "}.lst-kix_s0bij7o2r512-5>li:before{content:"\0025a0  "}ol.lst-kix_o1wsnr4piq1b-8{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-1{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-0{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-3{list-style-type:none}ul.lst-kix_mpnpv7ekeo7q-0{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-2{list-style-type:none}ol.lst-kix_o1wsnr4piq1b-5{list-style-type:none}.lst-kix_zhrwxtd2sngd-1>li:before{content:"\0025cb  "}ol.lst-kix_4tgbgh9bb7g-5.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-5 0}ol.lst-kix_o1wsnr4piq1b-4{list-style-type:none}.lst-kix_9bzwod9gvkvp-0>li:before{content:"\0025cf  "}.lst-kix_mpnpv7ekeo7q-0>li:before{content:"\0025cf  "}ol.lst-kix_o1wsnr4piq1b-7{list-style-type:none}ol.lst-kix_ur253ckj31qe-7.start{counter-reset:lst-ctn-kix_ur253ckj31qe-7 0}ol.lst-kix_o1wsnr4piq1b-6{list-style-type:none}ul.lst-kix_4h54ot51egiu-2{list-style-type:none}ul.lst-kix_4h54ot51egiu-1{list-style-type:none}ul.lst-kix_4h54ot51egiu-0{list-style-type:none}ul.lst-kix_4h54ot51egiu-6{list-style-type:none}.lst-kix_cpm4oaqfjy6b-1>li:before{content:"\0025cb  "}ul.lst-kix_4h54ot51egiu-5{list-style-type:none}ul.lst-kix_4h54ot51egiu-4{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-7.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-7 0}ul.lst-kix_4h54ot51egiu-3{list-style-type:none}ul.lst-kix_zvvafajkk2a-2{list-style-type:none}ul.lst-kix_zvvafajkk2a-3{list-style-type:none}ul.lst-kix_4h54ot51egiu-8{list-style-type:none}ul.lst-kix_zvvafajkk2a-4{list-style-type:none}.lst-kix_o9ptu63d1d74-5>li:before{content:"\0025a0  "}ul.lst-kix_4h54ot51egiu-7{list-style-type:none}ul.lst-kix_zvvafajkk2a-5{list-style-type:none}ul.lst-kix_zvvafajkk2a-6{list-style-type:none}ul.lst-kix_zvvafajkk2a-7{list-style-type:none}ul.lst-kix_zvvafajkk2a-8{list-style-type:none}ul.lst-kix_x30zykguitxy-8{list-style-type:none}.lst-kix_o9ptu63d1d74-7>li:before{content:"\0025cb  "}ol.lst-kix_jqqhsyijt7h5-3.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-3 0}.lst-kix_uhfsw7q6s458-6>li:before{content:"\0025cf  "}.lst-kix_cpm4oaqfjy6b-7>li:before{content:"\0025cb  "}.lst-kix_7ka97cx080q5-0>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-0,decimal) ". "}.lst-kix_uhfsw7q6s458-4>li:before{content:"\0025cb  "}ul.lst-kix_x30zykguitxy-0{list-style-type:none}ul.lst-kix_x30zykguitxy-1{list-style-type:none}ul.lst-kix_x30zykguitxy-2{list-style-type:none}ul.lst-kix_x30zykguitxy-3{list-style-type:none}ul.lst-kix_x30zykguitxy-4{list-style-type:none}ul.lst-kix_x30zykguitxy-5{list-style-type:none}ul.lst-kix_x30zykguitxy-6{list-style-type:none}ul.lst-kix_x30zykguitxy-7{list-style-type:none}.lst-kix_ontx89k1fqr9-4>li:before{content:"\0025cb  "}.lst-kix_p4k358qyzf3y-2>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-2}.lst-kix_ontx89k1fqr9-6>li:before{content:"\0025cf  "}.lst-kix_dkq97x3e7jfk-5>li:before{content:"\0025a0  "}ol.lst-kix_uthm435lclg8-0{list-style-type:none}.lst-kix_7ka97cx080q5-6>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-6,decimal) ". "}.lst-kix_7ka97cx080q5-8>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-8,lower-roman) ". "}.lst-kix_dkq97x3e7jfk-7>li:before{content:"\0025cb  "}.lst-kix_nr7rbh2hfyf0-3>li:before{content:"\0025cf  "}.lst-kix_25j83v4xa4ig-2>li:before{content:"\0025a0  "}.lst-kix_x7o2gv80my2k-0>li:before{content:"\0025cf  "}.lst-kix_x7o2gv80my2k-2>li:before{content:"\0025a0  "}.lst-kix_bv0w79unf8ye-6>li:before{content:"\0025cf  "}.lst-kix_bv0w79unf8ye-8>li:before{content:"\0025a0  "}.lst-kix_nr7rbh2hfyf0-1>li:before{content:"\0025cb  "}.lst-kix_25j83v4xa4ig-0>li:before{content:"\0025cf  "}ul.lst-kix_i0q09d67i790-8{list-style-type:none}.lst-kix_x7o2gv80my2k-8>li:before{content:"\0025a0  "}ul.lst-kix_i0q09d67i790-7{list-style-type:none}ul.lst-kix_i0q09d67i790-6{list-style-type:none}ul.lst-kix_i0q09d67i790-5{list-style-type:none}ul.lst-kix_i0q09d67i790-4{list-style-type:none}ul.lst-kix_i0q09d67i790-3{list-style-type:none}ul.lst-kix_i0q09d67i790-2{list-style-type:none}ul.lst-kix_i0q09d67i790-1{list-style-type:none}ul.lst-kix_i0q09d67i790-0{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-2.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-2 0}ol.lst-kix_jqqhsyijt7h5-8.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-8 0}.lst-kix_qqf2twdkbiw0-1>li:before{content:"\0025cb  "}.lst-kix_xiiy75ihy1il-6>li:before{content:"\0025cf  "}.lst-kix_xzp0xel2onox-1>li:before{content:"\0025cb  "}.lst-kix_xzp0xel2onox-3>li:before{content:"\0025cf  "}.lst-kix_qqf2twdkbiw0-3>li:before{content:"\0025cf  "}.lst-kix_xiiy75ihy1il-4>li:before{content:"\0025cb  "}.lst-kix_vxbaz02t98k3-7>li:before{content:"\0025cb  "}.lst-kix_bv0w79unf8ye-0>li:before{content:"\0025cf  "}ol.lst-kix_7ka97cx080q5-5.start{counter-reset:lst-ctn-kix_7ka97cx080q5-5 0}ul.lst-kix_lux5kw5rxj0b-8{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-7{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-6{list-style-type:none}ul.lst-kix_zvvafajkk2a-0{list-style-type:none}.lst-kix_ur253ckj31qe-8>li{counter-increment:lst-ctn-kix_ur253ckj31qe-8}.lst-kix_ygjhinrwstz-6>li:before{content:"\0025cf  "}ul.lst-kix_zvvafajkk2a-1{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-1{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-0{list-style-type:none}ol.lst-kix_4roymkz9vma-2.start{counter-reset:lst-ctn-kix_4roymkz9vma-2 0}ul.lst-kix_lux5kw5rxj0b-5{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-4{list-style-type:none}ul.lst-kix_lux5kw5rxj0b-3{list-style-type:none}.lst-kix_e21xjkb2e68h-7>li:before{content:"\0025cb  "}ul.lst-kix_lux5kw5rxj0b-2{list-style-type:none}.lst-kix_qswbihp1i06v-8>li:before{content:"\0025a0  "}ul.lst-kix_a0ivi9b1qp8i-8{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-7{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-6{list-style-type:none}.lst-kix_nwfayb939eln-7>li:before{content:"\0025cb  "}ul.lst-kix_k08o05q2crtr-1{list-style-type:none}ul.lst-kix_k08o05q2crtr-0{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-1{list-style-type:none}ul.lst-kix_k08o05q2crtr-5{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-0{list-style-type:none}ul.lst-kix_k08o05q2crtr-4{list-style-type:none}ul.lst-kix_k08o05q2crtr-3{list-style-type:none}ul.lst-kix_k08o05q2crtr-2{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-5{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-4{list-style-type:none}ul.lst-kix_k08o05q2crtr-8{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-3{list-style-type:none}ul.lst-kix_k08o05q2crtr-7{list-style-type:none}ul.lst-kix_a0ivi9b1qp8i-2{list-style-type:none}ul.lst-kix_k08o05q2crtr-6{list-style-type:none}.lst-kix_xwc7ibkfiucl-0>li:before{content:"\0025cf  "}ul.lst-kix_ontx89k1fqr9-7{list-style-type:none}.lst-kix_m3f8ler1ak4c-4>li:before{content:"\0025cb  "}ul.lst-kix_ontx89k1fqr9-8{list-style-type:none}ul.lst-kix_ontx89k1fqr9-5{list-style-type:none}ul.lst-kix_ontx89k1fqr9-6{list-style-type:none}ul.lst-kix_54trtu6df9f8-8{list-style-type:none}.lst-kix_o1wsnr4piq1b-8>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-8,lower-roman) ". "}.lst-kix_n2keu386kfur-2>li:before{content:"\0025a0  "}ul.lst-kix_54trtu6df9f8-4{list-style-type:none}.lst-kix_nwfayb939eln-1>li:before{content:"\0025cb  "}.lst-kix_fv8y17g7myim-4>li:before{content:"\0025cb  "}ul.lst-kix_54trtu6df9f8-5{list-style-type:none}.lst-kix_v26ncletnxud-2>li:before{content:"\0025a0  "}ul.lst-kix_54trtu6df9f8-6{list-style-type:none}.lst-kix_o6fhz26vvidv-0>li:before{content:"\0025cf  "}ul.lst-kix_54trtu6df9f8-7{list-style-type:none}.lst-kix_6bb7qu9vjqr2-5>li:before{content:"-  "}ul.lst-kix_54trtu6df9f8-0{list-style-type:none}.lst-kix_m3f8ler1ak4c-2>li:before{content:"\0025a0  "}ul.lst-kix_54trtu6df9f8-1{list-style-type:none}.lst-kix_o1wsnr4piq1b-6>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-6,decimal) ". "}ul.lst-kix_54trtu6df9f8-2{list-style-type:none}ul.lst-kix_54trtu6df9f8-3{list-style-type:none}ul.lst-kix_2w2qcvvb39hd-5{list-style-type:none}ul.lst-kix_2w2qcvvb39hd-4{list-style-type:none}ul.lst-kix_2w2qcvvb39hd-7{list-style-type:none}.lst-kix_o1wsnr4piq1b-8>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-8}ul.lst-kix_2w2qcvvb39hd-6{list-style-type:none}.lst-kix_emh9tw6ivvnv-4>li:before{content:"\0025cb  "}ul.lst-kix_2w2qcvvb39hd-8{list-style-type:none}.lst-kix_ouaz0d8j7xj-8>li:before{content:"\0025a0  "}ol.lst-kix_7ka97cx080q5-0.start{counter-reset:lst-ctn-kix_7ka97cx080q5-0 0}.lst-kix_qswbihp1i06v-0>li:before{content:"\0025cf  "}.lst-kix_oli7yc1w3gpg-7>li:before{content:"\0025cb  "}ol.lst-kix_p4k358qyzf3y-3.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-3 0}.lst-kix_emh9tw6ivvnv-2>li:before{content:"\0025a0  "}ul.lst-kix_2w2qcvvb39hd-1{list-style-type:none}ul.lst-kix_2w2qcvvb39hd-0{list-style-type:none}ul.lst-kix_2w2qcvvb39hd-3{list-style-type:none}.lst-kix_qswbihp1i06v-2>li:before{content:"\0025a0  "}ul.lst-kix_2w2qcvvb39hd-2{list-style-type:none}.lst-kix_n2keu386kfur-8>li:before{content:"\0025a0  "}.lst-kix_ouaz0d8j7xj-2>li:before{content:"\0025a0  "}.lst-kix_h9p9lo20tlpj-3>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-3,decimal) ". "}.lst-kix_oli7yc1w3gpg-1>li:before{content:"\0025cb  "}.lst-kix_ouaz0d8j7xj-0>li:before{content:"\0025cf  "}.lst-kix_c4nzmsldypra-6>li:before{content:"\0025cf  "}.lst-kix_o6fhz26vvidv-8>li:before{content:"\0025a0  "}.lst-kix_25j83v4xa4ig-8>li:before{content:"\0025a0  "}.lst-kix_p4k358qyzf3y-6>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-6}ul.lst-kix_ric95po2pkzv-8{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-2.start{counter-reset:lst-ctn-kix_h9p9lo20tlpj-2 0}ul.lst-kix_ric95po2pkzv-7{list-style-type:none}.lst-kix_tcwoy9lfuqk4-4>li:before{content:"\0025cb  "}ul.lst-kix_xprjjxgruvg-2{list-style-type:none}.lst-kix_fv8y17g7myim-2>li:before{content:"\0025a0  "}ul.lst-kix_xprjjxgruvg-3{list-style-type:none}ul.lst-kix_xprjjxgruvg-0{list-style-type:none}ul.lst-kix_xprjjxgruvg-1{list-style-type:none}.lst-kix_z9n4e0cpc149-7>li:before{content:"\0025cb  "}ul.lst-kix_ric95po2pkzv-2{list-style-type:none}ul.lst-kix_xprjjxgruvg-6{list-style-type:none}ul.lst-kix_ric95po2pkzv-1{list-style-type:none}ul.lst-kix_xprjjxgruvg-7{list-style-type:none}ul.lst-kix_ric95po2pkzv-0{list-style-type:none}ul.lst-kix_xprjjxgruvg-4{list-style-type:none}.lst-kix_o6fhz26vvidv-2>li:before{content:"\0025a0  "}.lst-kix_w7vhjpenesq3-4>li:before{content:"\0025cb  "}.lst-kix_gayel7f32yde-5>li:before{content:"-  "}.lst-kix_gayel7f32yde-7>li:before{content:"-  "}.lst-kix_6bb7qu9vjqr2-7>li:before{content:"-  "}.lst-kix_tcwoy9lfuqk4-2>li:before{content:"\0025a0  "}ul.lst-kix_xprjjxgruvg-5{list-style-type:none}ul.lst-kix_ric95po2pkzv-6{list-style-type:none}.lst-kix_z9n4e0cpc149-5>li:before{content:"\0025a0  "}ul.lst-kix_ric95po2pkzv-5{list-style-type:none}.lst-kix_n2keu386kfur-0>li:before{content:"\0025cf  "}.lst-kix_c4nzmsldypra-4>li:before{content:"\0025cb  "}ul.lst-kix_ric95po2pkzv-4{list-style-type:none}ul.lst-kix_xprjjxgruvg-8{list-style-type:none}ul.lst-kix_ric95po2pkzv-3{list-style-type:none}.lst-kix_w7vhjpenesq3-6>li:before{content:"\0025cf  "}ol.lst-kix_jqqhsyijt7h5-5.start{counter-reset:lst-ctn-kix_jqqhsyijt7h5-5 0}.lst-kix_n2tc9z47p0ha-2>li:before{content:"\0025a0  "}ul.lst-kix_c85u756bkx0j-4{list-style-type:none}ol.lst-kix_4fkgc27ss5mq-4.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-4 0}ul.lst-kix_c85u756bkx0j-3{list-style-type:none}.lst-kix_n2tc9z47p0ha-4>li:before{content:"\0025cb  "}ul.lst-kix_c85u756bkx0j-6{list-style-type:none}ul.lst-kix_c85u756bkx0j-5{list-style-type:none}ul.lst-kix_c85u756bkx0j-8{list-style-type:none}ul.lst-kix_c85u756bkx0j-7{list-style-type:none}.lst-kix_xwc7ibkfiucl-6>li:before{content:"\0025cf  "}ul.lst-kix_c85u756bkx0j-0{list-style-type:none}ul.lst-kix_c85u756bkx0j-2{list-style-type:none}ul.lst-kix_c85u756bkx0j-1{list-style-type:none}.lst-kix_xwc7ibkfiucl-8>li:before{content:"\0025a0  "}.lst-kix_c85u756bkx0j-8>li:before{content:"\0025a0  "}.lst-kix_54trtu6df9f8-4>li:before{content:"\0025cb  "}.lst-kix_wxhgjniia6a-2>li:before{content:"\0025a0  "}.lst-kix_c85u756bkx0j-0>li:before{content:"\0025cf  "}.lst-kix_alje6q6hzub8-2>li:before{content:"-  "}.lst-kix_f3z068owedlc-7>li:before{content:"\0025cb  "}.lst-kix_fyzn24yyf01g-6>li:before{content:"\0025cf  "}.lst-kix_s0bij7o2r512-3>li:before{content:"\0025cf  "}.lst-kix_cm89sr4fo6m-3>li:before{content:"\0025cf  "}.lst-kix_e08effp12mqf-4>li:before{content:"\0025cb  "}.lst-kix_7tkvl1c78ae3-6>li:before{content:"\0025cf  "}.lst-kix_b96slznfb3gj-7>li:before{content:"-  "}.lst-kix_4h47981ce058-3>li:before{content:"\0025cf  "}.lst-kix_4fkgc27ss5mq-4>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-4}.lst-kix_jqqhsyijt7h5-6>li:before{content:"" counter(lst-ctn-kix_jqqhsyijt7h5-6,decimal) ". "}.lst-kix_1293ztc79klg-2>li:before{content:"\0025a0  "}.lst-kix_y5u0wek48eld-2>li:before{content:"-  "}.lst-kix_4tgbgh9bb7g-1>li:before{content:"\0025cb  "}.lst-kix_i2rq8qzg0efa-3>li:before{content:"+  "}.lst-kix_p4k358qyzf3y-2>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-2,lower-roman) ". "}ol.lst-kix_p4k358qyzf3y-1.start{counter-reset:lst-ctn-kix_p4k358qyzf3y-1 0}.lst-kix_icjmrj4tgidb-5>li:before{content:"-  "}.lst-kix_yk52z9n95trk-4>li:before{content:"\0025cb  "}.lst-kix_x3ptir2y8jgm-8>li:before{content:"\0025a0  "}.lst-kix_x30zykguitxy-1>li:before{content:"\0025cb  "}.lst-kix_o1wsnr4piq1b-0>li:before{content:"" counter(lst-ctn-kix_o1wsnr4piq1b-0,decimal) ") "}.lst-kix_aninp2c52sy3-5>li:before{content:"\0025a0  "}.lst-kix_hxvvdzewxqs2-4>li:before{content:"\0025cb  "}.lst-kix_ieipvto6nfrd-1>li:before{content:"-  "}.lst-kix_4roymkz9vma-3>li:before{content:"\0025cf .\0025cb ." counter(lst-ctn-kix_4roymkz9vma-2,decimal) "." counter(lst-ctn-kix_4roymkz9vma-3,decimal) ". "}ul.lst-kix_y5u0wek48eld-7{list-style-type:none}ul.lst-kix_y5u0wek48eld-6{list-style-type:none}ul.lst-kix_y5u0wek48eld-8{list-style-type:none}ol.lst-kix_ontx89k1fqr9-0.start{counter-reset:lst-ctn-kix_ontx89k1fqr9-0 0}ul.lst-kix_y5u0wek48eld-3{list-style-type:none}.lst-kix_ns4c34lwftpb-7>li:before{content:"\0025cb  "}.lst-kix_a76b6w8pnkyv-1>li:before{content:"\0025cb  "}ul.lst-kix_y5u0wek48eld-2{list-style-type:none}ul.lst-kix_y5u0wek48eld-5{list-style-type:none}ul.lst-kix_y5u0wek48eld-4{list-style-type:none}.lst-kix_d1mamalvi3wu-3>li:before{content:"\0025cf  "}.lst-kix_x3ptir2y8jgm-0>li:before{content:"\0025cf  "}ul.lst-kix_y5u0wek48eld-1{list-style-type:none}ul.lst-kix_y5u0wek48eld-0{list-style-type:none}.lst-kix_ia5tg78e6pm4-6>li:before{content:"\0025cf  "}.lst-kix_k08o05q2crtr-2>li:before{content:"\0025a0  "}.lst-kix_f7u4w07k18eh-6>li:before{content:"\0025cf  "}.lst-kix_2w2qcvvb39hd-6>li:before{content:"\0025cf  "}.lst-kix_s27gvee6ygqb-5>li:before{content:"\0025a0  "}.lst-kix_mpnpv7ekeo7q-2>li:before{content:"\0025a0  "}.lst-kix_9tajp1agqiyw-3>li:before{content:"\0025cf  "}.lst-kix_ock170xmc4e5-6>li:before{content:"\0025cf  "}.lst-kix_vxbaz02t98k3-1>li:before{content:"\0025cb  "}.lst-kix_ur253ckj31qe-5>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-5,lower-roman) ". "}.lst-kix_eu4b93xn98yk-7>li:before{content:"\0025cb  "}.lst-kix_abgphud9refj-4>li:before{content:"\0025cb  "}ul.lst-kix_jduurcgheuo-8{list-style-type:none}ul.lst-kix_jduurcgheuo-7{list-style-type:none}ul.lst-kix_jduurcgheuo-6{list-style-type:none}.lst-kix_i0q09d67i790-0>li:before{content:"\0025cf  "}.lst-kix_g6dv2r60jppe-0>li:before{content:"\0025cf  "}.lst-kix_b6qjs7hmfdgs-2>li:before{content:"\0025a0  "}.lst-kix_ejnvxy8b1e9w-3>li:before{content:"\0025cf  "}.lst-kix_np2y6n1468n7-3>li:before{content:"\0025cf  "}.lst-kix_i0q09d67i790-8>li:before{content:"\0025a0  "}.lst-kix_g6dv2r60jppe-8>li:before{content:"\0025a0  "}ol.lst-kix_4roymkz9vma-4.start{counter-reset:lst-ctn-kix_4roymkz9vma-4 0}.lst-kix_4h54ot51egiu-5>li:before{content:"\0025a0  "}.lst-kix_jza88dihngw3-6>li:before{content:"\0025cf  "}ol.lst-kix_4roymkz9vma-7.start{counter-reset:lst-ctn-kix_4roymkz9vma-7 0}.lst-kix_uthm435lclg8-4>li:before{content:"\0025cb  "}.lst-kix_zhrwxtd2sngd-7>li:before{content:"\0025cb  "}.lst-kix_tgtwnecgfrou-3>li:before{content:"+  "}.lst-kix_k0c3ozmp0pnp-3>li:before{content:"\0025cf  "}.lst-kix_ric95po2pkzv-6>li:before{content:"\0025cf  "}.lst-kix_uh3r4fp09cs-5>li:before{content:"\0025a0  "}ul.lst-kix_jduurcgheuo-1{list-style-type:none}ul.lst-kix_jduurcgheuo-0{list-style-type:none}ol.lst-kix_7ka97cx080q5-7.start{counter-reset:lst-ctn-kix_7ka97cx080q5-7 0}ul.lst-kix_jduurcgheuo-5{list-style-type:none}ul.lst-kix_jduurcgheuo-4{list-style-type:none}ul.lst-kix_jduurcgheuo-3{list-style-type:none}ul.lst-kix_jduurcgheuo-2{list-style-type:none}.lst-kix_l4298avyh9v6-7>li:before{content:"\0025cb  "}.lst-kix_9ft7dqndoe5p-3>li:before{content:"\0025cf  "}.lst-kix_tfsdypepg13u-7>li:before{content:"\0025cb  "}.lst-kix_tnmatnpjv4t2-5>li:before{content:"\0025a0  "}.lst-kix_a0ivi9b1qp8i-5>li:before{content:"\0025a0  "}.lst-kix_9bzwod9gvkvp-2>li:before{content:"\0025a0  "}.lst-kix_gmyqb4k4mims-2>li:before{content:"\0025a0  "}ul.lst-kix_zhrwxtd2sngd-8{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-7{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-6{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-5{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-4{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-3{list-style-type:none}.lst-kix_4tgbgh9bb7g-5>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-5}ul.lst-kix_zhrwxtd2sngd-2{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-1{list-style-type:none}ul.lst-kix_zhrwxtd2sngd-0{list-style-type:none}.lst-kix_4fkgc27ss5mq-3>li{counter-increment:lst-ctn-kix_4fkgc27ss5mq-3}.lst-kix_tczavuk46t7o-7>li:before{content:"\0025cb  "}.lst-kix_8rc65o4ywyr4-1>li:before{content:"\0025cb  "}.lst-kix_tczavuk46t7o-8>li:before{content:"\0025a0  "}.lst-kix_xqkxylhicw49-6>li:before{content:"\0025cf  "}.lst-kix_8rc65o4ywyr4-2>li:before{content:"\0025a0  "}.lst-kix_tczavuk46t7o-5>li:before{content:"\0025a0  "}.lst-kix_xqkxylhicw49-7>li:before{content:"\0025cb  "}.lst-kix_8rc65o4ywyr4-5>li:before{content:"\0025a0  "}ol.lst-kix_7ka97cx080q5-4.start{counter-reset:lst-ctn-kix_7ka97cx080q5-4 0}.lst-kix_8rc65o4ywyr4-4>li:before{content:"\0025cb  "}.lst-kix_8rc65o4ywyr4-8>li:before{content:"\0025a0  "}.lst-kix_8f57fsr386yf-8>li:before{content:"\0025a0  "}.lst-kix_8rc65o4ywyr4-7>li:before{content:"\0025cb  "}.lst-kix_4tgbgh9bb7g-3>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-3}ul.lst-kix_cxongm456yyi-5{list-style-type:none}ul.lst-kix_cxongm456yyi-6{list-style-type:none}ul.lst-kix_cxongm456yyi-3{list-style-type:none}ul.lst-kix_cxongm456yyi-4{list-style-type:none}.lst-kix_xqkxylhicw49-0>li:before{content:"\0025cf  "}.lst-kix_8f57fsr386yf-6>li:before{content:"\0025cf  "}ul.lst-kix_cxongm456yyi-7{list-style-type:none}ul.lst-kix_cxongm456yyi-8{list-style-type:none}.lst-kix_h9p9lo20tlpj-0>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-0}.lst-kix_8f57fsr386yf-5>li:before{content:"\0025a0  "}ul.lst-kix_cxongm456yyi-1{list-style-type:none}.lst-kix_7ka97cx080q5-3>li{counter-increment:lst-ctn-kix_7ka97cx080q5-3}ul.lst-kix_cxongm456yyi-2{list-style-type:none}.lst-kix_xqkxylhicw49-1>li:before{content:"\0025cb  "}ul.lst-kix_cxongm456yyi-0{list-style-type:none}.lst-kix_xqkxylhicw49-4>li:before{content:"\0025cb  "}.lst-kix_8f57fsr386yf-2>li:before{content:"\0025a0  "}.lst-kix_xqkxylhicw49-3>li:before{content:"\0025cf  "}ul.lst-kix_xqkxylhicw49-1{list-style-type:none}ul.lst-kix_xqkxylhicw49-0{list-style-type:none}.lst-kix_vlcqhp8hs5e3-0>li:before{content:"\0025cf  "}.lst-kix_1rk6yybn5non-3>li:before{content:"-  "}.lst-kix_qwci6b6x7du4-8>li:before{content:"-  "}.lst-kix_qwci6b6x7du4-7>li:before{content:"-  "}ul.lst-kix_xqkxylhicw49-5{list-style-type:none}ul.lst-kix_f3z068owedlc-8{list-style-type:none}ul.lst-kix_xqkxylhicw49-4{list-style-type:none}ul.lst-kix_xqkxylhicw49-3{list-style-type:none}ul.lst-kix_f3z068owedlc-6{list-style-type:none}ul.lst-kix_xqkxylhicw49-2{list-style-type:none}ul.lst-kix_f3z068owedlc-7{list-style-type:none}ul.lst-kix_f3z068owedlc-4{list-style-type:none}ul.lst-kix_xqkxylhicw49-8{list-style-type:none}.lst-kix_1rk6yybn5non-7>li:before{content:"-  "}ul.lst-kix_f3z068owedlc-5{list-style-type:none}ul.lst-kix_gmyqb4k4mims-1{list-style-type:none}ul.lst-kix_xqkxylhicw49-7{list-style-type:none}ul.lst-kix_f3z068owedlc-2{list-style-type:none}ul.lst-kix_gmyqb4k4mims-0{list-style-type:none}ul.lst-kix_xqkxylhicw49-6{list-style-type:none}ul.lst-kix_f3z068owedlc-3{list-style-type:none}ul.lst-kix_gmyqb4k4mims-3{list-style-type:none}ul.lst-kix_f3z068owedlc-0{list-style-type:none}ul.lst-kix_gmyqb4k4mims-2{list-style-type:none}.lst-kix_vlcqhp8hs5e3-5>li:before{content:"\0025a0  "}.lst-kix_vlcqhp8hs5e3-6>li:before{content:"\0025cf  "}.lst-kix_1rk6yybn5non-6>li:before{content:"-  "}ul.lst-kix_f3z068owedlc-1{list-style-type:none}ul.lst-kix_gmyqb4k4mims-5{list-style-type:none}ul.lst-kix_gmyqb4k4mims-4{list-style-type:none}.lst-kix_nzxgmnmwjatv-3>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-3,decimal) ". "}.lst-kix_qwci6b6x7du4-1>li:before{content:"-  "}.lst-kix_qwci6b6x7du4-4>li:before{content:"-  "}.lst-kix_yqx6kz4lvbt4-6>li:before{content:"\0025cf  "}.lst-kix_yqx6kz4lvbt4-5>li:before{content:"\0025a0  "}.lst-kix_nzxgmnmwjatv-6>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-6,decimal) ". "}ul.lst-kix_gmyqb4k4mims-7{list-style-type:none}ul.lst-kix_gmyqb4k4mims-6{list-style-type:none}ul.lst-kix_w7vhjpenesq3-8{list-style-type:none}ul.lst-kix_gmyqb4k4mims-8{list-style-type:none}.lst-kix_nzxgmnmwjatv-0>li:before{content:"" counter(lst-ctn-kix_nzxgmnmwjatv-0,decimal) ". "}ul.lst-kix_w7vhjpenesq3-2{list-style-type:none}ul.lst-kix_w7vhjpenesq3-3{list-style-type:none}.lst-kix_vlcqhp8hs5e3-3>li:before{content:"\0025cf  "}.lst-kix_1rk6yybn5non-0>li:before{content:"-  "}ul.lst-kix_w7vhjpenesq3-0{list-style-type:none}ul.lst-kix_w7vhjpenesq3-1{list-style-type:none}ul.lst-kix_w7vhjpenesq3-6{list-style-type:none}ul.lst-kix_w7vhjpenesq3-7{list-style-type:none}.lst-kix_vlcqhp8hs5e3-2>li:before{content:"\0025a0  "}ul.lst-kix_w7vhjpenesq3-4{list-style-type:none}.lst-kix_tiumq34nhjs2-6>li:before{content:"\0025cf  "}ul.lst-kix_w7vhjpenesq3-5{list-style-type:none}.lst-kix_ur253ckj31qe-2>li{counter-increment:lst-ctn-kix_ur253ckj31qe-2}.lst-kix_mi7vyzrjrpd8-8>li:before{content:"\0025a0  "}.lst-kix_tiumq34nhjs2-3>li:before{content:"\0025cf  "}ol.lst-kix_ur253ckj31qe-1.start{counter-reset:lst-ctn-kix_ur253ckj31qe-1 0}.lst-kix_xm7ynypsh0z5-8>li:before{content:"\0025a0  "}.lst-kix_tiumq34nhjs2-1>li:before{content:"\0025cb  "}.lst-kix_l9w5fm8md2s9-2>li:before{content:"\0025a0  "}.lst-kix_mi7vyzrjrpd8-4>li:before{content:"\0025cb  "}.lst-kix_wawbbtzgf6il-6>li:before{content:"\0025cf  "}.lst-kix_mi7vyzrjrpd8-2>li:before{content:"\0025a0  "}.lst-kix_wawbbtzgf6il-4>li:before{content:"\0025cb  "}.lst-kix_flatlsyq6jr0-7>li:before{content:"\0025cb  "}.lst-kix_yqx6kz4lvbt4-2>li:before{content:"\0025a0  "}ul.lst-kix_tfsdypepg13u-8{list-style-type:none}ul.lst-kix_tfsdypepg13u-5{list-style-type:none}ul.lst-kix_tfsdypepg13u-4{list-style-type:none}ul.lst-kix_tfsdypepg13u-7{list-style-type:none}.lst-kix_yqx6kz4lvbt4-0>li:before{content:"\0025cf  "}.lst-kix_b6qjs7hmfdgs-3>li:before{content:"\0025cf  "}.lst-kix_np2y6n1468n7-2>li:before{content:"\0025a0  "}ul.lst-kix_tfsdypepg13u-6{list-style-type:none}ul.lst-kix_tfsdypepg13u-1{list-style-type:none}ul.lst-kix_tfsdypepg13u-0{list-style-type:none}.lst-kix_p4k358qyzf3y-0>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-0}ul.lst-kix_tfsdypepg13u-3{list-style-type:none}.lst-kix_i0q09d67i790-1>li:before{content:"\0025cb  "}ul.lst-kix_tfsdypepg13u-2{list-style-type:none}ul.lst-kix_fv8y17g7myim-1{list-style-type:none}.lst-kix_np2y6n1468n7-6>li:before{content:"\0025cf  "}ul.lst-kix_fv8y17g7myim-0{list-style-type:none}.lst-kix_xm7ynypsh0z5-4>li:before{content:"\0025cb  "}.lst-kix_l9w5fm8md2s9-6>li:before{content:"\0025cf  "}.lst-kix_i0q09d67i790-7>li:before{content:"\0025cb  "}.lst-kix_xm7ynypsh0z5-2>li:before{content:"\0025a0  "}ul.lst-kix_fv8y17g7myim-7{list-style-type:none}.lst-kix_wawbbtzgf6il-0>li:before{content:"\0025cf  "}ul.lst-kix_fv8y17g7myim-6{list-style-type:none}.lst-kix_l9w5fm8md2s9-8>li:before{content:"\0025a0  "}.lst-kix_np2y6n1468n7-8>li:before{content:"\0025a0  "}ul.lst-kix_fv8y17g7myim-8{list-style-type:none}ul.lst-kix_fv8y17g7myim-3{list-style-type:none}ul.lst-kix_fv8y17g7myim-2{list-style-type:none}ul.lst-kix_fv8y17g7myim-5{list-style-type:none}.lst-kix_cxongm456yyi-1>li:before{content:"\0025cb  "}ol.lst-kix_4fkgc27ss5mq-6.start{counter-reset:lst-ctn-kix_4fkgc27ss5mq-6 0}ul.lst-kix_fv8y17g7myim-4{list-style-type:none}.lst-kix_e5u94rrvoqs-4>li:before{content:"\0025cb  "}ul.lst-kix_b6qjs7hmfdgs-0{list-style-type:none}.lst-kix_cxongm456yyi-5>li:before{content:"\0025a0  "}ul.lst-kix_7tkvl1c78ae3-0{list-style-type:none}.lst-kix_e5u94rrvoqs-8>li:before{content:"\0025a0  "}.lst-kix_cxongm456yyi-7>li:before{content:"\0025cb  "}ul.lst-kix_7tkvl1c78ae3-6{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-5{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-8{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-7{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-2{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-1{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-4{list-style-type:none}ul.lst-kix_7tkvl1c78ae3-3{list-style-type:none}.lst-kix_zvvafajkk2a-0>li:before{content:"\0025cf  "}.lst-kix_7ev7v5cpgtkc-3>li:before{content:"\0025cf  "}.lst-kix_flatlsyq6jr0-5>li:before{content:"\0025a0  "}.lst-kix_tnmatnpjv4t2-0>li:before{content:"\0025cf  "}.lst-kix_tnmatnpjv4t2-2>li:before{content:"\0025a0  "}.lst-kix_zvvafajkk2a-4>li:before{content:"\0025cb  "}.lst-kix_fcckwvdxoljx-5>li:before{content:"-  "}.lst-kix_7ev7v5cpgtkc-7>li:before{content:"\0025cb  "}.lst-kix_flatlsyq6jr0-1>li:before{content:"\0025cb  "}.lst-kix_tnmatnpjv4t2-6>li:before{content:"\0025cf  "}.lst-kix_flatlsyq6jr0-0>li{counter-increment:lst-ctn-kix_flatlsyq6jr0-0}.lst-kix_tczavuk46t7o-4>li:before{content:"\0025cb  "}ul.lst-kix_b6qjs7hmfdgs-3{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-4{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-1{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-2{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-7{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-8{list-style-type:none}ul.lst-kix_b6qjs7hmfdgs-5{list-style-type:none}.lst-kix_tczavuk46t7o-2>li:before{content:"\0025a0  "}ul.lst-kix_b6qjs7hmfdgs-6{list-style-type:none}.lst-kix_wxhgjniia6a-0>li:before{content:"\0025cf  "}ol.lst-kix_ur253ckj31qe-5.start{counter-reset:lst-ctn-kix_ur253ckj31qe-5 0}.lst-kix_wxhgjniia6a-3>li:before{content:"\0025cf  "}.lst-kix_uthm435lclg8-0>li{counter-increment:lst-ctn-kix_uthm435lclg8-0}ul.lst-kix_xiiy75ihy1il-5{list-style-type:none}ul.lst-kix_xiiy75ihy1il-4{list-style-type:none}ul.lst-kix_xiiy75ihy1il-7{list-style-type:none}ul.lst-kix_xiiy75ihy1il-6{list-style-type:none}.lst-kix_4tgbgh9bb7g-2>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-2,lower-roman) ". "}.lst-kix_ur253ckj31qe-1>li{counter-increment:lst-ctn-kix_ur253ckj31qe-1}ul.lst-kix_xiiy75ihy1il-1{list-style-type:none}.lst-kix_hxvvdzewxqs2-7>li:before{content:"\0025cb  "}ul.lst-kix_xiiy75ihy1il-0{list-style-type:none}ul.lst-kix_xiiy75ihy1il-3{list-style-type:none}ul.lst-kix_xiiy75ihy1il-2{list-style-type:none}.lst-kix_y5u0wek48eld-0>li:before{content:"-  "}ol.lst-kix_4tgbgh9bb7g-3.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-3 0}ul.lst-kix_oli7yc1w3gpg-0{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-1{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-2{list-style-type:none}.lst-kix_k08o05q2crtr-8>li:before{content:"\0025a0  "}.lst-kix_p4k358qyzf3y-0>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-0,lower-latin) ". "}.lst-kix_k08o05q2crtr-7>li:before{content:"\0025cb  "}ul.lst-kix_oli7yc1w3gpg-7{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-8{list-style-type:none}ul.lst-kix_uhfsw7q6s458-8{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-3{list-style-type:none}.lst-kix_a76b6w8pnkyv-7>li:before{content:"\0025cb  "}ul.lst-kix_oli7yc1w3gpg-4{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-5{list-style-type:none}ul.lst-kix_oli7yc1w3gpg-6{list-style-type:none}ul.lst-kix_uhfsw7q6s458-4{list-style-type:none}ul.lst-kix_uhfsw7q6s458-5{list-style-type:none}ul.lst-kix_uhfsw7q6s458-6{list-style-type:none}ul.lst-kix_uhfsw7q6s458-7{list-style-type:none}ul.lst-kix_qswbihp1i06v-8{list-style-type:none}ul.lst-kix_uhfsw7q6s458-0{list-style-type:none}ul.lst-kix_qswbihp1i06v-7{list-style-type:none}ul.lst-kix_uhfsw7q6s458-1{list-style-type:none}.lst-kix_hxvvdzewxqs2-3>li:before{content:"\0025cf  "}ul.lst-kix_qswbihp1i06v-6{list-style-type:none}ul.lst-kix_uhfsw7q6s458-2{list-style-type:none}ul.lst-kix_qswbihp1i06v-5{list-style-type:none}ul.lst-kix_uhfsw7q6s458-3{list-style-type:none}ul.lst-kix_qswbihp1i06v-4{list-style-type:none}ul.lst-kix_qswbihp1i06v-3{list-style-type:none}.lst-kix_hxvvdzewxqs2-6>li:before{content:"\0025cf  "}ul.lst-kix_qswbihp1i06v-2{list-style-type:none}ul.lst-kix_qswbihp1i06v-1{list-style-type:none}ul.lst-kix_qswbihp1i06v-0{list-style-type:none}ul.lst-kix_ontx89k1fqr9-3{list-style-type:none}ul.lst-kix_ontx89k1fqr9-4{list-style-type:none}ul.lst-kix_ontx89k1fqr9-1{list-style-type:none}ul.lst-kix_ontx89k1fqr9-2{list-style-type:none}.lst-kix_d1mamalvi3wu-5>li:before{content:"\0025a0  "}.lst-kix_ns4c34lwftpb-4>li:before{content:"\0025cb  "}.lst-kix_d1mamalvi3wu-6>li:before{content:"\0025cf  "}.lst-kix_a76b6w8pnkyv-0>li:before{content:"\0025cf  "}.lst-kix_d1mamalvi3wu-2>li:before{content:"\0025a0  "}ol.lst-kix_4tgbgh9bb7g-2.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-2 0}.lst-kix_ns4c34lwftpb-5>li:before{content:"\0025a0  "}.lst-kix_ia5tg78e6pm4-0>li:before{content:"\0025cf  "}.lst-kix_ia5tg78e6pm4-1>li:before{content:"\0025cb  "}.lst-kix_a76b6w8pnkyv-4>li:before{content:"\0025cb  "}ul.lst-kix_c4nzmsldypra-8{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-8.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-8 0}ul.lst-kix_c4nzmsldypra-7{list-style-type:none}ul.lst-kix_c4nzmsldypra-6{list-style-type:none}ul.lst-kix_c4nzmsldypra-5{list-style-type:none}ul.lst-kix_c4nzmsldypra-4{list-style-type:none}.lst-kix_k08o05q2crtr-4>li:before{content:"\0025cb  "}ul.lst-kix_c4nzmsldypra-3{list-style-type:none}.lst-kix_ns4c34lwftpb-1>li:before{content:"\0025cb  "}ul.lst-kix_c4nzmsldypra-2{list-style-type:none}.lst-kix_a76b6w8pnkyv-3>li:before{content:"\0025cf  "}.lst-kix_ia5tg78e6pm4-4>li:before{content:"\0025cb  "}ul.lst-kix_c4nzmsldypra-1{list-style-type:none}ul.lst-kix_c4nzmsldypra-0{list-style-type:none}.lst-kix_tiumq34nhjs2-7>li:before{content:"\0025cb  "}.lst-kix_xm7ynypsh0z5-1>li:before{content:"\0025cb  "}.lst-kix_tiumq34nhjs2-4>li:before{content:"\0025cb  "}.lst-kix_ur253ckj31qe-3>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-3,decimal) ". "}.lst-kix_mi7vyzrjrpd8-5>li:before{content:"\0025a0  "}.lst-kix_ock170xmc4e5-4>li:before{content:"\0025cb  "}.lst-kix_mi7vyzrjrpd8-1>li:before{content:"\0025cb  "}.lst-kix_tiumq34nhjs2-0>li:before{content:"\0025cf  "}.lst-kix_ock170xmc4e5-0>li:before{content:"\0025cf  "}.lst-kix_ns4c34lwftpb-8>li:before{content:"\0025a0  "}.lst-kix_wawbbtzgf6il-7>li:before{content:"\0025cb  "}.lst-kix_n2keu386kfur-3>li:before{content:"\0025cf  "}ul.lst-kix_cm89sr4fo6m-5{list-style-type:none}.lst-kix_yqx6kz4lvbt4-3>li:before{content:"\0025cf  "}ul.lst-kix_cm89sr4fo6m-6{list-style-type:none}ul.lst-kix_cm89sr4fo6m-3{list-style-type:none}ul.lst-kix_cm89sr4fo6m-4{list-style-type:none}ul.lst-kix_cm89sr4fo6m-1{list-style-type:none}.lst-kix_flatlsyq6jr0-8>li:before{content:"\0025a0  "}ul.lst-kix_cm89sr4fo6m-2{list-style-type:none}ul.lst-kix_cm89sr4fo6m-0{list-style-type:none}.lst-kix_p4k358qyzf3y-8>li{counter-increment:lst-ctn-kix_p4k358qyzf3y-8}ol.lst-kix_4tgbgh9bb7g-8{list-style-type:none}.lst-kix_wawbbtzgf6il-3>li:before{content:"\0025cf  "}ul.lst-kix_cm89sr4fo6m-7{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-6{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-7.start{counter-reset:lst-ctn-kix_4tgbgh9bb7g-7 0}ul.lst-kix_cm89sr4fo6m-8{list-style-type:none}.lst-kix_6bb7qu9vjqr2-4>li:before{content:"-  "}ol.lst-kix_4tgbgh9bb7g-7{list-style-type:none}.lst-kix_l9w5fm8md2s9-5>li:before{content:"\0025a0  "}ul.lst-kix_bv0w79unf8ye-0{list-style-type:none}ul.lst-kix_bv0w79unf8ye-2{list-style-type:none}ul.lst-kix_bv0w79unf8ye-1{list-style-type:none}.lst-kix_xm7ynypsh0z5-5>li:before{content:"\0025a0  "}.lst-kix_nwfayb939eln-6>li:before{content:"\0025cf  "}ul.lst-kix_bv0w79unf8ye-4{list-style-type:none}.lst-kix_o1wsnr4piq1b-5>li{counter-increment:lst-ctn-kix_o1wsnr4piq1b-5}.lst-kix_np2y6n1468n7-5>li:before{content:"\0025a0  "}ul.lst-kix_bv0w79unf8ye-3{list-style-type:none}ul.lst-kix_bv0w79unf8ye-6{list-style-type:none}ul.lst-kix_bv0w79unf8ye-5{list-style-type:none}ul.lst-kix_bv0w79unf8ye-8{list-style-type:none}.lst-kix_4roymkz9vma-2>li{counter-increment:lst-ctn-kix_4roymkz9vma-2}ul.lst-kix_bv0w79unf8ye-7{list-style-type:none}.lst-kix_b6qjs7hmfdgs-0>li:before{content:"\0025cf  "}ul.lst-kix_x3ptir2y8jgm-1{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-0{list-style-type:none}.lst-kix_uthm435lclg8-2>li:before{content:"\0025a0  "}ul.lst-kix_x3ptir2y8jgm-7{list-style-type:none}.lst-kix_cxongm456yyi-4>li:before{content:"\0025cb  "}ul.lst-kix_x3ptir2y8jgm-6{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-8{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-3{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-2{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-5{list-style-type:none}ul.lst-kix_x3ptir2y8jgm-4{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-4{list-style-type:none}.lst-kix_cxongm456yyi-8>li:before{content:"\0025a0  "}.lst-kix_h9p9lo20tlpj-8>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-8,lower-roman) ". "}ol.lst-kix_4tgbgh9bb7g-5{list-style-type:none}ol.lst-kix_ur253ckj31qe-6.start{counter-reset:lst-ctn-kix_ur253ckj31qe-6 0}ol.lst-kix_4tgbgh9bb7g-2{list-style-type:none}.lst-kix_ric95po2pkzv-0>li:before{content:"\0025cf  "}ol.lst-kix_4tgbgh9bb7g-3{list-style-type:none}ol.lst-kix_4tgbgh9bb7g-0{list-style-type:none}.lst-kix_tcwoy9lfuqk4-1>li:before{content:"\0025cb  "}.lst-kix_tcwoy9lfuqk4-5>li:before{content:"\0025a0  "}.lst-kix_e5u94rrvoqs-7>li:before{content:"\0025cb  "}.lst-kix_dkq97x3e7jfk-0>li:before{content:"\0025cf  "}.lst-kix_ric95po2pkzv-4>li:before{content:"\0025cb  "}.lst-kix_7ka97cx080q5-4>li{counter-increment:lst-ctn-kix_7ka97cx080q5-4}ul.lst-kix_xiiy75ihy1il-8{list-style-type:none}.lst-kix_o6fhz26vvidv-3>li:before{content:"\0025cf  "}.lst-kix_6bb7qu9vjqr2-8>li:before{content:"-  "}.lst-kix_fcckwvdxoljx-8>li:before{content:"-  "}.lst-kix_flatlsyq6jr0-4>li:before{content:"\0025cb  "}.lst-kix_tnmatnpjv4t2-3>li:before{content:"\0025cf  "}.lst-kix_zvvafajkk2a-1>li:before{content:"\0025cb  "}.lst-kix_7ev7v5cpgtkc-6>li:before{content:"\0025cf  "}.lst-kix_a0ivi9b1qp8i-7>li:before{content:"\0025cb  "}.lst-kix_tczavuk46t7o-1>li:before{content:"\0025cb  "}.lst-kix_gmyqb4k4mims-0>li:before{content:"\0025cf  "}.lst-kix_cpm4oaqfjy6b-3>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-3>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-3}.lst-kix_o9ptu63d1d74-3>li:before{content:"\0025cf  "}.lst-kix_uhfsw7q6s458-5>li:before{content:"\0025a0  "}.lst-kix_o9ptu63d1d74-6>li:before{content:"\0025cf  "}.lst-kix_7ka97cx080q5-1>li{counter-increment:lst-ctn-kix_7ka97cx080q5-1}.lst-kix_cpm4oaqfjy6b-6>li:before{content:"\0025cf  "}.lst-kix_ontx89k1fqr9-0>li:before{content:"" counter(lst-ctn-kix_ontx89k1fqr9-0,decimal) ". "}.lst-kix_ur253ckj31qe-4>li{counter-increment:lst-ctn-kix_ur253ckj31qe-4}.lst-kix_ontx89k1fqr9-7>li:before{content:"\0025cb  "}.lst-kix_7ka97cx080q5-4>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-4,lower-latin) ". "}.lst-kix_uhfsw7q6s458-8>li:before{content:"\0025a0  "}.lst-kix_dkq97x3e7jfk-6>li:before{content:"\0025cf  "}.lst-kix_cpm4oaqfjy6b-0>li:before{content:"\0025cf  "}.lst-kix_7ka97cx080q5-7>li:before{content:"" counter(lst-ctn-kix_7ka97cx080q5-7,lower-latin) ". "}ol.lst-kix_ur253ckj31qe-2.start{counter-reset:lst-ctn-kix_ur253ckj31qe-2 0}.lst-kix_25j83v4xa4ig-3>li:before{content:"\0025cf  "}.lst-kix_nr7rbh2hfyf0-4>li:before{content:"\0025cb  "}.lst-kix_nzxgmnmwjatv-2>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-2}.lst-kix_x7o2gv80my2k-6>li:before{content:"\0025cf  "}.lst-kix_nr7rbh2hfyf0-7>li:before{content:"\0025cb  "}.lst-kix_4roymkz9vma-7>li{counter-increment:lst-ctn-kix_4roymkz9vma-7}.lst-kix_xzp0xel2onox-4>li:before{content:"\0025cb  "}.lst-kix_uhfsw7q6s458-2>li:before{content:"\0025a0  "}.lst-kix_xiiy75ihy1il-5>li:before{content:"\0025a0  "}.lst-kix_xiiy75ihy1il-8>li:before{content:"\0025a0  "}.lst-kix_xzp0xel2onox-7>li:before{content:"\0025cb  "}.lst-kix_xiiy75ihy1il-2>li:before{content:"\0025a0  "}.lst-kix_vxbaz02t98k3-8>li:before{content:"\0025a0  "}.lst-kix_vxbaz02t98k3-2>li:before{content:"\0025a0  "}.lst-kix_bv0w79unf8ye-2>li:before{content:"\0025a0  "}.lst-kix_ygjhinrwstz-5>li:before{content:"\0025a0  "}.lst-kix_qqf2twdkbiw0-7>li:before{content:"\0025cb  "}.lst-kix_e21xjkb2e68h-8>li:before{content:"\0025a0  "}.lst-kix_qqf2twdkbiw0-4>li:before{content:"\0025cb  "}.lst-kix_vxbaz02t98k3-5>li:before{content:"\0025a0  "}.lst-kix_ygjhinrwstz-8>li:before{content:"\0025a0  "}ol.lst-kix_jqqhsyijt7h5-0{list-style-type:none}ul.lst-kix_4tgbgh9bb7g-1{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-1{list-style-type:none}.lst-kix_ur253ckj31qe-6>li:before{content:"" counter(lst-ctn-kix_ur253ckj31qe-6,decimal) ". "}.lst-kix_9tajp1agqiyw-4>li:before{content:"\0025cb  "}.lst-kix_xwc7ibkfiucl-2>li:before{content:"\0025a0  "}ol.lst-kix_jqqhsyijt7h5-6{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-7{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-8{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-2{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-3{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-4{list-style-type:none}ol.lst-kix_jqqhsyijt7h5-5{list-style-type:none}ol.lst-kix_4roymkz9vma-6{list-style-type:none}.lst-kix_fv8y17g7myim-8>li:before{content:"\0025a0  "}ol.lst-kix_4roymkz9vma-7{list-style-type:none}ol.lst-kix_4roymkz9vma-8{list-style-type:none}.lst-kix_g6dv2r60jppe-1>li:before{content:"\0025cb  "}.lst-kix_n2keu386kfur-6>li:before{content:"\0025cf  "}.lst-kix_emh9tw6ivvnv-0>li:before{content:"\0025cf  "}ol.lst-kix_4roymkz9vma-2{list-style-type:none}ol.lst-kix_4roymkz9vma-3{list-style-type:none}.lst-kix_v26ncletnxud-0>li:before{content:"\0025cf  "}.lst-kix_c4nzmsldypra-0>li:before{content:"\0025cf  "}ol.lst-kix_4roymkz9vma-4{list-style-type:none}ol.lst-kix_4roymkz9vma-5{list-style-type:none}.lst-kix_emh9tw6ivvnv-6>li:before{content:"\0025cf  "}.lst-kix_6bb7qu9vjqr2-1>li:before{content:"-  "}.lst-kix_qswbihp1i06v-4>li:before{content:"\0025cb  "}.lst-kix_ock170xmc4e5-7>li:before{content:"\0025cb  "}.lst-kix_25j83v4xa4ig-6>li:before{content:"\0025cf  "}.lst-kix_g6dv2r60jppe-7>li:before{content:"\0025cb  "}.lst-kix_l4298avyh9v6-0>li:before{content:"\0025cf  "}.lst-kix_nwfayb939eln-3>li:before{content:"\0025cf  "}.lst-kix_4h54ot51egiu-6>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-0>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-0}.lst-kix_uthm435lclg8-5>li:before{content:"\0025a0  "}.lst-kix_oli7yc1w3gpg-5>li:before{content:"\0025a0  "}.lst-kix_ouaz0d8j7xj-4>li:before{content:"\0025cb  "}.lst-kix_tcwoy9lfuqk4-8>li:before{content:"\0025a0  "}.lst-kix_h9p9lo20tlpj-5>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-5,lower-roman) ". "}.lst-kix_gmyqb4k4mims-3>li:before{content:"\0025cf  "}.lst-kix_z9n4e0cpc149-1>li:before{content:"\0025cb  "}.lst-kix_ric95po2pkzv-7>li:before{content:"\0025cb  "}.lst-kix_dkq97x3e7jfk-3>li:before{content:"\0025cf  "}ol.lst-kix_ur253ckj31qe-0.start{counter-reset:lst-ctn-kix_ur253ckj31qe-0 0}.lst-kix_o6fhz26vvidv-6>li:before{content:"\0025cf  "}ul.lst-kix_i2rq8qzg0efa-7{list-style-type:none}.lst-kix_k0c3ozmp0pnp-2>li:before{content:"\0025a0  "}.lst-kix_a0ivi9b1qp8i-4>li:before{content:"\0025cb  "}ul.lst-kix_i2rq8qzg0efa-8{list-style-type:none}.lst-kix_w7vhjpenesq3-2>li:before{content:"\0025a0  "}.lst-kix_l4298avyh9v6-6>li:before{content:"\0025cf  "}.lst-kix_jqqhsyijt7h5-6>li{counter-increment:lst-ctn-kix_jqqhsyijt7h5-6}ul.lst-kix_i2rq8qzg0efa-3{list-style-type:none}ul.lst-kix_i2rq8qzg0efa-4{list-style-type:none}ul.lst-kix_i2rq8qzg0efa-5{list-style-type:none}ul.lst-kix_i2rq8qzg0efa-6{list-style-type:none}ul.lst-kix_i2rq8qzg0efa-0{list-style-type:none}ul.lst-kix_i2rq8qzg0efa-1{list-style-type:none}.lst-kix_w7vhjpenesq3-8>li:before{content:"\0025a0  "}.lst-kix_gayel7f32yde-3>li:before{content:"-  "}ul.lst-kix_i2rq8qzg0efa-2{list-style-type:none}.lst-kix_m3f8ler1ak4c-8>li:before{content:"\0025a0  "}.lst-kix_n2tc9z47p0ha-0>li:before{content:"\0025cf  "}.lst-kix_9bzwod9gvkvp-3>li:before{content:"\0025cf  "}.lst-kix_n2tc9z47p0ha-6>li:before{content:"\0025cf  "}.lst-kix_cm89sr4fo6m-7>li:before{content:"\0025cb  "}.lst-kix_54trtu6df9f8-3>li:before{content:"\0025cf  "}.lst-kix_nzxgmnmwjatv-5>li{counter-increment:lst-ctn-kix_nzxgmnmwjatv-5}ul.lst-kix_1293ztc79klg-0{list-style-type:none}ul.lst-kix_1293ztc79klg-1{list-style-type:none}ul.lst-kix_1293ztc79klg-2{list-style-type:none}ul.lst-kix_1293ztc79klg-3{list-style-type:none}ul.lst-kix_1293ztc79klg-4{list-style-type:none}ul.lst-kix_1293ztc79klg-5{list-style-type:none}ul.lst-kix_1293ztc79klg-6{list-style-type:none}ul.lst-kix_1293ztc79klg-7{list-style-type:none}.lst-kix_b96slznfb3gj-3>li:before{content:"-  "}ul.lst-kix_1293ztc79klg-8{list-style-type:none}.lst-kix_b96slznfb3gj-6>li:before{content:"-  "}.lst-kix_4tgbgh9bb7g-8>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-8,lower-roman) ". "}.lst-kix_4r80w4zgcdjq-6>li:before{content:"\0025cf  "}.lst-kix_alje6q6hzub8-1>li:before{content:"-  "}.lst-kix_7tkvl1c78ae3-7>li:before{content:"\0025cb  "}.lst-kix_e08effp12mqf-3>li:before{content:"\0025cf  "}.lst-kix_4tgbgh9bb7g-5>li:before{content:"" counter(lst-ctn-kix_4tgbgh9bb7g-5,lower-roman) ". "}.lst-kix_p4k358qyzf3y-6>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-6,decimal) ". "}.lst-kix_y5u0wek48eld-6>li:before{content:"-  "}.lst-kix_y5u0wek48eld-3>li:before{content:"-  "}.lst-kix_p4k358qyzf3y-3>li:before{content:"" counter(lst-ctn-kix_p4k358qyzf3y-3,decimal) ". "}.lst-kix_1293ztc79klg-6>li:before{content:"\0025cf  "}.lst-kix_54trtu6df9f8-0>li:before{content:"\0025cf  "}.lst-kix_h9p9lo20tlpj-2>li:before{content:"" counter(lst-ctn-kix_h9p9lo20tlpj-2,lower-roman) ". "}.lst-kix_yk52z9n95trk-0>li:before{content:"\0025cf  "}.lst-kix_x30zykguitxy-0>li:before{content:"\0025cf  "}.lst-kix_hxvvdzewxqs2-0>li:before{content:"\0025cf  "}.lst-kix_yk52z9n95trk-3>li:before{content:"\0025cf  "}.lst-kix_s27gvee6ygqb-1>li:before{content:"\0025cb  "}.lst-kix_v26ncletnxud-6>li:before{content:"\0025cf  "}ul.lst-kix_qqf2twdkbiw0-8{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-7{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-6{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-5{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-4{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-3{list-style-type:none}ul.lst-kix_qqf2twdkbiw0-2{list-style-type:none}.lst-kix_aninp2c52sy3-6>li:before{content:"\0025cf  "}ul.lst-kix_qqf2twdkbiw0-1{list-style-type:none}.lst-kix_v26ncletnxud-3>li:before{content:"\0025cf  "}.lst-kix_icjmrj4tgidb-6>li:before{content:"-  "}ul.lst-kix_qqf2twdkbiw0-0{list-style-type:none}.lst-kix_ieipvto6nfrd-0>li:before{content:"-  "}.lst-kix_4tgbgh9bb7g-0>li{counter-increment:lst-ctn-kix_4tgbgh9bb7g-0}.lst-kix_e08effp12mqf-0>li:before{content:"\0025cf  "}.lst-kix_4h47981ce058-7>li:before{content:"\0025cb  "}.lst-kix_k08o05q2crtr-1>li:before{content:"\0025cb  "}.lst-kix_ia5tg78e6pm4-7>li:before{content:"\0025cb  "}.lst-kix_f7u4w07k18eh-7>li:before{content:"\0025cb  "}.lst-kix_wxhgjniia6a-6>li:before{content:"\0025cf  "}ul.lst-kix_dkq97x3e7jfk-8{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-7{list-style-type:none}.lst-kix_9bzwod9gvkvp-6>li:before{content:"\0025cf  "}ul.lst-kix_4roymkz9vma-0{list-style-type:none}ul.lst-kix_4roymkz9vma-1{list-style-type:none}.lst-kix_eu4b93xn98yk-3>li:before{content:"\0025cf  "}.lst-kix_xprjjxgruvg-2>li:before{content:"\0025a0  "}.lst-kix_b6qjs7hmfdgs-6>li:before{content:"\0025cf  "}.lst-kix_ygjhinrwstz-2>li:before{content:"\0025a0  "}.lst-kix_ur253ckj31qe-7>li{counter-increment:lst-ctn-kix_ur253ckj31qe-7}.lst-kix_a0ivi9b1qp8i-1>li:before{content:"\0025cb  "}.lst-kix_i0q09d67i790-4>li:before{content:"\0025cb  "}.lst-kix_g6dv2r60jppe-4>li:before{content:"\0025cb  "}.lst-kix_nwfayb939eln-0>li:before{content:"\0025cf  "}.lst-kix_l4298avyh9v6-3>li:before{content:"\0025cf  "}ul.lst-kix_dkq97x3e7jfk-0{list-style-type:none}.lst-kix_qswbihp1i06v-1>li:before{content:"\0025cb  "}ul.lst-kix_dkq97x3e7jfk-2{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-1{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-4{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-3{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-6{list-style-type:none}ul.lst-kix_dkq97x3e7jfk-5{list-style-type:none}.lst-kix_oli7yc1w3gpg-8>li:before{content:"\0025a0  "}.lst-kix_9tajp1agqiyw-7>li:before{content:"\0025cb  "}.lst-kix_h9p9lo20tlpj-6>li{counter-increment:lst-ctn-kix_h9p9lo20tlpj-6}.lst-kix_emh9tw6ivvnv-3>li:before{content:"\0025cf  "}ol.lst-kix_h9p9lo20tlpj-8{list-style-type:none}.lst-kix_ouaz0d8j7xj-1>li:before{content:"\0025cb  "}ol.lst-kix_h9p9lo20tlpj-7{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-8{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-6{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-7{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-5{list-style-type:none}.lst-kix_e5u94rrvoqs-1>li:before{content:"\0025cb  "}ol.lst-kix_h9p9lo20tlpj-4{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-3{list-style-type:none}.lst-kix_tgtwnecgfrou-7>li:before{content:"+  "}ol.lst-kix_h9p9lo20tlpj-2{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-1{list-style-type:none}ol.lst-kix_h9p9lo20tlpj-0{list-style-type:none}ul.lst-kix_xzp0xel2onox-1{list-style-type:none}ul.lst-kix_xzp0xel2onox-0{list-style-type:none}.lst-kix_4fkgc27ss5mq-5>li:before{content:"" counter(lst-ctn-kix_4fkgc27ss5mq-5,lower-roman) ". "}ul.lst-kix_xzp0xel2onox-3{list-style-type:none}ul.lst-kix_xzp0xel2onox-2{list-style-type:none}ol.lst-kix_flatlsyq6jr0-0{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-2{list-style-type:none}.lst-kix_uthm435lclg8-8>li:before{content:"\0025a0  "}.lst-kix_9ft7dqndoe5p-7>li:before{content:"\0025cb  "}ul.lst-kix_ejnvxy8b1e9w-1{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-0{list-style-type:none}.lst-kix_gmyqb4k4mims-6>li:before{content:"\0025cf  "}ul.lst-kix_ejnvxy8b1e9w-6{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-5{list-style-type:none}ul.lst-kix_ejnvxy8b1e9w-4{list-style-type:none}.lst-kix_gayel7f32yde-6>li:before{content:"-  "}ul.lst-kix_ejnvxy8b1e9w-3{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-8{list-style-type:none}ul.lst-kix_alje6q6hzub8-0{list-style-type:none}.lst-kix_4r80w4zgcdjq-3>li:before{content:"\0025cf  "}ul.lst-kix_alje6q6hzub8-2{list-style-type:none}ul.lst-kix_alje6q6hzub8-1{list-style-type:none}ul.lst-kix_alje6q6hzub8-4{list-style-type:none}.lst-kix_uh3r4fp09cs-1>li:before{content:"\0025cb  "}.lst-kix_w7vhjpenesq3-5>li:before{content:"\0025a0  "}ul.lst-kix_6bb7qu9vjqr2-8{list-style-type:none}ul.lst-kix_alje6q6hzub8-3{list-style-type:none}ul.lst-kix_alje6q6hzub8-6{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-1{list-style-type:none}ul.lst-kix_alje6q6hzub8-5{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-0{list-style-type:none}ul.lst-kix_alje6q6hzub8-8{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-3{list-style-type:none}ul.lst-kix_alje6q6hzub8-7{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-2{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-5{list-style-type:none}.lst-kix_abgphud9refj-0>li:before{content:"\0025cf  "}ul.lst-kix_k9nxvn8n4dkv-4{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-7{list-style-type:none}ul.lst-kix_k9nxvn8n4dkv-6{list-style-type:none}ul.lst-kix_xzp0xel2onox-5{list-style-type:none}.lst-kix_n2tc9z47p0ha-3>li:before{content:"\0025cf  "}ul.lst-kix_xzp0xel2onox-4{list-style-type:none}ul.lst-kix_xzp0xel2onox-7{list-style-type:none}ul.lst-kix_xzp0xel2onox-6{list-style-type:none}ul.lst-kix_xzp0xel2onox-8{list-style-type:none}ul.lst-kix_6bb7qu9vjqr2-7{list-style-type:none}.lst-kix_7ev7v5cpgtkc-0>li:before{content:"\0025cf  "}ul.lst-kix_6bb7qu9vjqr2-6{list-style-type:none}ul.lst-kix_6bb7qu9vjqr2-5{list-style-type:none}ul.lst-kix_6bb7qu9vjqr2-4{list-style-type:none}ul.lst-kix_6bb7qu9vjqr2-3{list-style-type:none}.lst-kix_fcckwvdxoljx-2>li:before{content:"-  "}ul.lst-kix_6bb7qu9vjqr2-2{list-style-type:none}ul.lst-kix_6bb7qu9vjqr2-1{list-style-type:none}.lst-kix_zvvafajkk2a-7>li:before{content:"\0025cb  "}ul.lst-kix_6bb7qu9vjqr2-0{list-style-type:none}ol{margin:0;padding:0}.c90{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:270pt;border-top-color:#000000;border-bottom-style:solid}.c103{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c81{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:98.2pt;border-top-color:#000000;border-bottom-style:solid}.c67{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:105pt;border-top-color:#000000;border-bottom-style:solid}.c73{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:77.2pt;border-top-color:#cccccc;border-bottom-style:solid}.c69{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:226.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c89{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c83{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:173.2pt;border-top-color:#cccccc;border-bottom-style:solid}.c88{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:102.7pt;border-top-color:#000000;border-bottom-style:solid}.c106{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:81.8pt;border-top-color:#000000;border-bottom-style:solid}.c61{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c2{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:60.8pt;border-top-color:#000000;border-bottom-style:solid}.c25{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:94.9pt;border-top-color:#000000;border-bottom-style:solid}.c111{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#000000;border-bottom-style:solid}.c92{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#000000;border-bottom-style:solid}.c78{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:55.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c76{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:155.2pt;border-top-color:#000000;border-bottom-style:solid}.c62{border-right-style:solid;padding:3pt 3pt 3pt 3pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:middle;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:61.5pt;border-top-color:#000000;border-bottom-style:solid}.c11{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c41{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:81.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c86{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:107.2pt;border-top-color:#000000;border-bottom-style:solid}.c115{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#000000;border-bottom-style:solid}.c71{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:96.8pt;border-top-color:#000000;border-bottom-style:solid}.c68{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:79.5pt;border-top-color:#000000;border-bottom-style:solid}.c91{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:111.8pt;border-top-color:#000000;border-bottom-style:solid}.c84{border-right-style:solid;padding:3pt 3pt 3pt 3pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:middle;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:48.8pt;border-top-color:#000000;border-bottom-style:solid}.c79{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:121.5pt;border-top-color:#000000;border-bottom-style:solid}.c55{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c46{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:205.5pt;border-top-color:#000000;border-bottom-style:solid}.c85{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:201.8pt;border-top-color:#000000;border-bottom-style:solid}.c66{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:97.7pt;border-top-color:#000000;border-bottom-style:solid}.c80{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:81.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c96{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#000000;border-bottom-style:solid}.c101{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#cccccc;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#cccccc;border-bottom-style:solid}.c105{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c114{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#cccccc;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#cccccc;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:175.5pt;border-top-color:#cccccc;border-bottom-style:solid}.c98{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:172.6pt;border-top-color:#000000;border-bottom-style:solid}.c29{background-color:#ffffff;color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-size:10.5pt;font-family:"Arial";font-style:normal}.c27{color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c31{color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Arial";font-style:normal}.c44{color:#000000;font-weight:normal;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c36{color:#000000;font-weight:normal;vertical-align:baseline;font-size:9pt;font-family:"Arial";font-style:normal}.c1{margin-left:36pt;orphans:2;widows:2;padding-left:0pt;direction:ltr}.c45{color:#000000;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c6{margin-left:72pt;orphans:2;widows:2;padding-left:0pt;direction:ltr}.c51{color:#980000;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c14{font-size:12pt;font-style:italic;color:#1155cc;text-decoration:underline}.c5{orphans:2;widows:2;direction:ltr;height:11pt}.c10{orphans:2;widows:2;direction:ltr}.c112{color:#000000;vertical-align:baseline;font-style:normal}.c21{padding-top:0pt;padding-bottom:0pt;line-height:1.15}.c42{padding-top:0pt;padding-bottom:0pt;line-height:1.4285714285714286}.c43{padding-top:0pt;padding-bottom:0pt;line-height:1.0}.c72{border-collapse:collapse;margin-right:auto}.c52{padding-top:0pt;line-height:1.15}.c17{background-color:#ffffff;color:#222222}.c0{padding-bottom:3pt;direction:ltr}.c58{font-size:18pt;font-family:"Arial"}.c113{max-width:540pt;padding:72pt 36pt 72pt 36pt}.c4{margin-left:36pt;padding-left:0pt}.c9{color:inherit;text-decoration:inherit}.c3{padding:0;margin:0}.c7{font-size:14pt;font-weight:bold}.c39{font-size:10pt;color:#ff00ff}.c23{font-size:9pt;font-family:"Courier New"}.c28{font-size:10pt;color:#3a5bdc}.c33{background-color:#ffffff;font-size:10pt}.c94{padding-top:11pt;line-height:1.4285714285714286}.c8{direction:ltr;height:11pt}.c13{color:#ff0000;font-size:12pt}.c93{padding-top:24pt;padding-bottom:6pt}.c47{margin-left:72pt;padding-left:0pt}.c34{font-size:10pt;color:#0000ff}.c108{padding-top:18pt;padding-bottom:4pt}.c63{font-weight:normal}.c95{font-family:"Arial"}.c82{color:#ff0000}.c100{text-decoration:line-through}.c110{font-size:16pt}.c49{color:#9900ff}.c87{padding-left:0pt}.c32{direction:ltr}.c16{text-decoration:underline}.c35{background-color:#ffffff}.c22{font-weight:bold}.c70{color:#980000}.c38{font-size:14pt}.c37{color:#cc0000}.c60{font-size:8pt}.c26{text-indent:36pt}.c64{margin-left:30.8pt}.c116{font-size:13pt}.c54{font-size:10.5pt}.c57{margin-left:108pt}.c109{height:16pt}.c102{color:#222222}.c12{text-align:center}.c40{font-size:10pt}.c104{height:14pt}.c77{page-break-after:avoid}.c15{text-align:left}.c74{color:#0000ff}.c99{line-height:1.4285714285714286}.c53{margin-left:36pt}.c97{font-size:11pt}.c48{font-style:italic}.c50{margin-left:72pt}.c65{font-size:9pt}.c30{line-height:1.38}.c19{height:11pt}.c107{font-size:19.5pt}.c59{font-family:"Courier New"}.c20{color:#1155cc}.c75{height:0pt}.c56{padding-bottom:3pt}.c18{font-size:12pt}.c24{text-align:right}.title{padding-top:0pt;color:#000000;font-size:21pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:13pt;padding-bottom:10pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:10pt;color:#000000;font-size:16pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:10pt;color:#000000;font-weight:bold;font-size:13pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:8pt;color:#666666;font-weight:bold;font-size:12pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:8pt;color:#666666;text-decoration:underline;font-size:11pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:8pt;color:#666666;font-size:11pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:8pt;color:#666666;font-size:11pt;padding-bottom:0pt;font-family:"Trebuchet MS";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c35 c113"><h1 class="c10 c77"><a name="h.heev1udpsdfl"></a><span class="c58 c22">Splicing/Network, 2015-10-06</span></h1><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Created a REST API running on the beagle head node that allows jobs to be submitted using POST command and mutation results to be obtained using a GET command.</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Setup</span><span>:</span></p><p class="c10"><span class="c23">&gt;&gt; import requests</span></p><p class="c10"><span class="c23">&gt;&gt; url = &lsquo;http://192.168.6.19/elaspic/api/v1.0/submitjob&rsquo;</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Structures in the database</span></p><p class="c10"><span class="c23 c37"># Can submit multiple mutations separated by commas</span></p><p class="c10"><span class="c23">&gt;&gt; import requests</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.post(url, {&lsquo;uniprot_id&rsquo;: &lsquo;A0AV96&rsquo;, &lsquo;mutations&rsquo;: &lsquo;P79L,G51W,Y83A,R96A,R96H,A185T&lsquo;}</span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;status&#39;: &#39;Submitted&#39;}</span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c10"><span class="c23 c37"># Can submit more mutations for the same protein; provean and model will NOT be calculated twice</span></p><p class="c10"><span class="c23">&gt;&gt; import requests</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.post(url, {&lsquo;uniprot_id&rsquo;: &lsquo;A0AV96&rsquo;, &lsquo;mutations&rsquo;: I262F,S309L,A336T}</span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;status&#39;: &#39;Submitted&#39;}</span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c10"><span class="c23 c37"># When we request results, first we get job ids of provean and model running on the cluster</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.get(url, {&lsquo;uniprot_id&rsquo;: &lsquo;A0AV96&rsquo;, &lsquo;mutations&rsquo;: &#39;P79L,}</span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;sequence&#39;: {&#39;is_missing&#39;: True, &#39;status&#39;: &lsquo;Running&rsquo;, &lsquo;job_id&rsquo;: &#39;3316107&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;model&#39;: {&#39;is_missing&#39;: True, &#39;status&#39;: &lsquo;Running&rsquo;, &#39;job_id&#39;: &#39;3316108&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;P79L&#39;: {&#39;status&#39;: &#39;Unknown&#39;, &#39;result&#39;: []}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;R96H&#39;: {&#39;status&#39;: &#39;Unknown&#39;, &#39;result&#39;: []}</span></p><p class="c10"><span class="c23">}</span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c10"><span class="c23 c37"># When provean and model are done, the mutations get put on the cluster</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.get(url, {&lsquo;uniprot_id&rsquo;: &lsquo;A0AV96&rsquo;, &lsquo;mutations&rsquo;: &lsquo;P79L,R96H&rsquo;}</span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;sequence&#39;: {&#39;is_missing&#39;: False, &#39;status&#39;: Done, &lsquo;job_id&rsquo;: &#39;3316107&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;model&#39;: {&#39;is_missing&#39;: False, &#39;status&#39;: Done&rsquo;, &#39;job_id&#39;: &#39;3316108&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;P79L&#39;: {&#39;status&#39;: &#39;Running&#39;, &#39;job_id&#39;: &#39;3316109&#39;},</span></p><p class="c10"><span class="c23">&nbsp;&#39;R96H&#39;: {&#39;status&#39;: &#39;Running&#39;, &#39;job_id&#39;: &#39;3316110&#39;}</span></p><p class="c10"><span class="c23">}</span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c10"><span class="c23 c37"># Can get results for one or more mutations</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.get(url, {&lsquo;uniprot_id&rsquo;: &lsquo;A0AV96&rsquo;, &lsquo;mutations&rsquo;: &lsquo;P79L,R96H&rsquo;}</span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;sequence&#39;: {&#39;is_missing&#39;: False, &#39;status&#39;: Done, &lsquo;job_id&rsquo;: &#39;3316107&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;model&#39;: {&#39;is_missing&#39;: False, &#39;status&#39;: Done&rsquo;, &#39;job_id&#39;: &#39;3316108&#39;}, </span></p><p class="c10"><span class="c23">&nbsp;&#39;P79L&#39;: {&#39;status&#39;: &#39;Done&#39;, &#39;job_id&#39;: &#39;3316109&#39;, &#39;result&#39;: [{&#39;alignment_score&#39;: 0.449285, &nbsp; &nbsp; &nbsp;&#39;alignment_def&#39;: &#39;73:137&#39;, &#39;domain_def&#39;: &#39;51:151&#39;, &#39;provean_errors&#39;: None, &#39;secondary_structure_mut&#39;: &#39;T&#39;, &#39;mutation_errors&#39;: None, &#39;mutation_modeller&#39;: &#39;P29L&#39;, ... </span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c5"><span class="c23"></span></p><p class="c10"><span class="c22">User-provided structures (and target sequences)</span></p><p class="c10"><span>The API is the same as with user-provided structures. The structure (and target sequences), should be put in a unique subfolder in </span><span class="c23">/home/kimlab1/database_data/elaspic_v2/user_input</span><span class="c48">.</span><span>&nbsp;Then, a job can be submitted by specifying a </span><span class="c23 c16">unique_id</span><span>&nbsp;(the name of the subfolder), &nbsp;</span><span class="c23 c16">pdb_file</span><span>&nbsp;(the name of the structure), </span><span class="c23 c16">sequence_file</span><span>&nbsp;(optional, fasta file containing the sequences of all chains to be modelled), </span><span class="c23 c16">mutations</span><span>&nbsp;(a comma separated list of mutations. If only </span><span class="c23 c16">pdb_file</span><span>&nbsp;is provided, mutations are in PDB coordinates (e.g. &ldquo;A_M2A,B_G3A&rdquo;). If a </span><span class="c23 c16">sequence_file</span><span>&nbsp;is provided, mutations are in sequence coordinates (e.g. &ldquo;1_M1A,2_G2A&rdquo;).</span></p><p class="c5"><span></span></p><p class="c10"><span class="c23"># Use &lsquo;pdb_file&rsquo;,</span></p><p class="c10"><span class="c23">&gt;&gt; r = requests.post(url, {&lsquo;unique_id&rsquo;, &lsquo;pdb_file&rsquo;:, &lsquo;4DKL.pdb&rsquo;, &lsquo;sequence_file&rsquo;: &lsquo;mytargets.fasta&rsquo;, &lsquo;mutations&rsquo;: </span></p><p class="c10"><span class="c23">&gt;&gt; print(r.json())</span></p><p class="c10"><span class="c23">{&#39;status&#39;: &#39;Submitted&#39;}</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c7">Thesis</span></p><p class="c10"><span>To make the thesis a bit more computer science-ie and complete, &nbsp;I could say I implemented a RESTful &ldquo;web service&rdquo; that runs elaspic jobs on a local Sun Grid Engine cluster and outputs results to the user. For this to be 100% true, there are still minor changes that have to be made, like </span><span class="c16">passing actual file data </span><span>rather than filenames?</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><hr style="page-break-before:always;display:none;"><h1 class="c10 c77 c109"><a name="h.ax4ezl95i8sn"></a></h1><h1 class="c10 c77"><a name="h.m8jazsjxyr12"></a><span class="c58 c22">Splicing/Network, 2015-10-06</span></h1><p class="c5"><span></span></p><p class="c10"><span class="c7">ELASPIC Webserver</span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Allowing the user to input raw PDB structures</span></p><p class="c5"><span></span></p><p class="c10"><span>This turned out to be more complicated than expected:</span></p><ul class="c3 lst-kix_9bzwod9gvkvp-0 start"><li class="c1"><span>The </span><span class="c22">d</span><span>&nbsp;object (a Domain or DomainPair) is used throughout the pipeline as the primary data type. </span></li></ul><ul class="c3 lst-kix_9bzwod9gvkvp-1 start"><li class="c6"><span>e.g., to get the domain sequence it&rsquo;s </span><span class="c22">`</span><span class="c22">d.uniprot.uniprot_sequence` </span><span>for a Domain, and </span><span class="c22">`d.domain_1.uniprot.uniprot_sequence`</span><span>&nbsp;for a DomainPair. </span></li><li class="c6"><span>to get a domain model, it&rsquo;s </span><span class="c22">d.template.model.model_filename </span><span>for both Domain and DomainPair.</span></li><li class="c6"><span>etc&hellip;</span></li></ul><ul class="c3 lst-kix_9bzwod9gvkvp-0"><li class="c1"><span>Tried fixing it, but I was either putting in a if / else statements whenever I was using `</span><span class="c22">d`</span><span>&nbsp;or duplicating code and changing `d` to something else.</span></li><li class="c1"><span>Ended up refactoring the code to make it more modular and testable. </span></li></ul><p class="c5"><span></span></p><p class="c10"><span>&nbsp;</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 504.50px; height: 405.00px;"><img alt="" src="images/image99.png" style="width: 720.00px; height: 405.00px; margin-left: -107.50px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10 c12"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTZzdoTDdLdFYtUDQ&amp;sa=D&amp;usg=AFQjCNFRDf0mvEeARlKXfrZR-DvbapmVKw">https://drive.google.com/open?id=0B1FXQpQb4bbTZzdoTDdLdFYtUDQ</a></span></p><p class="c5"><span></span></p><p class="c5"><span class="c17 c7"></span></p><p class="c10"><span class="c17 c7">Thesis</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17 c22">ELASPIC</span></p><p class="c10"><span class="c17 c22">(Predict the structural effect of mutations on a proteome-wide scale)</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c16">This was mainly an implementation, not original work&hellip;?</span><span>&nbsp;(phrasing&hellip;)</span></p><p class="c5"><span></span></p><p class="c10"><span class="c17">Core</span></p><p class="c10"><span class="c17">Training: Protherm</span></p><p class="c10"><span class="c17">Validation: Taipale histone interaction data (but FoldX alone does better&hellip;) &nbsp;(</span><span class="c16 c35 c20"><a class="c9" href="#id.pqo49xh38dgp">Figure</a></span><span class="c17">)</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Interface</span></p><p class="c10"><span class="c17">Training: Skempi</span></p><p class="c10"><span class="c17">Validation: ? (Taipale </span><span>luciferase protein complementation </span><span class="c17">data was not good) (</span><span class="c16 c35 c20"><a class="c9" href="#id.4ryvwfmrt2oq">Figure</a></span><span class="c17">)</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Database schema</span></p><p class="c10"><span class="c17">Cron job that updates downloadable database dumps every month.</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Statistics</span></p><p class="c10"><span class="c17">Number of proteins / PPIs that we were able to calculate.</span></p><p class="c10"><span class="c17">Mutations: Uniprot disease vs. polymorphism, COSMIC driver vs. passenger, Clinvar, GWAS</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Training / Variable elimination </span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17 c22">Drug combinations</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">If we include the DREAM challenge, can present it as an application of the pipeline? Mapping mutations to the PPI network in order to predict the susceptibility of different cell lines to different drug combinations.</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Design of the predictor</span></p><p class="c10"><span class="c17">(Mention Alex K.&rsquo;s validation)</span></p><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c17">Dream Challenge (Validation)?</span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><p class="c5"><span class="c17"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c17"></span></p><p class="c10"><span class="c7">Next weeks</span></p><p class="c5"><span></span></p><ul class="c3 lst-kix_x3ptir2y8jgm-0 start"><li class="c1"><span>Finish the LocalPipeline / DatabasePipeline classes, and update jobsubmitter scripts on beagle so that the webserver can submit jobs with the updated pipeline (including raw PDB structures as input).</span></li><li class="c1"><span>Create a PPI network for different cell lines, based on their mutations (already scored all missense mutations with ELASPIC) and gene expression levels (keep the threshold used, and possibly try changing it later in cross-validation to see if we can improve results).</span></li><li class="c1"><span>Calculate network properties for each cell line network.</span></li><li class="c1"><span>Train 6 predictors:</span></li></ul><ul class="c3 lst-kix_x3ptir2y8jgm-1 start"><li class="c6"><span>Regressors predicting the </span><span class="c48">change</span><span>&nbsp;in synergy score between two cell lines, using drug + target or target only features.</span></li><li class="c6"><span>Regressors predicting the </span><span class="c48">change</span><span>&nbsp;in synergy score between two cell lines, using drug + target or target only features </span><span class="c48">but no gene expression data</span><span>.</span></li><li class="c6"><span>Classifier predicting presence / absence of synergy, using drug + target or target only features. Combine with Clare&rsquo;s training set (using weight &lt; 1)?</span></li></ul><ul class="c3 lst-kix_x3ptir2y8jgm-0"><li class="c1"><span>Many other data sources can be used:</span></li></ul><ul class="c3 lst-kix_x3ptir2y8jgm-1 start"><li class="c6"><span>LINCS: a lot of drug &rArr; gene expression perturbation profiles. Targets with similar perturbation profiles are likely to be synergistic.</span></li><li class="c6"><span>Data from previous challenges</span></li><li class="c6"><span>Pathway data</span></li><li class="c6"><span>Similarity of drug monotherapy responses from other sources.</span></li></ul><p class="c5 c50"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5 c94"><span class="c35 c22 c107"></span></p><p class="c5 c94"><span class="c35 c22 c107"></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span></span></p><p class="c10"><span class="c40">Reviewer: 1</span></p><p class="c10"><span class="c40">Comments to the Author</span></p><p class="c10"><span class="c40">This manuscript presents the ELASPIC web server which can be used to predict and analyze the effect of mutations on protein domains and domain-domain interactions. This topic is of great importance. The web server is functional and very user-friendly (I particularly appreciated the available help, downloads and software, and all the available options including an HTML5 alternative to Java-based visualization).</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40 c102">However, as detailed below, some important results are missing</span><span class="c40">&nbsp;</span><span class="c40">(especially about the &quot;improved structural domain definitions&quot; that were not used in the original ELASPIC paper)</span><span class="c102 c40">, some aspects of the manuscript could be improved and some additional information could be added to the web interface to make it even more useful.</span></p><p class="c10"><span class="c40">Major comments</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">1. There are already some web servers that allow users to predict the effect of mutations and that use homology modelling, for instance HOPE (Venselaar et al, BMC Bioinformatics 2010) or SNPeffect (De Baets et al, Nucleic Acids Res 2012). The authors should include those in their introduction/references and briefly discuss in what way ELASPIC is different from them. </span></p><p class="c10"><span class="c49 c22 c40">OK.</span></p><p class="c10"><span class="c49 c40">HOPE</span></p><ul class="c3 lst-kix_s8oehts9vu79-0 start"><li class="c1"><span class="c49 c40">Predicts the effect of mutations on protein CORE but not on PPIs. </span></li><li class="c1"><span class="c49 c40">Do not predict ddG.</span></li></ul><p class="c10"><span class="c49 c40">SNPeffect</span></p><ul class="c3 lst-kix_m3f8ler1ak4c-0 start"><li class="c1"><span class="c49 c40">Also no interfaces.</span></li></ul><p class="c5"><span class="c39 c22"></span></p><p class="c5"><span class="c39 c22"></span></p><p class="c10"><span class="c40">2. The Profs pipeline seems to be a really crucial element in the version of the ELASPIC pipeline that is available through the web server. It is highlighted as such in the abstract and main paper. Profs is described in the supplementary material in terms of (detailed) methods, which is very useful; however no results are presented.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">2a. Are there some validation results showing that Profs actually improves domain definitions over Pfam, as the abstract and paper say? I am aware that Profs is not the main subject of the present paper but since it seems so important to it, some evidence to support it would have its place in the supplementary material of the current manuscript, since the Profs pipeline in itself does not appear to have been published yet.</span></p><p class="c10"><span class="c49 c22 c40">OK. </span><span class="c49 c40">Can include the plot I made comparing Profs, Pfam, and Gene3D. Link: </span><span class="c16 c40 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTMGlITVoyY2JvbjQ&amp;sa=D&amp;usg=AFQjCNHkZSHM09XLaH0KfhZbR_oSe8oFIg">https://drive.google.com/open?id=0B1FXQpQb4bbTMGlITVoyY2JvbjQ</a></span></p><p class="c10"><span class="c49 c22 c40">Andres sent some case studies: </span><span class="c16 c40 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D1Q-aL1WMDOAnv4iE4iLLSo5-ds0kQKhl_QbXNpwYkxyM&amp;sa=D&amp;usg=AFQjCNEaNBg8h9NDd4i_BLMmmlcq0AKyVQ">https://drive.google.com/open?id=1Q-aL1WMDOAnv4iE4iLLSo5-ds0kQKhl_QbXNpwYkxyM</a></span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">2b. Even more importantly, how does the use of Profs instead of Pfam domains affect the performance of ELASPIC? The original publication (Berliner et al PLoS One 2014) described the results of ELASPIC using Pfam domains only.</span></p><p class="c10"><span class="c49 c40">Original publication publication was using (Sebastian&rsquo;s) PDBfam for homology modelled structures as well. The PDBfam (Profs) pipeline underwent majour development since then but the results should remain applicable. </span></p><p class="c10"><span class="c40 c82">Comparing ELASPIC performance using Pfam vs. Profs would be difficult&hellip; maybe we can just say that the longer domain definitions obtained with Profs are good enough reason to use Profs instead of Pfam?</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">3. It is very interesting that the web server provides a structural representation of the mutated protein/interface (as well as downloadable PDB files). However it seems that those structures are obtained by just replacing the side chain of the wt residue. For instance, the structure displayed in the main figure shows a covalent bond (&lt;=3A) between the mutated W and the neighboring F. I view this as a sign that the structure/interface has not been relaxed/optimized after the mutation has been performed. Could the authors provide some more details about this? For instance how is the rotamer picked? Would it be possible to perform a minimum amount of relaxation so that the structure containing the mutated residue is more realistic?</span></p><p class="c10"><span class="c49 c16 c40">We rely on FoldX to optimize the structure, and it optimizes the wild type and mutant structure in the same way... Quote:</span></p><p class="c10"><span class="c49 c40 c48">This command ensures that whenever you are mutating a protein you always move the same neighbours in the WT and in the mutant producing for each mutant a corresponding PDB for its WT (each mutation will move different neighbours and therefore you need different WT references). </span></p><p class="c10"><span class="c49 c40">+ good suggestion but not relevant for a </span><span class="c49 c22 c40">webserver</span><span class="c49 c40">&nbsp;paper, and can be implemented after the paper is accepted :).</span></p><p class="c10"><span class="c49 c22 c40">Can add this information to the supplemental material?</span></p><p class="c5"><span class="c16 c39"></span></p><p class="c10"><span class="c17 c40 c95">4. Still regarding the web server, it would be most useful if there were some clear indications of </span><span class="c40">whether a PDB structure was used or a homology model was built</span></p><p class="c10"><span class="c40">- which PDB structure was used (if applicable) or which template(s) for homology modelling</span></p><p class="c10"><span class="c34">Daniel</span><span class="c49 c40">. This information is in the database and alignment downloads. Can add this field in the output.</span></p><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- </span><span class="c40">whether the mutation is present in another database (COSMIC, etc.) </span></p><p class="c10"><span class="c49 c40">A bit more difficult. Add another field in the mutation output page to Ensembl / COSMIC / Clinvar / Uniprot?</span></p><p class="c10"><span class="c49 c40">(May need to create a new table in the database if the queries are slow). </span></p><ol class="c3 lst-kix_jqqhsyijt7h5-0 start" start="1"><li class="c1"><span class="c49 c40">Uniprot humsavar.txt (~70k mutations) -&gt; SwissVar (and OMIM) -&gt; dbSNP / Ensembl </span></li><li class="c1"><span class="c49 c40">Uniparc -&gt; Ensembl db (&gt; 2 million mutations) -&gt; dbSNP / COSMIC / Ensembl website</span></li></ol><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- </span><span class="c40">I am assuming the mutations highlighted on the sequence representation that appears when you enter an input name are those that were retrieved from SNP/SNV databases, but could not verify this and find out exactly which database each mutation belongs to. A link to a description of the mutation in the relevant database would be even better.</span></p><p class="c10"><span class="c49 c40">Maybe </span><span class="c22 c34">Daniel </span><span class="c49 c40">has some suggestions&hellip;</span><span class="c34">&nbsp;If we implement the above request it shouldn&rsquo;t be difficult.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Minor comments</span></p><p class="c10"><span class="c40">- Especially for pre-calculated mutations (such as the R2088W in EP300 described in the main figure), why are there several result lines showing very similar results? In the case of R2088W, the mutation has 5 result lines annotated as &quot;interface with NCOA3&quot; and almost (but not 100%) identical values.</span></p><p class="c10"><span class="c49 c40">This is the splice variant issue&hellip; </span></p><ul class="c3 lst-kix_6bb7qu9vjqr2-0 start"><li class="c1"><span class="c16 c28"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/03525d/&amp;sa=D&amp;usg=AFQjCNHLU_r9bBV73TisZupA-wuC15ESXQ">http://elaspic.kimlab.org/result/03525d/</a></span></li></ul><ul class="c3 lst-kix_icjmrj4tgidb-0 start"><li class="c1"><span class="c49 c40">Can remove interactions between non-canonical uniprots, but then external identifiers (e.g. Ensembl) will often not have any interactions (as they map to splice variants). </span></li><li class="c1"><span class="c49 c40">Maybe just display the uniprot id with the splice variant indicated by -1, -2, -3?</span></li><li class="c1"><span class="c33 c74">Daniel</span><span class="c33 c49">: fix &ldquo;Interface w/ HGNCIdentifier object&rdquo;...</span></li></ul><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">- In the methods, there are many tools or data sources that are cited without a reference: HGNC, OMIM, COSMIC, etc.</span></p><p class="c10"><span class="c49 c40">Will do. Left this out because we ran out of space.</span></p><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- In section 2.3 I am not sure what is meant by &quot;impact scores&quot;. Is it something more than the predicted deltadeltaG?</span></p><p class="c10"><span class="c49 c40">No, it&rsquo;s just ddG (for now). Will change.</span></p><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- In legend of Figure 1: severally -&gt; severely.</span></p><p class="c10"><span class="c49 c40">Thanks. Will change.</span></p><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- In section 3.2, the sentence &quot;We applied our approach to a list of somatic mutations found in cancer&quot; somehow leads to the expectation that the results for this list would be available somewhere and discussed (regarding availability, I assume this list is among the available downloads - pointing to the exact location would be most helpful).</span></p><p class="c10"><span class="c22 c40 c49">Will make ddG predictions Ensembl / COSMIC / Clinvar / Uniprot available for download?</span></p><p class="c5"><span class="c49 c40"></span></p><p class="c10"><span class="c40">- The discussion in the supplementary material is actually not much more &quot;in-depth&quot; than the discussion in section 3.2. Maybe some extra figures describing the other interfaces and/or the representation of the associated interaction network would help in this regard.</span></p><p class="c10"><span class="c34 c22">Daniel</span><span class="c49 c40">.</span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Reviewer: 2</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Comments to the Author</span></p><p class="c10"><span class="c40">The authors designed a webserver that predicts mutation effects on protein stability and binding affinity. Given a protein sequence (Uniprot ID), domains are identified. For each domain, homology models are built. And homology models for all interactions involving this protein are also built. The structural features together with other features are used by a machine learning predictor to predict binding affinity for every mutation.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">The user interface is very friendly and nicely designed. Users are allowed to specify mutations by either clicking on the protein figure or by choosing from the drop-down menu. Batch processing mode is also available. The results are clear and informative. The Help page is also informative.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">I have, however, four comments:</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">1) It would be nice to offer the possibility to give a PDB entry rather than a UNIPROT code.</span></p><p class="c10"><span class="c49 c40">Have to either train another predictor using raw structures, or create some hack where the structures still go through MODELLER? (The latter is probably better, since maintaining several predictors is a mess and passing a structure through MODELLER does not change much and is quick if the alignment is 100% identical).</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">2) It would be nice to allow for upload of structures. Allowing the uploading structures can skip the homology modeling step, and can provide more precise information to the machine learning predictor.</span></p><p class="c10"><span class="c34 c22">Daniel</span><span class="c49 c40">. If we decide to implement step 1) above.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">3) Could not see how to target / define a mutation within the interface of a complex. It would be nice to show the interface (from the homology models) on the protein sequence figure, too.</span></p><p class="c10"><span class="c34 c22">Daniel</span><span class="c34">.</span><span class="c49 c40">&nbsp;Difficult! Cytoscape PPI map at mutation input?</span></p><p class="c10"><span class="c49 c40">(ELASPIC already allows specifying a particular uniprot_domain_pair_id).</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">4) Figure 4 in the Suppl. Material provides critical details about the pipeline. Maybe consider putting it in the main text or on the </span><span class="c34">web server</span><span class="c40">.</span></p><p class="c10"><span class="c49 c22 c40">No space&hellip;?</span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Reviewer: 3</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Comments to the Author</span></p><p class="c10"><span class="c40">The Applications Note describes the ELASPIC web-server that predicts the effect of mutations on protein folding and protein-protein interaction. The server is the first server that uses homology models to predict the effect of mutations on protein stability and binding affinity and hence, it is justified to present it to a wider audience. The web server has a simple and clean user interface, making the usage very easy for non-experts. The explanation and documentation are adequate. The performance of the pipeline has been demonstrated in a separate publication. The manuscript is well written, it cites related work, briefly describes the algorithms behind the server and shows a screenshot of the results page for a structural model of a case study. The web server works as described, however some minor points have to be addressed.</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">Minor:</span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c40">1. &nbsp; &nbsp; &nbsp;Uploading of a file to the server is not working. Firefox displays the message: &ldquo;File could not be uploaded - try again&rdquo; . Chromium does not report any problems but no file is uploaded. This has to be fixed. It would also be beneficial if an example file is provided to the user.</span></p><p class="c10"><span class="c34">Daniel.</span></p><p class="c10"><span class="c40">2. &nbsp; &nbsp; &nbsp;If there are many previously run mutations, it is almost impossible to read the labels in the protein figure at the submit job web page due to the overlapping text. It would be nice if this information could also be presented in a table or similar format to the user.</span></p><p class="c10"><span class="c34 c22">Daniel. This is probably the most difficult.</span></p><p class="c10"><span class="c40">3. &nbsp; &nbsp; &nbsp;The single mutation submit form allows the use to select the protein ID O00522 as an example input by clicking on it. However, the protein ID is not highlighted and can therefore easily be missed by the user.</span></p><p class="c10"><span class="c34">Daniel.</span></p><p class="c5"><span class="c49 c22 c40"></span></p><p class="c10"><span class="c40">----------------------------------------</span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c10"><span class="c49 c22 c40">Majour question:</span></p><ul class="c3 lst-kix_g84y3pe344o9-0 start"><li class="c1"><span class="c49 c40">Is it OK if we pass all structures through Modeller (even if 100% sequence identity) as a way to &ldquo;relax&rdquo; the structure? Building and maintaining a second predictor that is based on raw PDBs would be a lot of work (and since the PDBs can vary so much from structure to structure, maybe using Modeller is a good way to prevent overfitting on particular idiosyncrasies of the training set?)</span></li></ul><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><p class="c5"><span class="c40"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c40"></span></p><h1 class="c10 c77"><a name="h.yna5w7pmyaeb"></a><span class="c58 c22">Splicing/Network, 2015-09-22</span></h1><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c7">Dream Challenge Outline:</span></p><p class="c5"><span></span></p><p class="c10"><span>Challenge 1a: Predict synergy scores of drug pairs in particular cell lines, given, as training data, synergy scores of </span><span class="c22">the same drug </span><span>pairs in different cell lines. </span></p><ul class="c3 lst-kix_ia5tg78e6pm4-0 start"><li class="c1"><span>Provided with mutation / CNV / gene expression data for most cell lines.</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Challenge 1b: Same as 1a, but without using gene expression data (only mutations / CNVs).</span></p><ul class="c3 lst-kix_9ft7dqndoe5p-0 start"><li class="c1"><span>Supposed to levarage interesting patterns that we notice in challenge 1a</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Challenge 2:</span><span>&nbsp;Predict synergy scores of drug pairs in particular cell lines, given, as training data, synergy scores of </span><span class="c22">different drug pairs </span><span>in different cell lines. (The most similar to Clare&rsquo;s approach).</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Final submission &nbsp;deadline extended until January&hellip; (~300 participants, biggest DREAM challenge ever)</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c7">Exploring the Dataset for Meaningful Features:</span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Synergy Scores</span></p><p class="c10"><span>Correlation between synergy scores in different cell lines ~ 0.25. </span></p><p class="c10"><span>Correlation between synergy scores in the same cell lines ~ 0.50 (as reported on the forums).</span></p><p class="c10"><span>So this is probably the strongest feature for </span><span class="c16">challenge 1</span><span>&nbsp;(but some pairs of cells may be consistently more correlated than others!)</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 401.00px; height: 274.00px;"><img alt="" src="images/image39.png" style="width: 401.00px; height: 274.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c10"><span>Features:</span></p><ul class="c3 lst-kix_krjlquulz7zd-0 start"><li class="c1"><span>Max Synergy Score from association network + confidence</span></li><li class="c1"><span>Mean Synergy Score from association network + confidence</span></li><li class="c1"><span>Weighted mean Synergy Score from association network + confidence</span></li></ul><p class="c10"><span>Correlation between monotherapy responses:</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Monotherapy Scores</span></p><p class="c5 c99"><span class="c35 c22 c54"></span></p><a href="#" name="76a3634b907b475cdc87ccf535596f809dc00b0f"></a><a href="#" name="0"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c84" colspan="1" rowspan="1"><p class="c5 c42 c15"><span class="c29"></span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c35 c22 c54">ic50_2</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c35 c22 c54">h_2</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c35 c22 c54">e_inf_2</span></p></td></tr><tr class="c75"><td class="c84" colspan="1" rowspan="1"><p class="c10 c42 c12"><span class="c35 c22 c54">ic50_1</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c45 c35 c22 c54">0.251497</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c15 c42"><span class="c29">-0.046592</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">-0.082847</span></p></td></tr><tr class="c75"><td class="c84" colspan="1" rowspan="1"><p class="c10 c42 c12"><span class="c35 c22 c54">h_1</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">-0.046592</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">0.020994</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">0.044669</span></p></td></tr><tr class="c75"><td class="c84" colspan="1" rowspan="1"><p class="c10 c42 c12"><span class="c35 c22 c54">e_inf_1</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">-0.082847</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c29">0.044669</span></p></td><td class="c62" colspan="1" rowspan="1"><p class="c10 c42 c15"><span class="c35 c54">0.</span><span class="c35 c54">138592</span></p></td></tr></tbody></table><p class="c5"><span></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 241.33px;"><img alt="" src="images/image57.png" style="width: 720.00px; height: 241.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span class="c22">Cross-validation:</span></p><ul class="c3 lst-kix_d1mamalvi3wu-0 start"><li class="c1"><span>Leave one drug pair / cell line out (as long as there is the same drug pair in different cell lines)</span></li><li class="c1"><span>Leave one drug pair out (all cell lines)</span></li><li class="c1"><span>Leave one drug pair out (all cell lines, and all drug pairs that contain either of the two drugs)</span></li></ul><h1 class="c10 c77"><a name="h.kyy9rg6q3mab"></a><span class="c22 c58">Splicing/Network, 2015-09-15</span></h1><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span class="c18">Make all mutations for P41250 variants&hellip;</span></p><ul class="c3 lst-kix_mpnpv7ekeo7q-0 start"><li class="c1"><span class="c18">Use protein names P41250-DM1, P41250-DM6, P41250-DM9 in </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/&amp;sa=D&amp;usg=AFQjCNHQrssidsCRml0EWikB_aWe3W_G2Q">elaspic</a></span><span class="c18">.</span></li><li class="c1"><span class="c18">Plots: &nbsp;</span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/a/kimlab.org/folderview?id%3D0B1FXQpQb4bbTfkxHMHkteE1kaEIyT21taXhGVDdScWpBTTNOUGNDVEpIYTBQb3gxcHJ2WEE%26usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNHuO90eQ157oROoDA4YQesQMIg8BA">google drive folder</a></span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Updated the ELASPIC virtual machine</span></p><ul class="c3 lst-kix_qffkk51zafur-0 start"><li class="c1"><span class="c18">Run jobs on the beagle cluster instead of locally on the VM</span></li></ul><ul class="c3 lst-kix_g6dv2r60jppe-0 start"><li class="c1"><span class="c18">Transferred metadata for previously-completed jobs to the new VM.</span></li><li class="c1"><span class="c18">Fixed some Python 3 issues that haven&rsquo;t been detected previously.</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Mapped drop-off score vs. NormDOPE score, and HIPPIE (physical) and String (functional) interaction networks</span></p><ul class="c3 lst-kix_4r80w4zgcdjq-0 start"><li class="c1"><span class="c18">Peptides are unlikely to have high drop-off score when String edge betweenness is high.</span></li><li class="c1"><span class="c18">Next page...</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://www.synapse.org/%23!Synapse:syn4231880/wiki/&amp;sa=D&amp;usg=AFQjCNEwHZkCTxVCL_GiUUcZ8v-EYAPcYw">DREAM Drug Combination Challenge</a></span></p><ul class="c3 lst-kix_cm89sr4fo6m-0 start"><li class="c1"><span class="c18">Updating our PPI database</span></li></ul><ul class="c3 lst-kix_cm89sr4fo6m-1 start"><li class="c6"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://www.ebi.ac.uk/Tools/webservices/psicquic/view/main.xhtml&amp;sa=D&amp;usg=AFQjCNEF10cnMrC1ZC1Z12NgRGk04CxSXQ">PSICQUIC View</a></span><span class="c18">&nbsp;(download raw PPI data from most databases in a standard format)</span></li><li class="c6"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://netbio.bgu.ac.il/myproteinnet/&amp;sa=D&amp;usg=AFQjCNGHdXdHwUDVhFhaL9lbiHfnK0YFKQ">MyProteinNet</a></span><span class="c18">&nbsp;(tissue s</span><span class="c18">pecific PPI networks)</span></li></ul><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Updating the uniprot sequence and cross-reference database </span></p><ul class="c3 lst-kix_hxvvdzewxqs2-0 start"><li class="c1"><span class="c18">No easy way to update the 10+ GB uniprot identifier table other than to recreate it with the new data.</span></li></ul><ul class="c3 lst-kix_hxvvdzewxqs2-1 start"><li class="c6"><span class="c18">Faster to recreate the table using new data then update it with missing entries</span></li></ul><ul class="c3 lst-kix_l9w5fm8md2s9-0 start"><li class="c1"><span class="c18">Use uniparc, which cross-references </span><span class="c48 c18">sequences</span><span class="c18">&nbsp;from different databases using the CRC64 hash of the sequence.</span></li><li class="c1"><span class="c18">Ideal for mapping mutations to / from Ensembl and NCBI because we know that the sequences are identical.</span></li><li class="c1"><span class="c18">The XML document contains a &ldquo;last updated&rdquo; field, which makes updating the database easy</span></li></ul><ul class="c3 lst-kix_l9w5fm8md2s9-1 start"><li class="c6 c19"><span class="c18"></span></li></ul><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 202.67px;"><img alt="" src="images/image98.png" style="width: 720.00px; height: 202.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 202.67px;"><img alt="" src="images/image123.png" style="width: 720.00px; height: 202.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 202.67px;"><img alt="" src="images/image127.png" style="width: 720.00px; height: 202.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 558.67px;"><img alt="" src="images/image134.png" style="width: 720.00px; height: 558.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span class="c18">Correlation is weak but p-value is higher than Predictor / FoldX / Provean (e.g. </span><span class="c16 c20 c18"><a class="c9" href="#kix.orrw8pwmmbup">Interface properties vs. drop-off score</a></span><span class="c18">).</span></p><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 202.67px;"><img alt="" src="images/image111.png" style="width: 720.00px; height: 202.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 202.67px;"><img alt="" src="images/image132.png" style="width: 720.00px; height: 202.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22 c18"></span></p><hr><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 201.33px;"><img alt="" src="images/image64.png" style="width: 720.00px; height: 201.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 201.33px;"><img alt="" src="images/image93.png" style="width: 720.00px; height: 201.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><h1 class="c10 c109 c77"><a name="h.qmplrvqzvvgg"></a></h1><hr style="page-break-before:always;display:none;"><h1 class="c10 c109 c77"><a name="h.p6gl0mqh3sry"></a></h1><h1 class="c10 c77"><a name="h.24zfgsvqb447"></a><span class="c58 c22">Splicing/Network, 2015-09-01</span></h1><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c38"></span></p><p class="c10"><span class="c22 c110">Progress</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c22 c18">Jobsubmitter for the ELASPIC webserver</span></p><ul class="c3 lst-kix_ric95po2pkzv-1 start"><li class="c1"><span class="c18">Allow the webserver to submit jobs onto the cluster instead of having a limited amount of resources that most of the time are wasted and / or overwhelmed.</span></li></ul><ul class="c3 lst-kix_ric95po2pkzv-2 start"><li class="c6"><span class="c18">This way people in the lab can use it too without overloading the virtual machine :).</span></li></ul><ul class="c3 lst-kix_ric95po2pkzv-1"><li class="c1"><span class="c18">If a protein does not have precalculated provean supporting sets or structure, don&rsquo;t start running provean and modeller for every submitted mutation in that protein! </span></li></ul><ul class="c3 lst-kix_ric95po2pkzv-2 start"><li class="c6"><span class="c18">Create a lock for each provean and modeller job that is running. Release the lock once the alignment and structure have been calculated. (Mark&rsquo;s suggestion).</span></li></ul><ul class="c3 lst-kix_ric95po2pkzv-1"><li class="c1"><span class="c18">Provean and modeller can be running for the same protein at the same time.</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Done. Just need some more testing and then I will swap it with the current ELASPIC virtual machine (or use a docker container&hellip;).</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c22 c18">Nice-to-haves...</span></p><ul class="c3 lst-kix_ric95po2pkzv-1"><li class="c1"><span class="c18">Limit the load on the MySQL database by processing several mutations inside one job (using one query to the database). Maybe </span></li></ul><p class="c5"><span class="c18"></span></p><h1 class="c10 c93"><a name="h.xzzswv21ouhn"></a><span class="c95 c22">Glycine--tRNA ligase (P41250)</span></h1><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Created a copy of the P41250 protein with only a single domain that we are interested in, with 2PMF as the structural template (it can be queried in the ELASPIC webserver as </span><span class="c22 c18">P41250-BCG101</span><span class="c18">). </span></p><p class="c10"><span class="c22 c18">PDB </span><span class="c18">551-674; </span><span class="c22 c18">Uniprot</span><span class="c18">: 607-728</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 316.81px; height: 228.50px;"><img alt="" src="images/image01.png" style="width: 316.81px; height: 228.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span class="c18">Created a histogram of SASA scores, and decided to use 30% SASA as a threshold for interface vs. core amino acids.</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">This gave me 60 residues. I mutated each of those residues to each of the 10 polar amino acids: </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/518dd5/&amp;sa=D&amp;usg=AFQjCNGW4rhiQBQe5Nm5TWntWnYTyR_fpg">http://elaspic.kimlab.org/result/518dd5/</a></span><span class="c18">. &lt;--</span></p><p class="c10"><span class="c16 c22 c18">Carles manually inspected the structure </span><span class="c16 c18">and came up with some leads</span></p><p class="c10"><span class="c48 c18">Manual inspection of the structure</span></p><ol class="c3 lst-kix_p4k358qyzf3y-1 start" start="1"><li class="c6 c30"><span class="c48 c18">MET16 [574] (628) </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/836b78/&amp;sa=D&amp;usg=AFQjCNFtJvEXLYi91sm4tZEyJ2zlUrStcw">http://elaspic.kimlab.org/result/836b78/</a></span></li><li class="c6 c30"><span class="c48 c18">PRO17 [575] (629) </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/7ead46/&amp;sa=D&amp;usg=AFQjCNFuRuqfKPFq4MV60SSmMPVeHaZ6gw">http://elaspic.kimlab.org/result/7ead46/</a></span></li><li class="c6 c30"><span class="c48 c18">VAL35 [593] (647) </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/b67722/&amp;sa=D&amp;usg=AFQjCNGp0oz9GpBJHIkrDuoRWy2qKM0RFg">http://elaspic.kimlab.org/result/b67722/</a></span></li><li class="c6 c30"><span class="c48 c18">MET80 [638] (692) </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/0ae75d/&amp;sa=D&amp;usg=AFQjCNGsCzivnzXfClRZpZji-tPRCkuLNA">http://elaspic.kimlab.org/result/0ae75d/</a></span></li><li class="c6 c30"><span class="c48 c18">LEU113 [671] (725) </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/889eeb/&amp;sa=D&amp;usg=AFQjCNHcgHGcF2nMylRMZe1l1mlGTCHVOQ">http://elaspic.kimlab.org/result/889eeb/</a></span><span class="c48 c18">&nbsp;</span><span class="c22 c48 c18">(***)</span></li></ol><ol class="c3 lst-kix_p4k358qyzf3y-0 start" start="1"><li class="c1 c30"><span class="c48 c18">subs arginine by lysine</span></li></ol><ol class="c3 lst-kix_p4k358qyzf3y-1 start" start="1"><li class="c6 c30"><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/result/227cd5/&amp;sa=D&amp;usg=AFQjCNExzXtJlBQjy8zlAVH_3RAMSIoNyg">http://elaspic.kimlab.org/result/227cd5/</a></span><span class="c18">&nbsp;</span><span class="c22 c48 c18">(***)</span></li></ol><p class="c5 c30"><span class="c18"></span></p><p class="c10 c30"><span class="c16 c18">Mutations to </span><span class="c16 c22 c18">K/E/Q </span><span class="c16 c18">usually are predicted to be the least disruptive or even stabilizing&hellip;</span></p><p class="c10 c30"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 229.33px;"><img alt="" src="images/image42.png" style="width: 720.00px; height: 229.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c30"><span class="c16 c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 229.33px;"><img alt="" src="images/image109.png" style="width: 720.00px; height: 229.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c10"><span>Select all amino acids with SASA scores &gt; 30% and mutate them to 10 polar amino acids (DESTYQNKRH).</span></p><ul class="c3 lst-kix_i2rq8qzg0efa-0 start"><li class="c1"><span>all arginines to lysine </span></li></ul><p class="c5"><span></span></p><ul class="c3 lst-kix_i2rq8qzg0efa-0"><li class="c1"><span class="c7">Splicing/Network, 2015-08-25</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span class="c22 c18">Mapping the </span><span class="c22 c48 c18">entire </span><span class="c22 c18">disorderome library to ELASPIC</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 365.30px; height: 301.50px;"><img alt="" src="images/image143.png" style="width: 365.30px; height: 301.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 322.50px; height: 236.37px;"><img alt="" src="images/image89.png" style="width: 322.50px; height: 236.37px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 350.04px; height: 264.50px;"><img alt="" src="images/image106.png" style="width: 350.04px; height: 264.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 352.50px; height: 265.13px;"><img alt="" src="images/image07.png" style="width: 352.50px; height: 265.13px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span class="c22 c18">Mapping mutations to interfaces</span></p><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 349.50px; height: 233.31px;"><img alt="" src="images/image90.png" style="width: 349.50px; height: 233.31px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.50px; height: 236.00px;"><img alt="" src="images/image20.png" style="width: 354.50px; height: 236.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 353.50px; height: 236.00px;"><img alt="" src="images/image61.png" style="width: 353.50px; height: 236.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span class="c7">Correlation between </span><span class="c7 c48">dropoff score</span><span class="c7">&nbsp;and interface features</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><a href="#" name="99349da45baa57268307a17b1e73fea9b6db897e"></a><a href="#" name="1"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c5 c21 c15"><span class="c45 c63 c38"></span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">SCORE_rwp</span></p></td><td class="c103" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">SCORE_hek</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">SCORE_rwp</span></p></td><td class="c106" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">1</span></p></td><td class="c92" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.116051</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">SCORE_hek</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.116051</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">1</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">uniprot_sequence_length</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.02027</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.030715</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">match_pc</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.086333</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.01859</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">pc_peptide_covered_by_interface</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.009508</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.007789</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">pc_interface_covered_by_peptide</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.002261</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.019852</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">norm_dope</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.024184</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.035112</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">interface_area_hydrophobic</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.053904</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.021387</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">interface_area_hydrophilic</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.030405</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.020782</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c22 c40 c45">interface_area_total</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.048425</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.021678</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c51 c63 c40">predictor_ddg_sum</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c51 c63 c40">-0.062794</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c51 c63 c40">-0.021631</span></p></td></tr><tr class="c75"><td class="c11" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c51 c63 c40">foldx_ddg_sum</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c22 c40 c51">-0.066751</span></p></td><td class="c103" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c51 c63 c40">-0.018237</span></p></td></tr><tr class="c75"><td class="c111" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c51 c40 c63">provean_score_sum</span></p></td><td class="c106" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c51 c22 c40">0.069049</span></p></td><td class="c96" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c51 c22 c40">0.020643</span></p></td></tr><tr class="c75"><td class="c114" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">predictor_ddg_abssum</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c45 c22 c40">-0.064016</span></p></td><td class="c89" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c45 c22 c40">-0.021777</span></p></td></tr><tr class="c75"><td class="c105" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c40">foldx_ddg_abssum</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.063576</span></p></td><td class="c101" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c45 c22 c40">-0.024342</span></p></td></tr><tr class="c75"><td class="c115" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">provean_score_abssum</span></p></td><td class="c106" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.069092</span></p></td><td class="c92" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.020856</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">predictor_ddg_mean</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.00322</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.000505</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">foldx_ddg_mean</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.030116</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.013136</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">provean_score_mean</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.002456</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.014325</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">predictor_ddg_min</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.031324</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.021758</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">foldx_ddg_min</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.026806</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.016937</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">provean_score_min</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.039117</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0.005405</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">predictor_ddg_max</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.022403</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.015544</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">foldx_ddg_max</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c70 c40">-0.057418</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c70 c40">-0.021062</span></p></td></tr><tr class="c75"><td class="c61" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c31">provean_score_max</span></p></td><td class="c41" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.040273</span></p></td><td class="c55" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">-0.01125</span></p></td></tr></tbody></table><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c38"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 350.50px; height: 240.12px;"><img alt="" src="images/image14.png" style="width: 350.50px; height: 240.12px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 343.25px; height: 240.50px;"><img alt="" src="images/image40.png" style="width: 343.25px; height: 240.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c38"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 348.58px; height: 238.50px;"><img alt="" src="images/image128.png" style="width: 348.58px; height: 238.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.50px; height: 235.69px;"><img alt="" src="images/image30.png" style="width: 354.50px; height: 235.69px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c38"></span></p><p class="c5"><span class="c7"><br></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Splicing/Network, 2015-08-18</span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c16 c18"></span></p><p class="c10"><span class="c22 c18">Progress:</span></p><ul class="c3 lst-kix_a76b6w8pnkyv-0 start"><li class="c1"><span class="c18">We are still missing 170,000 interface mutations (~17% of all interface mutations)</span></li></ul><ul class="c3 lst-kix_a76b6w8pnkyv-1 start"><li class="c6"><span class="c18">Looking at the failed examples, many of those mutations fail because the 7zip archives are outdated</span></li><li class="c6"><span class="c18">Updated the 7zip archvies on the </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download&amp;sa=D&amp;usg=AFQjCNEQpfPsvSAT9UL-PhMgAIxq_rV25Q">elaspic downloads page</a></span><span class="c18">&nbsp;(were .tar.gz before)</span></li></ul><ul class="c3 lst-kix_a76b6w8pnkyv-0"><li class="c1"><span class="c18">Finally compiled graph-tool</span></li><li class="c1"><span class="c18">Was figuring out how to run virtual machines on the webserver</span></li></ul><ul class="c3 lst-kix_a76b6w8pnkyv-1 start"><li class="c6"><span class="c18">I was using VirtualBox + Vagrant on my desktop, but the webserver uses Xen Server (more performant)</span></li><li class="c6"><span class="c18">Converting VirtualBox -&gt; Xen images and figuring out how xen works</span></li></ul><ul class="c3 lst-kix_a76b6w8pnkyv-0"><li class="c1"><span class="c18">Drug combinations</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c16 c18"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-08-11</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c16 c18">Taipale Y2H protein-protein interaction dataset:</span></p><p class="c5"><span class="c18"></span></p><p class="c10 c12"><span class="c22 c18">Mapping mutations to uniprot</span></p><p class="c10"><span class="c18">Number of unique mutations (starting point): &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;527<br>Number of unique mutations &nbsp;(after mapping to uniprot): &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;657</span></p><p class="c10"><span class="c18">Number of mutated proteins:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;285</span></p><p class="c10"><span class="c18">Number of unique interfaces &nbsp;(after mapping to uniprot):&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2197</span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Run Profs (PDBfam) on the new interfaces to find structural templates</span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10 c12"><span class="c22 c18">Number of unique interface / mutation combinations</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 634.83px; height: 228.50px;"><img alt="" src="images/image21.png" style="width: 634.83px; height: 228.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c10 c12"><span class="c22 c18">Taking the best interface for each mutation</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 633.96px; height: 229.50px;"><img alt="" src="images/image84.png" style="width: 633.96px; height: 229.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><a href="#" name="id.4ryvwfmrt2oq"></a><p class="c10"><span class="c22 c18">GPCA</span><span class="c18">: </span><span class="c48 c18">Gaussia princeps</span><span class="c18">&nbsp;luciferase protein complementation assay</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 600.00px;"><img alt="" src="images/image52.png" style="width: 720.00px; height: 600.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">The only ok feature is Provean, but even that disappears when look at all interface mutations and not just those for which we could build a structural model.</span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Looking at the change in GPCA scores between wildtype and mutant isn&rsquo;t much different....</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 600.00px;"><img alt="" src="images/image72.png" style="width: 720.00px; height: 600.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Looking at patterns in the loss or gain of interactions wasn&rsquo;t much better...</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 600.00px;"><img alt="" src="images/image54.png" style="width: 720.00px; height: 600.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 816.00px;"><img alt="" src="images/image17.png" style="width: 720.00px; height: 816.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-08-05</span></p><p class="c5"><span class="c7"></span></p><ul class="c3 lst-kix_ejnvxy8b1e9w-0 start"><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8000&amp;sa=D&amp;usg=AFQjCNHHEZlqN9nJviXZf25Cfc1nE-fgmg">http://kimweb1.ccbr.utoronto.ca:8000</a></span></li><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8001&amp;sa=D&amp;usg=AFQjCNG69JABLIuEk66TvfxKmK14BlPOpg">http://kimweb1.ccbr.utoronto.ca:8001</a></span></li></ul><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Documentation and Wiki</span></p><p class="c10"><span class="c18">username: kimguest</span></p><p class="c10"><span class="c18">password: 630kim630</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Created two shared folders:</span></p><ul class="c3 lst-kix_ejnvxy8b1e9w-0"><li class="c1"><span class="c48 c18 c70">/kimlab1/share_www/docs</span><span class="c18">&nbsp;-&gt; </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca/docs&amp;sa=D&amp;usg=AFQjCNGkONkcgEzuCVUFg-OjUpWjbRSkww">http://kimweb1.ccbr.utoronto.ca/docs</a></span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-1 start"><li class="c6"><span class="c18">Lab members can copy their static html files here if they want to make them accessible to other people in the lab. </span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-0"><li class="c1"><span class="c70 c48 c18">/</span><span class="c70 c48 c18">kimlab1/share_www/wiki</span><span class="c70 c18">&nbsp;</span><span class="c18">-&gt; </span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8201&amp;sa=D&amp;usg=AFQjCNH5m4_w0rzloX65Zjx1CcQZ_p4s6Q">http://kimweb1.ccbr.utoronto.ca:8201</a></span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-1 start"><li class="c6"><span class="c18">A django wiki for editing documentation interactively.</span></li><li class="c6"><span class="c18">Files organised in a hierarchical folder structure</span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-2 start"><li class="c10 c87 c57"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8201/bioinformatics&amp;sa=D&amp;usg=AFQjCNHhWhtsM2fzTcU984Spzvsg-Fk3WQ">/bioinformatics.rst</a></span></li><li class="c10 c87 c57"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8201/database/example_queries&amp;sa=D&amp;usg=AFQjCNGCt2AmBkYFIMSqvB3GdFqCdpxKog">/database/example_queries.rst</a></span></li><li class="c10 c57 c87"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8201/database/schema/elaspic&amp;sa=D&amp;usg=AFQjCNGyw-G71jS4suPlssYRbZZzAQlmDQ">/database/schema/elaspic.rst</a></span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-1"><li class="c6"><span class="c18">Attachments also in a hierarchical folder structure, followed by &ldquo;attachment&rdquo;</span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-2 start"><li class="c10 c87 c57"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimweb1.ccbr.utoronto.ca:8201/database/schema/attachment/elaspic_schema.png&amp;sa=D&amp;usg=AFQjCNHbALTLrRsCyPOhZl_v2NC4_PDNcQ">/database/schema/attachment/elaspic_schema.png</a></span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-1"><li class="c6"><span class="c18">Getting everything to work required a pull request to waliki: </span></li></ul><ul class="c3 lst-kix_ejnvxy8b1e9w-2 start"><li class="c10 c87 c57"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://github.com/mgaitan/waliki/pull/108&amp;sa=D&amp;usg=AFQjCNHSRumd_mTJe6KIiwObzaUuXxYDUw">https://github.com/mgaitan/waliki/pull/108</a></span></li></ul><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Can use either method depending on preference (plus the </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://kimlab.org&amp;sa=D&amp;usg=AFQjCNGUT6LVjlYKl0lEIoVKPN_AIuqYnw">http://kimlab.org</a></span><span class="c18">&nbsp;</span><span class="c18">wiki for critical info)</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span>There was </span><span class="c22">a lot</span><span>&nbsp;of headache setting up a folder that by default has write permissions for every person in the </span><span class="c48">kimlab </span><span>group. In NFSv4, the </span><span class="c48">umask</span><span>&nbsp;takes precedence over ACL permissions, so you have to mount </span><span class="c48">kimlab </span><span>with NFSv3 to get the shared folders to work: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://marc.info/?t%3D123739823200003%26r%3D1%26w%3D2&amp;sa=D&amp;usg=AFQjCNGlSxN7zdJwIg9qn-qnZ3p0LKoonw">http://marc.info/?t=123739823200003&amp;r=1&amp;w=2</a></span><span>. </span></p><p class="c5"><span></span></p><p class="c10"><span>Remounted the ELASPIC archive folder (holding all the pdb files) with NFSv3, so the permissions are by default </span><span class="c22">rwX</span><span>&nbsp;for </span><span class="c22">user</span><span>&nbsp;and </span><span class="c22">group </span><span>for all structures computed by the webserver (before we were setting umask with each call, but it was still not working consistently).</span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c7">Updated ELASPIC Webserver</span></p><p class="c10"><span class="c18">Introduced a hack into the call to celery to make the webserver work with the updated pipeline:</span></p><ul class="c3 lst-kix_ouaz0d8j7xj-0 start"><li class="c1"><span class="c18">e.g.: </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://localhost:8202/result/dd6ecf/&amp;sa=D&amp;usg=AFQjCNESwUWbgOqT3ApceiHmRZLzM1m6ag">http://localhost:8202/result/dd6ecf/</a></span></li><li class="c1"><span class="c18">Will create a new virtualmachine and migrate </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org&amp;sa=D&amp;usg=AFQjCNFCE5OD6J-OR6H3_7urBXi5BjoxQA">http://elaspic.kimlab.org</a></span><span class="c18">&nbsp;after talking to Mark.</span></li></ul><p class="c10"><span class="c7">Plan</span></p><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_4h54ot51egiu-0 start"><li class="c1"><span class="c18">No more webserver / documentation stuff!!! </span></li><li class="c1"><span class="c18">Alanine scanning vs. dropoff score</span></li><li class="c1"><span class="c18">Drug combinations challenge (has not started yet&hellip;)</span></li><li class="c1"><span class="c18">Thesis</span></li></ul><p class="c5"><span class="c18"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-07-15</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c18">Set up local Django servers: </span></p><ul class="c3 lst-kix_a0ivi9b1qp8i-0 start"><li class="c1 c30"><span class="c18">ELASPIC webserver (running the updated ELASPIC pipeline): </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://192.168.6.167:8202/&amp;sa=D&amp;usg=AFQjCNHyjHtAdDoCoPnO4BKBD0wcXbcpTA">http://192.168.6.167:8202</a></span></li></ul><ul class="c3 lst-kix_a0ivi9b1qp8i-1 start"><li class="c6 c30"><span class="c18">Waiting for an </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://github.com/celery/celery/issues/1709&amp;sa=D&amp;usg=AFQjCNHVvaac3BzBerlAPzfkB1_qW7vorg">issue with celery</a></span><span class="c18">&nbsp;to be resolved before upgrading the server</span></li><li class="c6 c30"><span class="c18">Fix an error in a web-server css stylesheet which led to bad display in Firefox:</span></li></ul><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 346.50px; height: 188.36px;"><img alt="" src="images/image114.png" style="width: 565.71px; height: 334.16px; margin-left: -19.64px; margin-top: -55.17px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 361.20px; height: 193.50px;"><img alt="" src="images/image129.png" style="width: 581.80px; height: 344.72px; margin-left: -15.35px; margin-top: -56.91px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_xqkxylhicw49-0 start"><li class="c1"><span class="c18">Lical wiki: </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://192.168.6.167:8201&amp;sa=D&amp;usg=AFQjCNGAIok8i4NTXrHS7KYzslxHJMEC5A">http://192.168.6.167:8201</a></span></li><li class="c1"><span class="c18">Disorderome peptides vs. &Delta;&Delta;G and Provean score (next page)</span></li><li class="c1"><span class="c18">&Delta;&Delta;G and Provean score vs. Histone interaction data</span></li></ul><ul class="c3 lst-kix_xqkxylhicw49-1 start"><li class="c6"><span class="c18">working on the interaction (Y2H) data now...</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c18"></span></p><p class="c10"><a href="#" name="kix.orrw8pwmmbup"></a><span class="c22 c18">Interface properties vs. drop-off score</span></p><p class="c5 c12"><span class="c18"></span></p><p class="c10 c12"><span class="c18">No correlation between FoldX &Delta;&Delta;G and peptide z-score</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 346.00px;"><img alt="" src="images/image08.png" style="width: 720.00px; height: 872.00px; margin-left: 0.00px; margin-top: -172.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 328.00px;"><img alt="" src="images/image49.png" style="width: 720.00px; height: 816.00px; margin-left: 0.00px; margin-top: -164.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><hr style="page-break-before:always;display:none;"></p><p class="c10 c12"><span class="c18">Some correlation between predictor &Delta;&Delta;G and peptide z-score</span></p><p class="c10 c12"><span class="c18">(but it&rsquo;s mostly due to Provean)</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 170.00px;"><img alt="" src="images/image08.png" style="width: 720.00px; height: 872.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 172.00px;"><img alt="" src="images/image08.png" style="width: 720.00px; height: 872.00px; margin-left: 0.00px; margin-top: -700.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 165.00px;"><img alt="" src="images/image49.png" style="width: 720.00px; height: 816.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 165.00px;"><img alt="" src="images/image49.png" style="width: 720.00px; height: 816.00px; margin-left: 0.00px; margin-top: -651.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><a href="#" name="id.pqo49xh38dgp"></a><p class="c10 c12"><span class="c38">ELASPIC features vs. Histone interaction data </span><span class="c7">(CORE)</span></p><p class="c5 c15"><span class="c38"></span></p><p class="c5 c15"><span class="c38"></span></p><p class="c10 c15"><span class="c18">number of unique uniprots / mutations: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2,963<br>number of unique mutations inside domains:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2,009<br>number of unique mutations with ddg scores: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1,855</span></p><p class="c5 c15"><span class="c38"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 600.00px;"><img alt="" src="images/image75.png" style="width: 720.00px; height: 600.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c15"><span class="c18"></span></p><p class="c5 c15"><span class="c18"></span></p><p class="c10 c12"><span class="c38">Our FoldX scores make sense...</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 469.70px; height: 335.50px;"><img alt="" src="images/image46.png" style="width: 469.70px; height: 335.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><p class="c5 c12"><span class="c7"></span></p><p class="c10 c12"><span class="c38">Histone interaction data is predictive of disease...</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 541.00px; height: 382.00px;"><img alt="" src="images/image125.png" style="width: 541.00px; height: 382.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-07-09</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c22 c18">Progress</span></p><ul class="c3 lst-kix_eu4b93xn98yk-0 start"><li class="c1"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTZTdmQ1djUmRMcms&amp;sa=D&amp;usg=AFQjCNEEHtXtqq4FYyxx-6zKRkPVOKoFBA">Lab meeting</a></span></li><li class="c1"><span>70% of alanine scanning</span></li><li class="c1"><span>Looking at overlap between ClinVar mutations from ClinVar and Ensembl databases</span></li></ul><p class="c5"><span></span></p><p class="c10"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4h47981ce058-0 start"><li class="c1"><span>Scores for ALL disorderome peptides</span></li><li class="c1"><span>Thesis</span></li><li class="c1"><span>Dream Challenge</span></li></ul><p class="c5"><span></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 810.67px;"><img alt="" src="images/image119.png" style="width: 720.00px; height: 810.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 810.67px;"><img alt="" src="images/image120.png" style="width: 720.00px; height: 810.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-06-24</span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span class="c18">Revised drafts for the paper are in the </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTfk9iTEhMNi0yWE9OOUR6V1U4RjhmQlpIQ0ljY0dUdGZ2V1FIcmNmQmdqd0U%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNEUrCVeO59p1TqpAao3rDpIqwMK5g">webserver</a></span><span class="c18">&nbsp;subfolder</span></p><ul class="c3 lst-kix_k0c3ozmp0pnp-0 start"><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTcmhMZ0Zra1l1YlU%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNHXZoOMv7C-RW5CxChkLnE4SUUJhw">letter to the editor</a></span></li><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTWmFRZlZPQ1dqX3M%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNG4h61RO1-FKbgfjTUovn1M7hM85g">paper</a></span></li><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTZkhBOHdXbUFIdDQ%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNFEZZV3PEZu8vT1WIJFtyDmvlUehw">supplemental material</a></span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Drug combination prediction challenge: </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://www.synapse.org/%23!Synapse:syn4231880/wiki/&amp;sa=D&amp;usg=AFQjCNEwHZkCTxVCL_GiUUcZ8v-EYAPcYw">https://www.synapse.org/#!Synapse:syn4231880/wiki/</a></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 153.33px;"><img alt="" src="images/image25.png" style="width: 720.00px; height: 153.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span class="c33 c22">Registration Open: </span><span class="c33">25 June 2015</span></p><p class="c10"><span class="c33 c22">Launch: </span><span class="c33">July 2015 (Exact date to be announced)</span></p><p class="c10"><span class="c33 c22">Close: </span><span class="c33">September 2015 (Exact date to be announced)</span></p><p class="c10"><span class="c33">As the basis of the Challenge, AstraZeneca is releasing ~30k experimentally tested drug combinations measuring cell viability over 140 drugs and 120 cancer cell lines (primarily colon, lung, and breast), and monotherapy drug response data for each drug and cell line. Moreover, in coordination with the </span><span class="c28 c35"><a class="c9" href="https://www.google.com/url?q=http://www.cancerrxgene.org/&amp;sa=D&amp;usg=AFQjCNG4dxfVLkANwSmlETzUGZCeSiWpxA">Genomics of Drug Sensitivity in Cancer </a></span><span class="c33">and </span><span class="c28 c35"><a class="c9" href="https://www.google.com/url?q=http://cancer.sanger.ac.uk/cosmic&amp;sa=D&amp;usg=AFQjCNGsSFnLj-WnvnkaOCnV9F4sfV7sSw">COSMIC</a></span><span class="c33">&nbsp;teams at the Sanger Institute, genomic data including gene expression, somatic mutation, copy-number alterations, and methylation data will be released into the public domain and made accessible for Challenge participants through the Challenge data repository. </span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-06-17</span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c10 c12"><span class="c22">Provean vs. peptide z-score</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 588.15px; height: 354.50px;"><img alt="" src="images/image118.png" style="width: 588.15px; height: 354.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10 c12"><span class="c22">FoldX vs. peptide z-score</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 579.50px; height: 346.56px;"><img alt="" src="images/image87.png" style="width: 579.50px; height: 346.56px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10 c12"><span class="c22">Metapredictor vs. peptide z-score</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 581.50px; height: 347.44px;"><img alt="" src="images/image15.png" style="width: 581.50px; height: 347.44px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Alanine scanning of all human PPIs</span></p><ul class="c3 lst-kix_oli7yc1w3gpg-0 start"><li class="c1"><span>The structure archive is &gt; 1 million files, so even without any calculations it breaks scinet.</span></li><li class="c1"><span>Can&rsquo;t use tar because takes a long time to decode single files</span></li><li class="c1"><span>Trying p7zip&hellip; seems to work ok</span></li></ul><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span class="c22">Documentation for the lab</span></p><ul class="c3 lst-kix_ns4c34lwftpb-0 start"><li class="c1"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://192.168.6.167/database&amp;sa=D&amp;usg=AFQjCNGRK7xzcYC0VMg3LVVikwxLaI53CQ">http://192.168.6.167/database</a></span></li><li class="c1"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://192.168.6.167/notes&amp;sa=D&amp;usg=AFQjCNFTvcq_7PrXWkWga1QXzoj9Ut9a0g">http://192.168.6.167/notes</a></span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-06-10</span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span>Disorderome</span></p><ul class="c3 lst-kix_oli7yc1w3gpg-0"><li class="c1"><span>All done, but haven&rsquo;t done analysis yet.</span></li></ul><ul class="c3 lst-kix_54trtu6df9f8-0 start"><li class="c1"><span>Transferring data back to </span><span class="c48">kimstg</span></li></ul><ul class="c3 lst-kix_54trtu6df9f8-1 start"><li class="c6"><span>Divide tar into many small files -&gt; works much better. I guess scinet dev nodes terminate long-running processes, so you have to split up all your tasks into smaller chunks.</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Alanine scanning of human uniprot</span></p><ul class="c3 lst-kix_f7u4w07k18eh-0 start"><li class="c1"><span>Running on Scinet and Guillimin </span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Ensembl mutations</span></p><ul class="c3 lst-kix_zvvafajkk2a-0 start"><li class="c1"><span>Running on Beagle and Banting</span></li></ul><p class="c5"><span></span></p><hr><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Making PBS / SGE scripts</span></p><p class="c5"><span></span></p><p class="c10"><span>Beagle and Banting </span></p><ul class="c3 lst-kix_b6qjs7hmfdgs-0 start"><li class="c1"><span>Mark noticed that my jobs were not maxing out the CPU</span></li></ul><ul class="c3 lst-kix_b6qjs7hmfdgs-1 start"><li class="c6"><span>Turns out when I submit jobs on both beagle and banting with one job per mutation, the database can&rsquo;t keep up with the queries and becomes the limiting factor</span></li><li class="c6"><span>Wrote a SGE script that runs mutations in chunks using a separate SQLite database per node. This is using all the resources.</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Scinet</span></p><ul class="c3 lst-kix_xwc7ibkfiucl-0 start"><li class="c1"><span>Overshot the 1 million files mark. </span></li><li class="c1"><span>Modifying code to keep the old and new files separate, so that I can easily compress all the new files into a single tar.</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Guillimin</span></p><ul class="c3 lst-kix_25j83v4xa4ig-0 start"><li class="c1"><span>Got an email saying that I am maxing out the /tmp folder on some nodes. </span></li><li class="c1 c19"><span></span></li></ul><p class="c5"><span></span></p><p class="c10"><span>MP2</span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-27</span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22 c18"></span></p><ul class="c3 lst-kix_wxhgjniia6a-0 start"><li class="c1"><span>Disorderome</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1 start"><li class="c6"><span>78,058 mutations calculated, 17,760 remaining (19%, same as last week)</span></li><li class="c6"><span>submitting jobs now (modified ELASPIC so that I don&rsquo;t recalculate domain pairs that have already been calculated)</span></li><li class="c6"><span>need a good way to transfer files back (Globus times out and tar-ing everything takes hours and I exceed time limit on dev nodes)</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-2 start"><li class="c10 c87 c57"><span>will probably create a set of files containing the filenames of everything that has to be transferred back</span></li></ul><p class="c5 c21 c15"><span></span></p><p class="c5 c21 c15"><span></span></p><p class="c10 c21 c15"><span>Disease mutations from different sources (restricted to only those mutations that fall inside domains)</span></p><a href="#" name="8a058abaa1a6312405d378d2f887e9407f04d3ec"></a><a href="#" name="2"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c15 c21"><span class="c22 c40">variation_set_id</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c22 c40">name</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c22 c40">description</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c22 c40">number of proteins</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">9</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">All phenotype-associated variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants that have been associated with a phen...</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">10</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">OMIM phenotype variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variations linked to entries in the Online Men...</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">2494</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">11</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Uniprot phenotype variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variations with phenotype annotations provided...</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">1195</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">12</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">NHGRI catalog phenotype variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants associated with phenotype data from t...</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">269</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">13</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">PhenCode</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants from the PhenCode Project</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">634</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">17</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">COSMIC phenotype variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Phenotype annotations of somatic mutations fou...</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">31003</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">18</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">HGMD-PUBLIC variants</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants annotated by HGMD</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">0</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">62</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">All ClinVar</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants with ClinVar annotation</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">7493</span></p></td></tr><tr class="c75"><td class="c78" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">64</span></p></td><td class="c83" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">clinically associated</span></p></td><td class="c69" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">Variants described by ClinVar as being probabl..</span></p></td><td class="c73" colspan="1" rowspan="1"><p class="c10 c21 c24"><span class="c31">3553</span></p></td></tr></tbody></table><p class="c5 c21 c15"><span></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-20</span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22 c18"></span></p><ul class="c3 lst-kix_wxhgjniia6a-0"><li class="c1"><span>Disorderome</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1 start"><li class="c6"><span>78,058 mutations calculated, 17,760 remaining (19%)</span></li><li class="c6"><span>10 to 20 domain mutations per hour per node</span></li><li class="c6"><span>Add option to evaluate mutation in a specific domain (or domain-domain pair) rather than uniprot</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-2 start"><li class="c10 c87 c57"><span>End up calculating duplicates if mutations has been evaluated for one domain but not another</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1"><li class="c6 c19"><span></span></li><li class="c6"><span>Getting strange errors copying data back; need to email scinet about it</span></li></ul><p class="c5"><span></span></p><ul class="c3 lst-kix_wxhgjniia6a-0"><li class="c1 c21 c15"><span>ELASPIC (rerunning for all missing domains (&lt;200) and domain-domain interactions (~2000)</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1 start"><li class="c6 c21 c15"><span>POPS was failing when there were steric clashes in the model, making the whole pipeline crash</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-2 start"><li class="c10 c21 c87 c57 c15"><span>Replaced POPS values with NaNs in those cases</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1"><li class="c6 c21 c15"><span>Biopython is crashing for some PDB structures</span></li><li class="c6 c21 c15"><span>Need to do something about combinatorial explosion of domain-domain interactions in some cases</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-15</span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span class="c22 c18">News</span></p><p class="c5"><span class="c22 c18"></span></p><ul class="c3 lst-kix_wxhgjniia6a-0"><li class="c1"><span>Daniel says he was too busy to write the cover letter. Hopefully this week&hellip; (not really my job&hellip;)</span></li></ul><p class="c5"><span></span></p><ul class="c3 lst-kix_wxhgjniia6a-0"><li class="c1"><span>Compiled all ELASPIC dependencies on a CentOS 5.11 virtual machine using conda, and uploaded the binaries to binstar (</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://binstar.org/ostrokach&amp;sa=D&amp;usg=AFQjCNE6d9rdSNmYmWQDw4edlPD2aq7Nzg">https://binstar.org/ostrokach</a></span><span>).</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1 start"><li class="c6"><span>Anyone can now install ELASPIC by running: &nbsp;`</span><span class="c59">conda install elaspic`</span><span>, and setting a </span><span class="c59">MODELLER_LICENSE</span><span>&nbsp;environmental variable.</span></li><li class="c6"><span>Tested this on SciNet and it works. </span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-2 start"><li class="c10 c87 c57"><span>Some clusters run very old versions of linux, and these binaries should still work without recompiling...</span></li></ul><ul class="c3 lst-kix_wxhgjniia6a-1"><li class="c6"><span>If I can set up a Mac virtual machine, should be possible to compile this to work on a Mac (so Michael does not have to compile all the stuff himself).</span></li></ul><p class="c5"><span></span></p><ul class="c3 lst-kix_s27gvee6ygqb-0 start"><li class="c1"><span>Sent the new instructions and a link to modelled domain-domain interactions to Michael Dong&hellip;</span></li></ul><ul class="c3 lst-kix_s27gvee6ygqb-1 start"><li class="c6"><span>But this might not have been what he was looking for? I mistakenly thought he was from Regis Pommes&rsquo; lab, not Nicholoas Provart&hellip; Was he looking for raw pdb structures?</span></li></ul><p class="c5"><span></span></p><ul class="c3 lst-kix_2w2qcvvb39hd-0 start"><li class="c1"><span>10794 out of 59288 alanine scanning mutations done</span></li></ul><ul class="c3 lst-kix_2w2qcvvb39hd-1 start"><li class="c6"><span>Still have to see whether the jobs simply timed out or if there&rsquo;s something wrong with how I made this alanine scanning set.</span></li><li class="c6"><span>I think the 10 k jobs finished overnight, so pretty quick..</span></li><li class="c6"><span>Do alanine scanning for everything or just the disorderome?</span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22 c18">Summary of all previous to-dos</span></p><p class="c5"><span class="c22 c18"></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0 start" start="1"><li class="c1"><span>Bug in ELASPIC when the structural template contains phosphorylated residues</span></li></ol><ul class="c3 lst-kix_flatlsyq6jr0-1 start"><li class="c6"><span>Carles found CHARMM phosphoserine and phosphotyrosine force fields (patches...) that I will try adding to modeller.</span></li><li class="c6"><span>Remove the phosphate group and add it later if necessary using FoldX?</span></li><li class="c6"><span>Treat phosphorylated residues as BLK residues (need to alter a bit the alignment pipeline)?</span></li></ul><p class="c5"><span></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0" start="2"><li class="c1"><span>There are likely other bugs as well. About 1 thousand domains and few thousand domain pairs don&rsquo;t have models because of bad templates but also other errors.</span></li></ol><ul class="c3 lst-kix_flatlsyq6jr0-1 start"><li class="c6"><span>Add more specificity in the error messages, storing errors that are OK so I don&rsquo;t have to rerun.</span></li><li class="c6"><span>For domain pairs, some have a combinatorial explosion. Have to think of a way to fix this.</span></li></ul><p class="c5"><span></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0" start="3"><li class="c1"><span>Revisit the training set, particularly the interaction training set, and put the figures in the thesis and online documentation.</span></li></ol><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 410.50px; height: 287.74px;"><img alt="" src="images/image117.png" style="width: 410.50px; height: 287.74px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c21 c15"><span></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0" start="4"><li class="c1"><span>Analysing disease mutations</span></li></ol><ul class="c3 lst-kix_flatlsyq6jr0-1 start"><li class="c6"><span>Create a new COSMIC driver / passenger dataset with proteins divided based on mutation density instead of mutation than count. Can keep the old Uniprot disease / polymorphism dataset.</span></li><li class="c6"><span>&#8710;&#8710;G predictions vs. Provean</span></li><li class="c6"><span>&#8710;&#8710;G predictions vs. Disease (high negative, mid negative, neutral, mid positive, high positive)</span></li><li class="c6"><span>(-0.5 &lt; ddG &lt; 0.5 for neutral)</span></li><li class="c6"><span>Presumably &#8710;&#8710;G predictions should be particularly useful for interface mutations, since surface amino acids are much less conserved. Interface mutations are also over-represented in disease.</span></li></ul><p class="c5"><span></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0" start="5"><li class="c1"><span>Train an interface yes / no classifier using yeast two hybrid studies and other data.</span></li></ol><p class="c5"><span></span></p><ol class="c3 lst-kix_flatlsyq6jr0-0" start="6"><li class="c1"><span>Alter the webserver so that it can submit jobs to beagle rather than running them locally?</span></li></ol><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c22"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Thesis topics (2015-05-16)</span></p><p class="c5"><span class="c7"></span></p><ul class="c3 lst-kix_i0q09d67i790-0 start"><li class="c1"><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://www.sciencedirect.com/science/article/pii/S0092867415004304%23&amp;sa=D&amp;usg=AFQjCNGdRb_290tYiB9FJ-Z8VR1mMsmt-w">Widespread Macromolecular Interaction Perturbations in Human Genetic Disorders</a></span></li></ul><ul class="c3 lst-kix_i0q09d67i790-1 start"><li class="c6"><span class="c18">&ldquo;Majority of missense disease mutants do not dramatically impact protein structure or folding&rdquo; </span></li><li class="c6"><span class="c18">&ldquo;we obtained interaction profiles for 460 mutant proteins and their 220 WT counterparts and found 521 perturbed interactions out of 1,316 PPIs.&rdquo;</span></li><li class="c6"><span class="c18">&ldquo;mutations perturbing PPIs are preferentially located within structural domains&rdquo;</span></li><li class="c6"><span class="c18">Output: Predict all interaction-perturbing mutations in OMIM / COSMIC / ClinVar&hellip; (just a probability instead of ddG, but way more training data&hellip;)</span></li></ul><p class="c5"><span class="c22 c18"></span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 243.50px; height: 243.50px;"><img alt="" src="images/image79.png" style="width: 243.50px; height: 243.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_n2tc9z47p0ha-0 start"><li class="c1"><span class="c18">Collect more mutation data</span></li></ul><ul class="c3 lst-kix_n2tc9z47p0ha-1 start"><li class="c6"><span class="c18">I didn&rsquo;t get many results when I tried this previously, but I was just querying Pubmed and google with simple keywords.</span></li><li class="c6"><span class="c18">Uniprot sometimes annotates mutations with ddG effects (e.g. </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://www.uniprot.org/uniprot/O00330%23pathology_and_biotech&amp;sa=D&amp;usg=AFQjCNEf79EtYQtfvwW6uqZ7PExcAcmgZQ">Strongly decreased DLD binding</a></span><span class="c18">). Could also mine Pubmed titles and abstracts downloaded by Alex. Often a mutation results in &ldquo;no detectable binding&rdquo;, but maybe we could still use this?.. </span></li><li class="c6"><span class="c18">Look up other methods that people use to calculate ddG (ELISA, TI, chaperone binding?). Can simplify it into a classification problem.</span></li></ul><p class="c5 c53"><span class="c18"></span></p><ul class="c3 lst-kix_mi7vyzrjrpd8-0 start"><li class="c1"><span class="c18">Use Omnia and </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://www.omnia.md/blog/2014/11/6/how-to-train-your-force-field&amp;sa=D&amp;usg=AFQjCNFb_W_-ZrkEzcV4hGC0_IEas_kTfw">ForceBalancer</a></span><span class="c18">&nbsp;to train our own forcefield and use it instead of FoldX. This way we could correctly cross-validate our method, and compare it to the </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://github.com/Kortemme-Lab/ddg&amp;sa=D&amp;usg=AFQjCNHR4wWn-YEZo0FgbiQJuzDZYuwgng">rosetta benchmark</a></span><span class="c18">. Also, FoldX is closed source, and the authors can complain about us using it in a webserver (same with modeller, but no way to change that&hellip;).</span></li></ul><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_mi7vyzrjrpd8-0"><li class="c1"><span class="c18">Each domain should be annotated with a PSI-BLAST(</span><span class="c13">hmmscan / jackhmmer</span><span class="c18">) multiple sequence alignment</span></li></ul><ul class="c3 lst-kix_mi7vyzrjrpd8-1 start"><li class="c6"><span class="c18">Measure conservation / covariation of residues that are in close proximity to the mutated amino acid.</span></li><li class="c6"><span class="c18">Measure covariation between </span><span class="c48 c18">surface</span><span class="c18">&nbsp;amino acids on different chains (using </span><span class="c13 c16"><a class="c9" href="https://www.google.com/url?q=https://github.com/soedinglab/CCMpred&amp;sa=D&amp;usg=AFQjCNGAUCpVttup3S3-YUeMpsZUFmyz5A">CCMpred</a></span><span class="c18">, which can calculate the pseudo-likelihood measure).</span></li></ul><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_mi7vyzrjrpd8-0"><li class="c1 c21 c15"><span class="c18">...</span></li></ul><p class="c5 c53"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-15</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span>disorderome: </span></p><p class="c10"><span>3756 domain pairs</span></p><p class="c10"><span>3013 uniprot pairs</span></p><p class="c10"><span>1224 uniprot pairs where all splice variants that have a corresponding canonical version have been removed</span></p><p class="c10"><span>59288 mutations</span></p><p class="c5"><span></span></p><p class="c10"><span>Talked to Michael about testing out elaspic.</span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-15</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c22">Progress</span></p><ul class="c3 lst-kix_vxbaz02t98k3-0 start"><li class="c1"><span>Manuscript: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTMnNqbXNsTTVpTGM%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNFLv96GBEDYrwBk83RV82dqb7Wa1g">pdf</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTLWpRWWgyakNqVWc%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNEN2Fxt5vCnYleD9BOYV9dARaw6RQ">docx</a></span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span class="c17">Daniel was busy last week and had a presentation on Monday. He said he&rsquo;ll changing the figure in the manuscript and add more detail about to the case study ASAP.</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Supplemental: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTTVJya3pUZzJfeU0%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNGWv7YwHEesvdyfKDoOuxUQsjwQlg">pdf</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTLWxLYmRzWm1rMG8%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNE0igfYdJmucogPrOJyaveGs5Rbbw">docx</a></span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span>Good to go. Just need more detail about the case study.</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Description of every column in every table added to the </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://elaspic.readthedocs.org/en/latest/database_schema/database_schema.html&amp;sa=D&amp;usg=AFQjCNFYZ-1p11AvX4Zax9FT9vGIT901Vg">documentation</a></span><span>.</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span>(still need to proof-read this though)</span></li><li class="c6"><span>Should I add this to the supplemental.</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span class="c22">Backing up external files</span></p><ul class="c3 lst-kix_jm1l4y2hlnt3-0 start"><li class="c1"><span>Compressing all external files generates archives that are too large</span></li></ul><ul class="c3 lst-kix_jm1l4y2hlnt3-1 start"><li class="c6"><span>We don&rsquo;t need to provide FoldX wildtype and mutant structures.</span></li><li class="c6"><span>We don&rsquo;t need to provide pickled backup of the information for each model and mutation.</span></li></ul><ul class="c3 lst-kix_jm1l4y2hlnt3-0"><li class="c1"><span>Couldn&rsquo;t find an easy way to compress a subset of the files in our folder structure</span></li></ul><ul class="c3 lst-kix_jm1l4y2hlnt3-1 start"><li class="c6"><span>We want to include files in: </span></li></ul><ul class="c3 lst-kix_jm1l4y2hlnt3-2 start"><li class="c10 c87 c57"><span class="c59">human/up/up/up/*</span></li><li class="c10 c87 c57"><span class="c59">human/up/up/up/dom/*</span></li><li class="c10 c87 c57"><span class="c59">human/up/up/up/dom/dom/*</span></li></ul><p class="c10 c50"><span>but not in any other directory .</span></p><ul class="c3 lst-kix_jm1l4y2hlnt3-1"><li class="c6"><span>Before was using: </span><span class="c59">tar czf human_structures.tar.gz human</span></li><li class="c6"><span>Using: </span><span class="c59">tar czf human_structures.tar.gz human/up/up/up/*</span><span>&nbsp;and similar variants generates an error saying </span><span class="c59">argument list too large</span><span>.</span></li><li class="c6"><span>Same error when using the </span><span class="c59">--excl</span><span class="c59">u</span><span class="c59">de</span><span>&nbsp;option.</span></li></ul><ul class="c3 lst-kix_jm1l4y2hlnt3-0"><li class="c1"><span>Ended up using information in the database to create lists of files that need to the archive, and creating an archive with only those files (e.g. </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/current_release/Homo_sapiens/provean/&amp;sa=D&amp;usg=AFQjCNH7suDe9BzhKlUGfOXwx_CfYGBfOA">provean</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/current_release/Homo_sapiens/uniprot_domain/&amp;sa=D&amp;usg=AFQjCNHNGEko2Zy9MFJkzMKuSfLkYTq3bw">uniprot_domain</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/current_release/Homo_sapiens/uniprot_domain_pair/&amp;sa=D&amp;usg=AFQjCNENxhaSZHMw35vHJiCQx5v4Tank_Q">uniprot_domain_pair</a></span><span>).</span></li></ul><p class="c5 c53"><span></span></p><p class="c5 c53"><span></span></p><p class="c5"><span class="c22 c18"></span></p><p class="c5"><span class="c22 c18"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c22 c18"></span></p><p class="c10"><span class="c22 c18">Alanine scanning of interface mutations</span></p><ul class="c3 lst-kix_7tkvl1c78ae3-0 start"><li class="c1"><span class="c18">We have 373,361 mutations in 6,296 proteins </span></li></ul><ul class="c3 lst-kix_7tkvl1c78ae3-1 start"><li class="c6"><span class="c18">Have templates for another ~1,200 proteins, so could extend this dataset in the future&hellip;</span></li><li class="c6"><span class="c18">Many more templates for splice variants and a few more for TrEMBL.</span></li></ul><ul class="c3 lst-kix_7tkvl1c78ae3-0"><li class="c1"><span class="c18">Have models and provean supporting sets for all of these proteins</span></li><li class="c1"><span class="c18">Should pre-calculate Provean scores before sending in to SciNet...</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 702.38px; height: 425.50px;"><img alt="" src="images/image95.png" style="width: 702.38px; height: 425.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-05-08</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c22 c18">Progress:</span></p><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Manuscript: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTMnNqbXNsTTVpTGM%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNFLv96GBEDYrwBk83RV82dqb7Wa1g">pdf</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTLWpRWWgyakNqVWc%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNEN2Fxt5vCnYleD9BOYV9dARaw6RQ">docx</a></span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span class="c17">I think it&#39;s good. My only concern is that the case study at the end may be a bit vague, so it might be good to add more detail about it in the supplemental. Daniel said he&rsquo;ll look into it today. I&rsquo;ll fill in the details that require calculating mutations, etc.</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Supplemental: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTSV9KX2RBYkM5ek0%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNFym9iNhMRMBdvP4Rbb2vLHxqF90w">pdf</a></span><span>, </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTODFUSVJNeGNxVzQ%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNGY9bl1lK2nRZvUz4SkV-KY63IYRg">docx</a></span></li><li class="c1"><span>Added explicit links to bitbucket and readthedocs on the </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/help/&amp;sa=D&amp;usg=AFQjCNGolfPpNhaAkwneudQ42jYTFuInOA">ELASPIC website</a></span><span>.</span></li><li class="c1"><span>Bug in ELASPIC when the structural template contains phosphorylated residues</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span>Carles found CHARMM phosphoserine and phosphotyrosine force fields (patches...) that I will try adding to modeller.</span></li><li class="c6"><span>Remove the phosphate group and add it later if necessary using FoldX?</span></li><li class="c6"><span>Treat phosphorylated residues as BLK residues (need to alter a bit the alignment pipeline)?</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-04-01</span></p><p class="c5"><span class="c7"></span></p><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Daniel selected a test case for the paper </span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTMjE2b2RXWTJmQ0k%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNH8B9d2Tvbrd455dMDxpuadMD-55g">https://drive.google.com/open?id=0B1FXQpQb4bbTMjE2b2RXWTJmQ0k&amp;authuser=1</a></span></li><li class="c6"><span>EP300/GBP interacting with GRIP1/NCOA2.</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1 c21 c15"><span>Guess I&rsquo;ll spend the tomorrow (and Friday) in the library.</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-03-25</span></p><p class="c10"><span class="c16 c18">Webserver paper:</span></p><ul class="c3 lst-kix_vxbaz02t98k3-0"><li class="c1"><span>Daniel wants to submit the paper to Bioinformatics Applications Note (</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTV3JqdWF3ZUpIRFE%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNH1qwg6LDl6_7wJDPhAunkbFjLF2w">Draft</a></span><span>)</span></li><li class="c1"><span>We need an example to put in the paper...</span></li></ul><ul class="c3 lst-kix_vxbaz02t98k3-1 start"><li class="c6"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D1-Q49z56i6HCAqJ7aFGySad-71IP5D__UuLoxBZ0g9qk%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNGPeSC80Swdhj8HFEqZ_3JnGbBxVw">Cosmic cancer mutations</a></span></li><li class="c6"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D1--Hrwor_87S5XL7I8JNhtucsVDSWH-zECcEo1KVyNh0%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNGo0aK0DazVPLYApnYVKTWLCciYgg">ClinVar pathogenic mutations</a></span><span>: </span></li><li class="c6"><span>Maybe P53, Cyclin 2A, MDM2, etc.?</span></li></ul><p class="c5 c53"><span class="c38"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c16 c18">Previous to-dos:</span></p><ol class="c3 lst-kix_4tgbgh9bb7g-0 start" start="1"><li class="c1"><span>Create a script that saves all required ELASPIC tables from the database to text files:</span></li></ol><ul class="c3 lst-kix_4tgbgh9bb7g-1 start"><li class="c6"><span>Created a script that takes a snapshot of the database and makes the </span><span class="c22">tables</span><span>&nbsp;and </span><span class="c22">models</span><span>&nbsp;for different organisms available from the ELASPIC webserver downloads page: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/&amp;sa=D&amp;usg=AFQjCNHDWGcCb76UNbdNBZ1riRON_KNIzg">http://elaspic.kimlab.org/static/download/</a></span><span>.</span></li></ul><ol class="c3 lst-kix_4tgbgh9bb7g-0" start="2"><li class="c1"><span>Create a script that can create all the required ELASPIC tables and fill them with data from text files:</span></li></ol><ul class="c3 lst-kix_4tgbgh9bb7g-1 start"><li class="c6"><span>Moved database username and password from the source code into an encrypted configuration file.</span></li><li class="c6"><span>Made it possible to specify the database type from within the configuration file.</span></li><li class="c6"><span>Added test cases that create a </span><span class="c48">MySQL</span><span>, </span><span class="c48">PostgreSQL</span><span>, or </span><span class="c48">SQLite</span><span>&nbsp;database, uploads to it a small subset of the data, and calculate a core and an interface mutation.</span></li><li class="c6"><span>Can now run ELASPIC on Scinet?</span></li></ul><ol class="c3 lst-kix_4tgbgh9bb7g-0" start="3"><li class="c1"><span>Update documentation to include the ELASPIC database schema:</span></li></ol><ul class="c3 lst-kix_4tgbgh9bb7g-1 start"><li class="c6"><span>Updated documentation can be found here: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.readthedocs.org&amp;sa=D&amp;usg=AFQjCNFhMMOcGPngkbnKEGfJhgRLZKZgxA">http://elaspic.readthedocs.org</a></span><span>. </span></li><li class="c6"><span>Description of the database schema is here: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.readthedocs.org/en/latest/database_schema.html&amp;sa=D&amp;usg=AFQjCNE_JLgR6u4pd9bcZ5ccjdo9DLTeTQ">http://elaspic.readthedocs.org/en/latest/database_schema.html</a></span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>To do...&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c10"><span>Analysing disease mutations (still to do):</span></p><ul class="c3 lst-kix_dkq97x3e7jfk-0 start"><li class="c1"><span>&#8710;&#8710;G predictions vs. Protherm</span></li><li class="c1"><span>&#8710;&#8710;G predictions vs. Disease (high negative, mid negative, neutral, mid positive, high positive)</span></li></ul><ul class="c3 lst-kix_dkq97x3e7jfk-1 start"><li class="c6"><span>(-0.5 &lt; ddG &lt; 0.5 for neutral OK?)</span></li></ul><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-03-11</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span>ELASPIC</span><span>:</span></p><ul class="c3 lst-kix_nr7rbh2hfyf0-0 start"><li class="c1"><span>Add final </span><span>classifiers</span><span>&nbsp;to the source code (</span><span class="c22">done</span><span>)</span></li><li class="c1"><span>Fix python 2 / 3 compatibility issues (</span><span class="c22">done</span><span>)</span></li><li class="c1"><span>Write the manuscript (in progress)</span></li></ul><ul class="c3 lst-kix_nr7rbh2hfyf0-1 start"><li class="c6"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTdmZVY1U5WTZMMnM&amp;sa=D&amp;usg=AFQjCNHeSczNqsi_VRHTmz4xyyoiRVRuSg">https://drive.google.com/open?id=0B1FXQpQb4bbTdmZVY1U5WTZMMnM</a></span></li></ul><ul class="c3 lst-kix_nr7rbh2hfyf0-0"><li class="c1"><span>Update documentation to include the ELASPIC database schema</span></li><li class="c1"><span>Create a script that saves all required ELASPIC tables from the database to a text file</span></li><li class="c1"><span>Create script that can create all the required ELASPIC tables and fill them with data from text files</span></li></ul><p class="c10"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c10"><span>Analysing disease mutations (still to do):</span></p><ul class="c3 lst-kix_dkq97x3e7jfk-0"><li class="c1"><span>&#8710;&#8710;G predictions vs. Protherm</span></li><li class="c1"><span>&#8710;&#8710;G predictions vs. Disease (high negative, mid negative, neutral, mid positive, high positive)</span></li></ul><ul class="c3 lst-kix_dkq97x3e7jfk-1 start"><li class="c6"><span>(-0.5 &lt; ddG &lt; 0.5 for neutral OK?)</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>SciNet:</span></p><ul class="c3 lst-kix_o9ptu63d1d74-0 start"><li class="c1"><span>You must enter the Compute Canada Role Identifier (CCRI) of your sponsor or supervisor. They will be asked to confirm your role. Your sponsor can find their CCRI on their login page at https://ccdb.computecanada.ca/me. A CCRI is an identifier of the form </span><span class="c22">abc-123-01</span><span>.</span></li></ul><p class="c5"><span></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><hr style="page-break-before:always;display:none;"><h2 class="c10 c77 c108"><a name="h.ev2k4j4jvnyq"></a><span class="c95 c18">Provean predicts disease states better than FoldX</span></h2><p class="c10"><span>Provean score distribution stays roughly the same for core mutations with different </span><span class="c48">solvent accessibilities</span><span>&nbsp;and for interface mutations with different </span><span class="c48">minimum contact distance</span><span>&nbsp;cutoffs. FoldX </span><span class="c116">&Delta;&Delta;</span><span class="c48 c116">G</span><span>&nbsp;scores show much larger differences between disease and polymorphism when the </span><span class="c48">solvent accessibility</span><span>&nbsp;is low for core mutations or when the </span><span class="c48">minimum contact distance</span><span>&nbsp;is low for interface mutations.</span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span>Provean score distributions for all Uniprot disease / polymorphism mutations.<br></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 317.33px;"><img alt="" src="images/image83.png" style="width: 720.00px; height: 317.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c10"><span>Provean score distributions for Uniprot disease / polymorphism mutations </span></p><p class="c10"><span>(</span><span class="c48">core</span><span>&nbsp;mutations with </span><span class="c48">solvent_accessibility</span><span>&nbsp;&lt; 20 and </span><span class="c48">interface</span><span>&nbsp;mutations with </span><span class="c48">contact_distance</span><span>&nbsp;&lt; 4 A)</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 317.33px;"><img alt="" src="images/image13.png" style="width: 720.00px; height: 317.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span>FoldX &Delta;&Delta;G distributions all Uniprot disease / polymorphism mutations.<br></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 317.33px;"><img alt="" src="images/image38.png" style="width: 720.00px; height: 317.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Foldx &Delta;&Delta;G distributions for Uniprot disease / polymorphism </span></p><p class="c10"><span>(</span><span class="c48">core</span><span>&nbsp;mutations with </span><span class="c48">solvent_accessibility</span><span>&nbsp;&lt; 20 and </span><span class="c48">interface</span><span>&nbsp;mutations with </span><span class="c48">contact_distance</span><span>&nbsp;&lt; 4 A)</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 317.33px;"><img alt="" src="images/image130.png" style="width: 720.00px; height: 317.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c5"><span class="c22"></span></p><p class="c10"><span>&Delta;&Delta;G predictions by FoldX and ELASPIC &nbsp;for Uniprot Disease / Polymorphism (updated)</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 828.50px;"><img alt="" src="images/image22.png" style="width: 720.00px; height: 828.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-03-04</span></p><p class="c5"><span class="c7"></span></p><ul class="c3 lst-kix_yk52z9n95trk-0 start"><li class="c1"><span class="c18">15,401 domain-domain interactions added by using protein-protein interaction data from the </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://dx.doi.org/10.1016/j.cell.2014.10.050&amp;sa=D&amp;usg=AFQjCNF1WlQRkBougDQ4wwZidBecyx5-LQ">Rolland </a></span><span class="c14"><a class="c9" href="https://www.google.com/url?q=http://dx.doi.org/10.1016/j.cell.2014.10.050&amp;sa=D&amp;usg=AFQjCNF1WlQRkBougDQ4wwZidBecyx5-LQ">et. al.</a></span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://dx.doi.org/10.1016/j.cell.2014.10.050&amp;sa=D&amp;usg=AFQjCNF1WlQRkBougDQ4wwZidBecyx5-LQ">&nbsp;2014</a></span><span class="c18">&nbsp;cell paper.</span></li></ul><p class="c5"><span class="c18"></span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 570.76px; height: 533.50px;"><img alt="" src="images/image55.png" style="width: 570.76px; height: 533.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c18"></span></p><ul class="c3 lst-kix_1293ztc79klg-0 start"><li class="c1"><span class="c18">Hard drive broke down, took a while to figure out how to get it to work with the spare 3TB one (needed a different partition table because &gt; 2TB, etc.)...</span></li><li class="c1"><span class="c18">Added </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=https://bitbucket.org/ostrokach/elaspic/src/ae7b46c1202c69783a1f4c2d4bb25e196480f238/code/tests/test_elaspic.py?at%3Dmaster&amp;sa=D&amp;usg=AFQjCNFnQMvA9bk-C7wVWYOQ2k13Y9YqiQ">unit tests </a></span><span class="c18">to the ELASPIC pipeline</span></li></ul><ul class="c3 lst-kix_1293ztc79klg-1 start"><li class="c6"><span class="c18">Select at random one uniprot domain and one uniprot domain pair with precalculated mutations and rerun the pipeline.</span></li></ul><ul class="c3 lst-kix_1293ztc79klg-0"><li class="c1 c21 c15"><span class="c18">Paper draft&hellip;</span></li></ul><ul class="c3 lst-kix_1293ztc79klg-1 start"><li class="c6 c21 c15"><span class="c18">Patrick Alloy mutation server: </span><span class="c16 c20 c18"><a class="c9" href="https://www.google.com/url?q=http://dsysmap.irbbarcelona.org/&amp;sa=D&amp;usg=AFQjCNH-r_JbH9SIC91T6wMCOCuNgzP9JQ">http://dsysmap.irbbarcelona.org/</a></span></li></ul><p class="c5"><span class="c38"></span></p><p class="c10"><span class="c38">Distribution of ddG scores at different Provean thresholds</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 555.50px; height: 405.83px;"><img alt="" src="images/image122.png" style="width: 555.50px; height: 405.83px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 557.50px; height: 409.30px;"><img alt="" src="images/image104.png" style="width: 557.50px; height: 409.30px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-02-18</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c22">Progress</span></p><p class="c5"><span></span></p><p class="c10"><span>Classifiers</span></p><ul class="c3 lst-kix_tcwoy9lfuqk4-0 start"><li class="c1"><span>Some of the features are 0 for all training examples. Removed those features.</span></li><li class="c1"><span>Reran the hyperparameter optimization / feature-elimination pipeline</span></li><li class="c1"><span>The CORE classifier looks OK. The interface classifier looks iffy. </span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Elaspic documentation:</span><span class="c65">&nbsp;</span><span class="c16 c20 c65"><a class="c9" href="https://www.google.com/url?q=http://elaspic.readthedocs.org&amp;sa=D&amp;usg=AFQjCNFhMMOcGPngkbnKEGfJhgRLZKZgxA">http://elaspic.readthedocs.org</a></span></p><p class="c10"><span>Databases documentation: </span><span class="c16 c65 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTVXdUa2g5b3psd2s%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNF3dASXTs7wwnLLuoOyakCLMXTJ6Q">https://drive.google.com/open?id=0B1FXQpQb4bbTVXdUa2g5b3psd2s&amp;authuser=1</a></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">To do:</span></p><ul class="c3 lst-kix_lm4lwqvvvsaj-0 start"><li class="c1"><span>Add protein-protein interaction data from the 2014 cell paper</span></li><li class="c1"><span>Make sure we have homology models for all proteins / interactions in the human proteome</span></li><li class="c1"><span>Integrate ELASPIC output / ELASPIC input / SIFT / Provean / GO terms / all the other features I presented before, in order to distinguish between disease / polymorphism mutations and driver / passenger mutations</span></li><li class="c1"><span>Write research paper (&ldquo;combining structural and sequence information in order to predict the deleteriousness of mutations&rdquo;)</span></li></ul><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-01-27</span></p><hr><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><a href="#" name="id.5yrdq8sb4dit"></a><p class="c10"><span class="c7">Feature elimination</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c18">After removing uninformative features:</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 502.67px;"><img alt="" src="images/image117.png" style="width: 720.00px; height: 502.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c18"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">Before</span><span class="c18">&nbsp;removing uninformative features:</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 502.67px;"><img alt="" src="images/image59.png" style="width: 720.00px; height: 502.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5 c15"><span class="c7"></span></p><p class="c5 c12"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5 c12"><span class="c7"></span></p><p class="c10 c12"><span class="c7">CORE</span></p><p class="c10"><span class="c18">After removing uninformative features</span></p><p class="c5"><span class="c18"></span></p><p class="c10 c26"><span class="c18">&nbsp; &nbsp;Feature importance without Provean&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Feature importance with Provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 349.50px; height: 304.95px;"><img alt="" src="images/image02.png" style="width: 699.00px; height: 632.23px; margin-left: -349.50px; margin-top: -17.48px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.43px; height: 309.50px;"><img alt="" src="images/image02.png" style="width: 706.89px; height: 641.67px; margin-left: 0.00px; margin-top: -17.74px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span class="c18">Before removing uninformative feature</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 373.00px;"><img alt="" src="images/image56.png" style="width: 720.00px; height: 720.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10 c12"><span class="c7">Interface</span></p><p class="c5"><span class="c38"></span></p><p class="c10"><span class="c18">After removing uninformative features</span></p><p class="c5"><span class="c18"></span></p><p class="c10 c26"><span class="c18">Interface mutations without Provean&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Interface mutations with Provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 303.00px;"><img alt="" src="images/image02.png" style="width: 720.00px; height: 651.00px; margin-left: 0.00px; margin-top: -348.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c18">Before removing uninformative features</span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 349.00px;"><img alt="" src="images/image56.png" style="width: 720.00px; height: 720.00px; margin-left: 0.00px; margin-top: -371.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">After feature elimination</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c18">After removing uninformative features</span></p><p class="c10"><span class="c18">Before removing uninformative features</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 678.50px; height: 678.50px;"><img alt="" src="images/image103.png" style="width: 678.50px; height: 678.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.38px; height: 283.50px;"><img alt="" src="images/image82.png" style="width: 354.38px; height: 283.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c7">&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 350.50px; height: 282.86px;"><img alt="" src="images/image86.png" style="width: 350.50px; height: 282.86px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Before feature elimination</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 720.00px;"><img alt="" src="images/image65.png" style="width: 720.00px; height: 720.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span class="c7">Cosmic Core</span></p><p class="c10"><span class="c7">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c16">FoldX reference&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; </span><span class="c16">Elaspic paper</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 353.50px; height: 175.75px;"><img alt="" src="images/image27.png" style="width: 353.50px; height: 885.58px; margin-left: 0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c7">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 301.30px; height: 228.50px;"><img alt="" src="images/image31.png" style="width: 619.81px; height: 536.63px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c10 c26 c50"><span class="c16">Without provean</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; </span><span class="c16">With provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 357.52px; height: 356.50px;"><img alt="" src="images/image27.png" style="width: 357.52px; height: 901.97px; margin-left: 0.00px; margin-top: -182.49px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 353.50px; height: 358.54px;"><img alt="" src="images/image27.png" style="width: 353.50px; height: 896.87px; margin-left: 0.00px; margin-top: -538.33px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10 c12"><span class="c7">Cosmic Interface</span></p><p class="c10"><span class="c7">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c16">FoldX reference&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; </span><span class="c16">Elaspic paper</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 349.86px; height: 173.50px;"><img alt="" src="images/image70.png" style="width: 349.86px; height: 880.41px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c7">&nbsp; &nbsp; &nbsp; &nbsp; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 280.66px; height: 229.47px;"><img alt="" src="images/image31.png" style="width: 580.68px; height: 499.21px; margin-left: 0.00px; margin-top: -213.37px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c10 c26 c50"><span class="c16">Without provean</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; </span><span class="c16">With provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.50px; height: 357.45px;"><img alt="" src="images/image70.png" style="width: 354.50px; height: 893.92px; margin-left: 0.00px; margin-top: -179.58px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 353.50px; height: 355.45px;"><img alt="" src="images/image70.png" style="width: 353.50px; height: 888.90px; margin-left: 0.00px; margin-top: -533.45px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c16">FoldX reference</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 376.50px; height: 187.83px;"><img alt="" src="images/image115.png" style="width: 376.50px; height: 948.44px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c10 c26 c50"><span class="c16">Without provean</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; </span><span class="c16">With provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 355.50px; height: 357.49px;"><img alt="" src="images/image115.png" style="width: 355.50px; height: 888.58px; margin-left: 0.00px; margin-top: -175.97px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 354.50px; height: 356.49px;"><img alt="" src="images/image115.png" style="width: 354.50px; height: 886.09px; margin-left: 0.00px; margin-top: -529.60px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c16">FoldX reference</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 359.50px; height: 180.31px;"><img alt="" src="images/image97.png" style="width: 359.50px; height: 903.79px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c10 c26 c50"><span class="c16">Without provean</span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; </span><span class="c16">With provean</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 347.53px; height: 348.50px;"><img alt="" src="images/image97.png" style="width: 347.53px; height: 871.00px; margin-left: 0.00px; margin-top: -174.75px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 347.50px; height: 347.50px;"><img alt="" src="images/image97.png" style="width: 347.50px; height: 868.50px; margin-left: 0.00px; margin-top: -521.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><hr><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Performance of mutation scoring algorithms:</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 902.00px;"><img alt="" src="images/image32.png" style="width: 720.00px; height: 1352.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 720.00px; height: 450.00px;"><img alt="" src="images/image32.png" style="width: 720.00px; height: 1352.00px; margin-left: 0.00px; margin-top: -902.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span class="c18"></span></p><p class="c10"><span class="c18">To do:</span></p><p class="c10"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PDBfam summary</span></p><p class="c10"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Double check the interface training set / classifier</span></p><p class="c10"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Meta-classifier + other features for phenotype prediction?</span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2015-01-14</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span>Progress:</span></p><ul class="c3 lst-kix_x7o2gv80my2k-0 start"><li class="c1"><span>Bugfixes</span><span>&nbsp;on the web server...</span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-1 start"><li class="c6"><span>FoldX expired in 2015. Had to hack it with libfaketime.so.1</span></li><li class="c6"><span>Daniel added a PDBfam info section: </span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-2 start"><li class="c10 c87 c57"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/help/&amp;sa=D&amp;usg=AFQjCNGolfPpNhaAkwneudQ42jYTFuInOA">http://elaspic.kimlab.org/help/</a></span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-1"><li class="c6"><span>Added a README to the data download page: </span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-2 start"><li class="c10 c87 c57"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/pdbfam/&amp;sa=D&amp;usg=AFQjCNFMHSa-oiLkjHxNS6-CLj1zTxj1fw">http://elaspic.kimlab.org/static/download/pdbfam/</a></span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-1"><li class="c6 c19"><span></span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-0"><li class="c1"><span>Added data to the domain-domain interaction table</span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-1 start"><li class="c6"><span>8,967,336 uniprot domain pairs </span></li><li class="c6"><span>255,145 had templates before, 459,385 have templates now</span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-0"><li class="c1"><span>Trained core and interface classifiers</span></li><li class="c1"><span>Precalculated data</span></li></ul><ul class="c3 lst-kix_x7o2gv80my2k-1 start"><li class="c6"><span>~100,000 core mutations</span></li><li class="c6"><span>~25,000 interface mutations</span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c16">Examples of high-impact mutations:</span></p><p class="c5"><span class="c16"></span></p><a href="#" name="e24af2c90c0c31661d62652a023d741984e09e76"></a><a href="#" name="3"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c44 c16">Protein</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c44 c16">Uniprot ID</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c16 c44">Mutation</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c44 c16">ddG</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>Ribonuclease pancreatic</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>P61823</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>Y123G</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c10 c21 c15"><span>-11.7 </span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>Ribonuclease pancreatic</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>P61823</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Y123A</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c27">-12.0</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Thioredoxin-1</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P0AA25</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">D27I</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c10 c21 c15"><span class="c45 c22 c97">12.1 ?</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Subtilisin inhibitor</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P01006</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">V44G</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">-9.805</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Lysozyme</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P00720</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">L91P</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">-9.555</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Lysozyme</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P00720</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">L66P</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">-12.23</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c32 c15 c43"><span class="c27">Bifunctional tail protein</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P12528</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">R382S</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c16 c22 c20"><a class="c9" href="https://www.google.com/url?q=http://www.jbc.org/content/264/18/10693.full.pdf&amp;sa=D&amp;usg=AFQjCNExUap-GpIXd9lmlsxlA_IlfZk3yQ">-17.4</a></span><span class="c22">&nbsp;?</span></p></td></tr><tr class="c75"><td class="c76" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Bifunctional tail protein</span></p></td><td class="c79" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">P12528</span></p></td><td class="c91" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">K286R</span></p></td><td class="c68" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c16 c22 c20"><a class="c9" href="https://www.google.com/url?q=http://www.jbc.org/content/264/18/10693.full.pdf&amp;sa=D&amp;usg=AFQjCNExUap-GpIXd9lmlsxlA_IlfZk3yQ">-10.3</a></span><span class="c45 c22 c97">&nbsp;?</span></p></td></tr></tbody></table><p class="c10"><span class="c48">etc...</span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c10"><span class="c16">All features:</span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c5"><span class="c16"></span></p><p class="c10"><span class="c16">Subset of features with correlation &gt; 0.10:</span></p><p class="c5"><span class="c16"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 450.67px;"><img alt="" src="images/image76.png" style="width: 624.00px; height: 450.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c16">All features:</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 456.00px;"><img alt="" src="images/image58.png" style="width: 624.00px; height: 456.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c15"><span class="c7"></span></p><p class="c5 c15"><span class="c7"></span></p><p class="c5 c12"><span class="c7"></span></p><p class="c10 c15"><span class="c16">Subset of features with correlation &gt; 0.10:</span></p><p class="c5"><span class="c7"></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 456.00px;"><img alt="" src="images/image03.png" style="width: 624.00px; height: 456.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2014-12-17</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span></span></p><p class="c10"><span>Progress:</span></p><ul class="c3 lst-kix_x7o2gv80my2k-0"><li class="c1"><span class="c100">40</span><span>&nbsp; &nbsp;42.3 million domains with templates</span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Core mutations training set:</span></p><ul class="c3 lst-kix_e5u94rrvoqs-0 start"><li class="c1"><span>191 proteins</span></li><li class="c1"><span>3948 mutations x 2</span></li><li class="c1"><span>xxx wild-type models + 11130 mutant models </span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>&nbsp; &nbsp; </span><span>&nbsp; </span><span class="c16">Templates for the core training set at different identity thresholds</span></p><p class="c5"><span></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.50px; height: 192.26px;"><img alt="" src="images/image33.png" style="width: 297.50px; height: 192.26px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 308.17px; height: 191.34px;"><img alt="" src="images/image137.png" style="width: 308.17px; height: 191.34px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c12"><span></span></p><p class="c10 c12"><span>It really is 11,130 mutations</span><span>...</span></p><p class="c10 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 420.50px; height: 272.65px;"><img alt="" src="images/image78.png" style="width: 420.50px; height: 272.65px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>Training the core classifier: </span></p><p class="c10"><span>Now the data is balanced and has 71642 points:</span></p><p class="c5"><span></span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 306.50px; height: 218.50px;"><img alt="" src="images/image73.png" style="width: 306.50px; height: 218.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 303.14px; height: 215.50px;"><img alt="" src="images/image63.png" style="width: 303.14px; height: 215.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span>The predicted scores ddG values the same &ldquo;dynamic range&rdquo; and FoldX (unlike previously)...</span></p><p class="c10"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 453.33px;"><img alt="" src="images/image131.png" style="width: 624.00px; height: 453.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Remaining...</span></p><p class="c5"><span></span></p><ul class="c3 lst-kix_zhrwxtd2sngd-0 start"><li class="c1"><span>Update the set of domain-domain interactions</span></li><li class="c1"><span>Retrain the domain-domain interaction classifier</span></li><li class="c1"><span>Daniel coming by on Monday to update the website and finish the 1-page proposal</span></li><li class="c1"><span>Look into ddG values &gt; 10 or &lt; 10.</span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span class="c7"></span></p><hr style="page-break-before:always;display:none;"><p class="c5"><span class="c7"></span></p><p class="c10"><span class="c7">Subgroup Splicing/Network, 2014-12-10</span></p><p class="c5"><span class="c7"></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Recalculate domain and domain-domain interaction data</span></p><p class="c5"><span></span></p><p class="c10"><span>Progress:</span></p><ul class="c3 lst-kix_x7o2gv80my2k-0"><li class="c1"><span>60 million domains (vs. &lt; 30 million)</span></li><li class="c1"><span>40 million domains with templates (vs. 17 million) </span></li><li class="c1"><span>374 thousand domain-domain interaction templates (365 thousand of those are human)</span></li><li class="c1"><span>Need PPI sources for other species (contacted the HIPPIE developers&hellip;)</span></li></ul><p class="c5"><span></span></p><p class="c10"><span>Problem:</span></p><p class="c10"><span>Some of the domains in our core training dataset were still absent from the database. The main reason is problems matching blast output with domain definitions in the database using a </span><span class="c48">(pdb_id, pdb_chain, pdb_pdbfam_name, pdb_pdbfam_idx) </span><span>tuple</span><span>. </span><span class="c48">pdb_pdbfam_name</span><span>&nbsp;can change between version and </span><span class="c48">pdb_pdbfam_idx</span><span>&nbsp;is assigned inconsistently and is absent in the most recent file of PDB domain definitions.</span></p><p class="c5"><span></span></p><p class="c10"><span>Solutions:</span></p><ul class="c3 lst-kix_xm7ynypsh0z5-0 start"><li class="c1"><span>Updated the domain table in the elaspic database with domain definitions from Andres&rsquo; most recent version of PDBfam</span></li></ul><ul class="c3 lst-kix_xm7ynypsh0z5-1 start"><li class="c6"><span>This is a bit of a hack because those domains were not passed through SCOWLP and thus are not included in the domain_contact table.</span></li><li class="c6"><span>Did not remove overlapping domains in order to maximize the match rate between blast output and the domain table.</span></li></ul><ul class="c3 lst-kix_xm7ynypsh0z5-0"><li class="c1"><span>Made some (hacky) fixes to the Errors when matching PDBfam uniprot domain definitions with their respective PDB templates </span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c10"><span class="c22">Putting precalculated data on the webserver...</span></p><ul class="c3 lst-kix_xm7ynypsh0z5-0"><li class="c1"><span>Daniel configured the elaspic webserver to make one of the folders on our storage cluster visible to the public: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://elaspic.kimlab.org/static/download/&amp;sa=D&amp;usg=AFQjCNHDWGcCb76UNbdNBZ1riRON_KNIzg">http://elaspic.kimlab.org/static/download/</a></span></li></ul><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c5"><span></span></p><p class="c32"><span class="c22">Getting the new version of elaspic to run on the virtual machine...</span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Subgroup Splicing/Network, 2014-12-03</span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">1) Elaspic</span></p><p class="c8"><span></span></p><p class="c32"><span>Finished running PDBfam v3 </span></p><ul class="c3 lst-kix_tnmatnpjv4t2-0 start"><li class="c4 c32"><span>v1 was merging domains that were not supposed to be merged, finding a structural template for only a small region of the resulting superdomain</span></li><li class="c4 c32"><span>v2 improved the domain merging behaviour, but in a small number of cases it for domains that had a structural template with v1</span></li><li class="c4 c32"><span>Andres made new libraries, which were supposed to fix the missing templates bug (v3). </span></li><li class="c4 c32"><span>Overall, we get templates for 2,881,625 domains for which we didn&rsquo;t have templates previously</span></li><li class="c4 c32"><span>But v3 does not find templates for 2,904,001 domains for which we had templates previously&hellip;</span></li></ul><p class="c8"><span></span></p><p class="c32"><span>Uploaded new domain definitions to the database (making sure not to have duplicates, etc.)</span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span>Andres made a pretty good wiki for PDBfam: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://bitbucket.org/afgiraldofo/pdbfam/wiki/Home&amp;sa=D&amp;usg=AFQjCNHVDPQSRhrTMKkTf0tDJn8k6Kqf7g">https://bitbucket.org/afgiraldofo/pdbfam/wiki/Home</a></span></p><p class="c8"><span></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">To do:</span></p><ul class="c3 lst-kix_tnmatnpjv4t2-0"><li class="c4 c32"><span>Daniel is coming by Thursday to fix a few bugs on the webserver and add a page to store precalculated data</span></li><li class="c4 c32"><span>Add PDBfam to the methods section of the manuscript</span></li><li class="c4 c32"><span>Recalculate domain domain interaction data</span></li><li class="c4 c32"><span>Retrain classifiers</span></li><li class="c4 c32"><span>Precalculate human uniprot structures and as many mutations as possible</span></li></ul><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">2) Decision making under uncertainty course project</span></p><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span class="c16 c22 c18">Mapping genes to diseases in the disease ontology (DO)</span></p><p class="c8"><span class="c16"></span></p><a href="#" name="e3f79cf9fba6f9340d48aa36408d3a6e9517e067"></a><a href="#" name="4"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c85" colspan="1" rowspan="1"><p class="c43 c8 c15"><span class="c27"></span></p></td><td class="c86" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">DisGeNet (gv)</span></p></td><td class="c81" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">HPO-D</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Overlap</span></p></td></tr><tr class="c75"><td class="c85" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number of unique genes</span></p></td><td class="c86" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">1839</span></p></td><td class="c81" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">3158</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">0</span></p></td></tr><tr class="c75"><td class="c85" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number of unique diseases</span></p></td><td class="c86" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">2376</span></p></td><td class="c81" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">4919</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">?</span></p></td></tr><tr class="c75"><td class="c85" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number of unique gene-disease pairs</span></p></td><td class="c86" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">2622</span></p></td><td class="c81" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">6481</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">0</span></p></td></tr><tr class="c75"><td class="c85" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number (and %) of diseases that were mapped to the disease ontology </span></p><p class="c43 c32 c15"><span class="c27">(DO contains ~6000 diseases)</span></p></td><td class="c86" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">233 (10%)</span></p></td><td class="c81" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">782 (16%)</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">220</span></p></td></tr></tbody></table><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span class="c16 c22 c18">Mapping EMR records to diseases in the disease ontology (DO)</span></p><p class="c8"><span class="c16 c22 c18"></span></p><p class="c32"><span>STRIDE (Stanford Translational Research Integrated Database Environment)</span></p><p class="c32"><span>(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.nature.com/articles/sdata201432&amp;sa=D&amp;usg=AFQjCNHSX1V3_Q390rPLOHxTfmhaKEmwEg">paper</a></span><span>) (</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://datadryad.org/resource/doi:10.5061/dryad.jp917&amp;sa=D&amp;usg=AFQjCNHnmdcbQYLvcnKsk0LnCoFq3kXAqQ">website</a></span><span>) </span></p><p class="c32"><span>Natural language processing of raw text in 20 million clinical notes spanning 19 years of data.</span></p><p class="c8"><span></span></p><p class="c32"><span>HuDiNe (Human Disease Network)</span></p><p class="c32"><span>(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://dx.doi.org/10.1371/journal.pcbi.1000353&amp;sa=D&amp;usg=AFQjCNEpKdnIo-GtPkQr4NOJf4AqMj3mQQ">paper</a></span><span>) (</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://datadryad.org/resource/doi:10.5061/dryad.jp917&amp;sa=D&amp;usg=AFQjCNHnmdcbQYLvcnKsk0LnCoFq3kXAqQ">website</a></span><span>)</span></p><p class="c32"><span>Medical claims data for elderly Americans enrolled in the Medicare program</span></p><p class="c8"><span></span></p><a href="#" name="927b607fe8be8c01ffa0050e319d40233c485a00"></a><a href="#" name="5"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c46" colspan="1" rowspan="1"><p class="c43 c8 c15"><span class="c27"></span></p></td><td class="c67" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>Stride</span></p></td><td class="c71" colspan="1" rowspan="1"><p class="c43 c32 c15"><span>Hudine</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Overlap</span></p></td></tr><tr class="c75"><td class="c46" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number of patients</span></p></td><td class="c67" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">~1.2 million</span></p></td><td class="c71" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">~13 million</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c8 c15"><span class="c27"></span></p></td></tr><tr class="c75"><td class="c46" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number of unique diseases (&lsquo;concepts&rsquo;)</span></p></td><td class="c67" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">18,787</span></p></td><td class="c71" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">15,983</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c8 c15"><span class="c27"></span></p></td></tr><tr class="c75"><td class="c46" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">Number (and %) of diseases that were mapped to the disease ontology </span></p><p class="c43 c32 c15"><span class="c27">(DO contains ~6000 diseases)</span></p></td><td class="c67" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">1635 (8.7%)</span></p><p class="c43 c8 c15"><span class="c27"></span></p><p class="c43 c8 c15"><span class="c27"></span></p></td><td class="c71" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">1888 (11.8%)</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c43 c32 c15"><span class="c27">702</span></p></td></tr></tbody></table><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 444.50px; height: 291.26px;"><img alt="" src="images/image96.png" style="width: 444.50px; height: 291.26px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span class="c16 c22 c18">Overlap between diseases mapped to genes and EMR records</span></p><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span class="c16">Disease ontology terms with patient and disease information</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 502.50px; height: 385.73px;"><img alt="" src="images/image138.png" style="width: 502.50px; height: 385.73px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32 c12"><span class="c16">Disease ontology terms with patient and disease information after DO was filtered to remove leaves without any information</span></p><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 519.50px; height: 402.11px;"><img alt="" src="images/image50.png" style="width: 519.50px; height: 402.11px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8 c12"><span class="c16 c22 c18"></span></p><p class="c32 c12"><span class="c16 c22 c18">The Bayesian network</span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 292.00px;"><img alt="" src="images/image136.png" style="width: 624.00px; height: 292.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 329.33px;"><img alt="" src="images/image92.png" style="width: 624.00px; height: 329.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Remaining:</span></p><ol class="c3 lst-kix_nzxgmnmwjatv-0 start" start="1"><li class="c4 c32"><span>Construct and export the gene - disase ontology Bayesian network in a Hudine .net format that can be loaded into R and UnBBayes</span></li><li class="c4 c32"><span>Analyse how tractable it is to perform inference on this network</span></li><li class="c4 c32"><span>Perform sensitivity analysis on query genes given one or more disease observations</span></li><li class="c4 c32"><span class="c16">Try using different utility functions to map from disease probabilities to disease rankings</span></li><li class="c4 c32"><span>Learn a better network using EMR data in order to get better results&hellip;?</span></li></ol><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><hr><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Subgroup Splicing/Network, 2014-11-19</span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">1) Elaspic</span></p><ul class="c3 lst-kix_cpm4oaqfjy6b-0 start"><li class="c4 c32"><span>Emailed Andres about some problems that I noticed with the new version of PDBfam (some of the domains that previously had structural templates lost those templates).</span></li></ul><ul class="c3 lst-kix_cpm4oaqfjy6b-1 start"><li class="c47 c32"><span>Andres updated the PDBfam libraries, which he says should fix those problems.</span></li><li class="c47 c32"><span>Also, Andres says he&rsquo;s happy if PDBfam appears in the methods section of the Elaspic webserver rather than as its own paper.</span></li></ul><ul class="c3 lst-kix_cpm4oaqfjy6b-0"><li class="c4 c32"><span>Finding templates for new domain-domain interactions</span></li></ul><ul class="c3 lst-kix_cpm4oaqfjy6b-1 start"><li class="c47 c32"><span>Making PDBfam libraries that are redundant and that are based on atom rather than seqres information (two pdbs might have the same sequence but different interaction partners)</span></li><li class="c47 c32"><span>Fix errors that come up.</span></li></ul><p class="c8"><span></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">2) Decision making under uncertainty course project</span></p><p class="c8"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0 start"><li class="c4 c32"><span>Medical expert system that uses gene-level information to predict diseases.</span></li><li class="c4 c32"><span>Noisy-OR disease nodes that integrate input from several proteins</span></li></ul><p class="c8"><span></span></p><ol class="c3 lst-kix_h9p9lo20tlpj-0 start" start="1"><li class="c4 c32"><span>Try to learn better weights for protein-disease edges using </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.nature.com/articles/sdata201432&amp;sa=D&amp;usg=AFQjCNHSX1V3_Q390rPLOHxTfmhaKEmwEg">EMR data</a></span><span>.</span></li><li class="c4 c32"><span>Examine how tractable it is to do inference on this network and compare inference using different algorithms.</span></li><li class="c4 c32"><span>Integrate different predictions in a gene ontology network and predict the most likely diseases based different specificity vs. misclassification utility functions.</span></li></ol><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 223.36px; height: 98.50px;"><img alt="" src="images/image80.jpg" style="width: 591.16px; height: 98.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 132.50px; height: 179.00px;"><img alt="" src="images/image35.png" style="width: 132.50px; height: 179.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 201.30px; height: 93.50px;"><img alt="Disease Ontology logo" src="images/image101.jpg" style="width: 686.40px; height: 93.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 416.00px; height: 116.00px;"><img alt="" src="images/image81.png" style="width: 416.00px; height: 116.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8 c24"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Subgroup Splicing/Network, 2014-11-12</span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">1) Uploading new domain definitions to the database.</span></p><p class="c32"><span>Created a scripts that checks whether the new domain definitions overlap perfectly, overlap partially, or do not overlap at all with the existing domain definitions in the database. </span></p><ul class="c3 lst-kix_gmyqb4k4mims-0 start"><li class="c4 c32"><span>For domains that overlap partially, I deleted their overlapping partners in the database and uploaded the new variants. </span></li><li class="c4 c32"><span>For domains that don&rsquo;t overlap at all, I just added the new domain to the database. </span></li><li class="c4 c32"><span>I did not change domains that overlap perfectly (in order to keep the models and mutations).</span></li></ul><p class="c8"><span></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span>Q03924</span></p><p class="c32"><span>New domain definitions:</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 69.33px;"><img alt="" src="images/image108.png" style="width: 624.00px; height: 69.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span>vs. before:</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 70.67px;"><img alt="" src="images/image11.png" style="width: 624.00px; height: 70.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span>Genome3D:</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 102.67px;"><img alt="" src="images/image18.png" style="width: 624.00px; height: 102.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c40"></span></p><p class="c8"><span class="c40"></span></p><p class="c8"><span class="c40"></span></p><p class="c32"><span class="c40">Provean</span></p><p class="c32"><span class="c40">Number of mutations: 3953</span></p><p class="c32"><span class="c40">Number of mutations with ddG entries: 3421</span></p><p class="c32"><span class="c40">Number of mutations with dTm entries: 1827</span></p><p class="c8"><span class="c18"></span></p><p class="c8"><span class="c22 c18"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Subgroup Splicing/Network, 2014-11-05</span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">1) Comparing Pfam domain definitions, Gene3D domain definitions and PDBfam domain definitions.</span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span>Histogram of domain lengths for domains obtained from different sources.</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 246.67px;"><img alt="" src="images/image41.png" style="width: 624.00px; height: 246.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTQkVpdTZPNUtBMGc%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNHlho2B8B73Dlm6nFSInZzLWBwKLw">https://drive.google.com/open?id=0B1FXQpQb4bbTQkVpdTZPNUtBMGc&amp;authuser=1</a></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span>Histograms of the fraction of domains from source A (rows) that are covered by domains from source B (columns).</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 313.33px;"><img alt="" src="images/image67.png" style="width: 624.00px; height: 313.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTUjc0a2R2Z1Z6dHM%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNEflhTjGUnM-IiP_XVFlDL-a7g47A">https://drive.google.com/open?id=0B1FXQpQb4bbTUjc0a2R2Z1Z6dHM&amp;authuser=1</a></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span>Histograms of the number of domains from source B (columns) that overlap with domains from source A (rows).</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 313.33px;"><img alt="" src="images/image71.png" style="width: 624.00px; height: 313.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTNzJmdUY2c2Rkakk%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNHYz6wE3HjxcoxX-zpHMU_U700wEA">https://drive.google.com/open?id=0B1FXQpQb4bbTNzJmdUY2c2Rkakk&amp;authuser=1</a></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c18"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">2) Correcting bad domain definitions</span></p><p class="c8"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>Ran a few test cases using an updated version of PDBfam scripts and libraries (found in the most up-to-date folder in Andres&rsquo; directory on beagle). The updated version of PDBfam appeared to have most of the errors resolved.</span></li><li class="c4 c32"><span>Downloaded Pfam domain definitions for all known protein sequences (as of 14-07-17) from the simap database. </span></li><li class="c4 c32"><span>Wrote R scripts that run simap domain definitions through the updated PDBfam pipeline.</span></li><li class="c4 c32"><span>Analysed all sequences that could be matched to a uniprot identifier.</span></li><li class="c4 c32"><span>42,608,671 vs. 18,217,258 unique uniprots (out of a total of 53,285,583 proteins in our uniprot database)</span></li></ul><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32"><span class="c18">Q03924</span></p><p class="c32"><span>Current definition:</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 70.67px;"><img alt="" src="images/image11.png" style="width: 624.00px; height: 70.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c18"></span></p><p class="c32"><span>Current definition:</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 117.33px;"><img alt="" src="images/image43.png" style="width: 624.00px; height: 117.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32"><span>Andres PDBfam definitions: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><a href="#" name="09846d7882df23eafcb843d99a82614e2f366e41"></a><a href="#" name="6"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c104"><td class="c66" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">97</span></p></td><td class="c25" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">143</span></p></td><td class="c88" colspan="1" rowspan="1"><p class="c21 c8 c15"><span class="c31"></span></p></td><td class="c98" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">zf-C2H2_6</span></p></td></tr><tr class="c104"><td class="c66" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">144</span></p></td><td class="c25" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">218</span></p></td><td class="c88" colspan="1" rowspan="1"><p class="c21 c8 c15"><span class="c31"></span></p></td><td class="c98" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">zf-H2C2_2</span></p></td></tr><tr class="c104"><td class="c66" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">219</span></p></td><td class="c25" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">280</span></p></td><td class="c88" colspan="1" rowspan="1"><p class="c21 c8 c15"><span class="c31"></span></p></td><td class="c98" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">zf-H2C2_2</span></p></td></tr><tr class="c104"><td class="c66" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">267</span></p></td><td class="c25" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">483</span></p></td><td class="c88" colspan="1" rowspan="1"><p class="c21 c8 c15"><span class="c31"></span></p></td><td class="c98" colspan="1" rowspan="1"><p class="c21 c32 c15"><span class="c31">zf-H2C2_2</span></p></td></tr></tbody></table><p class="c8"><span class="c18"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c22 c18">3) Decision making under uncertainty course project</span></p><p class="c8"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>Medical expert system that uses gene-level information to predict diseases.</span></li></ul><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 262.50px; height: 262.50px;"><img alt="" src="images/image35.png" style="width: 262.50px; height: 262.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><hr><p class="c8"><span class="c7"></span></p><p class="c8"><span class="c7"></span></p><p class="c32"><span class="c7">Subgroup Splicing/Network, 2014-10-29</span></p><p class="c8"><span class="c22 c18"></span></p><p class="c8"><span class="c22 c18"></span></p><p class="c32"><span class="c22 c18">Parsing the Protherm database</span></p><p class="c8"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>Found the uniprot id and uniprot name for rows where only the protein name is specified, making sure that the uniport sequence matches the mutated amino acids (4 cases).</span></li><li class="c4 c32"><span>For mutations that occur multiple times, removed rows that have a &ldquo;remarks&rdquo; entry (remarks often are about the presence of urea or guanidinium chloride)</span></li><li class="c4 c32"><span>If still there are multiple entries for a given mutation, took the median of the </span><span class="c48">ddG_H2O</span><span>, </span><span class="c48">ddG</span><span>&nbsp;and </span><span class="c48">dTm</span><span>&nbsp;values.</span></li><li class="c4 c32"><span>For the final &ldquo;change in Gibbs free energy&rdquo; value, took </span><span class="c48">ddG_H2O</span><span>&nbsp;if it is given, </span><span class="c48">ddG</span><span>&nbsp;otherwise. </span></li><li class="c4 c32"><span>Added 56 mutations with dTm values from another study.</span></li></ul><ul class="c3 lst-kix_n2keu386kfur-1 start"><li class="c47 c32"><span class="c40">A survey of proteins encoded by non-synonymous single nucleotide polymorphisms reveals a significant fraction with altered stability and activity. Allali-Hassani </span><span class="c40 c48">et. al.</span><span class="c40">&nbsp;2009.</span></li></ul><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32 c12"><span class="c40">Number of mutations that have ddG_H2O, ddG and dTm in the Protherm dataset</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 431.26px; height: 365.50px;"><img alt="" src="images/image04.png" style="width: 431.26px; height: 365.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32 c12"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTZ2NTRVJUZHF5Ums%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNHXtPAtFWXpEXxQYJxC3xzGBCVyDg">https://drive.google.com/open?id=0B1FXQpQb4bbTZ2NTRVJUZHF5Ums&amp;authuser=1</a></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32"><span>Updated version 14-11-14</span></p><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 437.50px; height: 359.10px;"><img alt="" src="images/image105.png" style="width: 437.50px; height: 359.10px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8 c12"><span></span></p><p class="c32 c12"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTcXlzSkNlNnJ6VDg%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNE2m49Ik1BBdZhgpCDCGhUstmo_rA">https://drive.google.com/open?id=0B1FXQpQb4bbTcXlzSkNlNnJ6VDg&amp;authuser=1</a></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32"><span>Successfully mapped 1357 mutations disrupting protein-protein interactions</span></p><p class="c32"><span>Mutations affect 61 distinct protein-protein pairs</span></p><p class="c8"><span></span></p><p class="c32"><span>Number of mutations: 3953</span></p><p class="c32"><span>Number of mutations with ddG entries: 3421</span></p><p class="c32"><span>Number of mutations with dTm entries: 1827</span></p><p class="c8"><span></span></p><p class="c32"><span>Number of unique (uniprot_id_1, uniprot_id_2, uniprot_mutation) tuples</span></p><p class="c32"><span>Overall: 1218</span></p><p class="c32"><span>Mutation matches the uniprot sequence : 1166</span></p><p class="c32"><span>Uniprot has at least one domain: 1122</span></p><p class="c32"><span>Mutation inside domain: 1094</span></p><p class="c32"><span>Interacting residues inside domain: 1109</span></p><p class="c32"><span>Both mutation and interacting residues inside domain: 1084</span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32"><span class="c22 c18">Converting &Delta;Tm values to &Delta;G values</span></p><p class="c8"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>Carles found a formula that approximates </span><span>&Delta;G from &Delta;Tm</span></li></ul><ul class="c3 lst-kix_n2keu386kfur-1 start"><li class="c47 c32"><img src="images/image00.png"></li><li class="c47 c32"><span class="c16 c40 c20"><a class="c9" href="https://www.google.com/url?q=http://www.biochem.oulu.fi/Biocomputing/juffer/Teaching/PhysicalBiochemistry/PhysBiochem-protein-folding.pdf&amp;sa=D&amp;usg=AFQjCNEVQYEdIlDyqtXtO0o_i6BniCAPUQ">http://www.biochem.oulu.fi/Biocomputing/juffer/Teaching/PhysicalBiochemistry/PhysBiochem-protein-folding.pdf</a></span></li></ul><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>This makes &Delta;G and &Delta;Tm more correlated, but the slope is off by a factor of ~ 5.</span></li><li class="c4 c32"><span>Not all rows with &Delta;Tm have &Delta;H...</span></li></ul><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32 c12"><span class="c40">Change in the Gibbs free energy vs. change in the melting temperature</span></p><p class="c32 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 400.50px; height: 285.20px;"><img alt="" src="images/image12.png" style="width: 400.50px; height: 285.20px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32 c12"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/open?id%3D0B1FXQpQb4bbTTnJmdHJ2WU16Qms%26authuser%3D1&amp;sa=D&amp;usg=AFQjCNGMZ4WO4mihag6zGxcwEOqwoCR6qQ">https://drive.google.com/open?id=0B1FXQpQb4bbTTnJmdHJ2WU16Qms&amp;authuser=1</a></span></p><p class="c8 c12"><span></span></p><p class="c8 c12"><span></span></p><p class="c8 c12"><span></span></p><p class="c32 c12"><span class="c40">&Delta;&Delta;G reported in Protherm vs. &Delta;&Delta;G calculated from &Delta;Tm</span></p><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 304.60px; height: 214.31px;"><img alt="" src="images/image102.png" style="width: 304.60px; height: 214.31px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 308.13px; height: 211.40px;"><img alt="" src="images/image09.png" style="width: 308.13px; height: 211.40px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c32 c15"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTVE8wYnNZcjhpRkE/view?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNF-QZHWenCrXOfFV-ADPtQ403OrUQ">https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTVE8wYnNZcjhpRkE/view?usp=sharing</a></span></p><p class="c32 c15"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTc2FwdDlfRUNvRms/view?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNGoQ7XWksKTCV3emIda7K2rutjgjg">https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTc2FwdDlfRUNvRms/view?usp=sharing</a></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32"><span class="c18">14-12-31 update:</span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c8"><span></span></p><p class="c32 c15"><span class="c22 c18">Making models having varying sequence identity with the templates</span></p><p class="c8 c15"><span></span></p><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>For each protein, found templates with sequence identity falling in the bins:</span></li></ul><ul class="c3 lst-kix_n2keu386kfur-1 start"><li class="c47 c32"><span>0 &lt; x&le; 40; &nbsp; &nbsp;40 &lt; x&le; 60; &nbsp; &nbsp;60 &lt; x&le; 80; &nbsp; &nbsp;80 &lt; x &le; 100;</span></li></ul><ul class="c3 lst-kix_n2keu386kfur-0"><li class="c4 c32"><span>Made homology models using the templates above for all wild-type sequences and all point mutant sequences.</span></li><li class="c4 c32"><span>Ran elaspic on all forward and reverse mutations</span></li></ul><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22 c18">Skempi</span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c16">Mapping mutations to interfaces:</span></p><p class="c0"><span>3047 entries describe mutations mapped to PDB structures (2795 unique PDB-mutation pairs):</span></p><p class="c0 c26"><span>730 entries -&gt; no interacting residues within 10 &Aring; of the mutation</span></p><p class="c0 c26"><span>509 entries -&gt; more than one partner chain within 10 &Aring; of the mutation</span></p><p class="c0 c26 c50"><span>(usually interaction between heterodimers)</span></p><p class="c0 c26"><span>1808 </span><span>entries</span><span>&nbsp;</span><span>-&gt; interaction could be mapped to two chains</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 310.67px;"><img alt="" src="images/image36.png" style="width: 624.00px; height: 310.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c16">Mapping mutations to uniprot:</span></p><p class="c0"><span>Out of 1808 entries with interactions that could be mapped to two chains:</span></p><p class="c0 c26"><span>1480 could be mapped to interactions between two uniprots</span></p><p class="c0 c26"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1407 mutations match the uniprot sequence</span></p><p class="c0 c26"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;71 mutations match the uniprot sequence after being reversed (change ddG sign)</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 302.67px;"><img alt="" src="images/image133.png" style="width: 624.00px; height: 302.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span><br></span><span class="c16">Mapping mutations to domains:</span></p><p class="c0 c19"><span></span></p><p class="c0"><span>End up with 1333 unique (uniprot_id_1, uniprot_id_2, mutation) tuples</span></p><p class="c0"><span>1331 where the mutation matches the uniprot sequence (SIFTS may have a different residue for the uniprot field and the pdb atom field)</span></p><p class="c0"><span>53 are reverses of existing mutations</span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">But&hellip; found problems with domain definitions&hellip;</span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c48">e.g.</span></p><p class="c0"><span>Superdomain: </span><span>SH3_1,PB1,SH3_1 </span></p><p class="c0"><span>Original domain definitions: 246:291,351:429,463:508 </span></p><p class="c0"><span>Final domain definition: 455:516</span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-10-22</span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">Making homology models for human uniprot:</span></p><ul class="c3 lst-kix_aninp2c52sy3-0 start"><li class="c0 c4"><span>Human protein domains</span></li></ul><ul class="c3 lst-kix_aninp2c52sy3-1 start"><li class="c0 c47"><span>141,151 domains </span></li><li class="c0 c47"><span>103,202 domains with structural templates (73%)</span></li><li class="c0 c47"><span>101,549 domains with homology models (98%)</span></li></ul><ul class="c3 lst-kix_aninp2c52sy3-0"><li class="c0 c4"><span>Human protein domain-domain pairs</span></li></ul><ul class="c3 lst-kix_aninp2c52sy3-1 start"><li class="c0 c47"><span>208,328 domain pairs</span></li><li class="c0 c47"><span>41,188 domain pairs with structural templates (20%)</span></li><li class="c0 c47"><span>35,582 domain pairs with homology models (86%)</span></li></ul><ul class="c3 lst-kix_aninp2c52sy3-0"><li class="c0 c4"><span>Errors for ~2500 proteins (another ~2500 proteins have &lt; 35% sequence identity with the template). Re-running elaspic for those proteins appears to fix the errors.</span></li></ul><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 305.33px;"><img alt="" src="images/image19.png" style="width: 624.00px; height: 305.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTV3B6WF9taUQyZmc/view?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNHc435HnyeJKtn_jZ-7QBbfEFScpA">https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTV3B6WF9taUQyZmc/view?usp=sharing</a></span></p><p class="c0 c19"><span></span></p><hr><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">Mapping Ensembl and Refeseq to Uniprot:</span></p><ul class="c3 lst-kix_emh9tw6ivvnv-0 start"><li class="c0 c4"><span>Added canonical Uniprot ids in place of the corresponding splice variant ids in the Uniprot mapping table.</span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-1 start"><li class="c0 c47"><span>The id mapping table provided by Uniprot uses the uniprot splice id (e.g. A0AUZ9-1) for all Ensembl-to-Uniprot and Refseq-to-Uniprot mapping. However, the fasta file containing all Uniprot sequences uses the base uniprot id (e.g. A0AUZ9) for the canonical splice variant. Previously, I was not able to map many Ensembl proteins to Uniprot because I could not find an entry for the canonical splice id in the Uniprot sequence table (</span><span class="c48">e.g.</span><span>&nbsp;I was looking for A0AUZ9-1 instead of A0AUZ9). In order to fix this, I downloaded the complete Uniprot dump, parsed it to get a list of canonical splice variants, and replaced those splice variant ids with the corresponding Uniprot ids in the id mapping table (e.g. A0AUZ9-1 --&gt; A0AUZ9).</span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-0"><li class="c0 c4"><span>Mapped all Human Ensembl and Refseq sequences to Uniprot. Performed global sequence alignments using MUSCLE for cases where the sequences are not identical. </span></li><li class="c0 c4"><span>Wrote a MySQL function to convert AA position from Ensembl or Refseq coordinates to Uniprot coordinates.</span></li></ul><p class="c0 c19"><span></span></p><hr><p class="c0 c19"><span></span></p><a href="#" name="fbb12dc97b272b377ca8ed20d75b5f46ec33759e"></a><a href="#" name="7"></a><table cellpadding="0" cellspacing="0" class="c72"><tbody><tr class="c75"><td class="c90" colspan="1" rowspan="1"><p class="c52 c0 c12"><span class="c45 c22 c97">Ensembl to Uniprot</span></p></td><td class="c90" colspan="1" rowspan="1"><p class="c43 c32 c12"><span class="c45 c22 c97">Refseq to Uniprot</span></p></td></tr><tr class="c75"><td class="c90" colspan="1" rowspan="1"><p class="c10 c43 c64 c15"><span class="c44 c16">83,037 exact matches</span></p><p class="c10 c43 c64 c15"><span class="c27">83,008 unique ensps (83.5%)</span></p><p class="c10 c43 c64 c15"><span class="c27">72,532 unique uniprots</span></p><p class="c5 c43 c64 c12"><span class="c27"></span></p><p class="c10 c43 c64 c15"><span class="c44 c16">6,110 partial matches</span></p><p class="c10 c43 c64 c15"><span class="c27">5,505 unique ensps (5.5%)</span></p><p class="c10 c43 c64 c15"><span class="c27">5,021 unique uniprots</span></p></td><td class="c90" colspan="1" rowspan="1"><p class="c43 c32 c64 c15"><span class="c44 c16">45,056 exact matches</span></p><p class="c43 c32 c64 c15"><span class="c27">42,916 unique ensps (60%)</span></p><p class="c43 c32 c64 c15"><span class="c27">32,550 unique uniprots</span></p><p class="c43 c8 c12 c64"><span class="c27"></span></p><p class="c43 c32 c64 c15"><span class="c44 c16">11,743 partial matches</span></p><p class="c43 c32 c64 c15"><span class="c27">9,391 unique ensps (13%)</span></p><p class="c43 c32 c64 c15"><span class="c27">7,582 unique uniprots</span></p><p class="c43 c8 c15"><span class="c27"></span></p></td></tr><tr class="c75"><td class="c90" colspan="1" rowspan="1"><p class="c52 c0 c12"><span class="c16 c65">Sequence identity of partial matches</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 206.67px;"><img alt="" src="images/image100.png" style="width: 297.00px; height: 206.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c90" colspan="1" rowspan="1"><p class="c10 c43 c12"><span class="c16 c65">Sequence identity of partial matches</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 206.67px;"><img alt="" src="images/image69.png" style="width: 297.00px; height: 206.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c75"><td class="c90" colspan="1" rowspan="1"><p class="c10 c43 c12"><span class="c36 c16">Sequence coverage of partial matches</span></p><p class="c52 c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 206.67px;"><img alt="" src="images/image51.png" style="width: 297.00px; height: 206.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c90" colspan="1" rowspan="1"><p class="c10 c43 c12"><span class="c16 c36">Sequence coverage of partial matches</span></p><p class="c10 c43 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 206.67px;"><img alt="" src="images/image10.png" style="width: 297.00px; height: 206.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr></tbody></table><p class="c0 c19"><span></span></p><hr><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">Loading the results of various mutation-scoring algorithms to the mysql database in hg38 coordinates:</span></p><ul class="c3 lst-kix_emh9tw6ivvnv-0"><li class="c0 c4"><span class="c16">Ensembl Variants database</span><span>&nbsp;(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.ensembl.org/info/docs/api/index.html&amp;sa=D&amp;usg=AFQjCNFaKr6YSBP8epLYUMf-RXrUQXjuOA">http://www.ensembl.org/info/docs/api/index.html</a></span><span>)</span><span>: </span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-1 start"><li class="c0 c47"><span>SIFT and Polyphen-2 results for dbSNP, COSMIC, hapMAP, 1000 genomes, </span><span class="c48">etc.</span><span>&nbsp;(&lt;7 million transcript mutations).</span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-0"><li class="c0 c4"><span class="c16">Provean</span><span>&nbsp;(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://provean.jcvi.org/index.php&amp;sa=D&amp;usg=AFQjCNEgxa_78OtE7R-hS0OoQTeqznsZgw">http://provean.jcvi.org/index.php</a></span><span>)</span><span>: </span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-1 start"><li class="c0 c47"><span>SIFT and Provean results for dbSNP (~3 million transcript mutations).</span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-0"><li class="c0 c4"><span class="c16">Mutation Assessor</span><span>&nbsp;(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://mutationassessor.org/&amp;sa=D&amp;usg=AFQjCNH536FG3MBc7hTTCGB-1n5d9sIczA">http://mutationassessor.org/</a></span><span>)</span><span>: </span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-1 start"><li class="c0 c47"><span>MutationAssessor scores for all Uniprot proteins (~76 million protein mutations).</span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-0"><li class="c0 c4"><span class="c16">FannsDB</span><span>&nbsp;(</span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://bg.upf.edu/fannsdb/&amp;sa=D&amp;usg=AFQjCNHPhFeK9d1H9Qu00lFXJwUHxWj63A">http://bg.upf.edu/fannsdb/</a></span><span>)</span><span>: </span></li></ul><ul class="c3 lst-kix_emh9tw6ivvnv-1 start"><li class="c0 c47"><span>SIFT, Polyphen-2, MutationAssessor, FATHMM, and CONDEL scores (where available) (~240 million transcript mutations).</span></li></ul><p class="c52 c0 c15 c19"><span></span></p><p class="c52 c0 c15"><span>Converting Ensembl Variants mutations to Uniprot coordinates leaves ~6.2 million transcript mutations.</span></p><p class="c52 c0 c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 300.00px;"><img alt="" src="images/image26.png" style="width: 624.00px; height: 300.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c52 c0 c15"><span class="c16 c60 c20"><a class="c9" href="https://www.google.com/url?q=https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTWldwT0NKdUZHZTA/view?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNH0blhaZRwUGWWgJeDh1VXdhr-5eg">https://drive.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTWldwT0NKdUZHZTA/view?usp=sharing</a></span></p><p class="c52 c0 c15 c19"><span></span></p><p class="c0 c19"><span></span></p><hr><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">Machine learning for predicting the energetic impact of mutations:</span></p><ul class="c3 lst-kix_tfsdypepg13u-0 start"><li class="c0 c4"><span>Reparsing the Protherm data file. There are cases where the pdb id is incorrect, or no pdb id is given. Have to map those cases to uniprot individually.</span></li><li class="c0 c4"><span>Add the inverse of the training mutations to the training set, and use models in different bins of sequence identity.</span></li><li class="c0 c4"><span>Add data from </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www-02.all-portland.net/bj/424/0015/4240015.pdf&amp;sa=D&amp;usg=AFQjCNHlpnE6Z5tvO9gJbv9X6nk51o_OZA">http://www-02.all-portland.net/bj/424/0015/4240015.pdf</a></span><span>&nbsp;to the dTm training set.</span></li><li class="c0 c4"><span>See how much SIFT / Provean improve classifier performance.</span></li><li class="c0 c4"><span>Train a predictor for both ddG values and dTm values?</span></li><li class="c0 c4"><span>Add a feature quantifying the conservation of amino acid in the ~5A vicinity of the mutated residue. </span></li></ul><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 448.00px; height: 413.00px;"><img alt="" src="images/image16.png" style="width: 448.00px; height: 413.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span>After removing rows with multiple mutations, and rows where mutation is null:</span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 449.00px; height: 413.00px;"><img alt="" src="images/image112.png" style="width: 449.00px; height: 413.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c22"></span></p><hr><p class="c0 c19"><span class="c22"></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-10-xx</span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0 start"><li class="c0 c4"><span>Look for a larger ddG training set of domain and interface mutations</span></li></ul><ul class="c3 lst-kix_wawbbtzgf6il-0 start"><li class="c0 c47"><span>ASEdb: the Alanine Scanning Energetics database (</span><span><a class="c9" href="https://www.google.com/url?q=http://www.asedb.org&amp;sa=D&amp;usg=AFQjCNF2yffmiQlDF-CHt9lXJV2lkPqLGg">http://www.asedb.org)</a></span><span>.</span></li><li class="c0 c47"><span>Reparse the Protherm and Skempi datasets to include proteins without structures</span></li></ul><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Submitted all human uniprot to protherm / skempi</span></li><li class="c0 c4"><span>Look for a larger ddG training set of domain and interface mutations</span></li></ul><ul class="c3 lst-kix_s0bij7o2r512-0 start"><li class="c0 c47"><span>ASEdb: the Alanine Scanning Energetics database (</span><span><a class="c9" href="https://www.google.com/url?q=http://www.asedb.org&amp;sa=D&amp;usg=AFQjCNF2yffmiQlDF-CHt9lXJV2lkPqLGg">http://www.asedb.org)</a></span><span>.</span></li><li class="c0 c47"><span>Reparse the Protherm and Skempi datasets to include proteins without structures</span></li></ul><p class="c0 c15 c19"><span></span></p><p class="c0 c15 c19"><span></span></p><hr><p class="c0 c15 c19"><span></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-10-xx</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 329.33px;"><img alt="" src="images/image66.png" style="width: 624.00px; height: 329.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span>Predicted ddG scores (top) and FoldX ddG scores (bottom) for uniprot disease mutations and polymorphisms (left) and cosmic drivers and passengers (right).</span></p><p class="c0"><span class="c16 c40 c20"><a class="c9" href="https://www.google.com/url?q=https://docs.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTNHk0cmo5SlRIczQ/edit&amp;sa=D&amp;usg=AFQjCNEGsLRrs8yjmtBa1Q7Ca9VNVS9JNQ">https://docs.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTNHk0cmo5SlRIczQ/edit</a></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 300.00px;"><img alt="" src="images/image45.png" style="width: 624.00px; height: 300.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span>Feature importances for the protherm regressor.</span></p><p class="c0"><span class="c16 c40 c20"><a class="c9" href="https://www.google.com/url?q=https://docs.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTclBVSUtQdHlFTk0/edit&amp;sa=D&amp;usg=AFQjCNEoMoy9qkPTbto2lrmrhoHrDpYBoQ">https://docs.google.com/a/kimlab.org/file/d/0B1FXQpQb4bbTclBVSUtQdHlFTk0/edit</a></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-09-24</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Have calculated ~50,000 domain mutations and ~8,000 interface mutations.</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_z9n4e0cpc149-0 start"><li class="c0 c4"><span>Look for a larger ddG training set for domain and interface mutations.</span></li><li class="c0 c4"><span>Retrain the classifiers and update ddG predictions for the calculated mutations.</span></li><li class="c0 c4"><span>Update the script that parses log files to find and categorize errors. Look through the errors that occur and try to fix them.</span></li><li class="c0 c4"><span>For errors that are due to incorrect PDBfam definitions, look into the new version of PDBfam to make sure that they don&rsquo;t occur, and email Andres if they still occur.</span></li><li class="c0 c4"><span>Ask Andres if he can change the libraries that we use for finding templates for Uniprot to include </span><span class="c48">all </span><span>sequences, rather than only the non-redundant sequences.</span></li><li class="c0 c4"><span>Compute more protein-level features: </span></li></ul><ul class="c3 lst-kix_z9n4e0cpc149-1 start"><li class="c0 c47"><span>(average?) position of the protein in the pathways in which it is involved (with pathways transformed into directed graphs using the </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://graphite&amp;sa=D&amp;usg=AFQjCNHtRJp0L1z15IkGud5b5G39Krl21w">graphite</a></span><span>&nbsp;R package). </span></li><li class="c0 c47"><span>a vector indicating the &ldquo;type&rdquo; of pathways in which the protein is involved</span></li><li class="c0 c47"><span>average coexpression of a protein with all other proteins in the cell</span></li></ul><ul class="c3 lst-kix_z9n4e0cpc149-0"><li class="c0 c4"><span>Create disease-specific protein expression profiles using </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.lincscloud.org/l1000/&amp;sa=D&amp;usg=AFQjCNHkyQukgdzZT4ucwH-e9OxW59uMvA">the connectivity map</a></span><span>&nbsp;gene expression data.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-09-17</span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16 c22">Mutation training sets</span></p><p class="c0"><span class="c16">Cosmic drivers:</span><span>&nbsp;</span></p><p class="c0 c26"><span>&gt;= 10 publed ids</span></p><p class="c0 c26"><span>&nbsp;&gt;= 30 mutations</span></p><p class="c0 c26"><span>&nbsp;&gt;= 1 &ldquo;driver&rdquo; mutation predicted by FATHMM </span></p><p class="c0"><span class="c16">Cosmic passengers:</span></p><p class="c0 c26"><span>&lt; 10 publed ids</span></p><p class="c0 c26"><span>&nbsp;&lt; 30 mutations</span></p><p class="c0 c26"><span>&nbsp;0 &ldquo;driver&rdquo; mutation predicted by FATHMM</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 308.00px; height: 201.00px;"><img alt="" src="images/image37.png" style="width: 308.00px; height: 394.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 308.00px; height: 199.00px;"><img alt="" src="images/image37.png" style="width: 308.00px; height: 394.00px; margin-left: 0.00px; margin-top: -195.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c16">Uniprot disease and polymorphic proteins</span></p><p class="c0"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Parsed &lsquo;humsavar.txt&rsquo; file from unipot. </span><span class="c16">Disease proteins</span><span class="c22">&nbsp;</span><span>are those that have at least one mutation marked as &ldquo;Disease&rdquo;. </span><span class="c16">Polymorphic proteins </span><span>are those that only have mutations marked as &ldquo;Polymorphism&rdquo;.</span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Cosmic</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 278.50px; height: 278.50px;"><img alt="" src="images/image91.png" style="width: 278.50px; height: 278.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 280.50px; height: 279.48px;"><img alt="" src="images/image135.png" style="width: 280.50px; height: 279.48px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Uniprot</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 274.50px; height: 273.56px;"><img alt="" src="images/image113.png" style="width: 274.50px; height: 273.56px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 273.50px; height: 273.50px;"><img alt="" src="images/image29.png" style="width: 273.50px; height: 273.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Gene essentiality</span></p><p class="c0 c19"><span></span></p><p class="c0 c50"><span>&nbsp; &nbsp; &nbsp;Uniprot&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; Cosmic</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 281.96px; height: 204.50px;"><img alt="" src="images/image47.png" style="width: 281.96px; height: 204.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 282.88px; height: 201.50px;"><img alt="" src="images/image88.png" style="width: 282.88px; height: 201.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Physical interaction network (Hippie)</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 520.00px;"><img alt="" src="images/image77.png" style="width: 624.00px; height: 520.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Genetic interaction network (human radiation hybrid study)</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 521.33px;"><img alt="" src="images/image53.png" style="width: 624.00px; height: 521.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Functional interaction network (String)</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 521.33px;"><img alt="" src="images/image141.png" style="width: 624.00px; height: 521.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span class="c16">Functional interaction network (String)</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 400.00px;"><img alt="" src="images/image139.png" style="width: 624.00px; height: 400.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 52.00px;"><img alt="" src="images/image60.png" style="width: 624.00px; height: 52.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 360.00px;"><img alt="" src="images/image116.png" style="width: 624.00px; height: 360.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-09-10</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Updated to the new version of FoldX (version 3 beta). Changed the pipeline to include water molecules in domain structures (including interfaces). Had to find and set some parameters in modeller in order for modeller to work.</span></li><li class="c0 c4"><span>Noticed that tcoffee was running for an unusually long amount of time for some structures. Eventually discovered that in order to perform structural alignments, tcoffee is querying the pdb online. While we supplied tcoffee with the path to the local pdb, it wasn&rsquo;t using it because it expects pdbs to be split into individual chains. Replaced the &lsquo;expresso&rsquo; mode where tcoffee tries to find the best structural template with a structural alignment where both structures use the structure of the template (tmalign_pair, muscle_pair, sap_pair). The alignments between distant homologues can be improved using hidden markov model profiles (to do?).</span></li><li class="c0 c4"><span>Added quotes around file names in the code because the asterisk in </span><span class="c48">&ldquo;elaspic_v2/human/P09/00/P09001/Ribosomal_L3</span><span class="c22 c48">*</span><span class="c48">103-300&rdquo; </span><span>would cause buggy behaviour. Replaced the asterisk with dot so that now it is &ldquo;</span><span class="c48">Ribosomal_L3.103-300&rdquo;</span><span>.</span></li><li class="c0 c4"><span>Incorrect domain definitions for some domains in PDBfam. The end of the domain is smaller than the start. Did manual correction for some of the biggest culprits (GPCRs). Emailed Andres. He is working on it&hellip;</span></li><li class="c0 c4"><span>Uploaded splicing variants to the database. Still running elaspic on them&hellip; (servers are packed)</span></li></ul><p class="c0"><span class="c22">Known elaspic errors</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c52 c0 c4 c15"><span>Sometimes you end up with poor alignments because MUSCLE puts in too many gaps, and later on modeller crashes when it has to model long loops.</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>All from below</span></li><li class="c0 c4"><span>Expand pfam profiles or create new hidden alignments for PDBfam domains. This could be used by FATHMM for predicting the pathogenicity of mutations, for creating better alignments, for predicting coevolving residues, etc.</span></li></ul><p class="c0 c19"><span></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-08-27</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Mapping </span><span class="c16">99,459</span><span>&nbsp;human protein sequences from Ensembl to Uniprot:</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Using the id conversion table provided by Uniprot, find an exact sequence match for </span><span class="c16">70,013</span><span>&nbsp;sequences (plus 32 duplicates).</span></li><li class="c0 c47"><span>Using sequence hash tables, find an exact sequence match for </span><span class="c16">90,351 </span><span>sequences. However, there are </span><span class="c16">44,442</span><span>&nbsp;duplicates, and choosing the correct match is problematic.</span></li><li class="c0 c47"><span>Using the external references provided with the Ensembl database, find an HGNC match for </span><span class="c16">96,928</span><span>&nbsp;sequences, a Uniprot match for </span><span class="c16">48,938</span><span>&nbsp;sequences and an exact Uniprot sequence match for only </span><span class="c16">6,878</span><span>&nbsp;sequences (have to map through the ensembl &lsquo;gene&rsquo; identifier).</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>For COSMIC v70, 1,378,894 mutations have a ENST accession, and 1,002,253 of those can be mapped to a uniprot with an identical sequence (can increase this a bit if I also include refseq accessions).</span></li><li class="c0 c4"><span>Ran 24,774 COSMIC &ldquo;driver&rdquo; mutations that I identified last week. </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>The structural modelling is done but I don&rsquo;t have ddG for most of them because Provean is still running (will be done in a couple of hours)</span></li></ul><p class="c0 c19"><span></span></p><p class="c10 c53"><span>COSMIC_DRIVER_V70</span></p><p class="c10 c53"><span class="c16">Domain errors:</span></p><p class="c10 c53"><span>0. Protein does not have any pfam domains: </span><span class="c22">10122</span><span>&nbsp;(1466 proteins, 1008 of which are splice variants...)</span></p><p class="c10 c53"><span>1. The mutation is outside of a domain but not all domains have structural templates: 0</span></p><p class="c10 c53"><span>2. The mutation is outside of a domain for all domains: 2914</span></p><p class="c10 c53"><span>7. A model of the domain containing the mutation could not be created: </span><span class="c22">357</span></p><p class="c10 c53"><span>8. The energetic impact of the mutation could not be calculated: 4047</span></p><p class="c10 c53"><span>Completed: 727 (66 proteins)</span></p><p class="c10 c53"><span class="c16">Interface errors:</span></p><p class="c10 c53"><span>2. The mutation is outside of a domain for all domains: 2498</span></p><p class="c10 c53"><span>5. Could not build a model of the interaction: </span><span class="c22">303</span></p><p class="c10 c53"><span>6. Mutation is outside the interaction interface: 2829</span></p><p class="c10 c53"><span>8. The energetic impact of the mutation could not be calculated: 1013</span></p><p class="c10 c53"><span>Completed: 365 (18 proteins)</span></p><p class="c5 c53"><span></span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Finally fixed the Provean bug that lead to insanely-high temp space usage (a bug in cd-hit made it go in an infinite loop if &ldquo;long&rdquo; sequences were present in the alignment)</span></li><li class="c0 c4"><span>Mined the Ensembl database to get mutation sets from Uniprot, ClinVar, PhenCode and LSDB. The mutation set for HGMD does not include the changed amino acid (I guess you have to pay HGMD to get this information&hellip;)</span></li><li class="c0 c4"><span>Downloaded the gene ontology and ~6000 gene enrichment datasets for pathways, etc. from GSEA (excluding GO): </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>The idea is to use a log-odds score for the GO terms, and a set of principal component vectors for the GSEA protein sets as some of the features.</span></li></ul><p class="c0 c12 c53"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 265.00px; height: 43.00px;"><img alt="" src="images/image85.png" style="width: 265.00px; height: 43.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Finish running Provean and calculate ddG for COSMIC mutations. Run HMMER on splicing variants (email Joan and Andres)?.. Check the data for consistency and look for interesting changes / pathway rewiring, etc.</span></li><li class="c0 c4"><span>Read the FATHMM paper and play with their code. A good way of dividing COSMIC mutations into drivers and passengers?</span></li><li class="c0 c4"><span>Divide iRefIndex data into organism-specific protein-protein interaction and genetic interaction networks. </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Compare the human PPI network with Biogrid to make sure that it is more complete. </span></li><li class="c0 c47"><span>Create two PPI networks, one with high-confidence interactions and the other with low confidence interactions?</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Create a functional interaction network using STRING.</span></li><li class="c0 c4"><span>Calculate some network-based features for the nodes and edges in those networks.</span></li><li class="c0 c4"><span>Think about better ways of using pathway information?</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-08-20</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c52 c0 c4 c15"><span>Found templates for all uniprot domain pairs.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c52 c0 c47 c15"><span>In total we have templates for 17629 domain-domain interactions (out of 78839 interactions between domains that have a structure of the corresponding clan-clan interaction). This is up from 16204 interaction templates that we had before.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c52 c0 c4 c15"><span>Parsed the new version of COSMIC (v70)</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c52 c0 c47 c15"><span>The new version has 1,424,054 (redundant) missense variants</span></li><li class="c52 c0 c47 c15"><span>1,274,335 of the variants (90%) could be mapped to an ensembl or refesq sequence</span></li><li class="c52 c0 c47 c15"><span>765,745 of the variants (54%) could also be mapped to a uniprot sequence </span></li><li class="c52 c0 c47 c15"><span>For 708,477 of the variants (50%) the ensembl and the uniprot sequence are the same.</span></li><li class="c52 c0 c47 c15"><span>This corresponds to 489,228 non-redundant mutations in 13,516 proteins</span></li><li class="c52 c0 c47 c15"><span>24,774 mutations (in 3,560 proteins) are labelled as drivers by fathmm. </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c52 c0 c4 c15"><span>Downloaded tables for the </span><span class="c16">ENSEMBL core </span><span>database and the </span><span class="c16">ENSEMBL variants </span><span>database and uploaded them to the MySQL server. </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c52 c0 c47 c15"><span>The </span><span class="c16">ENSEMBL core</span><span>&nbsp;database contains ENSG -&gt; ENST -&gt; ENSP mapping, alternative splicing, mapping to external identifiers</span></li><li class="c52 c0 c47 c15"><span>The </span><span class="c16">ENSEMBL variants</span><span>&nbsp;database contains curated variants from a large number of databases, including dbSNP, COSMIC, OMIM, Orphanet, HGMD, and ClinVar, together with SIFT and PolyPhen-2 scores for missense mutations, allele frequency from the 1000 genomes project, and the associated disorders.</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Run a large set of disease variants and negative controls (maybe using SciNet)?</span></li><li class="c0 c4"><span>Load Reactome and GO information to the database. Look into updates to iRefIndex and ways of filtering it to create a reliable protein-protein interaction network.</span></li><li class="c0 c4"><span>...Run 2 more steps in </span><span class="c48">scowlp</span><span>&nbsp;(structural alignment and clustering of domains and Rosetta PeptideDeriver to find the peptides with the highest binding energy for each domain).</span></li></ul><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-08-13</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>There was a small bug in the template finding code (one of the uniprot sequences was missing), but it took a long time to track this down.</span></li><li class="c0 c4"><span>PDBfam still leaves some overlapping domains, so I corrected those manually. </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>If the domains overlap by &lt; 50% of their length, I truncated them at the halfway point in their overlap</span></li><li class="c0 c47"><span>If the domains overlap by &gt; 50% of their length, I removed the shortest domain.</span></li><li class="c0 c47"><span>This leads to the removal of 459184 domains (&lt; 2%)</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>After including pdbs without CATH, we have structural templates for 73% of all domains (up from 68% when pdbs without CATH were not used).</span></li><li class="c0 c4"><span>Switched from DSSP to Stride for predicting secondary structure (DSSP and Stride are reported to give the same results in </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.cse.fau.edu/~xqzhu/courses/cot6930_08spring/Handout/ProteinSecondary.pdf&amp;sa=D&amp;usg=AFQjCNEmrDG521m5Fnn168SEa8Jy_2vo2Q">95% of the cases</a></span><span>, and Stride may be better because it uses hydrogen bonding in addition to torsion angles?)</span></li><li class="c0 c4"><span>Running this for the Protherm training set we now get 3180 out of 3384 mutations (94%).</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>17 mutations are in proteins without pfam domains</span></li><li class="c0 c47"><span>179 mutations are outside all of the pfam domains</span></li><li class="c0 c47"><span>9 mutations need closer examination</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Find domain-domain interaction templates</span></li><li class="c0 c4"><span>Preparing a set of cosmic driver and passenger mutations (mapping amino acid positions from ENST to Uniprot&hellip;)</span></li><li class="c0 c4"><span>Run 2 more steps in </span><span class="c48">scowlp</span><span>&nbsp;(structural alignment and clustering of domains and Rosetta PeptideDeriver to find the peptides with the highest binding energy for each domain).</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-08-06</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Decided to fix some of the bugs in </span><span class="c48">scowlp</span><span>&nbsp;code and rerun it for the pdb (part 2)...</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>For each pdb, put all the communication between the program and the database inside a transaction, so that interrupting the program part-way does not result in data &ldquo;corruption&rdquo;.</span></li><li class="c0 c47"><span>Introduced all the required unique constraints (e.g. domain_1 and domain_2 for a domain_contact table) and foreign_key constraints (e.g. atom -&gt; residue) into the database and fixed any inconsistencies.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Rerunning trembl using the new pdbfam library (with pdbs without cath information) is ~80% complete.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>R crashes in some cases and this has to be fixed. </span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Find templates for uniprot domains and uniprot domain pairs</span></li><li class="c0 c4"><span>Fix the dssp bug</span></li><li class="c0 c4"><span>Run elaspic on the training sets and a large set of disease mutations</span></li><li class="c0 c4"><span>Run additional steps in </span><span class="c48">scowlp</span><span>&nbsp;(run rosetta peptide-deriver and the structural classification of interfaces)</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-07-30</span></p><p class="c0"><span class="c22">Progress:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Decided to fix some of the bugs in </span><span class="c48">scowlp</span><span>&nbsp;code and rerun it for the pdb</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Fixed the a bug that caused a crash if domain definitions involve a letter (1A:6D)</span></li><li class="c0 c47"><span>Changed scowlp code to allow for parallel processing on multiple nodes. (However, if too many jobs are submitted at the same time, the MySQL database causes problems).</span></li><li class="c0 c47"><span>Ran the code for finding domain-domain interactions and for filtering those interactions using Noxclass</span></li><li class="c0 c47"><span>After rerunning scowlp we have 330,756 domain-domain interactions involving 58,079 pdbs (vs. 343,133 domain-domain interactions involving 57,672 pdbs that we had before??)</span></li><li class="c0 c47"><span>Trying to figure out why I have fewer domain-domain interactions in the new version. Pdbs that appear at least once are not included when you rerun the program&hellip;</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Ran the elaspic pipeline on the Protherm dataset using templates precalculated with pdbfam</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Back to 1404 out of 3384 mutations </span></li><li class="c0 c47"><span>Correlation coefficient between FoldX ddG and observed ddG is 0.351, which is similar to what it was before</span></li><li class="c0 c47"><span>Judging by the logs, most of the failed mutations are due to an error raised by the dssp (</span><span class="c48">boost::thread_resource_error</span><span>) on the new nodes. This is due to boost starting too many threads on the node. I had this error before and just put in a &ldquo;retry in a minute statement&rdquo;, but seems like this is not enough.</span></li></ul><p class="c10 c53"><span class="c16">PROTHERM</span></p><p class="c10 c53"><span>Domain errors:</span></p><p class="c10 c53"><span>0. Protein does not have any pfam domains: 17</span></p><p class="c10 c53"><span>1. The mutation is outside of a domain but not all domains have structural templates: 0</span></p><p class="c10 c53"><span>2. The mutation is outside of a domain for all domains: 179</span></p><p class="c10 c53"><span>7. A model of the domain containing the mutation could not be created: 0</span></p><p class="c10 c53"><span>8. The energetic impact of the mutation could not be calculated: 1784</span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Fix the inconsistencies with the scowlp domain-domain interactions</span></li><li class="c0 c4"><span>Run pdbfam using new libraries (both with and without cath) and using new scowlp output.</span></li><li class="c0 c4"><span>Find templates for domain pairs</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-07-23</span></p><p class="c0"><span class="c22">Elaspic:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Wrote code for finding the best structural template for uniprot domain pairs. Sort templates based on the product of the overall and interface scores.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c47 c32"><span>score = (alpha * (identical + 0.5*conserved) * coverage) - (1 - alpha) * coverage</span></li><li class="c47 c32"><span>score_if = identical_if + 0.5 * conserved_if</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Made changes to the pipeline and it now runs well using the new domain and domain pair templates.</span></li><li class="c0 c4"><span>The trembl dataset contains </span><span class="c22">18,187,173</span><span>&nbsp;uniprots and </span><span class="c22">23,743,525 </span><span>pdbfam domains. Found structural templates for </span><span class="c22">16,188,914 </span><span>of those domains (last week).</span></li><li class="c0 c4"><span>There are </span><span class="c22">61,821 </span><span>unique uniprot domain-domain interactions with a structural template involving the same pdbfam pair.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Found domain-domain interaction templates for </span><span class="c22">17,101 </span><span>of those uniprot domain pairs.</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>There are </span><span class="c22">108,159 </span><span>unique uniprot domain-domain interactions with a structural template involving the same clan pair (</span><span class="c22">46,338</span><span>&nbsp;interactions found using pfam clan pairs and not using pdbfam pairs).</span></li><li class="c0 c4"><span>But! Realised that I did not remove from the new version of scowlp domain-domain interactions that occur on the same chain in the pdb. (Previously I was ignoring those pairs inside the pipeline, but here it is easiest if I remove them from the database).</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Instead of </span><span class="c22">343,133 </span><span>domain-domain contact templates we now have </span><span class="c22">179,739</span></li><li class="c0 c47"><span>This lowers the number of &ldquo;uniprot domain-domain interactions with a structural template involving the same clan pair&rdquo; to </span><span class="c22">89,846</span><span>.</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4 c15 c52"><span>Find templates for domains and domain pairs using the new libraries that include pdbs without CATH (Andres&rsquo; made the new libraries).</span></li><li class="c52 c0 c4 c15"><span>Find templates for uniprot domain pairs that have a pfam clan match for their interaction.</span></li><li class="c52 c0 c4 c15"><span>Make sure that the interface scores match up with the interface amino acids, and so some sanity checks on produced alignments and models.</span></li><li class="c52 c0 c4 c15"><span>Retrain the classifiers using models obtained with the new template-finding algorithm</span></li><li class="c52 c0 c4 c15"><span>Edit the </span><span class="c48">scowlp</span><span>&nbsp;code to fix the errors thrown when the domain starts or ends with a residue that has a letter in the resnum.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-07-16</span></p><p class="c0"><span class="c22">SCOWLP:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c52 c0 c4 c15"><span>Changed the SQL query to prevent the table from being written to disk and sorted. This made it way faster, especially when splitting the job into multiple subprocesses. (Joining domain_contact with two domain tables and sorting by domain contact id). </span></li><li class="c52 c0 c4 c15"><span>Changed Noxclass code to prevent crashes caused by multiple naccess *.rsa files being present in the same folder.</span></li><li class="c52 c0 c4 c15"><span>In total, the new version of scowlp has 343,133 domain-domain contacts (the old version using non-redundant domain contacts from pdb biounits had 237,620 domain-domain contacts). </span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c52 c0 c47 c15"><span>32,553 domain contacts were not analysed by Noxclass (mainly because they are domain-ligand contacts)</span></li><li class="c52 c0 c47 c15"><span>6,847 domain contacts led to a crash because the interface area according to NACCESS is zero? This is probably a bug in scowlp...</span></li><li class="c52 c0 c47 c15"><span>243 timeouts</span></li><li class="c52 c0 c47 c15"><span>319 other errors</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>For the elaspic domain contact table, filter the results using a 70% NOXclass cutoff (didn&rsquo;t do this in the previous version of elaspic&hellip;)<br></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 494.00px; height: 340.81px;"><img alt="noxclass_hist.png" src="images/image34.png" style="width: 494.00px; height: 340.81px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></li></ul><p class="c0"><span class="c22">PDFfam:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Removed overlapping domains in the uniprot domain definition file from pfam.</span></li><li class="c0 c4"><span>Used Andres&rsquo; PBfam to join superdomains and repeating domains and to find structural temples for the entire uniprot. However, the latest version of pfam uses the</span><span class="c22">&nbsp;June 2012 uniprot release...</span><span>&nbsp;</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>&nbsp;Made scripts to interact with R code from python. Could integrate R into the elaspic pipeline</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Encountered a few bugs with the PDBfam package (missing blast databases for a few families and minor inconsistency in family naming). Emailed Andres and he fixed this. </span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">ELASPIC:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Changing the structure of the database and code to work with the new version of PDBfam (don&rsquo;t need a template table since templates and domains are found at the same time, etc.).</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Works for domain mutations</span></li><li class="c0 c47"><span>Still have to make it work for interface mutations (write code for finding the best templates for the interaction using blast).</span></li></ul><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">Why did the old way of finding templates take so long?</span></p><p class="c52 c0 c15"><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://pfam.xfam.org/protein/P21817&amp;sa=D&amp;usg=AFQjCNEHbNI1or-_ABIRBS7c6nQj5GU7Ag">P21817</a></span><span>&nbsp;(Ryanodine receptor 1) 13 Pfam domains</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 17.33px;"><img alt="" src="images/image24.png" style="width: 624.00px; height: 17.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 20.00px;"><img alt="" src="images/image126.png" style="width: 624.00px; height: 20.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 21.33px;"><img alt="" src="images/image44.png" style="width: 624.00px; height: 21.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c12"><span class="c16">Finding domain templates</span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 489.00px; height: 343.03px;"><img alt="" src="images/image68.png" style="width: 489.00px; height: 343.03px; margin-left: 0.00px; margin-top: -20.31px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span>115 templates analysed&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;takes ~ 2 hours </span></p><p class="c0 c12 c19"><span class="c22"></span></p><p class="c0 c12"><span class="c16">Finding domain pair templates</span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 486.39px; height: 365.50px;"><img alt="" src="images/image124.png" style="width: 486.39px; height: 365.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span>620 templates analysed&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;takes ~13 hours</span></p><p class="c0"><span>41 uniprot proteins interact with Ryanodine receptor 1. Have to find the best template for each of those interactions. &gt; 10 family cluster pairs for each interaction. Results from previous runs are not reused, so a lot of redundant work. </span></p><p class="c0 c19"><span class="c22"></span></p><hr><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-07-02</span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">New scowlp:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Script for running Noxclass crashes</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Was trying to make it run in parallel, but couldn&rsquo;t get it to work</span></li><li class="c0 c47"><span>Even as a single thread it stops part-way through</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Setting up python and R on the cluster</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c0 c47"><span>Had trouble making R and python work together on the cluster</span></li><li class="c0 c47"><span>Have to recompile a separate version of R (the one on banting is outdated)</span></li><li class="c0 c47"><span>R requires the readline library which was not installed on banting, so had to install it in a local folder.</span></li><li class="c0 c47"><span>The readline library is different of banting and beagle, so need separate python installations for the two clusters.</span></li><li class="c0 c47"><span>Making rpy2 (the interface between python and r) link against the new R version&hellip;&hellip;..</span></li><li class="c0 c47"><span>In the end, switched to conda, which offers binary package management for python. Conda included a local version of readline and other required system libraries, and so the same package can be run on banting and beagle. Downloaded R binaries which were compiled using conda, and set up build scripts to install all the required packages for Andres&rsquo; PDBfam and PDBfam itself. The advantage of this approach is that elaspic can be installed together with all of its R and python dependencies using `conda install -c path_to_repo elaspic`</span></li></ul><p class="c0 c53 c19"><span></span></p><p class="c0"><span class="c22">Slow domain expansion:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c0 c4"><span>Many pfam domains that take forever (e.g. </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://pfam.xfam.org/protein/Q8WZ42&amp;sa=D&amp;usg=AFQjCNHlWHx_UEt3qAR9Dh89jM_9Hv7sBw">Q8WZ42</a></span><span>)</span></li><li class="c0 c4"><span>Structural alignments take minutes</span></li></ul><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-06-18</span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c16 c22 c18">Debugging the pipeline</span></p><p class="c0"><span class="c22">Protherm:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>2957 out of 3384 mutations (</span><span class="c22">87%</span><span>). </span></li><li class="c1 c56"><span>Correlation between FoldX ddG and experimental ddG: </span><span class="c22">0.30</span></li><li class="c1 c56"><span>Correlation between FoldX ddG and experimental ddG using structures (Niklas): </span><span class="c22">0.47</span></li></ul><ul class="c3 lst-kix_4roymkz9vma-1 start"><li class="c6 c56"><span>(The previous training set had 30 duplicates involving the same mutation in the same protein but mapped to a different chaind and / or a different pdb)</span></li></ul><p class="c10 c12 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 605.00px; height: 340.00px;"><img alt="" src="images/image06.png" style="width: 605.00px; height: 340.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">Skempi:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>426 out of 862 mutations (</span><span class="c22">49%</span><span>). </span></li><li class="c1 c56"><span>Correlation between FoldX ddG and experimental ddG: </span><span class="c22">0.47</span></li><li class="c1 c56"><span>Correlation between FoldX ddG and experimental ddG using structures (Niklas): </span><span class="c22">0.44</span></li></ul><p class="c10 c12 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 552.50px; height: 356.54px;"><img alt="" src="images/image05.png" style="width: 552.50px; height: 356.54px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c22"></span></p><p class="c0"><span class="c22">Disease:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>1944 out of 2721 mutations (</span><span class="c22">71%</span><span>) </span></li></ul><p class="c10 c12 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 605.00px; height: 340.00px;"><img alt="" src="images/image28.png" style="width: 605.00px; height: 340.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c10 c12 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 542.50px; height: 381.76px;"><img alt="" src="images/image121.png" style="width: 542.50px; height: 381.76px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c56"><span></span></p><p class="c0"><span class="c16 c22 c18">Machine learning</span></p><p class="c0"><span class="c22">Protherm:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>Leave-one-out cross-validation while excluding members of the same pfam family and clan from the training set results in a correlation coefficient of </span><span class="c22">0.68</span><span>&nbsp;(without any optimisation).</span></li></ul><p class="c10 c12 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 564.50px; height: 423.38px;"><img alt="" src="images/image48.png" style="width: 564.50px; height: 423.38px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c56"><span></span></p><p class="c0"><span class="c22">Skempi:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>A very repetitive training set, involving only 37 different uniprots from 34 different pfam families.</span></li><li class="c1 c56"><span>Performing leave-one-out cross-validation where members of the same pfam family and clan are excluded from the training set results in 87 or 103 training data points in the majority of cases.</span></li><li class="c1 c56"><span>Correlation coefficient is only </span><span class="c22">0.37</span><span>&nbsp;(without optimization and provean)</span></li><li class="c1 c56"><span>Correlation coefficient is </span><span class="c22">0.74</span><span>&nbsp;when using plain leave-one-out cross-validation.</span></li></ul><p class="c5 c56"><span></span></p><p class="c10 c56"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 322.67px;"><img alt="" src="images/image74.png" style="width: 624.00px; height: 322.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c56"><span></span></p><p class="c0"><span class="c16 c22 c18">Hanif&rsquo;s scripts</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>Finished making all the domain contact and ligand contact tables, together with the associated residue-level and atom-level mapping tables (took half a week to run on a single node).</span></li><li class="c1 c56"><span>Need to run noxclass, and will then have all the information required to use the new pdbfam for the pipeline.</span></li><li class="c1 c56"><span>Errors when the pdb domain starts with a mixed number/letter (e.g. 10A)</span></li></ul><p class="c5 c56"><span></span></p><p class="c10 c56"><span class="c16 c22 c18">To do:</span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>Change the template-finding / alignment stage of the pipeline to use Andres&rsquo; approach with blast libraries for each pdbfam family.</span></li><li class="c1 c56"><span>Ask Andres to edit his scripts to make it possible to use pdb annotations from pfam?</span></li><li class="c1 c56"><span>Need a larger training set, particularly for interface mutations. Do not need an exact crystal structure. Maybe even use peptide-binding information?</span></li><li class="c1 c56"><span>Edit Hanif&rsquo;s / Joan&rsquo;s scripts to fix bugs and to make it take into account small molecules (drugs). Could then use chemical similarity (Tanimoto coef., etc.) to find similar drugs that would target the same protein. </span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-06-04</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>&ldquo;fn3|1&rdquo; from pdb 3r8qA fixed from Sebastian&rsquo;s version.</span></li><li class="c1 c56"><span>Protherm: </span><span class="c22">3077</span><span>&nbsp;out of 3384 mutations; (dG_wt, dG_mut, ddG): (0.025, 0.027, 0.446)</span></li><li class="c1 c56"><span>Skempi: </span><span class="c22">475</span><span>&nbsp;out of 862 mutations; (dG_wt, dG_mut, ddG): (0.092, 0.079, 0.679)</span></li><li class="c1 c56"><span>Disease: </span><span class="c22">2209</span><span>&nbsp;out of 2721 mutations; (dG_wt, dG_mut, ddG): (0.408, 0.408, 0.467)</span></li></ul><p class="c5 c56"><span class="c22"></span></p><p class="c10 c56"><span class="c22">To do:</span></p><p class="c10 c56"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;new pdbfam</span></p><p class="c10 c56"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hanif&rsquo;s scripts</span></p><p class="c0 c19"><span class="c7"></span></p><hr><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-05-28</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_4roymkz9vma-0"><li class="c1 c56"><span>Good structural templates do not necessarily come from the same pfam family (</span><span class="c48">e.g. </span><span>no structural templates for &ldquo;GSH_synthase&rdquo; but get a good match for &ldquo;GSH_synth_ATP&rdquo;). This goes back to the problem Andres was trying to solve in the previous subgroup meeting, by constructing blast libraries by clan name rather that family name. </span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c6 c56"><span>I modified the pipeline so that, if a structural template that exactly matches the pfam name cannot be found, the pipeline looks for structural templates for pfam names that have the same prefix only. For example, for the superdomain &lsquo;A_1+B_2+C_3&rsquo;, the pipeline would first try to find a structural template that has the exact name &lsquo;A_1+B_2+C_3&rsquo;. If it does not find any hits, it would then try to find unique structural templates for &lsquo;A&rsquo;, &lsquo;A+%&rsquo;, A_%, &lsquo;%+A&rsquo;, &lsquo;%+A+%&rsquo;, or &lsquo;%+A_%&rsquo; (% matches anything) and likewise for &lsquo;B&rsquo; and &lsquo;C&rsquo;. </span></li></ul><p class="c5 c53 c56"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0 start"><li class="c1 c56"><span>Should we use pfam definitions downloaded from the pfam website or pfam definitions obtained by parsing hmmer output? </span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c6 c56"><span>The two do not always match and this leads to missing domains and other inconsistencies when finding structures for uniprot domains.</span></li><li class="c6 c56"><span>For example, below are pfam and hmmer domain definitions for chain A in the pdb 1m6i:</span></li></ul><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 324.00px; height: 62.00px;"><img alt="" src="images/image142.png" style="width: 324.00px; height: 62.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 372.00px; height: 106.00px;"><img alt="" src="images/image94.png" style="width: 372.00px; height: 106.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c56"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c1 c56"><span>Added a 25% hard cutoff for sequence identity (for either of the two chains) to speed up computation.</span></li></ul><p class="c5 c56"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c1 c56"><span>FoldX fails when the PDB has methylated residues: MLZ (N-Methyl-Lysine), MLY (N-Dimethyl-Lysine) and M3L (N-Trimethyllysine). Emailed the FoldX group but they said that this is expected. Modified the pipeline so that the methyl carbons are removed and the residue is renamed to LYS.</span></li></ul><p class="c5 c56"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c1 c56"><span>&nbsp;Additional errors:</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c6 c56"><span>Difficult to know how much memory would be required for each job. FoldX and dssp sometime quit with some form of a thread error, (&ldquo;thread_resource_error&rdquo;, could not start a new thread, etc.)</span></li><li class="c6 c56"><span>Some structures still fail the check to make sure that the uniprot and pdb are aligned. This is difficult to track down. Sometimes the error is not reproduced when running the job locally.</span></li><li class="c6 c56"><span>Have difficult uniprots with many repeating domains, with each of those domains potentially responsible for the same set of interactions.</span></li></ul><p class="c5 c56"><span></span></p><p class="c10 c56"><span>Protherm data: 3089 out of 3414 (90%) but only 0.58 correlation coefficient in ddG.</span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Using pfam domain definitions downloaded from the pfam website rather than hmmer output? </span></li><li class="c0 c4"><span>Run the pipeline using raw structures to make sure that the output matches Niklas.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-05-21</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Genome3D: a compilation of UK resources for predicting and modelling CATH and SCOP domains for Uniprot proteins on the genome scale </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://www.genome3d.eu/uniprot/id/P04637/annotations&amp;sa=D&amp;usg=AFQjCNGZ8m2Pm_mS-EXbYT_UhRVL2BWRUg">http://www.genome3d.eu/uniprot/id/P04637/annotations</a></span></li><li class="c0 c4"><span>Interactome3D data is available for download: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://interactome3d.irbbarcelona.org/download.php&amp;sa=D&amp;usg=AFQjCNFGMbolKOMWuoqgqIu0z0uXRA8gBQ">http://interactome3d.irbbarcelona.org/download.php</a></span></li><li class="c0 c4"><span>SCOP got an update (SCOP2): </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://scop2.mrc-lmb.cam.ac.uk/&amp;sa=D&amp;usg=AFQjCNH8jqcdCvvQgdnuKtAhAjyIfJtzgA">http://scop2.mrc-lmb.cam.ac.uk/</a></span></li><li class="c0 c4"><span>Added the option to use mmCIF files. However, PDBFam (and interactome3D) is based on biounits, which are available for download only in pdb format, so went back to pdb. If we recreate biounit information from mmCIF file headers, then we can use mmCIF.</span></li><li class="c0 c4"><span>Changed the pipeline so that, if model building fails, the pdb template is added to the blacklist and the pipeline runs again to find the new best template. This should happen only when there is no other way to correct the error (</span><span class="c48">i.e.</span><span>&nbsp;the pdb file is hopeless), so I am not using this when debugging.</span></li><li class="c0 c4"><span>Rewrote the way heteroatoms are handled after parsing the pdb. All heteroatoms are moved to a separate chain which follows the chain(s) with all the amino acids for a single domain or a domain pair. This does not change the 3D structure, but makes it easier to create an alignment for modeller that does not give the &ldquo;Alignment sequence not found in PDB file&rdquo; error. This also simplifies other aspects of the pipeline as well: no need for heteroatom flags; a single domain can be found in chain [A]; a domain pair can be found in chains [A, B], with the order corresponding to the order of uniprots in the database (i.e. uniprot_id_1, uniprot_id_2).</span></li><li class="c0 c4"><span>Resubmitted the skempi and protherm jobs, and they are still running...</span></li></ul><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Go over the error logs for the new set of jobs...</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-05-14</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Removed duplicate items from `uniprot_domain_pair` (a table that lists all pairs of domains for uniprots that are known to interact and for which we have a structural template). Duplicate items were causing mismatches between alignments and models because alignment files would be saved in the same folder, overwriting alignments that are already there. Alignment for the same domain do not have to be the same because there is some randomness in the template expansion process.</span></li><li class="c0 c4"><span>Pipeline now removes H_ACE from the N-terminus of PDBs (not present in the uniprot sequence but present in the modeller pdb as part of the chain and not as a heteroatom).</span></li><li class="c0 c4"><span>Changed the python file that accesses the database to keep it&rsquo;s access to the database transient rather than maintaining a transaction the entire time that the pipeline is running (in some cases, I think this was causing missing values in the database if the pipeline was killed before completing its job, </span><span class="c48">i.e. </span><span>some mutations don&rsquo;t have model information).</span></li><li class="c0 c4"><span>Changed input to FoldX from 15 AA sequence snippets (which would cause errors for dimers, etc.) to a specific position in the chain.</span></li><li class="c0 c4"><span>Changed the template expansion and modelling stages of the pipeline from using the SEQRES (construct) sequence to using ATOM sequence. The two sequences sometimes report different amino acids, and this was causing modeller to throw the &#39;Alignment sequence not found in PDB file&#39; error.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span class="c22">To do:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Using the ATOM sequence leads to new errors due to different pdb eccentricities. Have to check individual cases and make exceptions.</span></li><li class="c0 c4"><span>Using mmCIF instead of PDB files? The biological assemblies are available from PDB only in PDB format, but both PDB and mmCIF files have a header that describes the transformations that are necessary to get the biological unit. This should be coordinated with PDBFam&hellip;</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-04-30</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Nothing new</span></li><li class="c0 c4"><span>Project due Friday...</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-04-23</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span>Obtained Provean scores for </span><span class="c16">all</span><span>&nbsp;Protherm and Skempi mutations that I was able to calculate (and &gt; 90% of disease mutations). In some cases, provean need </span><span class="c48">a lot</span><span>&nbsp;of disk space and memory. Put in checks so it doesn&rsquo;t crash nodes.</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c16">Protherm data:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span class="c16">2565 out of 3414 mutations</span><span>&nbsp;(previous value included duplicate domains from P06241 and P24821)</span></li><li class="c0 c4"><span>For the 849 failed mutations:</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c0 c47"><span>17 mutations: fall in a protein without any domains [P10762, P00094, P19614]</span></li><li class="c0 c47"><span>159 mutations: error finding and expanding a template</span></li><li class="c0 c47"><span class="c16">146 mutations</span><span>: mutation falls outside of a domain</span></li><li class="c0 c47"><span>9 mutations: error making a model</span></li><li class="c0 c47"><span class="c16">518 mutations</span><span>: error obtaining mutation properties (FoldX, etc.)</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span class="c22">0.13 </span><span>overall correlation in FoldX </span><span class="c22">dG</span><span>&nbsp;between PDB files (Niklas) and modelled domains </span></li><li class="c0 c4"><span>Average for top 500 mutations sorted by dG_wt correlation: </span><span class="c22">0.98</span></li><li class="c0 c4"><span>Average for top 1000 mutations sorted by dG_wt correlation:</span><span class="c22">&nbsp;0.85.</span><span>&nbsp;</span></li><li class="c0 c4"><span>Average for bottom 1000 mutations sorted by dG_wt correlation: </span><span class="c22">-0.05</span><span>.</span></li><li class="c0 c4"><span class="c22">0.65</span><span>&nbsp;overall correlation of FoldX </span><span class="c22">ddG</span><span>&nbsp;between PDB files (Niklas) and modelled domains</span></li><li class="c0 c4"><span>Average for top &nbsp;500 mutations sorted by dG_wt correlation: </span><span class="c22">0.65</span></li><li class="c0 c4"><span>Average for top 1000 mutations sorted by dG_wt correlation: </span><span class="c22">0.70</span><span>. </span></li><li class="c0 c4"><span>Average for bottom 1000 mutations sorted by dG_wt correlation: </span><span class="c22">0.61</span><span>&nbsp;correlation.</span></li></ul><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 349.50px; height: 255.30px;"><img alt="" src="images/image110.png" style="width: 349.50px; height: 255.30px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c16">Skempi data:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span class="c16">94 out of 862 mutations</span><span>&nbsp;(lower than before because, again, there were some duplicates)</span></li><li class="c0 c4"><span>22 of those mutation occur in interactions that are not reported in iRefIndex (or BioGrid)</span></li></ul><p class="c0 c57"><span>(P52799, Q96L35)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P01112, Q03386)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P06241, P03406)</span></p><p class="c0 c57"><span>(P11233, P15879)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P01006, P00782)</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>For the 768 failed mutations:</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c0 c47"><span>All mutations are in proteins that have at least one domain</span></li><li class="c0 c47"><span class="c16">478 mutations</span><span>: occur in interactions that are not present in iRefIndex (and BioGrid) + PDBFam. Need to investigate why these interactions were not added to the database when I added all contacts from PDBFam.</span></li></ul><p class="c0 c57"><span>(P00766,P01051)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(nan,P0A0L5)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P19909,P01857)</span></p><p class="c0 c57"><span>(P12643,Q5D734)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P01852,P01552)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Q9BYF1,P59594)</span></p><p class="c0 c57"><span>(P68390,P00777)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P78310,Q65914)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P01857,P38507)</span></p><p class="c0 c57"><span>(P01075,P00730)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(</span><span class="c37">P77173,P0A9A6</span><span>)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(</span><span class="c22">P0CH28,Q13191</span><span>)</span></p><p class="c0 c57"><span>(P04578,P01730)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P68390,P08246)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(</span><span class="c22 c37">P46672,P46655</span><span>)</span></p><p class="c0 c57"><span>(P00766,P80060)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(</span><span class="c22">Q03405,Q5PY49</span><span>)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(P38446,Q7A260)</span></p><p class="c0 c57"><span>(Q9Y5Y6,P00974)</span></p><ul class="c3 lst-kix_c85u756bkx0j-1"><li class="c0 c47"><span class="c16">35 mutations</span><span>: could not find and expand templates for (any of the) domains</span></li></ul><p class="c0 c53"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2i26,Q8JGG7,P00698)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1s1q,Q99816,P0CG48)</span></p><p class="c0 c57"><span>(1xd3,P0CG48,P15374)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1iar,P24394,P05112)</span></p><p class="c0 c57"><span>(1h9d,Q13951,Q01196)</span></p><ul class="c3 lst-kix_c85u756bkx0j-1"><li class="c0 c47"><span class="c16">31 mutations</span><span>: found and expanded at least one template, but the mutation falls outside of that domain</span></li><li class="c0 c47"><span class="c16">2 mutations</span><span>: could not make a homology model of the interaction</span></li><li class="c0 c47"><span class="c16">222 mutations</span><span>: error obtaining mutation properties (FoldX, etc.)</span></li><li class="c0 c47"><span>-------------------------------------------------------------------------------------------------------</span></li></ul><p class="c0 c53"><span>Looking at just the domains and not the interactions:</span></p><ul class="c3 lst-kix_c85u756bkx0j-1"><li class="c0 c47"><span class="c16">23 mutations</span><span>: error finding or making a domain template [P01075, P0CG48, Q7A260, Q8JGG7]</span></li><li class="c0 c47"><span class="c16">54 mutations</span><span>: have at least one domain template, but mutation falls outside of that domain</span></li><li class="c0 c47"><span class="c16">691 mutations</span><span>: </span><span class="c22">GOOD</span><span>. Fall inside a domain for which a template was found and was expanded. 600+ mutations were evaluated for interfaces involving a different partner than the partner in the Skempi dataset.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-do:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Check why so many fail at the FoldX / evaluation of mutation stage.</span></li><li class="c0 c4"><span>Check why some of the interactions are missing from the database.</span></li><li class="c0 c4"><span>Check why I get &lt; 95% sequence identity in some cases.</span></li><li class="c0 c4"><span>Borrow some code from Andres to deal with overlapping domains.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-04-16</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span>Disease data:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>2183 out of 2721 mutations</span></li><li class="c0 c4"><span>0.39 correlation coefficient on FoldX dG for wildtype and mutant (0.81 for the top 1000 mutations)</span></li><li class="c0 c4"><span>0.56 correlation coefficient on FoldX ddG (0.70 for the top 1000 mutations)</span></li></ul><p class="c0"><span>Protherm data:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>2619 out of 3414 mutations</span></li></ul><p class="c0"><span>Skempi data:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>101 out of 862 mutations</span></li></ul><p class="c0"><span>To-dos from before:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Sort out provean / sift / SNP-dryad issues </span></li><li class="c0 c4"><span>For both Protherm and Skempi, find the fraction of mutations that fall inside a domain and that can be calculated</span></li><li class="c0 c4"><span>Add an option to use original crystal structures from the pdb without modelling?</span></li><li class="c0 c4"><span>Retrain the classifier</span></li></ul><p class="c0"><span>Sharing PDBFam and other resources with Gary Bader&rsquo;s lab...</span></p><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c16">SNPdryad: predicting deleterious non-synonymous human SNPs using only orthologous protein sequences</span><span>&nbsp;(Ka-Chun Wong and Zhaolei Zhang)</span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 422.50px; height: 336.51px;"><img alt="http://bioinformatics.oxfordjournals.org/content/30/8/1112/F2.large.jpg" src="images/image107.jpg" style="width: 422.50px; height: 336.51px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c16"></span></p><p class="c0 c19"><span class="c16"></span></p><p class="c0 c19"><span></span></p><p class="c0"><span class="c16">Splicing:</span></p><p class="c0"><span>Collaborators sent </span><span class="c22">1450 scores</span><span>&nbsp;for peptides identified using Mascot / Maxquant</span></p><p class="c0"><span>In total, there are </span><span class="c22">919 unique peptides</span><span>&nbsp;(not filtered for overlap?)</span></p><p class="c0 c19"><span></span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 518.60px; height: 262.50px;"><img alt="" src="images/image140.png" style="width: 518.60px; height: 262.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 574.99px; height: 452.50px;"><img alt="" src="images/image23.png" style="width: 574.99px; height: 452.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c12"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 569.00px; height: 384.10px;"><img alt="" src="images/image62.png" style="width: 569.00px; height: 384.10px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-04-09</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Fixed most of the bugs in the pipeline (except for the modeller bug with the alignments)</span></li><li class="c0 c4"><span>Disease data: 1887 out of 2721 mutations (but this should be better now&hellip;)</span></li><li class="c0 c4"><span>Protherm: 3415 pdb mutations -&gt; 368</span></li></ul><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Add an option to use original crystal structures from the pdb without modelling?</span></li><li class="c0 c4"><span>Retrain the classifier</span></li></ul><p class="c0"><span>Discussed to-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>For both Protherm and Skempi, find the fraction of mutations that fall inside a domain and that can be calculated</span></li><li class="c0 c4"><span>Plot correlation for a single feature (like dG) over all mutations</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-03-26</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Fixed more bugs with indexing, and added assert statements to make sure the specified mutation matches the uniprot sequence.</span></li><li class="c0 c4"><span>The output I obtain for modelled disease mutations is now reasonably close to the output provided by Niklas (&gt; 99% correlation coefficient for 75% of mutations, &lt; 15% absolute difference for 60% of mutations)</span></li><li class="c0 c4"><span>But was able to calculate only 169 out of 2649 disease mutations provided by Niklas!</span></li></ul><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Add an option to use original crystal structures from the pdb without modelling.</span></li><li class="c0 c4"><span>Compare the results on the original structures with the results provided by Niklas.</span></li><li class="c0 c4"><span>Figure out SIFT / Provean scores.</span></li><li class="c0 c4"><span>Retrain classifiers (one for raw structures and one for modelled structures).</span></li><li class="c0 c4"><span>Figure out why I can calculate only so few mutations.</span></li><li class="c0 c4"><span>Switch alignments to use hidden markov model definitions rather that tcoffee.</span></li><li class="c0 c4"><span>Coevolution: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://docs.google.com/presentation/d/13KOmx6dKbXJT5AT6jdpY4wkYjZMCgxdsnHP3rzS58gI/edit?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNGSo8jVTVC8nuabFUAB-6x9_8mM7g">https://docs.google.com/presentation/d/13KOmx6dKbXJT5AT6jdpY4wkYjZMCgxdsnHP3rzS58gI/edit?usp=sharing</a></span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-03-26</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Fixed more bugs with indexing, and added assert statements to make sure the specified mutation matches the uniprot sequence.</span></li><li class="c0 c4"><span>The output I obtain for modelled disease mutations is now reasonably close to the output provided by Niklas (&gt; 99% correlation coefficient for 75% of mutations, &lt; 15% absolute difference for 60% of mutations)</span></li><li class="c0 c4"><span>But was able to calculate only 169 out of 2649 disease mutations provided by Niklas!</span></li></ul><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Add an option to use original crystal structures from the pdb without modelling.</span></li><li class="c0 c4"><span>Compare the results on the original structures with the results provided by Niklas.</span></li><li class="c0 c4"><span>Figure out SIFT / Provean scores.</span></li><li class="c0 c4"><span>Retrain classifiers (one for raw structures and one for modelled structures).</span></li><li class="c0 c4"><span>Figure out why I can calculate only so few mutations.</span></li><li class="c0 c4"><span>Switch alignments to use hidden markov model definitions rather that tcoffee.</span></li><li class="c0 c4"><span>Coevolution: </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=https://docs.google.com/presentation/d/13KOmx6dKbXJT5AT6jdpY4wkYjZMCgxdsnHP3rzS58gI/edit?usp%3Dsharing&amp;sa=D&amp;usg=AFQjCNGSo8jVTVC8nuabFUAB-6x9_8mM7g">https://docs.google.com/presentation/d/13KOmx6dKbXJT5AT6jdpY4wkYjZMCgxdsnHP3rzS58gI/edit?usp=sharing</a></span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-03-19</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Swissprot / trembl work fine in an sql server (I was wrong before, a transaction was in progress). </span></li><li class="c0 c4"><span>Fixed some bugs (fixed source and recompiled NACCESS, keeping track of the correct chain and residue, etc.) and recalculating COSMIC DRIVER mutations from Recep.</span></li><li class="c0 c4"><span>Calculated mutations don&rsquo;t match up with Recep&rsquo;s data.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Calculate more mutations, compare them to Recep&rsquo;s data and figure out why the two are different.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-03-12</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Pauline Ng sent me the new SIFT binaries, but they still don&rsquo;t work (segmentation fault).</span></li><li class="c0 c4"><span>Recep sent me the training set data.</span></li><li class="c0 c4"><span>Uploaded swissprot and trembl to the sql server, but now it is unstable. Will use a blast-formatted database with blastdbcmd lookup instead.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Merging pfam domains for trembl still in progress.</span></li><li class="c0 c4"><span>Calculate Protherm mutations and rerun machine learning to make sure everything still works.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-03-05</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>SIFT can run on beagle but not on banting or any of the lab computers.</span></li><li class="c0 c4"><span>PROVEAN seems like it would be (almost?) as good (see slides).</span></li><li class="c0 c4"><span>Integrated PROVEAN into the pipeline (alignments stored for easy retrieval later on).</span></li><li class="c0 c4"><span>Used CD-HIT to cluster all pfam domains in pdbfam and made changes to the pipeline so that it uses that information and does not do redundant alignments.</span></li><li class="c0 c4"><span>Pipeline now calculates a conservation score both for the entire sequence and for the interacting amino acids.</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-1 start"><li class="c0 c47"><span>Still am not sure which one is better or if conservation in interacting amino acids is a good feature.</span></li></ul><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Emailed Recep asking for files with raw results from the previous pipeline for the Protherm and Skempi databases.</span></li><li class="c0 c4"><span>Parsed Protherm data. Was running pfam_scan.pl on Swissprot+Trembl so that I would have domain definitions for non-human uniprot, but stopped this because I noticed that sequence and domain e-value cutoffs used by Sebastian (-e_seq 1e-4, -e_dom 1e-4) were not finding all the domains reported in pfam (in Q12788: Transducin beta-like protein 3, it finds only 7 WD40 repeats, but there should be 9).</span></li><li class="c0 c4"><span>Downloaded trembl domain definitions from and now parsing it to join domains.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Calculate Protherm mutations and rerun machine learning to make sure everything still works.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-02-12</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span>Finished:</span></p><ul class="c3 lst-kix_t6shmrihc7et-0 start"><li class="c0 c4"><span>The pipeline now uses naccess instead of dssp (or pops) to calculate sasa (before and after mutation).</span></li><li class="c0 c4"><span>Fixed a few bugs. Can now calculate mutations using precalculated structures.</span></li><li class="c0 c4"><span>The pipeline ignores interface mutations if the mutated residue is &gt; 5 A away from any atom in the partner chain.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>Further on:</span></p><ul class="c3 lst-kix_7ev7v5cpgtkc-0 start"><li class="c0 c4"><span>Compare the quality of the structures made using modeller alignments and tcoffee alignments.</span></li><li class="c0 c4"><span>Calculate training set mutations to optimise the pipeline and make sure everything still works.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-02-05</span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span>Finished:</span></p><ul class="c3 lst-kix_t6shmrihc7et-0"><li class="c0 c4"><span>Expand domain boundaries in alignments</span></li><li class="c0 c4"><span>For core domains: use SAP to get SASA</span></li><li class="c0 c4"><span>For interface domains: get interacting AA using 5 A cutoff</span></li><li class="c0 c4"><span>Get templates, structures and SASA for cosmic driver mutations</span></li><li class="c0 c4"><span>Log errors when something goes wrong (improved) (may be useful for the web-server&hellip;)</span></li><li class="c0 c4"><span>Kept re-running the code and fixing bugs that show up</span></li><li class="c0 c4"><span>Automated way to visualise SASA and mutations</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>Further on:</span></p><ul class="c3 lst-kix_7ev7v5cpgtkc-0"><li class="c0 c4"><span>Fix some bugs, including a big one with modeller that causes errors in &gt; 30% of interface structures.</span></li><li class="c0 c4"><span>Calculate dG of interaction for interface mutations (using FoldX) (and SASA too?)</span></li><li class="c0 c4"><span>Calculate the training set and use it as reference to optimise alignments / modelling</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2014-01-22</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_t6shmrihc7et-0"><li class="c0 c4"><span>Finished changing the input and output of the pipeline</span></li></ul><ul class="c3 lst-kix_t6shmrihc7et-1 start"><li class="c0 c47"><span>For input it now accesses a database (either a file-based sqlite3 database that gets copied to each node on &nbsp;a server, or a central postgresql database)</span></li><li class="c0 c47"><span>Four output, it puts copies of alignment and pdb files into an export folder. It also makes a text file containing the calculated information (for each template, each model and each mutation) in the same folder. The calculated information is also put back to the database (this works now).</span></li></ul><ul class="c3 lst-kix_t6shmrihc7et-0"><li class="c0 c4"><span>Need to run a few more test cases to make sure there are no bugs</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>Next week:</span></p><ul class="c3 lst-kix_t6shmrihc7et-0"><li class="c0 c4"><span>Expand domain boundaries in alignments</span></li><li class="c0 c4"><span>For core domains: get SASA (using SAP or NACCESS)</span></li><li class="c0 c4"><span>For interface domains: get interacting AA and dG of interaction (using FoldX)</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>Further on:</span></p><ul class="c3 lst-kix_7ev7v5cpgtkc-0"><li class="c0 c4"><span>cosmic drivers </span></li><li class="c0 c4"><span>Logging errors when something goes wrong</span></li><li class="c0 c4"><span>Calculate the training set and use it as reference to optimise alignments / modelling</span></li></ul><p class="c0 c19"><span class="c60"></span></p><p class="c0 c19"><span class="c60"></span></p><p class="c0"><span>Examples of folder structure:</span></p><p class="c0"><span class="c16 c65">For Q9UKG1 (DCC-interacting protein 13-alpha):</span></p><p class="c0"><span class="c65">human/Q9UKG1/PID*502-629/</span></p><p class="c0"><span class="c65">human/Q9UKG1/PID*502-629/PID*502-629/Q9UKG1/</span></p><p class="c0"><span class="c65">human/Q8NEU8/PID*494-609/PID*502-629/Q9UKG1/</span></p><p class="c0 c19"><span class="c65"></span></p><p class="c0"><span class="c16 c65">For Q9Y297 (F-box/WD repeat-containing protein 1A):</span></p><p class="c0"><span class="c65">human/Q9Y297/F-box-like*191-228/</span></p><p class="c0"><span class="c65">human/Q9Y297/Beta-TrCP_D*138-177/</span></p><p class="c0"><span class="c65">human/Q9Y297/WD40*334-581/</span></p><p class="c0 c19"><span class="c65"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-12-04</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Exam</span></li><li class="c0 c4"><span>How many mutations in COSMIC can we model? (preliminary, slide attached)</span></li><li class="c0 c4"><span>Making a pdb parser to get sequences for all the PDBFam domains, so I can start doing the alignments (many errors... will try using SIFTS to get better results). </span></li><li class="c0 c4"><span>Got the code from Niklas for compiling features from pipeline output.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Machine learning project (with Clare) will be a big part of the next two weeks</span></li><li class="c0 c4"><span>Incorporate the dG predictor (Recep) into the pipeline, so that everything is automated.</span></li><li class="c0 c4"><span>Use PDBFam, precalculate alignments, precalculate structures...</span></li></ul><p class="c0 c19"><span></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-11-27</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Spent time writing proposal / doing some preliminary work for the chemical interactions project (with Clare).</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Have an exam in the machine learning course next Tuesday.</span></li><li class="c0 c4"><span>Machine learning project should encompass a week of work.</span></li><li class="c0 c4"><span>On the weekend, Niklas will send me the code for feature post-processing.</span></li><li class="c0 c4"><span>Preliminary results about cosmic (using Sebastian&rsquo;s files)</span></li><li class="c0 c4"><span>Complete results about cosmic (using new interaction files)</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-11-20</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Fixed the bugs I made in the pipeline. Now it produces output for each unique pair or protein-protein interactions.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Use profile-profile alignments (e.g. </span><span class="c16 c20"><a class="c9" href="https://www.google.com/url?q=http://modorama.org/modexplorer/2013-11-20/edtkN8WTir/&amp;sa=D&amp;usg=AFQjCNEt3PVD5CiUp-jfDcOPk2MDK11aog">http://modorama.org/modexplorer/2013-11-20/edtkN8WTir/</a></span><span>)?</span></li><li class="c0 c4"><span>Cluster the alignments for similarity to speed up boundary expansion.</span></li><li class="c0 c4"><span>Run the pipeline on uniprot using PDBFam (first without interactions) to get correct boundaries.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-11-13</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Had an assignment due.</span></li><li class="c0 c4"><span>Still errors with the pipeline code for 2+ Pfam partners, and the code takes days to run</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><p class="c0"><span>Everything from the previous meeting, plus:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Have to start working on the machine learning assignment with Claire.</span></li><li class="c0 c4"><span>Have to give a presentation about it in two weeks.</span></li><li class="c0 c4"><span>The entire write-up is due December 16th.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-11-06</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>The interaction AA in Sebastian&rsquo;s file are incorrect (incomplete?), and Niklas was not relying on them when he was calculating interface ddGs. </span></li><li class="c0 c4"><span>Added the option to calculate ddG for multiple Pfam partners for a given Uniprot. &nbsp;</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Still have to email COSMIC results to collaborators.</span></li><li class="c0 c4"><span>Modify the pipeline to use PDBFam instead of 3DID to search for domain-domain and domain-peptide interactions (add ELMs?).</span></li><li class="c0 c4"><span>Improve domain-expansion and alignment stages of the pipeline, so that it works with original and un-expanded Pfam domains in input Uniprots (Niklas did most of this already).</span></li><li class="c0 c4"><span>Modify the pipeline to that, for a given Uniprot, it can generate correct domain definitions, homology models of all possible interactions and all core domains, lists of interacting amino acids and surface-exposed residues, and foldX scores.</span></li><li class="c0 c4"><span>Optimise the code above to run fast enough (currently, alignments take a very long time).</span></li><li class="c0 c4"><span>Run it on all human UniprotKB to have correct domain definitions and precalculated results for the webserver.</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-10-30</span></p><p class="c0 c19"><span class="c7"></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Made minor changes to the pipeline to output Modeller and FoldX pdb files with unique identifiers, so that we have a model for each prediction that the pipeline makes.</span></li><li class="c0 c4"><span>Looking at Sabastian&rsquo;s code, re-running HHMscan or pfamscan, etc. </span></li><li class="c0 c4"><span>Looked for measures of amino-acid conservation.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Obtain the machine learning component of the pipeline from Recep</span></li><li class="c0 c4"><span>Email the list of conservation scores and surface-exposed residues for high-scoring hits from the pipeline to collaborators?</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-10-23</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Most splicing events affecting a splicing site remove all interacting amino acids</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_ygjhinrwstz-0 start"><li class="c0 c4"><span>Handle errors in the pipeline better</span></li><li class="c0 c4"><span>Evaluate the probability that the interaction is lost for splicing events that take away a fraction of interacting AA?</span></li><li class="c0 c4"><span>Learn what Sabastian was doing</span></li><li class="c0 c4"><span>Expand the pipeline to regions where no template is found (Joan&rsquo;s suggestions):</span></li></ul><ul class="c3 lst-kix_ygjhinrwstz-1 start"><li class="c0 c47"><span>Surface-exposed AA for all proteins</span></li><li class="c0 c47"><span>Conservation of surface-exposed AA</span></li><li class="c0 c47"><span>Expand the set of protein-protein interactions by mapping interactions to proteins from a different family but with a similar structure (Joan)</span></li><li class="c0 c47"><span>Look for sequence covariance between partners that are known to interact</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-10-16</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>After filtering 15784 splicing events, only 74 interacting pFam pairs with structures are left</span></li><li class="c0 c4"><span>Judging by the initial analysis of the corresponding structures, it does not look promising</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_ygjhinrwstz-0"><li class="c0 c4"><span>Continue with splicing?</span></li></ul><p class="c0 c19"><span class="c7"></span></p><p class="c0 c19"><span class="c7"></span></p><p class="c0"><span class="c7">Subgroup Splicing/Network, 2013-09-25</span></p><p class="c0 c19"><span></span></p><ul class="c3 lst-kix_c85u756bkx0j-0"><li class="c0 c4"><span>Made alignments of splicing variants and canonical uniprotKB sequences</span></li><li class="c0 c4"><span>Trying to integrate this with the pipeline</span></li></ul><p class="c0 c19"><span></span></p><p class="c0"><span>To-dos:</span></p><ul class="c3 lst-kix_ygjhinrwstz-0"><li class="c0 c4"><span>Map splicing insertions / deletions to the uniprotKB numbering scheme.</span></li><li class="c0 c4"><span>Run deletions to see if any interaction partners are lost entirely (</span><span class="c48">i.e.</span><span>&nbsp;all AA in the pipeline interaction database are lost).</span></li><li class="c0 c4"><span>Make a heuristic for selecting pdb templates for splicing modelling</span></li><li class="c0 c4"><span>Use modeller to create structures where the interactions are lost partially, or where new domains are added.</span></li></ul><p class="c0 c19"><span></span></p><p class="c0 c19"><span></span></p></body></html>