\documentclass[11pt]{article}

\usepackage[margin=0.9in]{geometry}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage[document]{ragged2e}
\usepackage{mathtools}
\usepackage{booktabs}
\usepackage{float}
\usepackage{pbox}
\usepackage{setspace}
\usepackage{listings}

\setlength{\parskip}{3mm plus4mm minus3mm}

% Language, Fonts
\selectlanguage{english}
\usepackage{courier,amsmath,amsfonts} %fonts

% Referencing
\usepackage{xcolor} %
\usepackage{hyperref} %
\definecolor{dark-red}{rgb}{0.4,0.15,0.15}
\definecolor{dark-blue}{rgb}{0.15,0.15,0.4}
\definecolor{medium-blue}{rgb}{0,0,0.5}
\hypersetup{
	colorlinks, linkcolor={dark-red},
	citecolor={dark-blue}, urlcolor={medium-blue} %url can be magenta 
}

% % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % %

\usepackage[
    backend=biber,
    style=numeric,
    natbib=true,
    maxnames=2,
    sorting=nyt,
    sortcites=false,
    block=space,
    date=long,
    url=false, 
    doi=false,
    eprint=false,
    isbn=false,
    uniquename=false,
    uniquelist=false,
    terseinits=true,
    firstinits=false
]{biblatex}
%\usepackage[backend=biber,terseinits=true,sorting=none,isbn=false,doi=false]{biblatex}
\addbibresource{http://localhost:23119/better-bibtex/collection?/0/ZK67QXB9.biblatex}


% /Bioinformatics/Mutation scoring/
\addbibresource{http://localhost:23119/better-bibtex/collection?/0/RSRPJ2GB.biblatex}
% /Computer science
% \addbibresource{http://localhost:23119/better-bibtex/collection?/0/BHZ3WCDB.biblatex} 
% /Databases
%\addbibresource{http://localhost:23119/better-bibtex/collection?/0/HB6PP66S.biblatex}
% /Disease
%\addbibresource{http://localhost:23119/better-bibtex/collection?/0/2JWAVBHR.biblatex}
% /Mutation scoring
%\addbibresource{http://localhost:23119/better-bibtex/collection?/0/RSRPJ2GB.biblatex}


%\addbibresource{C:/Users/Dropbox/phd/ref/Exported Items.bib} % don't forget to writhe the path and extension of .bib file
\ExecuteBibliographyOptions{%
  citetracker=true,% Citation tracker enabled in order not to repeat citations, and have two lists.
  sorting=none,% Don't sort, just print in the order of citation
  alldates=long,% Long dates, so we can tweak them at will afterwards
  dateabbrev=false,% Remove abbreviations in dates, for same reason as ``alldates=long''
 % articletitle=true,% To have article titles in full bibliography
  maxcitenames=999% Number of names before replacing with et al. Here, everyone.
}


% % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % %

\title{Predicting the Effect of Mutations on a Genome-wide Scale}
\author{Alexey Strokach}
\date{December 01, 2015}
% \vspace{0.2cm} \vspace{0.2cm}\



\begin{document}

\onehalfspacing

\maketitle

\tableofcontents



\section*{Abstract}

\cite{pleasance_comprehensive_2010}

\cite{tennessen_evolution_2012}

\cite{lee_mutation_2010}



\newpage
\section{Introduction}

The computational methods for predicting protein stability upon mutation have been compared recently \cite{Potapov2009}.

Maximum correlation coefficient 0.86 \cite{Potapov2009}.

I-Mutant 2 and FoldX are trained

Rosetta can predict ddG as well, but the energy function has to be chosen carefully \cite{Kellogg2011}. In particular, a soft repulsion energy function should be used for repacking (combinatorial rotamer optimization carried out using Monte Carlo simulated annealing with Dunbrack backbone dependent rotamer library), optionally combined with a hard-repulsion energy function used during backbone and sidechain minimization with uniform constraints \cite{Kellogg2011}. However, optimization leads to only slightly improved accuracy for a single mutation (0.68 vs 0.69 correlation coefficient), and can be skipped in order to speed up predictions. 

When the reference energies for the 20 amino acids are fit to produce the best ddG correlation with experimental values, the correlation coefficient went up to 0.73. 
The source code for FoldX is not availible. FoldX is trained on the Protherm and Skempi datasets, and therefore our cross-validation is likely overfitting the 



\section{Methods}



\section{Results}

Untrained:

SIFT
Provean 
MutationAssessor
FATHMM (unweighted)


Trained:
Polyphen-2
MutationTaster
FATHMM (weighted)



\section{Discussion}

Depend on closed-source FoldX. Can use openMM to recreate most of the features that are used by FoldX and train a final classifier using those features.

Can use thermodynamic integration (TI) to increase the training set. Select a few mutations deemed to be the most important from each domain family. ...


\section{Future Directions}








% References

\printbibliography[title={References}]

\end{document}